---
layout: single 
title: "Genetics and multiregional evolution, meetings 2005" 
category: story
permalink: /weblog/reviews/genetics/genetics_mre_aapa_2005.html
tags: [Neandertals, out-of-Africa, Middle Pleistocene] 
comments: false 
author: John Hawks 
---

<p>
Several papers at the AAPA meetings presented evidence for deep Asian-specific lineages in the present human gene pool. For example, from Mike Hammer's abstract: 
</p>

<blockquote>Preliminary data from two loci that show evidence of ancient admixture will be discussed. A gene tree constructed from sequence data at the first locus roots in East Asia and has a most recent common ancestor ~2 million YBP. The pattern of nucleotide variation at the second locus reveals two major lineages that have not undergone recombination for over 2 million years, and statistically rejects the null hypothesis of panmixia during the early ancestry of modern humans (Hammer et al. 2005:115). </blockquote>

<p>
And from Shimada and Hey (2005:195): 
</p>

<blockquote>Most sequences [from a 10.1 kb region of the X] are quite similar to one another, however three sequences differed from the others at an average of 28.6 substitutions. Assuming a molecular clock, and a human/chimpanzee divergence time of 6 million years, the estimated age of the base of the human sequences is 1.1 million years ago, whereas the estimated base of the tree excluding these divergent human sequences is 290,000 years ago. These divergent sequences were found in samples from the Middle East (Druze and Bedouin populations) and North Africa (Mozabite population). The pattern is suggestive of admixture between non-African archaic humans and Modern Humans [sic]. </blockquote>

<p>
There were also more analytical papers by Hey and separately by Alan Rogers and colleagues (2005:182): 
</p>

<blockquote>The "ancestral allele" at a given locus is the allele thought to have been carried by the last common ancestor (LCA) of all humans. These are only estimates, of course, but they are often relatively good ones. Thus, it is interesting that human ancestral alleles are usually most common in Africa. Some claim that the ancestral allele should be most common in Africa, because it is the ancestral population. We argue otherwise. In the absence of selection or ascertainment bias, the expected frequency of the ancestral allele is the same in each modern population, regardless of the history of population size, subdivision, or gene flow. The observed tendency of ancestral alleles to cluster in Africa argues either for some form of ascertainment bias or for some form of selection. </blockquote>

<blockquote>We attribute the pattern to two forms of ascertainment bias, which affect different sorts of locus. These biases, together with a history of expansion out of Africa, are capable of producing the observed pattern. The only loci that are certainly free of bias are those that sequence arbitrary stretches of DNA far from known genes. In these bias-free systems, there is no tendency for ancestral alleles to be most common in Africa. </blockquote>

<p>
Out of these papers I found the last to be the most interesting. What the paper essentially said was that earlier work that had found the "ancestral allele" of most genes to lie in Africa was entirely irrelevant to the issue of modern human origins. This is because there is no reason to expect this ancestral allele should preferentially appear in any population, regardless of their demographic history. Rogers reported in the presentation that the last paragraph of the abstract was wrong: they now have good reason to believe that ascertainment bias is not responsible for the observed pattern. That would leave natural selection as apparently the only explanation, although what pattern of selection would create the excess of ancestral alleles in Africa is up for grabs. I have an idea, but I'm not sharing it just yet. 
</p>

<p>
The study is a powerful blow against the case for a recent, exclusively African origin of modern humans. It doesn't demolish the case, since the arguments for a recent African origin extend to other aspects of the genetic record, but it is damaging and suggests that a lot of work should be reevaluated. 
</p>

<p>
But if work <i>supporting</i> Out-of-Africa should be reevaluated, the study suggests that work <i>refuting</i> it should be reconsidered also. The genes examined by Hammer and colleagues and by Shimada and Hey both rely upon the finding of ancient genetic variation outside of Africa -- in essence, placing the "ancestral allele" for these genes in Eurasia. Rogers' and colleagues' analysis shows this logic to be misleading. 
</p>

<p>
The key is, that as we look at an increasing proportion of the genome, we are more likely to see a complex set of relationships. From my standpoint, that indicates that the pattern of human relationships was also likely complex, and that multiple forces (especially selection) were important in generating the current pattern of diversity. But for someone supporting a pure Out-of-Africa model, it means a retrenchment to a "consistency"-based argument: if you can't show the data are <b>inconsistent</b> with (i.e. absolutely falsify) a recent African origin, then you're just whistling in the wind. Despite the complete sequence of the human genome and a growing sample of individuals for many genomic regions, we can't seem to settle on whether the boundary case (no archaic input into human populations) has been falsified. 
</p>

<p>
Hammer and Jeff Wall are working on the problem; looking for analytical methods to definitively falsify an exclusive Out-of-Africa model. In short, they're looking for smoking gun evidence of archaic genes in the recent human gene pool, and they think they have two. One of these was described in Garrigan et al. (2005), <a href="/weblog/reviews/genetics/garrigan_2005.html">discussed in an earlier post</a>. Another shows a similar pattern of diversity outside of Africa. Together with the work by Shimada and Hey, and the earlier work of Alan Templeton (2002), these loci would appear to show very strong evidence of ancient genetic exchanges among human populations, beyond the timeframe predicted by the Out-of-Africa model. 
</p>

<p>
In my opinion, much of this logic is misleading. Many people studying the problem have a tendency to think that "archaic" genes should be hugely divergent from the genes of most living people. The search pattern to find them is to look for something very rare and different outside of Africa, or to place the root of the genealogy unambiguously in Asia or Europe. This is the premise behind Wall's 2000 study for example: 
</p>

<blockquote>To model archaic admixture, I use an infinite-sites coalescent model with recombination. I assume there is no selection and a constant rate of recombination per base pair per generation. Consider a panmictic population with diploid effective population size <i>N</i>. Going backward in time, suppose that at time <i>T<sub>0</sub></i> the remaining ancestors are placed randomly into one of two subpopulations with probabilities <i>c</i> and 1 - <i>c</i>. From time <i>T<sub>0</sub></i> to time <i>T<sub>1</sub></i>, these subpopulations are assumed to be completely isolated from each other and panmictic with diploid effective population sizes of <i>cN</i> and <i>N</i>, respectively. Then, at time <i>T<sub>1</sub></i>, all remaining lineages are placed into a single panmictic population with diploid effective size <i>N</i> (Wall 2000:1272-1273). </blockquote>

<p>
To translate a bit; the study assumed that archaic humans were arranged into completely isolated populations for a long period of time. This long period of isolation (described as a "deviation from panmixia") resulted in genetic sequences that were artificially elevated in their divergence from the mainstream of modern human origins, and these might be recognized by examining the genetic variation in living Eurasians. In Hammer and colleagues' abstract, the concept of "statistically rejects the null hypothesis of panmixia during the early ancestry of modern humans" comes straight from this sort of logic. 
</p>

<p>
But the hypothesis that archaic humans were completely (or even largely) isolated from each other is artificially extreme. It is a Pleistocene version of polygenism, except with a recent frenzy of interbreeding in the Upper Paleolithic. I don't believe it, and I don't know anybody who seriously would. 
</p>

<p>
Far more likely is the idea that archaic humans were always connected to each other by some level of gene flow. I would take as a null hypothesis that the level was consistent with the current <i>F<sub>ST</sub></i> of 0.1 to 0.15, which would mean an average of around two individuals moving between each pair of continents per generation. At this level, selected alleles can move relatively quickly across the human range. With the small effective size estimated for human populations, no ancient human group should have been very genetically divergent from another. This means that we shouldn't expect to find "archaic" alleles that are very different from most living people. And if we could have sequenced their DNA when they were alive, we would find that archaic people weren't very different from each other, either. 
</p>

<p>
DNA sequences from Neandertals confirm this expectation. The most recent common ancestor of the Neandertal mtDNA sequences and living humans may have lived as recently as 250,000 years ago. The initial estimate was around 600,000 years, which is elevated because of the anomolously high divergence of the Feldhofer 1 sequence. But even at this high date, the MRCA is significantly more recent than the earliest habitation of Europe, and far more recent than the initial dispersal of people from Africa. In other words, these archaic populations were connected to each other, and were exchanging genes. If there was any isolation of later Middle Pleistocene populations, such as the Neandertals, it was short compared to their shared ancestry. Their genes weren't very different from ours, and they were probably cycling toward greater similarity with us. 
</p>

<p>
If this is true, then we shouldn't expect to find many genes that indicate a strikingly divergent pattern for non-Africans. We should expect most genes to be more or less the same, although many will show greater African diversity for ecological and demographic reasons. Only those genes with exceptional histories of local adaptation will show highly divergent alleles in one place or another.  These genes might well be interesting because some of them will be related to the morphological features of archaic humans. But they will probably be exceptionally rare. 
</p>

<p>
So maybe that's why we are only now beginning to find them. 
</p>

<h4>References:</h4>

<p class="cite">Garrigan D, Mobasher Z, Severson T, Wilder JA, and Hammer MF. 2005. Evidence for archaic Asian ancestry on the human X chromosome. Mol Biol Evol 22:189-192. </p>

<p class="cite">Hammer MF, Garrigan D, Wilder JA, Mobasher Z, Severson T, and Kingan SB. 2005. Sequence data from the autosomes and X chromosome: evidence for ancient admixture in the history of <i>H. sapiens</i>? (abstract). Am J Phys Anthropol suppl 40:115. </p>

<p class="cite">Shimada MK, and Hey J. 2005. History of modern human population structure inferred from the worldwide survey on Xp11.22 sequences. Am J Phys Anthropol suppl 40:195. </p>

<p class="cite">Templeton AR. 2002. Out of Africa again and again. Nature 416:45-51. </p>

<p class="cite">Wall JD. 2000. Detecting ancient admixture in humans using sequence polymorphism data. Genetics 154:1271-1279. </p>

