---
layout: single 
title: "More assimilation, genetic-style" 
category: story
permalink: /weblog/reviews/genetics/garrigan_2005_structured_ancestral.html
tags: [MSA, Neandertals, Homo erectus] 
comments: false 
author: John Hawks 
---


<p>
Daniel Garrigan and colleagues (2005) have <a href="http://www.genetics.org/cgi/content/abstract/genetics.105.041095v1">an article</a> in press in <i>Genetics</i>, titled "Deep haplotype divergence and long-range linkage disequilibrium at Xp21.1 provide evidence that humans descend from a structured ancestral population." (via <a href="http://dienekes.blogspot.com/2005/06/research-on-x-chromosome-rejects-pure.html">Dienekes</a>). The research comes out of <a href="http://lifescience.arizona.edu/faculty.php?faculty_id=2891">Mike Hammer's</a> lab at the University of Arizona. The abstract includes the following: 
</p>

<blockquote>Aided by a novel experimental design, we present the first genetic evidence that statistically rejects the null hypothesis that our species descends from a single, historically panmictic population. In a global sample of 42 X chromosomes, two African individuals carry a lineage of non-coding 17.5 kilobase sequence that has survived for over one million years without any clear traces of ongoing recombination with other lineages at this locus. These patterns of deep haplotype divergence and long-range linkage disequilibrium are best explained by a prolonged period of ancestral population subdivision followed by relatively recent interbreeding. This inference supports human evolution models that incorporate admixture between divergent African branches of the genus Homo.</blockquote>

<p>
So what's the story here? After all, Alan Templeton has been talking about evidence for a non-panmictic ancestral population for humans for a long time. And Templeton has worked with Hammer on earlier papers.  So it clearly isn't true that this gene is the first evidence of non-panmixia.  
</p>

<p>
Instead, what this gene <i>might</i> show is something even more extreme.  Within the sample of X chromosomes in the study, there were two highly divergent haplotypes, separated by 10 mutational steps. One of these haplotypes is very common, found in most of the individuals globally.  The other appears to be rare, founded only two Mbuti pygmy individuals.  The interesting thing about these haplotypes is that there appeared to have been almost no recombination events between them -- despite the fact that the gene lies in a region of relatively high recombination, and there are many recombination events among different variants of the common haplotype.  Garrigan and colleagues (2005) shows that this result is highly unexpected if the population ancestral to all of the sampled individuals was panmictic.  But the very low likelihood of observing this result under panmixia indicates that it is likely that parts of the ancestral population were out of genetic contact for some period of time.  The paper illustrates this by showing an impermeable bar, representing reproductive isolation, separating the two haplotypes in the ancestral population.  The clear inference is that the persistence of two highly divergent lineages in this instance without recombination is indicative of some substantial period of reproductive isolation in the past.  
</p>

<p>
Unfortunately, the paper actually doesn't statistically test this hypothesis.  It focuses instead on testing the null hypothesis of panmixia.  It may well be that some kind of population structure including isolation by distance, instead of complete isolation, is consistent with the observations.  
</p>

<p>
In my view, this is an instance of people being less than careful about their assumptions.  The chart in the paper clearly shows a hypothesis of reproductive isolation.  But isolation is not tested in the paper.  This kind of isolation is a prediction of the "assimilation model" of modern human origins.  After all, without the isolation there would be nothing to assimilate -- instead, archaic humans would of been part of one regionally dispersed population. Garrigan and colleagues (2005:7) go so far as to formally call the model the "Isolation-and-Admixture (IAA) model," following Jeff Wall (2000) in the assumption that archaic humans may have represented divergent lineages with little (or at the boundary condition, no) interbreeding between them. 
</p>

<p>
Indeed, the findings of this paper might actually be consistent with the Out of Africa replacement model, in that they show evidence for a single dispersed population within Africa.  This kind of regionally differentiated African population has long been a prediction of some proponents of a recent African origin, sometimes called the "weak Garden of Eden model".  Insofar as strong regional isolation is not demonstrated by the study itself, such a model remains credible as an explanation for this gene.  The strength of refutation of the out of Africa model still depends on the combination of many different genes, which together are not consistent with a single recent origin in a small African population.  What remains to be demonstrated is the extent to which archaic-modern human contacts occurred.  
</p>

<p>
In that context, it is interesting that this paper raises the issue of archaic-modern contacts within Africa itself.  Africa was a highly diverse population in the past, it retains a strong degree of genetic diversity today, and there is every reason to expect that past African differentiation might have left traces in the present distribution of African genes. Garrigan and colleagues (2005:14-15) discuss the issue as follows: 
</p>

<blockquote>An interesting feature of our inference is that the putative isolation and admixture event likely occurred between ancient African subpopulations. The question of hominin admixture has typically focused on events either between AMH [anatomically modern humans] and Neanderthals in Europe or AMH and <i>Homo erectus</i> in Asia. Given recent fossil evidence, Africa may have provided the greatest opportunity for admixture between archaic subpopulations of <i>Homo</i>, simply because Africa harbored the highest levels of hominin taxonomic diversity (Wood 2002; Tattersall 2003). If the IAA [isolation and admixture] model is correct, it implies that subpopulations of archaic <i>Homo</i> existed in allopatry within Africa for much of the Pleistocene. Regardless, the Xp21.1 data re-iterate the key role of African hominin diversity in the evolution of our species (Jolly 2001). </blockquote>
</p>

<blockquote>If the AMH genome contains any degree of dual ancestry (i.e., archaic and modern), the Recent African Replacement model in its strictest definition (i.e., that of complete replacement) must be rejected. While the majority of the AMH genome may descend from a single African population, if further studies corroborate the inferences made from the Xp21.1 data, it would imply that the evolutionary lineage leading to AMH did not evolve reproductive isolation from other archaic hominin subpopulations and, thus, cannot be considered a distinct biological species. </blockquote>

<p>
The possibility of archaic-modern human interaction within Africa is often neglected, mainly because there is little African fossil evidence that could substantiate it.  But there are substantial regional differences within Africa in MSA traditions, possibly marking ancient population boundaries.  Moreover, some ancient populations within Africa may not yet be represented by fossil or archaeological evidence, including the ancestors of the pygmies considered in this study.  Genetics may have much to say in the next several years about the population variation within Africa in the past and the way that it may have contributed to the formation of the modern human phenotype. 
</p>

<h4>References:</h4>

<p class="cite">Garrigan D, Mobasher Z, Kingan SB, Wilder JA, Hammer MF. 2005. Deep haplotype divergence and long-range linkeage disequilibrium at Xp21.1 provide evidence that humans descend from a structured ancestral population. Genetics (online before print). <a href="http://www.genetics.org/cgi/content/abstract/genetics.105.041095v1">Abstract</a></p>

