---
layout: single 
title: "Continents of continuity" 
category: story
permalink: /weblog/reviews/genetics/harding_2005_continuity.html
tags: [Late Pleistocene, Middle Pleistocene, Neandertals] 
comments: false 
author: John Hawks 
---


<p>
Harding and McVean (2005) present a review of current genetic evidence addressing the origin of modern humans. Unlike other recent reviews, they do not present a litany of evidence in favor of a recent African origin. Instead they step away from the past to look at the prospects for more complex metapopulation models to explain all of the genetic data, rather than merely one part of it. Their basic theme is that contradictions between evidence that suggest a recent single origin and contrary evidence that suggests regional continuity may be resolved by considering a fuller range of demographic models. Such models encompass geographic structure in the ancestral population leading to modern humans. 
</p>

<p>
Harding and McVean never make explicit the difference between effective population size (N<sub>e</sub>) and census population size. N<sub>e</sub> describes the apparent rate of inbreeding in the ancient human population, while census population size describes the actual number of people that existed at any one time in the past. These values are widely divergent for most living animal species, including most mammals. This means that for most mammal populations, inbreeding is not merely a consequence of small population size, but also other evolutionary forces--chiefly natural selection. I think that Harding and McVean do not mean to confuse N<sub>e</sub> with census size, and indeed they do not make the equation between the two that has led to so many errors in other papers. But their failure to note the difference between them leads to some conceptual mistakes. Consider: 
</p>

<blockquote>The search for an ancestral history that can satisfactorily explain the genetic architecture of modern human phenotypies will require models that include positive selection within a structured population. Compelling genetic evidence has been found for geographically local adaptation from analysis of F<sub>ST</sub> values [citing Akey et al. 2002]. However, the relatively small N<sub>e</sub> values for humans and other primates, compared with <i>Drosophila</i> or rodents, implies weakened purifying selection and an expectation for some level of polymorphism among slightly deleterious variants. It will be easy to misinterpret the latter as evidence for positive selection in the form of local adaptation (671). </blockquote>

<p>
This quote brings up a really interesting idea--that human populations may appear to be locally adapted because their demography has exposed them more or less to a single global pattern of selection. But demography is not the only influence on N<sub>e</sub>. Although humans certainly are on a different scale from <i>Drosophila</i> or mice in terms of genetic drift, nevertheless, the most important influence on genetic diversity in all these species has probably been purifying selection and hitchhiking. 
</p>

<p>
Likewise, Harding and McVean (2005) confuse the issue of N<sub>e</sub> when referring to the demographic implications of particular gene genealogies: 
</p>

<blockquote>It has become easy to accept the recent age for mitochondrial Eve, and also to justify the many older TMRCA estimates for autosomal gene geneaologies, by assumng that N<sub>e</sub> has not been reduced from 10 000, but an NRY TMRCA estimate of 60 000 years, which is so much younger than mitochondrial Eve, has produced a quiet sense of unease (669). </blockquote>

<p>
They ignore the easiest explanation for these problems with young genealogies, which is positive selection. This again was necessitated by their focus on N<sub>e</sub> as a meaningful demographic parameter. In effect, they argue that the recent coalescence dates of NRY and mtDNA challenge the idea that there was a panmictic population before 100,000 years ago, because in such a population there should have been less heterogeneity of gene genealogies. 
</p>

<p>
To be honest, I can't follow this argument (on page 669) that the NRY somehow shows that:
</p>

<blockquote>African populations must have been more strongly subdivided and isolated from each other than non-African populations, and that some African populations were not a direct source for the range expansions out of Africa (669). </blockquote>

<p>
This idea seems completely in opposition to the Y chromosome evidence, taken at face value. A low variability for any genetic locus should be evidence for a recent origin in a small population that had <i>no</i> subdivision, <i>except in the case where the gene was subject to positive selection</i>. 
</p>

<p>
But of course if the Y chromosome, mtDNA, and other genes with recent coalescence dates were actually under selection, there would be no reason at all to oppose Harding and McVean's other scenario: 
</p>

<blockquote>An additional and more contentious possibility is that not all modern human diversity presently found outside of Africa evolved from recent African ancestry. The greater time-depth of autosomal and X chromosome loci, compared with mtDNA and Y chromosomes, allows subdivision in the ancestral population to date to a time when modern human morphology was evolving from an archaic form. Patterns in these genetic data do suggest admixture between the Late Pleistocene humans, whose range expansions are visible in mtDNA and Y chromosome data, and populations established earlier. Probably, most of this gene flow took place within sub-Saharan Africa, but we cannot rule out admixture elsewhere in the world (669). </blockquote>

<p>
The last part of this is not supported by the data; Templeton (2002),  <a href="weblog/reviews/genetics/garrigan_2005.html">Garrigan et al. (2005)</a> and others have shown evidence for ancient genetic contributions from Middle Pleistocene non-Africans. As far as I can tell, the idea that this contribution mysteriously occurred by the translocation of ancient Asians to sub-Saharan Africa is a fiction. Nor is the major issue time depth, since a deep time depth for autosomal DNA would easily be consistent with a purely African origin. The issue is geographic distribution, and the same observations that make a single panmictic population unlikely within a purely African context certainly cannot rule out a wider geographic context. And as Templeton (2002) points out, an origin limited exclusively to Africa may already be falsified by the genetic data. 
</p>

<p>
Is there any point to examining metapopulation models for human evolution? I speak as a believer in the idea that humans were a geographically structured metapopulation. But Harding and McVean (2005) do little but raise a few intriguing scenarios for past human metapopulations. They do not draw attention to the problems of metapopulation models. The most critical problem is deciding which parameters will be allowed to vary and which will be constant. It may be true that a more complex demographic scenario fits the pattern of data better than panmixia and constant population size. 
</p>

<p>
But it is probably true that multiple models provide a better fit. Certainly the best of all possible fits would be if natural selection was considered to act in a unique and independent way on every genetic locus. In this way, every gene would be maximally explained. But that hypothesis, we would object, is overparameterized -- in other words, it isn't parsimonious. The question is which of the potential parameters should we consider first to maximize both parsimony and explanatory power? The answer to this question is of course that we should test many, and include those that test out as potentially important. 
</p>

<p>
Moving to a metapopulation concept of ancient humans is undoubtedly a good idea, since ancient humans must have been a metapopulation. But I think that including selection as a parameter can explain more that we currently consider to be problematic with the panmixia model. This would include the recent coalescence dates of mtDNA, NRY, FoxP2, and other low-variation genes. It might also include the estimate of N<sub>e</sub> at 10,000. On the other hand, there are metapopulation models that can also explain human N<sub>e</sub>, and these may be part of the story. I tend to reject them because they would require that many other species also have distinctive metapopulation structures like ancient humans, and that seems less parsimonious than the idea that genomic selection in animal species is more common and powerful than usually thought. 
</p>

<p>
On the other hand, considering the distinctiveness of some fossil human populations, I think a metapopulation model makes a lot of sense. In particular, I am working on explaining many aspects of Neandertal evolution by reference to the idea that Pleistocene Europe was a population sink. This metapopulation concept is potentially very explanatory, and does have consequences for the interpretation of ancient DNA evidence and modern human variation. So metapopulation models must be considered as part of the overall explanation of human evolution. The question that we all face is which kinds of models we should turn to when the panmictic model is shown to be wrong. 
</p>

<h4>References:</h4>

<p class="cite">Harding RM and McVean G. 2005. A structured ancestral population for the evolution of modern humans. Curr Op Genet Devel 14:667-674. </p>

