---
layout: single
title: "The science of Helicobacter pylori"
description: "Looking at the way a human pathogen relates to human population history"
category: story
permalink: /weblog/reviews/genetics/h_pylori_2005.html
comments: false
author: John Hawks
modified: 2020-05-21
---



There is an article in the February 2005 <em>Scientific American</em> with the intriguing title "An endangered species in the stomach." The article fairly well covers the current science of <i>H. pylori</i>, especially focusing on its role in the ecosystem of the stomach.  For a little background, <i>H. pylori</i> exists in the stomachs of a large proportion of people in the world, and because of its tolerance of the acid environment of the stomach it is the only bacterium that can regularly survive there.

The frequency of <i>H. pylori</i> infection is high in most areas of the world.  However in the developed world, the frequency of infection has been decreasing, mainly due to routine administrations of antibiotics for other infections.  Evidently a short course of antibiotics is all that's necessary to wipe out <i>H. pylori</i> from the stomachs of most individuals.

This decline in <i>H. pylori</i> frequency in Western populations has had several effects, which the article points out are both good and bad.  Unmistakably good is the decline in stomach cancer, which has a much higher incidence in people with <i>H. pylori</i> infection.  The article points out that in 1900, stomach cancer was the most common form of cancer with the highest rate of mortality.  Today it is much less severe as a cause of mortality with a much lower incidence-lower than the incidence of many other kinds of cancers.

The article describes the bad effects of <i>H. pylori</i> disappearance as surrounding an increase in the incidence of acid reflux disease.  Acid reflux disease is a painful condition in its own right, but later in life it can lead to more serious complications including adinocarcinoma of the esophagus.  In the article, Blaser explores the ecosystem of different strains of <i>H. pylori</i> in order to explain why acid reflux disease has increased.

Different strains of <i>H. pylori</i> express different genes and some of these genes have more or less damaging effects on the tissues of the host.  Blaser finds that this diversity of genes actually has a function in feedback communication between host and the parasite.  For example, one gene that is particularly damaging in attacking stomach tissues can actually cause the body to reduce the level of acidity in the stomach. This reduction leads to a reduction in the irritation that leads to acid reflux.

Blaser is clearly very interested in the negative feedbacks between <i>H. pylori</i> and the host, and suggests that the understanding of this system may lead to advances in medical treatment.  He suggests that the use of probiotic treatments (applying live bacteria to the body instead of applying antibiotic agents) might become common if scientists could properly evaluate the best microbial environment for each individual.  This is an interesting perspective, but considering the complexity of drug interactions were only one substance is involved, it seems unlikely that scientists will be able to unravel the highly complex interactions of organisms any time soon.  Even so, the possibility does point to the necessity of the FDA or some other kind of oversight body considering what kind of regulations may be necessary on probiotic treatments.  Since live bacteria are clearly not drugs, it's not obvious that the regulations that currently apply to drugs and other medical substances will apply to this form of treatment.



====Parasites tracing human history



The article also mentions attempts to use the distribution of variation in the <i>H. pylori</i> around the world to trace the history of human migrations.  This general concept is not new-it has been applied to other human parasites before with varying levels of success.  The main benefit of using pathogens and other microorganisms as proxies for human variability is that there markedly more variable in their DNA sequences than are individual humans.  This means that the microorganisms more accurately trace events over relatively short time spans, like those involved with recent human migrations to different parts of the world.



On the other hand, there are disadvantages.  Microbes are not people, and they're not inherited a in the same manner as human genes. Some microorganisms have relatively vertical forms of transmission. This means that they are typically inherited by individuals from one of their parents or from another closely-related individual.  JC virus is an example of relatively vertical transmission, being inherited early in life predominantly from close relatives.  <i>H. pylori</i> is also inherited early in life, although it is not clear to what extent the microbial population may change over the course of a lifetime.



The other important difference between microbial evolution and human evolution is the intensity and form of selection. With their rapid generation times, microbes are under relatively intense selection, especially if they have some kind of effect on the host.  The interactions between pathogens in the host's immune system are very complex, and most genetic changes in microbes can be expected to be adaptive in one way or another.  In some pathogens, like JC virus, the effects on the host are absent or negligible, and in such cases we may argue that the evolution of their genes is---or at least might be---relatively neutral.  Other pathogens, like human papillomavirus, exert a greater effect on their host, including the possibility of mortality.  These microbes cannot be considered to have evolved neutrally.

It seems clear from the consideration of the effects of <i>H. pylori</i> on human health that it does have potentially major effects on its host.  If the microbial ecology of the stomach is actually affected by these kinds of interactions, then clearly the explanation for the diversity of these organisms around the world is more likely in the realm of natural selection then in the realm of neutrality.  This means that <i>H. pylori</i> probably is useless in yielding new information about human migrations.  Instead, it should be considered to give valuable information about the history of human gut adaptations.

Nevertheless, the mention in the <em>Scientific American</em> article prompted me to look back at the original work on <i>H. pylori</i> variation in <em>Science</em>, in a 2003 paper by Falush and colleagues.  As in other examples, this paper tries to match the variation found in different populations around the world with a model of human dispersals.  Like other articles, it uses the same model of dispersals, a model in which humans began in Africa and then dispersed to Asia and Europe and much later dispersed to the New World and the Pacific islands.


As with other papers along the same lines, the question is whether the new data in the paper (in this case <i>H. pylori</i> variation) actually add anything new to what we think we know about ancient migrations.  It helps to have some understanding of what form the new data take.  In this study, the authors sequenced small parts of eight genes.  Each of these genes is functional, meaning that the alleles carried by each of the microbes for these eight genes probably affect their survival and dispersal capacities.  After sequencing, the researchers plugged the data into a computer program with a one guiding assumption: that originally all of these <i>H. pylori</i> variants came from a small number of discrete populations.  In other words, the study assumes a humans once belong to a small number of pure races, each carrying a single <i>H. pylori</i> variant.  It is a mixture among these ancient traces that is assumed to have led to the current variation of human genes, as well as the variation of the <i>H. pylori</i> genes.



This is an extraordinary assumption, but unfortunately not a rare one. it is certainly possible to write a computer program to reconstruct putative ancestors under this assumption.  Most researchers who apply an assumption like this do it because of the ease of such application, not because they believe the assumption is valid.  But in this case, as in most other cases, the problems with the assumption are ignored once the result is in hand.  Humans did not once belonged to five pure races, the number concluded by this instance of computer analysis.  <i>H. pylori</i> did not descend from five ancient strains.  It is very likely that there was as much diversity 50,000 years ago in <i>H. pylori</i> as there is today, and that that diversity was equally scrambled as we now observe. What has happened in the interim is a small number of major human migrations-such as the migration of people to the New World-and a high level of natural selection in <i>H. pylori</i> distributions.  It is probably impossible to say exactly what pattern of natural selection would lead to the observed distribution, but to assume that that selection was absent, or that it had only minor effects on the current pattern of variation, is clearly wrong.



This is not to say that the current pattern of variation is without structure, or that human geography is irrelevant to <i>H. pylori</i> variation.  In fact human variation and dispersal is probably the main contributor to <i>H. pylori</i> variability over short time spans.  However, over spans of hundreds or thousands of generations, the effect of selection substantially outweighs the effects of population history.  Even over short time spans, changes in <i>H. pylori</i> variation may reflect forces other than human population movements, or mechanisms other than human movement.  The researchers find a high frequency of European haplotypes around the world, and attribute this to recent movements of European peoples, especially since the Age of Exploration.  But it is not just European people that have moved since that time, it is also European livestock.  How much of the spread of this <i>H. pylori</i> variation stems from the movement of people, and how much stems from their cultural and technological baggage? For <i>H. pylori</i>, as for most microbes, these questions are not merely unanswered, they're usually unasked.



I wonder if over the past 50,000 years, we're at the point in time where many kinds of genetic evidence retained just enough neutrality to partly match geographic movements, but not completely so.  Not everything has been under selection recently in humans, and human populations have been partially isolated over time.  But as we consider earlier in earlier times, it is much more likely that selection will have occurred for any particular gene, and it is much more likely that dispersal between regions will have allowed the exchange of advantageous variation.  Is there a critical time before which we will tell nothing about ancient history from genetic variation? I doubt that there is any single time comprising an impenetrable barrier to our vision into the past.  But clearly as we move further and further into the past our vision must become foggier.  So echoes of our current geographic distribution may be stretched into the past for a longer period than might actually be justifiable. And over long periods of time, tens of thousands of years in length, the geographic distribution of people does exert constraints on the evolution of their genes.



In this study as in so many others, the new data actually add nothing new to our understanding of human movements.  The researchers find that they are consistent with their understanding of human movements from other sources.  "Consistent with" means "Provide no independent test of". And that's the bottom line, in a very real sense.  Humans are geographically variable.  This variation is structured by distance.  Under an assumption that we had a single origin, this geographic structure is consistent with sequential migrations over time. And some large-scale migrations have happened.  Taking these facts as a baseline, is it surprising that any human gene-or any human-associated microbe-should have a pattern of variation consistent with this?



"New" information would be obtained if we had an independent estimate of the date of such movement, or an independent estimate of the order of dispersals, or an independent estimate of the population sizes of the different regions in the past.  So far, <i>H. pylori</i> has given us none of those.  Nor has any other microbe. And only too rarely have human genes been interpreted in this way.  Until we begin to consider whether different genes actually match each other's patterns of variation will we be testing hypotheses about human prehistory.  Until then, we're just playing consistency games.


<h4>References</h4>

<p class="cite">Blaser, Martin J. "An endangered species in the stomach." Scientific American 292.2 (2005): 38-45. <a href="https://www.jstor.org/stable/26060877">JSTOR</a></p>

<p class="cite">Falush, D., Wirth, T., Linz, B., Pritchard, J. K., Stephens, M., Kidd, M., ... & Yamaoka, Y. (2003). Traces of human migrations in Helicobacter pylori populations. Science, 299(5612), 1582-1585. <a href="https://doi.org/10.1126/science.1080857">doi:10.1126/science.1080857</a>

