---
layout: single 
title: "Evaluating selection and demography in human evolution" 
category: story
permalink: /weblog/reviews/genomics/selection/williamson_selection_demography_2005.html
tags: [Late Pleistocene, natural selection, genomics] 
comments: false 
author: John Hawks 
---


<p>
Williamson et al. (2005) present a new mathematical method for deriving information about population size change and selection from the allele frequency spectrum of variation taken at multiple genetic loci. Their method depends on separating sites that are selected from those that are neutral, and thereby isolating the effects of demography from those of selection. They then apply their technique to human genetic data to derive estimates of the average selection on selected sites, and the timing and magnitude of population size change from nonselected sites. 
</p>

<p>
Oh, if it were really so easy. 
</p>

<h4>Selection</h4>

<p>
To be fair, the paper states a major concern with accurately identifying selection in the context of species like humans and <i>Drosophila</i> that have experienced recent population growth. In other words, the main interest is not in deriving evidence about human prehistory, but instead about making sure that estimates of selection are not biased by population growth. 
</p>

<p>
With respect to selection, their major conclusion is as follows:
</p>

<blockquote>We find evidence that negative selection on nonsynonymous mutations is widespread, which implies that deleterious mutations make up a significant proportion of standing nonsynonymous variation. Exactly how this genetic variation contributes to phenotypic variation is a matter of considerable debate, especially for medically interesting phenotypes such as multifactorial genetic disease. Because deleterious mutations, by definition, have phenotypic effects, and because of the widespread nature of negative selection on nonsynonymous mutations, it seems likely that negatively selected, generally rare nonsynonymous SNPs have some negative impact on human health. If there is a general relationship between nonsynonymous polymorphism and human genetic disease, then our genomic estimates of the fitness effects of different types of mutations contain prior information about the likelihood that a mutation contributes to disease. It may be possible to use this information to aid in identifying SNPs that cause disease. Other studies have suggested this approach (e.g., Livingston et al. 2004), but it was unclear which of the many measures of exchangeability to use. We feel that the relative fitness of different amino acid changes is the best way to evaluate exchangeability, and we have done that here by using a model that includes demography and selection (Williamson et al. 2005:7887).</blockquote>

<p>
Readers may note that other studies have found evidence for a very high proportion of <i>positive</i> selection across the human genome (discussed <a href="weblog/reviews/genomics/selection_genome-wide.html">in this post</a>). The test applied in the current paper is not well suited to detecting evidence of positive selection, particularly if it is widespread, because it depends on the difference in frequency spectra between "selected" and "neutral" sites. Why the scare quotes? Because although noncoding sites or synonymous SNPs may well be neutral in the literal functional sense of not being targets of selection, it is impossible to verify that they are <i>unlinked</i> to selected sites. For the purposes of detecting negative (purifying) selection, this is not such a problem, because linkage will affect nearby sites only weakly (although this weak effect, called "background selection," may well influence the average level of variation in <i>Drosophila</i>). 
</p>

<p>
In any event, even if positive selection has been very common across the genome, most sites that have been subject to positive selection should have been fixed long ago. Only a few should still be under selection now, and these are predominantly very recent mutations. 
</p>

<p>
Consider the following scenario. The study considered 301 human genes. According to common knowledge, repeated here, positive selection leads to a relative excess of high-frequency alleles, compared to the predictions of neutrality (which predicts that there should be very few high-frequency alleles). But these high-frequency variants represent only a small proportion of the total number of genes currently under positive selection, since an allele being driven to fixation passes through every intermediate frequency, not merely the high ones. To detect evidence for positive selection, this study would have to find dozens of high-frequency variants in excess of neutral theory, representing scores of selected genes. But suppose instead that only one positively selected gene actually was in the sample. If so, then out of the human genome of approximately 20,000 genes, we might expect to find 60 or 70 genes currently under positive selection. In our fictive scenario it would be rash to extrapolate from a sample of 1, but in fact there are good reasons to think the true number is much higher. One such gene might take 1000 generations to transit from its appearance to fixation. There have been 100,000 generations in the 2 million years since the origin of our genus, and at least 300,000 since our divergence from chimpanzees. In other words, the complete transformation of the human genome by positive selection, altering thousands of genes -- or even all of them, multiple times -- would be far from detectable by this test. 
</p>

<p>
But remarkably, this test <i>does</i> find evidence for positive selection -- in <i>noncoding substitutions</i>! The authors put it less sensationally: "Interestingly, we find marginal evidence for weak positive selection on noncoding indel polymorphisms" (7885). I have no explanation for it. But if there actually is a statistically detectable excess of high-frequency variants for these polymorphisms, it may reflect selection at linked sites, or issues with the composition of the sample. If the level of positive selection is detectable, it is another strong evidence of the power of such selection over the long timespan of human evolution. 
</p>

<p>
In contrast to positive selection, even very strong purifying selection may leave low-frequency variants within the population for a long time. These variants are picked up within samples in large numbers. Low frequency variants are predicted to make up most genetic variation under neutrality, so the proportion of such variants is always a substantial part of the sample. High numbers make for powerful tests. For the human data examined in this study, the nonsynonymous coding sites have a higher proportion of low-frequency variants than do the noncoding, synonymous sites. Thus, they provide strong evidence of negative (purifying) selection. 
</p>

<h4>Human demography</h4>

<p>
So the results of the method applied to selection are mixed. It detects the weak force of purifying selection strongly; it detects the strong force of positive selection weakly. But as the authors perceptively note, the inference of demographic history and the inference of selection are not independent of each other. Therefore, the inferences about demography are in part subject to the weaknesses in detecting the effects of past selection. This study shares this problem with all previous work that has attempted to estimate past human population size from genetic evidence. 
</p>

<p>
How can selection affect interpretations of demography? Here's one way: Positive selection occurs rapidly relative to rate of recombination between sites. This means that a selective sweep may affect a relatively large section of a chromosome, including many "neutral" sites. This is the principle behind John Gillespie's (2002) pseudohitchhiking, or "genetic draft" model of neutral evolution. In a nutshell, if positive selection has been common, there is no reason to think that genetic variation at noncoding sites provides any indication of demographic parameters. The current study (by Williamson et al. 2005) assumes that positive selection has not had such an effect, nor has any other force significantly affected the variation of neutral sites. 
</p>

<p>
These are the kinds of influences that have been suggested to result in the large difference between census population sizes (the number of individuals within living species) and estimates of effective population sizes (measures of the rate of genetic drift) in nature. In humans and in most other animal species, the rate of genetic drift on neutral sites appears to have been much stronger than the census population sizes of those species would predict. This is a systematic difference that leads species to have much lower genetic variation than would be expected if they evolved under genetic drift alone. At present, the relative importance of selection and demographic factors in leading to this systematic difference is unknown. I suspect that selection has been strongly important in this difference, others argue that demographic factors have been the most important. 
</p>

<p>
In most previous genetic work, the effective population size (denoted as <i>N<sub>e</sub></i>) is around 10,000 individuals. Some scientists have suggested that the human population actually was once that small -- that only a few tens of thousands of people once comprised the entirety of humanity. If this were true, then the human population must have expanded in size massively sometime in the recent past. The evidence for a recent change in the mitochondrial DNA molecule was once suggested to be evidence for this change in population size, which was inferred to have occurred during the Late Pleistocene, perhaps 50,000 years ago. From these estimates comes the scenario of an expansion from a single small African population beginning after 100,000 years ago, reaching Europe and the Far East by 30,000 - 50,000 years ago.
</p>

<p>
Recently, it has become clear that a single massive expansion of a global human population cannot explain the pattern of genetic variation in living people. Simply put, the pattern of the 16,000 base pairs of the mtDNA molecule is not replicated by the 3 billion base pairs of the nuclear genome. 
</p>

<p>
To be sure, some genes do show a pattern of recent ancestry and apparent expansion. The <i>FoxP2</i> gene, for example, has a recent common ancestor for living people (within the past 200,000 years), and shows strong signs that it has not evolved neutrally. If all other genes looked like this, it would be strong evidence of massive population growth. 
</p>

<p>
But most genes do emphatically <i>not</i> look like this. This has been understood for several years, following reviews by Molly Przeworski and colleagues (2000), Jeff Wall (2000), and even my own dissertation (Hawks 1999). Many genes show no excess of rare variants, most show only a slight excess. The average gene shows no sign whatsoever of a massive population expansion during the Late Pleistocene. This has been concluded most powerfully by recent genome-wide studies of SNP variation by Marth and colleagues (2003; 2004; reviewed previously in <a href="weblog/reviews/genetics/marth_bottlenecks.html">this post</a>). 
</p>

<p>
Where does the current paper (Williamson et al. 2005) come in? Summarizing evidence from over 300 genes, this study does find evidence of a population expansion. Yes indeed -- a population expansion that happened 18,000 years ago! This expansion took the human population from a previous size of around 8000 individuals to a <i>current</i> size of around 50,000 individuals. 
</p>

<p>
Of course these estimates are far from realistic in anthropological terms. If anything, 18,000 years ago much of the human population should have been <i>contracting</i> rather than expanding. The idea that the human population could have been as amll as 8000 individuals (or very generously 100,000 individuals) during the LGM is simply ridiculous. By that time, the certain ancestors of living people were present from the western tip of Iberia to the edge of (or possibly well into) Beringia. If a genetic estimate cannot gauge a population that must have numbered several millions of people, it is time to stop talking about genetic estimates. 
</p>

<p>
To be fair, the demographic conclusions of the paper are phrased cautiously:
</p>

<blockquote>Therefore, although we find it striking that the time of population growth (18,200 years B.P.) roughly corresponds with events in human history that may have induced population growth, such as the end of the last ice age and the origin of agriculture, we feel that our demographic inferences should be interpreted cautiously until the full range of plausible demographic models has been explored in one coherent framework (7887). </blockquote>

<p>
At the same time, this apparently cautious discussion raises the more critical problem of a complete lack of communication or citation from any anthropologist. Hmm, I guess the <i>last glacial maximum</i> does roughly correspond to the "end of the last ice age and the origin of agriculture," in the usual manner of genetics confidence intervals. That is to say, it is only twice as old as either, so it might as well be the same. 
</p>

<p>
Speaking of confidence intervals, again in this paper there are none. No confidence intervals on the demographic estimates, no confidence intervals in the supporting text, no figure showing the likelihood surface, none, nothing, nada. 
</p>

<h4>What remains?</h4>

<p>
In a sense, the bone I am picking is different from that pursued by Williamson et al. (2005). What I care about is evidence for ancient demography. What they care about is better quantifying selection. I think that their paper is incomplete on their own terms, because of the problem quantifying positive selection, but that it is a credible theoretical effort. In particular, the insights about the frequency of genetic disorders based on their findings are a likely contribution to the future study of genetic variation in coding gene regions. 
</p>

<p>
But the inclusion of demography in this study confuses much more than it clarifies from the perspective of the anthropologist. Its estimates of demographic changes are clearly false, and the lack of detail about confidence intervals makes them impossible to evaluate. In the face of this fatal problem, it is fair to wonder whether the apparent insights about purifying selection have any value. 
</p>

<p>
The main importance of the data from these many genes is what they do not show. They do not show an expansion of many orders of magnitude. They do not show a current effective size that is anywhere near the current human population size (or a size sufficient to settle any large part of the world). They do not show evidence for expansion coincident with an "out of Africa" movement of people, over 50,000 years ago. 
</p>

<p>
Instead, the conclusion is concordant with the discussion of Eswaran and colleagues (2005:3): 
</p>

<blockquote>Thus, the nuclear data do not consistently signal expansion, and when they do, the signal is of a mild expansion, perhaps reflecting only post-Pleistocene population growth associated with the spread of agriculture. </blockquote>

<p>
The summary of current work is that we can completely exclude the hypothesis that "neutral" genetic variation in humans is explained <i>entirely</i> by past human population size. It simply cannot be true, because if it were, there should be strong signs of expansion that we do not in fact observe.
</p>

<p>
On the other hand, perhaps genetic data may tell us <i>something</i> about past human population size, even if population size is not the <i>only</i> explanation for genetic variation. We might expect that some demographic changes may have influenced genetic variation in distinctive ways that could be separated from the effects of selection. If so, then the results of the current paper may be relevant. If natural selection -- especially purifying selection -- explains most rare alleles at nonsynonymous coding sites, then perhaps the residue of rare alleles at synonymous or noncoding sites is a sign of recent changes in demographic patterns? 
</p>

<p>
This possibility is suggestive, but it appears at present to be fairly far from the data. If unknown factors (which may include selection) have altered "neutral" genetic variation by an order of magnitude or more from their neutral predictions, then it is hard to believe that a relatively small change in population size will be accurately measured by any genetic observations. 
</p>

<h4>References:</h4>

<p class="cite">Eswaran V, Harpending HC, and Rogers AR. 2005. Genomics refutes an exclusively African origin of humans. J Hum Evol Online advance before print. </p>

<p class="cite">Gillespie JH. 2000. Genetic drift in an infinite population: the pseudohitchhiking model. Genetics 155:909-919. </p>

<p class="cite">Williamson SH, Hernandez R, Fledel-Alon A, Zhu L, Nielsen R, and Bustamante CD. 2005. Simultaneous inference of selection and population growth from patterns of variation in the human genome. Proc Nat Acad Sci USA 102:7882-7887. <a href="http://www.pnas.org/cgi/content/full/102/22/7882">PNAS online</a></p>

