---
layout: single 
title: "Mitochondrial DNA adaptations in living human populations" 
category: story
permalink: /weblog/reviews/neandertals/neandertal_dna/ruiz-pesini_selection_mtdna_2004.html
tags: [out-of-Africa, Neandertals, natural selection] 
comments: false 
author: John Hawks 
---


<p>
I had read <a href="http://www.sciencemag.org/cgi/content/full/303/5655/223">this paper by Ruiz-Pesini et al. (2004)</a> before, but the particular combination of factors it suggests came together for me in a new way recently: 
</p>

<blockquote><b>Effects of Purifying and Adaptive Selection on Regional Variation in Human mtDNA</b></blockquote>

<blockquote>A phylogenetic analysis of 1125 global human mitochondrial DNA (mtDNA) sequences permitted positioning of all nucleotide substitutions according to their order of occurrence. The relative frequency and amino acid conservation of internal branch replacement mutations was found to increase from tropical Africa to temperate Europe and arctic northeastern Siberia. Particularly highly conserved amino acid substitutions were found at the roots of multiple mtDNA lineages from higher latitudes. These same lineages correlate with increased propensity for energy deficiency diseases as well as longevity. Thus, specific mtDNA replacement mutations permitted our ancestors to adapt to more northern climates, and these same variants are influencing our health today.</blockquote>

<p>
Here are the final two paragraphs of the paper: 
</p>

<blockquote>This combination of the increased predilection to energy deficiency diseases, but protection from neurodegenerative diseases and aging is consistent with the expectations for mtDNA coupling efficiency mutations. Uncoupling mutations would reduce ATP production, increasing the probability of energetic failure. However, they would also decrease mitochondrial ROS production by increasing the oxidation of the electron transport chain, thus reducing oxidative damage and apoptosis. This could decrease neuronal and other cell loss, thus increasing longevity.</blockquote>

<blockquote>Our observations support the hypothesis that certain ancient mtDNA variants permitted humans to adapt to colder climates, resulting in the regional enrichment of specific mtDNA lineages (haplogroups). Today these same variants result in differences in energy metabolism and altered mitochondrial oxidative damage, thus affecting health and longevity. Therefore, to understand individual predisposition to modern diseases, we must also understand our genetic past, the goal of the new discipline of evolutionary medicine (Ruiz-Pesini et al. 2004:226).</blockquote>

<p>
So in other words, populations in northern latitudes today are enriched for a number of mtDNA haplogroups that are likely adaptive to cold. Today, these haplogroups (as a class) are largely protective against degenerative diseases of aging, possibly because they reduce oxygen free radical production. But they are also more susceptible to disorders of energy metabolism, because they reduce ATP production. 
</p>

<p>
Needless to say, this says some interesting things about the relationship of longevity and energy metabolism in recent human populations. 
</p>

<p>
But at the moment, I'm thinking about Neandertals. They lived in a cold place, but their lifestyle suggests that energy metabolism was at a premium. At the same time, they had a much shorter maximum lifespan than living people. According to the model of mtDNA mutations outlined by Ruiz-Pesini et al. (2004), this would be a very odd combination: cold adaptation today is linked to longevity and lower energy metabolism; Neandertals required high metabolism but had lower longevity. 
</p>

<p>
Those functional considerations alone suggest that Neandertals needed a highly specialized mtDNA type that would have been unlike those of living people. 
</p>

<p>
But additionally, the increase in longevity and difference in lifestyle apparent in later Upper Paleolithic people gives a clear reason for the replacement of the Neandertal mtDNA type. These people lived longer, and they had markedly less energy expenditure than Neandertals did. Their dietary and cultural adaptations would have been much more similar to recent arctic peoples (and indeed, might well have been completely identical to the <i>ancestors</i> of recent arctic peoples). 
</p>

<p>
Would this have been an exceptional event? I don't really think so, because Ruiz-Pesini et al. (2004) outline how similar cold-adaptive mutations occurred in different macrohaplogroups that today are all present at higher latitudes. The occurrence of potentially adaptive mtDNA mutations appears to have been quite a common event throughout human prehistory, because today's haplogroups appear to be separated by many mutations that are adaptive in different contexts. 
</p>

<p>
The situation is reviewed in two papers by Douglas C. Wallace (2005a, 2005b). The thing that surprised me about these two reviews is that they embrace a positive selection hypothesis for mtDNA migration out of Africa. Consider: 
</p>

<blockquote>This mtDNA history is remarkable for the striking discontinuities that exist in mtDNA diversity between climatic zones. Of the extensive mtDNA variation present in Africa only two mtDNA lineages (M and N) succeeded in colonizing all of Eurasia. Of the plethora of Asian mtDNA types that subsequently accumulated, only three haplogroups (A, C, and D) and much later G came to occupy the extreme northeastern Chukotka Peninsula of Siberia. This strikingly correlation between mtDNA lineages and latitude and climate led to the hypothesis that mutations in certain mtDNAs that decreased the coupling efficiency increased mitochondrial heat production and permitted people to survive the cold of the more northern latitudes (Ruiz-Pesini et al., 2004) (Wallace 2005a:173).</blockquote>

<p>
This passage is followed by a section to support it, of which I find several parts very suggestive (my emphasis): 
</p>

<blockquote>This hypothesis is supported by the fact that missense mutations in mtDNA protein genes show regional specificity. Missense mutations are prevalent in the ATP6 gene in the arctic, in the cytb gene in Europe, and in the COI gene in Africa. Mutations in different ND genes also show regional correlation (Mishmar et al., 2003). <b>Moreover, many of the ancient missense mutations change amino acids that are as highly evolutionarily conserved as are most known pathogenic mutations, yet have been retained in the human population for tens of thousands of years. Hence, they could not be pathogenic in the environment in which they reside, but rather must be adaptive and thus beneficial.</b> Furthermore, an analysis of the missense mutations in cytb of complex III, for which the crystal structure is known, revealed that many of these missense mutations affected CoQ binding sites which would alter the Q-cycle, proton pumping, and thus OXPHOS coupling (Ruiz-Pesini et al., 2004).</blockquote>

<blockquote>Finally, when European mtDNA haplogroups were correlated with longevity and predisposition to Alzheimer Disease (AD) and Parkinson Disease (PD), it was found that mtDNAs harboring uncoupling variants were enriched in the elderly and deficient in AD and PD patients. This led to the conclusion that the uncoupling mutations must enhance the flow of electrons through the ETC keeping the ETC carriers oxidized. This, in turn, reduces the spurious transfer of electrons to O2 thus minimizing ROS production and reducing mitochondrial and cellular damage.</blockquote>

<blockquote>These same uncoupling mutations would also reduce the efficiency of ATP generation <b>which could then exacerbate ATP deficiencies resulting from milder mtDNA mutations</b>. This could account for the predilection of patients with Leber's Hereditary Optic Neuropathy (LHON) that harbor the milder mtDNA mutations to also have haplogroup J mtDNAs which harbor either the np 14798 or np 15257 cytb missense mutations. Thus ancient adaptive mtDNA variants are affecting individual predisposition to degenerative diseases and aging today.</blockquote>

<p>
This last one also would apply to milder slightly deleterious mutations of nuclear genomic loci that contribute to mitochondrial metabolism. One might even conclude that today's mitochondrial degenerative disorders may in part be a legacy of ancient adaptive mtDNA variants that no longer exist. 
</p>

<p>
Now, if we seriously accept the hypothesis that human mtDNA variation is regionally adaptive, then we have to conclude that a lot of literature that assumes mtDNA neutrality is just wrong. For example: 
</p>

<ol>
<li>If mtDNA is neutral, then the dates of mtDNA lineage divergences may tell us about the initial migrations of some human populations. If mtDNA is adaptive to different regions, then the dates of lineage divergences tell us about the times that adaptive mutations occurred. </li>
<li>If mtDNA is neutral, then it is surprising that archaic mtDNA variants are gone. If mtDNA is selected, then this is not at all surprising: the current global mtDNA variation is simply the product of the last globally adaptive mutation. </li>
<li>If mtDNA is neutral, then it is reasonable to explain the lack of ancestral haplogroup L outside of Africa as the consequence of an out-of-Africa population bottleneck. But if mtDNA is selected, this distribution is explained as Wallace (2005b:376) suggests: many mtDNA lineages may have entered Eurasia, but only a few survived local selection. </li>
</ol>

<p>
Is it possible that there have been no globally adaptive mutations? If the present pattern of variation is fine-tuned to climate and diet, it seems very unlikely that the massive life history, brain, and energetic changes during Pleistocene human evolution had no effect whatsoever. 
</p>

<p>
The present distribution of adaptive mtDNA variants suggests a scenario for the replacement of Neandertal mtDNA. Variants of human mtDNA that appear to be adaptive in Eurasia, and particularly in the northern parts of Eurasia, evolved recently upon an African background. The present variation of human mtDNA is comparatively recent, but it is ancient enough that some of today's variants were segregating within Africa over 100,000 years ago, and the haplogroup M dispersal from Africa appears to have occurred between 60 and 70 thousand years ago. Ultimately these gave rise to European haplogroups H, T, U, V, W, X, I, J, and K (Wallace 2005b), in the time period between 50,000 and 9,000 years ago. 
</p>

<p>
We know that these variants were superior to indigenous European mtDNA variants because the Neandertal mtDNA is gone today. Yet, we must suspect that the Neandertal mtDNA would have been very well adapted to their cold climate and high energetic requirements. The advantages of the incoming African-derived mtDNA variants were great, but they would not have been free of disadvantages -- especially with respect to either cold (which has historically restricted non-European mtDNA haplogroups to the south) or energy metabolism (which currently restricts European mtDNA haplogroups to the north). 
</p>

<p>
Thus, the replacement of Neandertal mtDNA could occur only upon the abandonment of Neandertal lifeways. Only a reduction in energy expenditure and exposure to cold could allow the spread of the African-derived mtDNA variants. Both these changes could be accomplished by a cultural transition, which additionally could increase dietary supply and thereby change selective constraints on energy efficiency. 
</p>

<p>
In this context, it is very significant that the latest Neandertals adopted Upper Paleolithic tool industries and other cultural elements usually associated with modern humans. This cultural transition may have decreased the selective advantages of endogenous European mtDNA variants and allowed the substitution of newer European variants of African derivation. In other words, it may have been the very process of adopting new cultural and demographic patterns that resulted in the selection against old Neandertal mtDNA, even within the European Neandertal population. 
</p>

<p>
So far, I have said nothing of what benefit the African-derived mtDNA variants may have provided. It seems likely that it was not related to cold (considering the Neandertals had plenty of time to become cold-adapted), energy (considering that the Neandertals appear to have had higher total energy expenditure than later people), or diet (since Upper Paleolithic people had broadly similar (if slightly different) diets to Neandertals). 
</p>

<p>
Instead, I would propose that the advantage of the African-derived mtDNA variants was in the one area (out of mtDNA-associated factors) where Neandertals and later humans significantly differ: longevity. It is not at all obvious that living longer is a better adaptation for humans as compared to the shorter lifespan of Neandertals. As a very recent adaptive change, it may have required fairly exceptional demographic conditions, such as large population sizes, a reliance on extensive trade networks, or other behavioral attributes of recent people. Only in such a cultural context can the survival of older individuals provide a fitness advantage to their younger kin. 
</p>

<p>
I do not think that the mtDNA change was the most important one; it probably followed many other genomic changes in favor of longevity. This would be similar to the effect of mtDNA variants in the face of climate or dietary differences today: no population was likely <i>restricted</i> from inhabiting the arctic by the lack of favorable mtDNA variants, but the fast mutation rate of mtDNA ensured that populations living in the arctic quickly gained new adaptive variants for their cold climates. Likewise, other genetic changes that led to a longer lifespan would quickly have led to mtDNA variants adaptive to the new demographic reality. The global human mtDNA variability likely reflects such trailing adaptive mutations. This might imply that the transition to greater longevity or other aspects of modern human life history would have been accompanied by not one, but multiple adaptive sweeps of global mtDNA variation. 
</p>

<h4>References:</h4>

<p class="cite">Ruiz-Pesini E, Mishmar D, Brandon M, Procaccio V, Wallace DC. 2004. Effects of purifying and adaptive selection on regional variation in human mtDNA. Science 303:223-226. <a href="http://www.sciencemag.org/cgi/content/full/303/5655/223">Full text (subscription)</a></p>

<p class="cite">Wallace DC. 2005a. The mitochondrial genome in human adaptive radiation and disease: On the road to therapeutics and performance enhancement. Gene 354:169-180. <a href="http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6T39-4GMS986-1&_coverDate=07%2F18%2F2005&_alid=314247286&_rdoc=1&_fmt=&_orig=search&_qd=1&_cdi=4941&_sort=d&view=c&_acct=C000020958&_version=1&_urlVersion=0&_userid=443835&md5=adcc94a3e466d6650ef2879617448ed1">Full text (subscription)</a></p>

<p class="cite">Wallace DC. 2005b. A mitochondrial paradigm of metabolic and degenerative diseases, aging, and cancer: a dawn for evolutionary medicine. Annu Rev Genet (<a href="http://arjournals.annualreviews.org/doi/abs/10.1146/annurev.genet.39.110304.095751">Online before print</a>)</p>

