---
layout: single 
title: "Selection, nuclear genetic variation, and mtDNA" 
category: story
permalink: /weblog/reviews/neandertals/neandertal_dna/weaver_roseman_2005.html
tags: [Neandertals, Late Pleistocene] 
comments: false 
author: John Hawks 
---


<p>
Weaver and Roseman (2005) review the case for Neandertal extinction based on ancient mtDNA. They present simulations that demonstrate that a Neandertal contribution to the modern human mtDNA pool is very unlikely. These simulations are simpler than those carried out by Currat and Excoffier (2004), but considering the publication schedule of <i>Current Anthropology</i>, they were probably completed earlier. Their conclusion is the same: if mtDNA has not undergone a selective sweep, then Neandertals almost certainly became extinct without issue. 
</p>

<p>
But that's not especially news: indeed, it was strongly suspected even before ancient Neandertal mtDNA sequences were discovered (e.g. Manderscheid and Rogers 1996). The question has been, and remains, is human mtDNA actually neutral? Or is its recent pattern of variation in living humans the result of recent selection within the human lineage? If there was a selective sweep in humans, then the mtDNA of Neandertals shouldn't look like modern human mtDNA for that reason. This wouldn't prove that Neandertals contributed other genes to later people, but it would make their mtDNA variation irrelevant to their ultimate fate. 
</p>

<p>
Unlike most papers on the topic, Weaver and Roseman (2005) review this issue. On the basis of their review, they conclude that mtDNA is almost certainly neutral. Here's how they put it: 
</p>

<blockquote>For the selective-sweep alternative to be correct there would have to have been virtually simultaneous selective sweeps in hundreds of unlinked genetic regions, which seems unlikely (Weaver and Roseman 2005:682).</blockquote>

<p>
That does indeed seem unlikely, so it might seem that they have made a solid case. 
</p>

<p>
But -- unsurprisingly to those who read the weblog often -- I think they have left out many important aspects of the story. There <i>is</i> a strong case for mtDNA selection, but Weaver and Roseman (2005) omit much of the data that point to that conclusion. And some of the data that they <i>include</i> actually indicates quite the opposite of what they claim. 
</p>

<p>
They make a very common assumption -- one that is widespread in the human genetics literature -- but one that is nonetheless wrong: every expansion is the same expansion. A review of the sources that Weaver and Roseman (2005) cite shows quite the opposite: expansions are not all alike, and expansions estimated for nuclear DNA may actually <i>prove</i> that mtDNA was selected. 
</p>

<p>
This is a very long post, and I have hidden most of it beneath the fold. Click on in if you want my take on selection on mtDNA....
</p>

<!-- more -->

<h4>Is it reasonable to think that mtDNA was selected?</h4>

<p>
Before embarking on a review of Weaver and Roseman's argument, it is important to tackle one central question: Is it even reasonable to think that mtDNA was selected? 
</p>

<p>
If this were an unreasonable idea, there would be no point to arguing for it. So what reason <i>might</i> one have to think that a selective sweep of mtDNA <i>might</i> have happened?
</p>

<p>
Within the past 30,000 to 1 million years, human populations have changed radically in longevity (Caspari and Lee 2004), brain size (Lee and Wolpoff 2003; Ruff et al. 1997), diet, and energetics (Leonard and Robertson 1997; Sorensen and Leonard 2001). Human mtDNA variants have been found to be associated with chronic diseases of aging , brain disorders (Zhu et al. 2004), performance in athletes (Niemi and Majamaa 2005), and longevity itself (Niemi et al. 2005). The present pattern of variation also appears to be correlated with climate (Ruiz-Pesini et al. 2004), and may affect the dietary energetics and insulin metabolism (Lowell and Schulman 2005). 
</p>

<p>
Simply put, variation in mtDNA is a strong target for further research into the effects of aging, metabolism, and disorders of the brain for a reason: it impacts all these areas strongly. 
</p>

<p>
Together, these facts strongly suggest that human mtDNA may have undergone multiple adaptive substitutions within the past million years. They don't prove that such selection happened, but they give abundant reason to suppose that it might have. Indeed, Ruiz-Pesini et al. (2004) suggest that adaptive selection has happened on mtDNA in some regions of the world in recent times. This suggestion is fully consistent with -- and even foreshadows -- the idea that mtDNA underwent many adaptive substitutions during human evolution. 
</p>

<p>
In fact, this is <i>exactly the same logic</i> by which many nuclear genes have been asserted to have been positively selected recently in human evolution. Consider the case of <i>FoxP2</i>. Enard et al. (2002) proposed that this gene had undergone a selective sweep within the past 200,000 years in humans, and Klein (2002) made it the centerpiece of his argument that language had evolved recently at the origin of modern humans. Like mtDNA, <i>FoxP2</i> is strongly out of mutation-drift equilibrium. But human <i>FoxP2</i> shows <i>many fewer</i> amino acid substitutions compared to chimpanzees than does human mtDNA (2 for <i>FoxP2</i>, 50-60 for mtDNA), meaning that the possibility of selection on human mtDNA should be <i>greater</i>, not less. And <i>FoxP2</i> is only one functional gene; mtDNA contains 13 peptide-coding regions (Arnason et al. 1996), selection on any one of which would affect the entire molecule. 
</p>

<p>
So what exactly is the difference that leads the same people to say that <i>FoxP2</i> is selected and mtDNA is not? There is no statistical test of selection that shows <i>FoxP2</i> to be selected and mtDNA not. Human mtDNA completely fails standard tests of neutrality such as Tajima's <i>D</i> (Merriwether et al. 1991), the ratio of synonymous to nonsynonymous substitutions (Wise et al. 1998), and comparison of within-species to between-species diversity (Wise et al. 1997). 
</p>

<p>
In fact, the only difference I can find is that there is a literature that <i>assumes</i> mtDNA neutrality and attempts to use its variation to determine what kind of <i>neutral event</i> could explain its variation without selection. The only kind of event that suffices is a massive expansion of the human population -- estimated to be a hundred-fold or greater -- from an initial effective population size of fewer than 10,000 individuals to many millions (Harpending et al. 1998; 1993; Sherry et al. 1994). This event is proposed to have occurred anytime from 40,000 years ago to as much as 150,000 years ago or longer -- although the data indicates that it must have occurred earlier in Africa and later in Europe and East Asia. 
</p>

<p>
There is no history of such an assumption for <i>FoxP2</i> (although it might equally be suggested to represent such an event), therefore its variation is logically assumed to represent a selective sweep. 
</p>

<h4>Population expansions</h4>

<p>
Thus, the issue of mtDNA selection cannot be separated from the issue of population expansion. It should be noted that an expansion of the human population does not <i>disprove</i> the hypothesis that a selective sweep occurred, but it does provide an <i>alternative hypothesis</i> that might explain the pattern of variation (although not the involvement of mtDNA in energy metabolism, diseases of aging, brain disorders, etc.). 
</p>

<p>
Happily, we can test this alternative. Other genes, in particular, all neutral regions of the nuclear genome, must reflect the same demographic history as mtDNA. Thus, these genes should also show evidence of a massive population expansion. 
</p>

<h4>What's an order of magnitude between friends?</h4>

<p>
Here's what Weaver and Roseman (2005:681-682) say about nuclear genomic evidence for population expansions: 
</p>

<blockquote>We believe that the close correspondence between studies of mtDNA and microsatellites and recent studies of autosomal and X-chromosome SNPs that controlled for ascertainment bias and population subdivision makes it reasonable to assume a model of rapid human population growth from a small size, starting 200,000-40,000 years ago. </blockquote>

<p>
The microsatellite study they cite is the review by Zhivotovsky et al. (2003). Here's what Weaver and Roseman (2005:681) say: 
</p>

<blockquote>Most populations show strong signatures of population growth with estimated start times that are consistent with those for mtDNA.</blockquote>

<p>
Here are the actual figures from Zhivotovsky et al. (2003:1179): 
</p>

<table border="1">
<tr><td>Estimate:</td>
<td>Africa <br />(Hunter-gatherers)</td>
<td>Africa <br />(farmers)</td>
<td>Eurasia</td>
<td>East Asia</td>
</tr>
<tr><td>Estimated expansion time (kya)</td>
<td>4.3</td><td>35.3</td><td>25.3</td><td>17.6</td></tr>
<tr><td>Effective population size before growth</td>
<td>2609</td><td>1883</td><td>1760</td><td>1688</td></tr>
</table>

<p>
In short, these populations are all estimated to have expanded from an effective size of less than 2000 at a time between 17,000 and 35,000 years ago (except hunter-gatherers, who are estimated to have expanded 4300 years ago). 
</p>

<p>
Weaver and Roseman (2005:681) cite a wide range of estimates for mtDNA expansion times: from 200,000 to 40,000 years ago. This range of estimates is wide mainly because of uncertainty about the mtDNA mutation rate. In their simulations, Weaver and Roseman (2005:679) assume a growth starting at 40,000 years ago; this is at or near the <i>latest possible</i> date of expansion for <i>Europe only</i> from other studies (e.g. Harpending et al. 1993; Sherry et al. 1994). Weaver and Roseman (2005:679) assume a preexpansion female effective size of 1000. 
</p>

<p>
Compared to these mtDNA estimates, the microsatellite estimates are not so bad. They are more recent than the <i>most recent possible</i> for mtDNA by a factor of two, but their preexpansion population size is consistent. 
</p>

<p>
What about the SNPs? The autosomal and X chromosomal SNP study they refer to is Marth et al. (2004). Weaver and Roseman (2005:681) say this: 
</p>

<blockquote>They concluded that all samples showed signatures of population growth consistent with the results for mtDNA. The East Asian and European-American diversity fit a model of a bottleneck followed by growth while the African-American sample fit a model of growth alone. Marth et al. estimated that post-bottleneck growth began 84,000-60,000 years ago for the East Asian sample and 86,000-54,000 years ago for the European-American sample, which overlaps the 200,000-40,000 years ago range for mtDNA. They estimated that growth in the African-American sample started earlier.</blockquote>

<p>
Here are the actual figures from Marth et al. (2004:360), for the best-fit models (bottleneck for European and Asian, expansion for Africa), with generations converted to years (assuming 20-year generations): 
</p>

<div style="text-align:center;">
<table border="1">
<tr><td>Estimate:</td><td>European</td><td>Asian</td><td>African</td></tr>
<tr><td>Original population size</td>
<td>10,000</td><td>10,000</td><td>10,000</td></tr>
<tr><td>Bottleneck size</td>
<td>2000</td><td>3000</td><td>N/A</td></tr>
<tr><td>Bottleneck duration</td>
<td>10,000</td><td>12,000</td><td>N/A></td></tr>
<tr><td>Expansion time</td>
<td>60,000</td><td>64,000</td><td>150,000</td></tr>
<tr><td>Final population size</td>
<td>20,000</td><td>25,000</td><td>18,000</td></tr>
</table>
</div>

<p>
The sharp-eyed will notice a couple of things about these tables. First, the initial population size between the microsatellite estimates and the SNP estimates is different by an order of magnitude. Now, that has an immense effect on the <i>coalescence times</i> expected for autosomal genes. The initial effective size of 10,000 in the SNP study recognizes that autosomal genes have coalescence times ranging from as little as 200,000 years (or less) to as ancient as 3 million years (or older). This range of dates is simply inconsistent with a long-term effective size of 1000-2000. This inconsistency alone means that the microsatellite estimates must be wrong. 
</p>

<p>
This is a more severe problem than it might appear, because the signature for population expansion from the microsatellites <I>requires</i> a very, very small initial population size. This is because the estimates are based on the variance of allele size taken among sites -- meaning that different loci must have nearly exactly the same coalescent time to show a sign of population expansion (Kimmel et al. 1998; Zhivotovsky et al. 2000). The methods used to substantiate an expansion on microsatellite data simply lack the power to detect an expansion from a initial size as great as 10,000. 
</p>

<p>
In other words, expansions are <i>not</i> all alike: the "expansions" estimated for the SNPs lie outside the statistical power of microsatellites entirely; the expansions estimated to explain microsatellite data are absolutely inconsistent with the large initial population size estimated for SNP data. Both these kinds of loci are autosomal: their evolution -- if neutral -- must follow precisely the same constraints. 
</p>

<p>
Of course, the dates are also different, by a factor of three or more. But the other key difference is that these SNPs indicate not a simple expansion, but a <i>bottleneck</i>. 
</p>

<p>
Can this bottleneck be consistent with the diversity of mtDNA? On the surface, it might seem that a post-bottleneck expansion and an expansion are the same thing. And Fay and Wu (1999) showed that certain kinds of bottlenecks might be consistent with mtDNA disequilibrium and nuclear DNA equilibrium. But the bottleneck estimated by Marth et al. (2004) is much less severe than the simulations of Fay and Wu (1999): [CORRECTION 9/5/05: these bottlenecks simulated by Fay and Wu (1999) are] three times longer (30,000 years) and involves one-half to one-third the population size during the bottleneck. In contrast, the simulations that are most consistent with the estimates of Marth et al. (2004) show that no large effect is expected upon mtDNA variation. 
</p>

<p>
The same conclusion may be drawn from the simulations presented by Ambrose (1998), who tested whether a bottleneck associated with the Toba volcanic event 71,000 years ago might be consistent with human mtDNA variation. These simulations found that such a bottleneck could not be excluded by mtDNA variation. But they also found that the bottleneck could not by itself <i>explain</i> the pattern of mtDNA variation. Instead, a more ancient reduction in population size must have occurred, if mtDNA is neutral. Such a reduction has not been found for nuclear genomic data. 
</p>

<p>
So, the evidence for population expansion from nuclear DNA is <i>not</i> consistent with mtDNA variation. Nor are estimates taken from microsatellites consistent with those taken from nuclear SNPs. All bottlenecks and expansions are simply <i>not alike</i>. Weaver and Roseman (2005) imply that these different sources of evidence are converging on a single answer. In fact, they are <i>diverging</i> from each other. 
</p>

<h4>Why isn't an expansion just an expansion?</h4>

<p>
It is important to keep in mind the parameters of the possible models, to understand why evidence for expansions is not all alike. The simplest model of ancient demography is a one-parameter model: a single population size, unchanging over time back to infinity. This is the hypothesis of "no expansion", and it is in fact an assertion: the assertion that a model with more parameters does not explain the data better than the one-parameter model. 
</p>

<p>
The next simplest model of demography is a three-parameter model. The parameters are the population size <i>before</i> a change, the population size <i>after</i> the change (or alternately, the magnitude of the change), and the <i>time</i> that the change happened. (A two-parameter model would include only time and magnitude of change; as long as the actual size of the population is important to us we are stuck with the third parameter.) This kind of model is often called a "two-epoch" model, meaning that the population was one size for some period of time, and another size for a second period of time. The only two-epoch demographic models are simple expansions and crashes. 
</p>

<p>
A population crash causes genetic drift to increase -- and genetic drift tends to eliminate rare alleles from the population. So populations that have undergone a crash are expected to show a deficit of rare (low-frequency) alleles, or a surplus of high-frequency alleles. (This, by the way, is also the prediction of balancing selection.) 
</p>

<p>
In contrast, a population expansion reduces the strength of genetic drift, meaning that rare (new) alleles should be more common than expected if population size had been constant. It takes a while for these new alleles to appear, so the strength of evidence depends on the time of the expansion -- that third parameter. 
</p>

<p>
Now, let's stop to notice a couple of things. First of all, microsatellites are different from SNPs in that SNPs are often unique mutations, whereas microsatellite alleles are length polymorphisms that can be arrived at by lots of different mutations. This means that "rare" microsatellite alleles do not provide the same kind of evidence for expansion that rare SNPs do. In practice, estimates of ancient demography from microsatellite data do not depend on rare alleles at all; instead, they depend on the pattern of allele size variation among different microsatellite loci. That's one reason why the results of the microsatellite and SNP studies look so different: they are using different, apparently incommensurable, observations of variation. 
</p>

<p>
Second, the consideration of the two-epoch model shows two options: a population crash or a population expansion. But one of these -- the crash -- is extraordinarily unlikely to be true for humans. 
</p>

<p>
For one thing, the human population <i>really did</i> expand in size recently. Not only was there an incredible explosion of populations after the advent of agricultural subsistence, but also there is good archaeological evidence for substantial population expansions in the Late Pleistocene (e.g. Stiner et al. 2000). 
</p>

<p>
For another, the human population is subdivided into many populations -- a structure that itself increases the proportion of rare alleles in the global population (Ptak and Przeworski 2002). So a population crash is essentially not an option. If the data significantly refute the one-parameter model (i.e. constant size), then expansion is the only kind of three-parameter model at play. 
</p>

<p>
In other words, human genetic data are biased. They <i>ought</i> to show evidence of strong population expansion. They <i>ought</i> to be inconsistent with constant population size back to infinity. 
</p>

<p>
So why, in so many cases, aren't they?
</p>

<h4>No expansion? Are you kidding?</h4>

<p>
As Weaver and Roseman (2005:681) note, Ptak and Przeworski (2002) reviewed more than 400 genomic regions and found no substantial evidence for expansion. And the "expansion" found by Marth et al. (2004; likewise Marth et al. 2003) is not the manyfold population growth that actually happened: it is an expansion from 10,000 people to 18,000 (at a minimum) or 25,000 (at a maximum) (!). As we have seen, only microsatellite data look remotely like the <i>magnitude</i> of human population growth, but they are completely wrong about the initial size and time of expansion. 
</p>

<p>
There is a simple answer: the proportion of rare alleles is affected by things besides population size. As Ptak and Przeworski (2002), population subdivision is one of these. As Polanski and Kimmel (2003) note, ascertainment bias may be another. And natural selection is very likely to be another influence on the proportion of rare alleles -- even at "neutral" sites, considering the effects of linkage to selected sites (Gillespie 2000). 
</p>

<p>
And as discussed by Marth et al. (2004), nuclear SNP data actually do not fit the three-parameter model. For non-African populations, they fit a <i>five</i>-parameter model: a bottleneck. This model reflects a mix of observations at different sites: there is an excess of low-frequency (rare) alleles, at the same time there is an excess of high-frequency alleles (Sherry 1996), as compared to both the three-parameter and one-parameter models. This excess of common alleles does not greatly reduce the appearance of a slight expansion in the three-parameter model, so it cannot account for the mismatch between genomic variation and archaeological data. But it may also result from the effects of selection, as certain SNPs may have been driven to high frequencies by positive or balancing selection. 
</p>

<p>
Eswaran et al. (2005) suggest another explanation for the signature of a bottleneck in nuclear SNPs. They find that such a pattern is the expected result of the assimilation of archaic human lineages into an expanding modern human population. This pattern contrasts strongly with the expected signature of population expansion under a replacement scenario of modern human origins. They conclude that archaic assimilation is more consistent with the pattern of genomic SNP data than replacement. 
</p>

<p>
As we can see, the explanation of nuclear genomic variation requires the consideration of many complexities that may affect the outcome. These complexities, taken together, mean that nuclear DNA variability bears no simple causal relationship with ancient population sizes. In particular, it cannot replicate the pattern of Upper Paleolithic and Holocene expansions that are reconstructed from archaeological data. Instead, estimates based on genetics (under assumptions of neutrality in up to five-parameter models) are generally more than an order of magnitude different. 
</p>

<p>
It is therefore incorrect to say that nuclear genomic evidence is consistent with mtDNA variation. In fact, it currently appears to be <i>inconsistent</i>, although it is fairer to say that we do not know the relevance (if any) of genomic variation to demographic reconstruction. Strikingly, one of the few ways to <i>make</i> these different sources of data consistent with each other may be to <i>require</i> the survival and proliferation of nuclear gene lineages from archaic humans (Eswaran et al. 2005). Populations did grow, and and this growth may have affected the pattern of mtDNA variation. But the inconsistencies are great enough that they cannot currently be explained by demography alone. 
</p>

<h4>Ancient genes and geography: the data Weaver and Roseman omit</h4>

<p>
Weaver and Roseman (2005) substantiated their assertion that mtDNA is neutral by using four primary references: Marth et al. (2004), Zhivotovsky et al. (2003), Ptak and Przeworski (2002), and Pritchard et al. (1999). As reviewed above, one of these (Ptak and Przeworski 2002) contradicts their argument, as do two others mentioned more briefly (Pereira et al. 2001; Hammer et al. 2003). 
</p>

<p>
But there are other relevant sources of information not included in the paper. Three of them are absolutely critical, since they bear directly on the hypothesis that genetic material from archaic humans survives in present human populations. 
</p>

<p>
The first category of "missing information" are the many studies of gene-geography relationships by Alan Templeton and colleagues (e.g., Templeton 2002). Based on the examination of the geographic variability of a dozen nuclear genes, these studies have concluded that an Out of Africa replacement of archaic humans cannot explain the pattern of human genetic diversity. Instead, the studies find significant evidence for ancient genetic structure including Europeans and East Asians. Templeton has argued (2002) that the pattern of mtDNA (and Y chromosomal) variation may represent one recent migration among many out of Africa; but the data are also consistent with a recent selective sweep. These data conclusively refute the hypothesis that no archaic gene lineages survived into living human populations. 
</p>

<p>
Templeton's studies have been critiqued on the grounds that they may not actually distinguish survival of archaic gene lineages <i>outside</i> Africa from survival of archaic lineages <i>inside</i> Africa. In other words, it has been claimed (Pearson 2003; Eswaran et al. 2005) that ancient population structure <i>within</i> Africa might mimic the survival of gene lineages from outside Africa. Eswaran et al. (2005) show that this scenario is likely not the case, as nuclear genomic data apparently reflect the survival of archaic non-African lineages. But this equivocation bears little importance to the explanation of mtDNA: even the survival of archaic lineages from <i>within</i> Africa challenges the idea that mtDNA variation reflects the expansion of one small African population and the displacement of others. Instead, survival of archaic African lineages suggests that the ancient population size of Africans was effectively much larger (perhaps many orders of magnitude larger) than a neutral mtDNA hypothesis would admit. Together Templeton's work and the analysis of Eswaran et al. (2005) indicate that the majority of genomic loci preserve allelic variation that originally characterized archaic human populations. 
</p>

<p>
The second category of evidence missing from Weaver and Roseman (2005) also bears on this issue of archaic survival. In addition to the genomewide analyses and Templeton's phylogeographic studies, both of which suggest the survival of a large proportion of archaic gene lineages, recent work has uncovered several genes that cannot be accommodated within the framework of a recent mtDNA expansion and replacement of archaic humans. These genes include (but are not limited to) <a href="weblog/reviews/genetics/garrigan_2005_structured_ancestral.html">the region around Xp21.1 (Garrigan et al. 2005)</a>, <a href="weblog/reviews/genetics/baird_2000_archaic_hybridization_telomeres.html">the Xp/Yp and 12q telomeric regions (Baird et al. 2000)</a>, and <a href="weblog/reviews/genetics/stefansson_2005_inversion.html">an inversion on 17q21.31 (Stefansson et al. 2005)</a>. Several other loci <a href="weblog/reviews/genetics/harding_2005_continuity.html">were discussed at the 2005 AAPA meetings</a>, and I know of a few more that are in press. 
</p>

<p>
In short, genomic data are not consistent with mtDNA neutrality, and a growing number of detailed studies have documented loci that represent the survival (and proliferation) of archaic human gene lineages. Much of this literature on specific loci has emerged in the last year; it is no surprise that it is missing from Weaver and Roseman (2005). 
</p>

<p>
But Templeton's analyses have been well known for years, and they bear directly on the issue. They are mentioned by Weaver and Roseman (2005:677) only as evidence that anthropological geneticists "favor a predominantly extra-European origin for the earliest modern populations in Europe" (!!!). 
</p>

<p>
The third category of missing information is the fossil and archaeological record. According to Weaver and Roseman (2005:682): 
</p>

<blockquote>Our results stress the importance of fully integrating archaeological, fossil, and genetic evidence in investigations of modern human origins.</blockquote>

<p>
It is therefore striking that the paper includes no acknowledgement of the fossil and archaeological evidence that supports Neandertal-modern reproductive continuity. This evidence includes (but is not limited to): 
</p>

<ol>
<li>The persistence of Neandertal traits in post-Neandertal populations (Frayer 1993; Duarte et al. 1999; Wolpoff et al. 2001; Trinkaus et al. 2003). Trinkaus (2005:218) concludes that the model of no interbreeding between Neandertals and modern humans is "intellectually dead." </li>
<li>The intergradation of Neandertal and contemporary populations, illustrated by their substantial morphological overlap in West Asia (Kramer et al. 2001; McCown and Keith 1939). </li>
<li>Shared directionality of morphological evolution in Neandertals and contemporary populations (Hawks and Wolpoff 2001). </li>
<li>The cognitive continuity of Neandertals and succeeding populations, evidenced by their substantially shared technical and symbolic ability (d'Errico 2003).</li>
</ol>

<p>
It seems clear that no genetic model that <i>excludes</i> a role for Neandertal genetic persistence can be considered to be "integrated" with these archaeological and fossil observations. Certainly considerable disagreement exists about the level of Neandertal contribution to later Europeans (as well as the level of Upper Paleolithic contribution to the more recent European gene pool). But evidence of intermixture is clear and recognized even by those who suspect that the actual level of such intermixture was low (Br&auml;uer et al. 2004). Together with the full pattern of genomic evidence, it seems clear that such intermixture is a potent explanation for the evolutionary pattern of the early Upper Paleolithic in Europe, as well as other regions during the Late Pleistocene. 
</p>

<h4>The bottom line</h4>

<p>
The issue here is not whether a population expansion occurred in the Late Pleistocene and Holocene. It certainly did. But does this expansion by itself explain the distinctive pattern of human mtDNA variation? And if so, does the fate of the Neandertals hinge on this demographic hypothesis? 
</p>

<p>
A consideration of a fuller set of genomic data indicates that the answer to both these questions is no. 
</p>

<p>
Human mtDNA has very likely been under positive selection. The evidence for this selection is as strong as for nearly any other selected locus. Although the specific target of the most recent selective sweep has not yet been identified, the same is true of other genes that are believed to have been under selection, such as <i>FoxP2</i>. The pattern of variation cannot be explained by population expansion, because other genomic regions are either inconsistent with mtDNA, inconsistent with each other, or inconsistent with any expansion at all. 
</p>

<p>
Determining whether Neandertals particularly contributed genetic material to the living human population is a challenge. Even if clear evidence of archaic lineages is found, it is difficult to substantiate that these lineages were found in a particular region of Europe over 40,000 years ago. 
</p>

<p>
Yet, substantial evidence of archaic lineages has been found. There is no question that some -- perhaps most -- human genes preserve allelic variation from archaic human populations. 
</p>

<p>
The morphological and archaeological evidence suggest strongly that Neandertal genetic lineages survived into later Upper Paleolithic populations. Ultimately, the genetic test of Neandertal survival may be carried out by finding nuclear DNA sequences from Neandertal fossils themselves. Until that time, we can say only that some Neandertal contribution to the modern human nuclear gene pool is consistent with the known evidence. 
</p>

<h4>References:</h4>

<p>
<p class="cite"><span class="biblabel">
<a name="XAmbrose:1998"></a></span>Ambrose SH. 1998.  Late Pleistocene human population bottlenecks, volcanic winter and differentiation of modern humans. J Hum Evol 34:623-652. </p>
 
<p class="cite"><span class="biblabel">
<a name="XArnason:1996"></a></span>Arnason U, Gullberg A, Janke A, Xu X. 1996. Pattern and timing of evolutionary divergences among hominoids based on analyses of complete mtDNAs. J Mol Evol 43:650-661.<br />
 </p>
 
<p class="cite"><span class="biblabel">
<a name="XBaird:2000"></a></span>Baird DM, Coleman J, Rosser ZH, Royle NJ. 2000.  High levels of sequence polymorphism and linkage disequilibrium at the telomere of 12q: Implications for telomere biology and human evolution. Am J Hum Genet 66:235-250. </p>
 
<p class="cite"><span class="biblabel">
<a name="XBrauer:2004"></a></span>Bruer G, Collard M, Stringer C. 2004.  On the reliability of recent tests of the Out of Africa hypothesis for modern human origins. Anat Rec 279A:701-707. </p>
 
<p class="cite"><span class="biblabel">
<a name="XCaspari:2004"></a></span>Caspari R, Lee SH. 2004.  Older age becomes common late in human evolution.  Proc Natl Acad Sci U S A 101:10,895-10,900. </p>
 
<p class="cite"><span class="biblabel">
<a name="XCurrat:2004"></a></span>Currat M, Excoffier L. 2004.  Modern humans did not admix with Neanderthals during their range expansion into europe. PLoS Biol 2:e421. </p>
 
<p class="cite"><span class="biblabel">
<a name="XdErrico:2003"></a></span>d&#8217;Errico F. 2003.  The invisible frontier: A multiple species model for the origin of behavioral modernity. Evol Anthropol 12:188-202. </p>
 
<p class="cite"><span class="biblabel">
<a name="XDuarte:1999"></a></span>Duarte C, Maurcio J, Pettitt PB, Souto P, Trinkaus E, van&nbsp;der Plicht H, Zilhao J. 1999.  The early Upper Paleolithic human skeleton from the Abrigo do Lagar Velho (Portugal) and modern human emergence in Iberia. Proc Natl Acad Sci U S A 96:7604-7609. </p>
 
<p class="cite"><span class="biblabel">
<a name="XEnard:2002"></a></span>Enard W, Przeworski M, Fisher SE, Lai CS, Wiebe V, Kitano T, Monasco AP, Pbo S. 2002. Molecular evolution of <i>FOXP2</i>, a gene involved in speech and language. Nature 418:869-872. </p>
 
<p class="cite"><span class="biblabel">
<a name="XEswaran:2005"></a></span>Eswaran V, Harpending H, Rogers AR. 2005.  Genomics refutes an exclusively African origin of humans. J Hum Evol 49:1-154. </p>
 
<p class="cite"><span class="biblabel">
<a name="XFay:1999"></a></span>Fay JC, Wu CI. 1999. A human population bottleneck can account for the discordance between patterns of mitochondrial versus nuclear DNA variation. Mol Biol Evol 16:1003-1005. </p>
 
<p class="cite"><span class="biblabel">
<a name="XFrayer:1993"></a></span>Frayer DW. 1993.   Evolution at the European edge: Neanderthal and Upper Paleolithic relationships. Prehistoire Europeenne 2:9-69. </p>
 
<p class="cite"><span class="biblabel">
<a name="XGarrigan:2005"></a></span>Garrigan D, Mobasher Z, Severson T, Wilder JA, Hammer MF. 2005.  Evidence for archaic Asian ancestry on the human X chromosome. Mol Biol Evol 22:189-192. </p>
 <p class="cite"><span class="biblabel">
<a name="XGillespie:2000"></a></span>Gillespie JH. 2000.   Genetic drift in an infinite population: the pseudohitchhiking model. Genetics 155:909-919. </p>
 
<p class="cite"><span class="biblabel">
<a name="XHammer:2003"></a></span>Hammer MF, Blackmer F, Garrigan D, Nachman MW, Wilder J. 2003.  Human population structure  and  its  effects  on  sampling  Y  chromosome  sequence  variation.    Genetics 164:1495-1509. </p>
 
<p class="cite"><span class="biblabel">
<a name="XHarpending:1998"></a></span>Harpending HC, Batzer MA, Gurven M, Jorde LB, Rogers AR, Sherry ST. 1998.  Genetic traces of ancient demography. Proc Natl Acad Sci U S A 95:1961-1967. </p>
 
<p class="cite"><span class="biblabel">
<a name="XHarpending:1993"></a></span>Harpending HC, Sherry ST, Rogers AR, Stoneking M. 1993.  The genetic structure of ancient human populations. Curr Anthropol 34:483-496. </p>
 
<p class="cite"><span class="biblabel">
<a name="Xaccretion"></a></span>Hawks J, Wolpoff MH. 2001.   The accretion model of Neandertal evolution.   Evolution 55:1474-1485. </p>
 
<p class="cite"><span class="biblabel">
<a name="XKimmel:1997"></a></span>Kimmel M, Chakraborty R, King J, Bamshad M, Watkins W, Jorde LB. 1997.  Signatures of population expansion in microsatellite repeat data. Genetics 148:1921-1930. </p>
 <p class="cite"><span class="biblabel">
<a name="XKlein:2002"></a></span>Klein R, Edgar B. 2002. The dawn of human culture. New York: John Wiley and Sons. </p>
 
<p class="cite"><span class="biblabel">
<a name="XKramer:2001"></a></span>Kramer A, Crummett TL, Wolpoff MH. 2001. Out of Africa and into the Levant: Replacement or admixture in Western Asia. Quaternary International 75:51-63. </p>
 
<p class="cite"><span class="biblabel">
<a name="XLee:2003"></a></span>Lee SH, Wolpoff MH. 2003.   The pattern of evolution in Pleistocene human brain size. Paleobiology 29:186-196. </p>
 
<p class="cite"><span class="biblabel">
<a name="XLeonard:1996"></a></span>Leonard WR, Robertson ML. 1996.  On diet, energy metabolism and brain size in human evolution. Curr Anthropol 37:125-128. </p>
 
<p class="cite"><span class="biblabel">
<a name="XLeonard:1997"></a></span>Leonard WR, Robertson ML. 1997.  Rethinking the energetics of bipedality.  Curr Anthropol 38:304-309. </p>
 
<p class="cite"><span class="biblabel">
<a name="XLowell:2005"></a></span>Lowell BB, Shulman GI. 2005.  Mitochondrial dysfunction and Type 2 diabetes.  Science 307:384-397. </p>
 <p class="cite"><span class="biblabel">
<a name="XManderscheid:1996"></a></span>Manderscheid EJ, Rogers AR. 1996.  Genetic admixture in the Late Pleistocene.  Am J Phys Anthropol 100:1-5. </p>
 
<p class="cite"><span class="biblabel">
<a name="XMarth:2003"></a></span>Marth G, Schuler G, Yeh R, Davenport R, Agarwala R, Church D, Wheelan S, Baker J, Ward M, Kholodov M, Phan L, Czabarka E, Murvai J, Cutler D, Wooding S, Rogers A, Chakravarti A, Harpending HC, Kwok PY, Sherry ST. 2003.  Sequence variations in the public human genome data reflect a bottlenecked population history. Proc Natl Acad Sci U S A 100:376-381. </p>
 
<p class="cite"><span class="biblabel">
<a name="XMarth:2004"></a></span>Marth  GT,  Czabarka  E,  Murvai  J,  Sherry  ST.  2004.   The  allele  frequency  spectrum  in genome-wide human variation data reveals signals of differential demographic history in three large world populations. Genetics 166:351-372. </p>
 
<p class="cite"><span class="biblabel">
<a name="XMcCown:1939"></a></span>McCown TD, Keith A. 1939. The stone age man of Mount Carmel: The fossil human remains from the Levalloiso-Mousterian, volume&nbsp;2. Oxford: Clarendon Press. </p>
 
<p class="cite"><span class="biblabel">
<a name="XMerriwether:1991"></a></span>Merriwether DA, Clark AG, Ballinger SW, Schurr TG, Soodyall H, Jenkins T, Sherry ST, Wallace  DC.  1991.   The  structure  of  human  mitochondrial  DNA  variation.   J  Mol  Evol 33:543-55. </p>
 <p class="cite"><span class="biblabel">
<a name="XNiemi:2005"></a></span>Niemi AK, Majamaa K. 2005.  Mitochondrial DNA and ACTN3 genotypes in Finnish elite endurance and sprint athletes. Eur J Hum Genet 13:965-969. </p>
 
<p class="cite"><span class="biblabel">
<a name="XNiemi:longevity:2005"></a></span>Niemi AK, Moilanen JS, Tanaka M, Hervonen A, Hurme M, Lehtimki T, Arai Y, Hirose N,  Majamaa  K.  2005.   A  combination  of  three  common  inherited  mitochondrial  DNA polymorphisms promotes longevity in Finnish and Japanese subjects 13:166-170. </p>
 
<p class="cite"><span class="biblabel">
<a name="XPearson:2003"></a></span>Pearson O. 2003.  Has the combination of genetic and fossil evidence solved the riddle of modern human origins? Evol Anthropol 13:145-159. </p>
 
<p class="cite"><span class="biblabel">
<a name="XPereira:2001"></a></span>Pereira  L,  Dupanloup  I,  Rossser  ZH,  Jobling  MA,  Barbujani  G.  2001.   Y-chromosome mismatch distributions in Europe. Mol Biol Evol 18:1259-1271. </p>
 
<p class="cite"><span class="biblabel">
<a name="XPolanski:2003"></a></span>Polanski  A,  Kimmel  M.  2003.    New  explicit  expressions  for  relative  frequencies  of single-nucleotide polymorphisms with applications to statistical inference on population growth. Genetics 165:427-436. </p>
 
<p class="cite"><span class="biblabel">
<a name="XPritchard:1999"></a></span>Pritchard JK, Seielstad MT, Perez-Lezaun A, Feldman MW. 1999. Population growth of human Y chromosomes: A study of Y chromosome microsatellites. Mol Biol Evol 16:1791-1798.<br />
 </p>
 
<p class="cite"><span class="biblabel">
<a name="XPtak:2002"></a></span>Ptak SE, Przeworski M. 2002.  Evidence for population growth in humans is confounded by fine-scale population structure 18:559-563. </p>
 
<p class="cite"><span class="biblabel">
<a name="XRuff:1997"></a></span>Ruff CB, Trinkaus E, Holliday TW. 1997.  Body mass and encephalization in Pleistocene <i>Homo</i>. Nature 387:173-176. </p>
 
<p class="cite"><span class="biblabel">
<a name="XRuiz-Pesini:2004"></a></span>Ruiz-Pesini E, Mishmar D, Brandon M, Procaccio V, Wallace DC. 2004.  Effects of purifying and adaptive selection on regional variation in human mtDNA. Science 303:223-226. </p>
 
<p class="cite"><span class="biblabel">
<a name="XSherry:1996"></a></span>Sherry S. 1996. Estimating human effective population sizes with genetic models incorporating demographic fluctuation. Ph.D. thesis, Pennsylvania State University. </p>
 
<p class="cite"><span class="biblabel">
<a name="XSherry:1994"></a></span>Sherry ST, Rogers AR, Harpending H, Soodyall H, Jenkins T, Stoneking M. 1994.  Mismatch distribution of mtDNA reveal recent human population expansions. Hum Biol 66:761-775. </p>
 
<p class="cite"><span class="biblabel">
<a name="XSorensen:2001"></a></span>Sorensen MV, Leonard WR. 2001. Neandertal energetics and foraging efficiency. J Hum Evol 40:483-495. </p>
 <p class="cite"><span class="biblabel">
<a name="XStefansson:2005"></a></span>Stefansson H, Helgason A, Steinthorsdottir GTV, Masson G, Bernard J, Baker A, Jonasdottir A, Ingason A, Gudnadottir VG, Desnica N, Hicks A, Gylfason A, Gudbjartsson DF, Jonsdittir GM, Sainz J, Agnarsson K, Birgisdottir B, Ghosh S, Olafsdottir A, Cazier JB, Kristjansson K, Frigge ML, Thorgeirsson TE, Gulcher JR, Kong A, Stefansson K. 2005.  A common inversion under selection in Europeans. Nature Genet 37:129-137. </p>
 
<p class="cite"><span class="biblabel">
<a name="XStiner:2000"></a></span>Stiner MC, Munro ND, Surovell TA. 2000.  The tortoise and the hare: Small-game use, the broad-spectrum revolution, and Paleolithic demography. Curr Anthropol 41:39-73. </p>
 
<p class="cite"><span class="biblabel">
<a name="XTempleton:2002"></a></span>Templeton AR. 2002. Out of Africa again and again. Nature 416:45-51. </p>
 
<p class="cite"><span class="biblabel">
<a name="XTrinkaus:2005"></a></span>Trinkaus E. 2005. Early modern humans. Annu Rev Anthropol 34:207-230. </p>
 
<p class="cite"><span class="biblabel">
<a name="XTrinkaus:2003"></a></span>Trinkaus E, &#350;tephan Milota, Rodrigo R, Mircea G, Moldovan O. 2003.  Early modern human cranial remains from the Pe&#351;tera cu Oase, Romania. J Hum Evol 45:245-253. </p>
 
<p class="cite"><span class="biblabel">
<a name="XWeaver:2005"></a></span>Weaver AH. 2005.  Reciprocal evolution of the cerebellum and neocortex in fossil humans. Proc Natl Acad Sci U S A 102:3576-3580. </p>
 <p class="cite"><span class="biblabel">
<a name="XWise:1998"></a></span>Wise CA, Sraml M, Easteal S. 1998.  Departure from neutrality at the mitochondrial <i>NADH</i> dehydrogenase subunit 2 gene in humans, but not in chimpanzees. Genetics 148:409-421. </p>
 
<p class="cite"><span class="biblabel">
<a name="XWise:1997"></a></span>Wise CA, Sraml M, Rubinsztein DC, Easteal S. 1997. Comparative nuclear and mitochondrial genome diversity in humans and chimpanzees. Mol Biol Evol 14:707-716. </p>
 
<p class="cite"><span class="biblabel">
<a name="XWolpoff:2001"></a></span>Wolpoff  MH,  Hawks  J,  Frayer  DW,  Hunley  K.  2001.   Modern  human  ancestry  at  the peripheries: A test of the replacement theory. Science 291:293-297. </p>
 
<p class="cite"><span class="biblabel">
<a name="XZhivotovsky:2000"></a></span>Zhivotovsky LA, Bennett L, Bowcock AM, Feldman MW. 2000. Human population expansion and microsatellite variation. Mol Biol Evol 17:757-767. </p>
 
<p class="cite"><span class="biblabel">
<a name="XZhivotovsky:2003"></a></span>Zhivotovsky LA, Rosenberg NA, Feldman MW. 2003.  Features of evolution and expansion of modern humans, inferred from genomewide microsatellite markers.   Am J Hum Genet 72:1171-1186. </p>
 
<p class="cite"><span class="biblabel">
<a name="XZhu:2004"></a></span>Zhu X, Smith MA, Perry G, Aliev G. 2004.  Mitochondrial failures in Alzheimer&#8217;s disease. American Journal of Alzheimers Disease and Other Dementias 19:345-352.
</p>


