---
layout: single 
title: "Selection on synonymous mutations" 
category: story
permalink: /weblog/reviews/genomics/selection/selection_synonymous_mrna_stability_2005.html
tags: [natural selection, genomics] 
comments: false 
author: John Hawks 
---


<p>
Here's an interesting thought: 
</p>

<blockquote>Background</blockquote>

<blockquote>In mammals, contrary to what is usually assumed, recent evidence suggests that synonymous mutations may not be selectively neutral. This position has proven contentious, not least because of the absence of a viable mechanism. Here we test whether synonymous mutations might be under selection owing to their effects on the thermodynamic stability of mRNA, mediated by changes in secondary structure.</blockquote>

<blockquote>Results</blockquote>

<blockquote>We provide numerous lines of evidence that are all consistent with the above hypothesis. Most notably, by simulating evolution and reallocating the substitutions observed in the mouse lineage, we show that the location of synonymous mutations is non-random with respect to stability. Importantly, the preference for cytosine at 4-fold degenerate sites, diagnostic of selection, can be explained by its effect on mRNA stability. Likewise, by interchanging synonymous codons, we find naturally occurring mRNAs to be more stable than simulant transcripts. Housekeeping genes, whose proteins are under strong purifying selection, are also under the greatest pressure to maintain stability.</blockquote>

<blockquote>Conclusion</blockquote>

<blockquote>Taken together, our results provide evidence that, in mammals, synonymous sites do not evolve neutrally, at least in part owing to selection on mRNA stability. This has implications for the application of synonymous divergence in estimating the mutation rate.</blockquote>

<p>
That's the abstract of <a href="http://genomebiology.com/2005/6/9/R75/?mkt=87259">a study (free text online)</a> in <i>Genome Biology</i> by J. V. Chamary and Laurence D. Hurst. 
</p>

<p>
What is the net effect of such selection? The short answer is nobody has any idea. Consider: 
</p>

<blockquote>The substitution rate at synonymous sites in exons is often used as a measure of the mutation rate [8,9]; however, this assumes neutral evolution of synonymous mutations [1,2]. By providing a parsimonious mechanism by which selection could act on synonymous sites, we can ignore the objection that prior evidence is indirect. Nevertheless, it is presently unclear to what degree synonymous mutations are favored or opposed by selection due to their effects on mRNA stability. Without being able to quantify the latter, as well as the net effect of other biases (for example, splice-associated), it will not be possible to directly estimate the extent to which use of the synonymous substitution rate leads to underestimates of the mutation rate and the mutational load.</blockquote>

<p>
And:
</p>

<blockquote>Indeed, it is quite possible that there exist no preferred codon within a gene while at the same time synonymous mutations are under selection. More generally, a complex set of trade-offs between different forms of selection and mutational biases may render interpretation of patterns of codon usage very difficult.</blockquote>

<p>
It seems possible to me that selection on mRNA stability may allow certain kinds of fine-tuning changes, analogous to selection on promotors or inhibitors. If the half-life of an mRNA within a cell could be decreased slightly, it might well have an adaptive (or conversely maladaptive) result. Would it make enough difference to be selected? Maybe not in most cases, but in some cases it might well do so. And Charmary and Hurst are able to show that genes subject to strong purifying selection appear to have greater constraint on mRNA stability -- in my view the most persuasive of their arguments for the effect. 
</p>

<p>
It also occurs to me that selection on mRNA structural stability would also be a consideration for the origin of the genetic code. Certain patterns of redundancy in silent sites would potentially be targets of selection, for long-lasting or shorter-lasting RNA codons as options for the same amino acid. 
</p>

<p>
It just goes to show how many real unknowns we still face when looking at molecular evolution. If synonymous mutations aren't really neutral, what will we discover next? 
</p>

<h4>References:</h4>

<p class="cite">Chamary JV and Hurst LD. 2005. Evidence for selection on synoymous mutations affecting stability of mRNA secondary structure in mammals. Genome Biol 6:R75. <a href="http://genomebiology.com/2005/6/9/R75/?mkt=87259">Free full text online</a></p>


