---
layout: single 
title: "The telomeres of the australopiths" 
category: story
permalink: /weblog/reviews/genetics/baird_2000_archaic_hybridization_telomeres.html
tags: [Australopithecus afarensis, Australopithecus africanus, Neandertals] 
comments: false 
author: John Hawks 
---


<p>
Speaking of <a href="weblog/reviews/genetics/kreitman_selection_review_2000.html">old papers</a>, I was just re-reading <a href="http://www.journals.uchicago.edu/AJHG/journal/issues/v66n1/991177/991177.html">this one</a> from Duncan Baird and colleagues (2000). 
</p>

<p>
What got me started was this line from the <a href="weblog/reviews/genetics/garrigan_2005_structures_ancestral.html">recent paper</a> by Garrigan and colleagues (2005:3): 
</p>

<blockquote>Aided by a novel experimental design, we present the first genetic evidence that statistically rejects the null hypothesis that our species descends from a single, historically panmictic population.</blockquote>

<p>
Of course, that didn't sound right to me, because people have been talking about evidence for archaic genes in recent humans for several years. The Baird study wasn't cited in that paper, and I returned to it to see what the evidence looked like. Here's the last line of the abstract (Baird et al. 2000:235): 
</p>

<blockquote>To explain the presence of a few diverged haplotypes adjacent to the Xp/Yp and 12q telomeres, we propose a model that involves the hybridization of two archaic hominoid [sic] lineages ultimately giving rise to modern <i>Homo sapiens</i>.</blockquote>

<p>
This more detailed consideration of the problem of divergent haplotypes comes from the discussion: 
</p>

<blockquote>Two alternative explanations for the presence of divergent haplotypes adjacent to two telomeres can be envisaged. First, the divergent haplotypes arose independently at separate subterminal loci within an archaic hominoid [sic] genome. The high level of exchange between subterminal repeat sequences then resulted in the relocation of one of the subterminal sequences with a telomere to the end of the same chromosome, thus creating two highly diverged haplotypes at one locus. We think, however, that this explanation is unlikely, since there is no evidence that "donor" loci exist in the modern genome. The results of linkage analysis indicate that the only copies of the sequences that can be amplified by the 12qA, 12qB, and 12qArev primers are linked to the end of 12q. Also, although a related copy of the 12q telomere-adjacent sequence is present on some copies of chromosomes 7q, the sequence in this location does not show more similarity to one 12q telomere-adjacent haplotype than to the other. In addition, there is no evidence that a second locus with homology to the Xp/Yp telomere-adjacent sequence is present in the human or in other great-ape genomes. It would therefore be necessary to assume that the "donor" loci for the ends of both chromosomes were present in an ancestral genome but have been lost from the modern human genome. Another explanation is that the diverged haplotypes arose, in separate archaic hominoid lineages, from a common ancestral sequence. These lineages would have to have been isolated for sufficient time to allow divergent haplotypes to arise as a result of sequential mutations and of fixation in each lineage for one predominant haplotype. The degree of sequence divergence between the haplotypes would be dependent on the mutation rates of the loci examined. The high mutation rate in the telomere-adjacent sequences would have resulted in rapid divergence of these sequences in the different lineages. A subsequent hybridization of two hominoid lineages would bring the highly diverged haplotypes together. The continued existence of the diverged haplotypes after the hybridization event would depend on factors such as recombination, drift, and founder effect, and it could vary between loci. This model implies that <i>Homo sapiens</i> may have evolved from a hybridization event between two hominoid [sic] lineages (Baird et al. 2000:247-248). </blockquote>

<p>
There's that "hominoid" again. Why hominoid instead of hominid? It turns out these polymorphisms are pretty ancient. Not hominoid-ancient, but, well, read for yourself: 
</p>

<blockquote>Since the timing of the proposed hybridization event is unknown, it is not possible to suggest which hominoid lineages may have been involved; however, the common ancestor to the lineages must have existed >2 million years ago, perhaps coinciding with one of the Australopithecine species. Additional analysis of the 12q telomere and its adjacent sequence in other human populations may distinguish between the explanations outlined above, but it is not unreasonable to suggest that hybridization between lineages separated by 1.9 million years could occur, because the orangutan subspecies <i>Pongo pygmaeus abelii</i> and <i>Pongo pygmaeus pygmaeus</i> are capable of producing fertile offspring, despite the existence of molecular data that suggests that the two subspecies diverged 1.5 -- 1.7 million years ago (Zhi et al. 1996). Of the two diverged haplotypes in the orangutan Xp/Yp telomere-adjacent sequence (discussed above), one haplotype (orang-lower) was detected in both the orangutan subspecies, but the second haplotype (orang-upper) was detected only in the <i>Pongo pygmaeus abelii</i> subspecies (2/18 alleles) (Baird and Royle 1997; Baird, unpublished data). Furthermore, the observation of homoplasy in skeletons of the Australopithecine species <i>A. africanus</i> and <i>A. afarensis</i> suggests that human evolution was more complex than is currently understood (McHenry and Berger 1998a; McHenry and Berger 1998b), and, recently, a skeleton with both human and Neanderthal characteristics was identified (Duarte et al. 1999) (Baird et al. 2000:248).</blockquote>

<p>
So, definitely hominid, but fairly ancient: they place the divergence of the haplotypes at at least 1.9 million years. The story here is not the time depth alone, but the lack of intermediate haplotypes between two extremes; which is the same story offered by Garrigan et al. (2005). Of course, it's not the "archaic" part that they claim is new, it's the "statistical test" part. They're starting to sound like paleoanthropologists!
</p>

<h4>References:</h4>

<p class="cite">Baird DM, Coleman J, Rosser ZH, Royle NJ. 2000. High levels of sequence polymorphism and linkage disequilbrium at the telomere of 12q: implications for telomere biology and human evolution. Am J Hum Genet 66:235-250. <a href="http://www.journals.uchicago.edu/AJHG/journal/issues/v66n1/991177/991177.html">Full text online</a></p>

<p class="cite">Garrigan D, Mobasher Z, Kingan SB, Wilder JA, Hammer MF. 2005. Deep haplotype divergence and long-range linkeage disequilibrium at Xp21.1 provide evidence that humans descend from a structured ancestral population. Genetics (online before print). </p>

