---
layout: single 
title: "Kreitman on human mtDNA selection" 
category: story
permalink: /weblog/reviews/genetics/kreitman_selection_review_2000.html
tags: [natural selection] 
comments: false 
author: John Hawks 
---


<p>
This is an <a href="http://arjournals.annualreviews.org/doi/full/10.1146/annurev.genom.1.1.539;jsessionid=nYPR1wD2wl6g">old paper</a> that I ran across today, a review of tests of selection with application to humans. Martin Kreitman is well known as a specialist in the population genetics of selection. The abstract says this: 
</p>

<blockquote>Attempts to understand the nonequilibrium configuration of silent polymorphism in human mitochondrial DNA illustrate the difficulty of distinguishing between selection and alternative demographic hypotheses. The range of plausible alternatives to selection will become better defined, however, as additional population genetic data sets become available, allowing better null models to be constructed.</blockquote>

<p>
And the conclusion says this: 
</p>

<blockquote>An instructive example of this problem lies in the interpretation of human mitochondrial nucleotide polymorphism. In a very insightful paper, Di Rienzo and Wilson reported that the genealogy of mitochondrial sequences in non-Africans was more starlike in shape than might be expected under neutrality and that the distribution of pairwise differences was Poisson shaped (20; also see 74). Di Rienzo interpreted this apparent departure from neutrality as an indication of recent population expansion. Theoretical treatment of the problem provided additional support for the expansion hypothesis (90), but a bottleneck at ~50,000 -- 100,000 years ago, possibly caused by the selective sweep of a favorable allele, could not be rejected.</blockquote>

<blockquote>Mitochondrial DNA has been assumed to be nonrecombining (but for evidence of recombination, see 4, 24); the sweep of a favorable mutation anywhere in the mitochondrial genome will cause the fixation of a single haplotype. Support for the selection hypothesis has come from the analysis of nuclear encoded genes. The nuclear genome shows little evidence for a skew towards rare alleles (18, 37, 38, 42, 83, 106), and thus towards a negative Tajima's D, as predicted under the population expansion hypothesis.</blockquote>

<blockquote>Theoretical investigation of bottlenecks and subsequent expansions (25) shows, however, that Tajima's D can be negative or positive depending on the size of the bottleneck and the timing and magnitude of an expansion. Given that the mitochondrial genome has a smaller effective population size (being maternally inherited and effectively haploid) than the nuclear genome, the conflicting portraits of polymorphism in the two genomes may be consistent with a population bottleneck (25). The exciting possibility of a selective sweep in the modern mitochondrial genome remains, unfortunately, an unresolved issue (Kreitman 2000:553).</blockquote>

<p>
I was happy to run across this reference that I previously missed, and so I'm posting it for others. It's good to read a review that appreciates the difficulties of detecting selection and distinguishing it from demography. The final paragraphs are sobering: 
</p>

<blockquote>The only current safeguard against gross misinterpretation of test results vis-a-vis selection vs historical demography is to have an a priori hypothesis about the type and direction of selection that are expected for the locus under investigation. The previously described work on Duffy provides a good example of this approach (37). There are two reasons to hope, however, that the situation for analyzing human polymorphism data sets will improve. First, as additional data sets accumulate, a reduction in the number of plausible historical demographic scenarios will be possible. The specific range of parameter values, for example, allowing mitochondrial genes but not nuclear genes to differ in the observed frequency spectrum of mutations may be shown to be unrealistic. Second, population history, whether it involves ancient bottlenecks, recent expansions, or specific population movements, affects the polymorphism of all nuclear genes equally. From a practical perspective, this means that the common signatures of human history on genetic variation should yield to the avalanche of data expected in future polymorphism studies. Better data mining techniques and sharper theoretical predictions are needed, however, to make this a reality (Kreitman 2000:553-554).</blockquote>

<blockquote>It should be possible, in principle, to construct a realistic neutral model of human variation that takes into account major features of human history. Such a model would then serve as a null hypothesis, a selectively neutral backdrop, against which to look for evidence of natural selection in individual genes. In no other organism is this possibility likely to be achieved at the high level of resolution possible for humans. Our species, despite its low levels of nucleotide polymorphism, issues in ethical sampling of native populations, and the inability to control matings, may thus replace <i>Drosophila</i> species as the poster child for molecular population genetics.</blockquote>

<p>
On a positive note, I think he's right that humans have become the best model for considering the molecular correlates of microevolution. We clearly know much more about ourselves than we do about other species, and it is hard to ignore evidence for long-term regional or local selective pressures. 
</p>

<p>
I wonder whether we are at a tipping point now, in 2005, compared to 2000. Have we reached the point where no single demographic hypothesis can explain both mtDNA and other genetic variation? Certainly we are, but there is more than that. Many autosomal genetic loci are inconsistent with each other in their pattern of variation. Selection has affected most areas of the genome in different ways. And some of those changes have been very recent. 
</p>

<p>
On the one hand, there is nothing surprising about this, since we know that humans have been evolving and still are. On the other hand, the universality of selection is not necessarily something anybody expected to find. We may need to ask ourselves, is there anything that doesn't bear the mark of selection on some linked site?
</p>

<h4>References:</h4>

<p class="cite">Kreitman M. 2000. Methods to detect selection in populations with applications to the human. Annu Rev Genom Hum Genet 1:539-559. <a href="http://arjournals.annualreviews.org/doi/full/10.1146/annurev.genom.1.1.539;jsessionid=nYPR1wD2wl6g">Full text online</a> </p>

