---
layout: single 
title: "Hunting the genes of the golden zebrafish" 
category: story
permalink: /weblog/reviews/genetics/populations/lamason_2005_slc24a5_skin_color.html
comments: false 
author: John Hawks 
---


<p>
In <i>Endless Forms Most Beautiful</i>, Sean Carroll describes work on pigmentation genes in species from butterflies to fruit flies and zebras to humans. He focuses mainly on <i>MC1R</i>, which happens to be behind much of the pigment patterning in vertebrates. But there are many other genes affecting skin coloration in humans, and these also might be illuminated by cross-species comparisons. 
</p>

<p>
One such study is the zebrafish story <a href="http://www.sciencemag.org/cgi/content/full/310/5755/1782">told by Rebecca Lamason et al. (2005) in <i>Science</i> (subscription)</a>. 
</p>

<blockquote>Lighter variations of pigmentation in humans are associated with diminished number, size, and density of melanosomes, the pigmented organelles of melanocytes. Here we show that zebrafish golden mutants share these melanosomal changes and that golden encodes a putative cation exchanger slc24a5 (nckx5) that localizes to an intracellular membrane, likely the melanosome or its precursor. The human ortholog is highly similar in sequence and functional in zebrafish. The evolutionarily conserved ancestral allele of a human coding polymorphism predominates in African and East Asian populations. In contrast, the variant allele is nearly fixed in European populations, is associated with a substantial reduction in regional heterozygosity, and correlates with lighter skin pigmentation in admixed populations, suggesting a key role for the SLC24A5 gene in human pigmentation.</blockquote>

<p>
The zebrafish is a common research proxy for vertebrate developmental biology, and there is a long history of tracking phenotypic variants like the one studied here. So that explains the connection -- people studying zebrafish realized that there was a phenotypic resemblance to humans, and when they found the gene, they assessed its variation in humans from public genome data. But the zebrafish themselves aren't central to the human story: 
</p>

<blockquote><b>Role of SLC24A5 in human pigmentation</b>. To evaluate the potential impact of SLC24A5 on the evolution of human skin pigmentation, we looked for polymorphisms within the gene. We noted that the G and A alleles of the single nucleotide polymorphism (SNP) rs1426654 encoded alanine or threonine, respectively, at amino acid 111 in the third exon of SLC24A5. This was the only coding SNP within SLC24A5 in the International Haplotype Map (HapMap) release 16c.1 (29). Sequence comparisons indicate the presence of alanine at the corresponding position in all other known members of the SLC24 (NCKX) gene family (fig. S5). The SNP rs1426654 had been previously shown to rank second (after the FY null allele at the Duffy antigen locus) in a tabulation of 3011 ancestry-informative markers (30). The allele frequency for the Thr111 variant ranged from 98.7 to 100% among several European-American population samples, whereas the ancestral alanine allele (Ala111) had a frequency of 93 to 100% in African, Indigenous American, and East Asian population samples (fig. S6) (29, 30). The difference in allele frequencies between the European and African populations at rs1426654 ranks within the top 0.01% of SNP markers in the HapMap database (29), consistent with the possibility that this SNP has been a target of natural or sexual selection.</blockquote>

<blockquote>A striking reduction in heterozygosity near SLC24A5 in the European HapMap sample (Fig. 5A) constitutes additional evidence for selection. The 150-kb region on chromosome 15 that includes SLC24A5, MYEF2, CTNX2, and part of SLC12A1 has an average heterozygosity of only 0.0072 in the European sample, which is considerably lower than that of the non-European HapMap samples (0.175 to 0.226). This region, which contains several additional SNPs with high-frequency differences between populations, was the largest contiguous autosomal region of low heterozygosity in the European (CEU) population sample (Fig. 5B). This pattern of variation is consistent with the occurrence of a selective sweep in this genomic region in a population ancestral to Europeans. For comparison, diminished heterozygosity is seen in a 22-kb region encompassing the 3' half of MATP (SLC45A2) in European samples, and more detailed analysis of this genomic region shows evidence for a selective sweep (31). However, the gene for agouti signaling protein (ASIP), which is known to be involved in pigmentation differences (32), shows no such evidence.</blockquote>

<p>
This is one of those cases where the HapMap adds to the perception of race differences instead of dispelling them. What is the cline of allele frequency variation for this gene? What is the strength of selection keeping the allele out of Africa (or possibly Asia)? These are questions that can't be answered with the small panels of individuals in the HapMap. So it's good for discovering polymorphisms, and even for finding variants that vary geographically, but it is less good for characterizing that geographic variation. 
</p>

<p>
The paper tests phenotypic effects of the SLC24A5 gene by measuring the skin reflectance in African-Americans, who are often polymorphic for the allele. This test shows that a considerable amount of skin color variation may be explained by the gene in this population; but there are evidently many other genetic factors that are more important in other populations, where the non-European allele is ubiquitous. 
</p>

<p>
Now, it would be interesting to see a gene tree for this one. 
</p>

<p>
UPDATE (12/16/05): Razib at Gene Expression <a href="http://www.gnxp.com/blog/2005/12/race-is-skin-deep.php">has much, much more</a>, including this: 
</p>

<blockquote>Since to the first approximation humans are creatures of Africa it makes sense that we would have fixed or constrained toward expression of genes that influence skin color toward a dark optimum. As non-Africans are a subset of Africans it seems plausible that the ancestral dark inducing forms of the genes will be shared. On the other hand, as the constraint for dark skin is released, because selection no longer favors it, the genes will start stumbling randomly in various directions via mutations. In the case of Europeans the MC1R locus seems to have walked in a random fashion and diversified greatly (30 alleles of greater than 1% frequency). On the hand, in East Asians the MC1R allele seems to have been selected toward one particular form that differs from the ancestral variant, in other words, constraint that limited the fitness toward those bearing the ancestral dark skin inducing allele(s) was released, but unlike Europeans selection now operated on a different allele and constrained diversification. This new finding makes that more intelligible: Europeans were given license to explore the range of MC1R variants because another locus was sufficient in hastening the induction of a light skin phenotype. Selection operates on genotype via phenotype, so the phenotype is sufficient to allow the individual to be fit and reproduce the genes, it is irrelevant what that particular conformation of genes that results in said phenotype is. An important point I am leaving out of this is that it is likely that MC1R has other fitness effects (recall the finding that redheads might be more sensitive to pain), and it is likely that this new locus is also implicated in other phenotypes, at least indirectly. In other words, loci don't explore the fitness landscape by their lonesome, but only in the context of changes and conditions on other loci.</blockquote>

<p>
That is more insightful than anything else you'll find anywhere. 
</p>

<h4>References:</h4>

<p class="cite">Lamason RL et al. 2005. SLC24A5, a Putative Cation Exchanger, Affects Pigmentation in Zebrafish and Humans. Science 310:1782-1786. <a href="http://www.sciencemag.org/cgi/content/full/310/5755/1782">Full text (subscription)</a></p>

