---
layout: single 
title: "Comparing human and chimpanzee promoters" 
category: story
permalink: /weblog/reviews/genomics/evo-devo/promoter_chimpanzee_human_2005.html
tags: [chimpanzees, genomics] 
comments: false 
author: John Hawks 
---


<p>
Current thinking on the nature of differences between humans and chimpanzees (or any other pair of closely related species, for that matter) holds that large phenotypic differences may be the result of relatively small differences in gene expression. 
</p>

<p>
Thus, the often-cited saw that humans and chimpanzees are 98 percent the same at the genetic level says almost nothing about the potential differences between the phenotypes of the two species, because identical genes may be expressed entirely differently by making slight changes to genetic promoters or inhibitors. This is not the only reason for the mismatch between genetic similarity and phenotypic difference, since a single point mutation may radically alter the amino acid sequence of a protein as well. But differences in gene expression are often assumed to be "tunable" to a greater extent than amino acid changes to proteins. Thus, certain slight changes to promoters might result in either slight or large differences in the quantities of a protein, the speed of transcription or post-transcription processing, or other biochemical attributes that would affect the phenotype. 
</p>

<p>
The possibility of many different possible responses to changes in promoters suggests a fertile drawing board for evolutionary change to work. Such changes might especially be important in the structural differences separating humans from other apes, since ontogenetic development depends sensitively on the concentration of certain proteins and peptides in the developing embryo. A slight increase in gene expression in part of the embryo might increase the length of the leg, or alter the form of the pelvis. Scientists are actively looking for the actual genes that may have influenced these processes in human evolution, and some they have found. 
</p>

<p>
So the story appears capable of explaining the phenotypic differences between humans and chimpanzees. The only problem is, nobody really knows how promoters work. 
</p>

<p>
The problem is illustrated in <a href="http://genomebiology.com/2005/6/7/R57/?mkt=83255">a current study</a> in <i>Genome Biology</i> by Florian Heissig and colleagues (2005), working from Svante Paabo's lab at the Max Planck Institute for Evolutionary Anthropology. The study examined genes with different patterns of expression (assessed by messenger RNA abundance) in chimpanzee and human tissues. They cloned the promoter regions of the genes and spliced them to a reporter gene to test directly the effect of the promoter on the gene expression. 
</p>

<p>
The results were surprising: 
</p>

<blockquote>Out of the 12 promoters tested, two (ACADSB, C10orf10) show a significant difference (ANOVA p-value < 0.05) in both cell lines whereas five (IMPA1, CGI-51, SH3BGR, UNG, TERF) show a significant difference in only one of the two cell lines. Five promoters show no significant difference in either cell line. The average sequence divergence (Table 1) for these five promoters (1.2%) and the remaining seven (1.3%) is not significantly different from each other, neither for the complete promoter fragment (two-tailed t-test, p = 0.65) nor for 220 bp around the transcription start site (p = 0.43) in which most of the conserved regulatory motifs are found [29]. One promoter (THEM2) contains a chimpanzee-specific Alu insertion, but does not show a significant difference in its activity in either of the two cell lines.</blockquote>

<blockquote>Three promoters (ACADSB, C10orf10, IMPA1) show activity differences in the promoter assays that go in the same direction as the expression differences of the corresponding genes in the tissues. Interestingly, the two promoters (ACADSB, C10orf10) that show qualitatively similar differences in the two cell lines are both in concordance with the tissue expression differences. For four promoters (CGI-51, SH3BGR, UNG, TERF) that show differences in only one of the cell lines, the difference goes in the opposite direction to the expression differences in the tissues (Heissig et al. 2005:R57).</blockquote>

<p>
The paper is very short and to the point, but the implications are striking. Humans and chimpanzees exhibit the same amount of sequence divergence for promoter regions with different activity levels versus promoters with the same activity level. Many genes that differ in activity exhibit no difference in promoter activity. Genes that have higher gene expression in one species <i>sometimes</i> have higher promoter activity, but <i>just as often</i> have <i>lower</i> promoter activity. In other words, the promoters are not an indication of the outcome. 
</p>

<p>
And the study only examined genes with substantial differences in expression between chimpanzees and humans. For genes with similar patterns of expression, the authors have this to say: 
</p>

<blockquote>If many genetic differences do indeed influence the expression of a single gene, the proximal promoters of these non-differentially expressed genes would be expected to differ in their activity almost as frequently as the promoters of differentially expressed genes (Heissig et al. 2005:R57).</blockquote>

<p>
Why should this be? Gene expression is the outcome of a cascade of events within the cell, including the production of enhancer molecules of various kinds, the binding of such enhancers to promoter sites, the metabolism of the gene product within the cell, including the speed with which it may be taken up by receptors, and the correlated effects of other genes with similar products. Again, this system is ideal for evolutionary fine-tuning. But it means that several different changes -- at different genomic sites -- may be necessary to optimize any given function. 
</p>

<p>
The really bad part of this is that it means a comparison of human and chimpanzee genomes may actually tell us nothing about how the phenotypic differences between the species arose. 
</p>

<p>
Now, these results may be considered as tentative, since the number of genes examined was only 12. But if the pattern holds true of other genes, even for a substantial minority of genes, it bodes ill for our ability to use genomic sequences to predict gene expression levels in tissues. And predicting phenotypic differences in body structure or behavior is many levels removed from gene expression. So for the high-level structural questions we are likely to ask about human evolution, no answers are likely to be forthcoming from genomic comparisons alone. 
</p>

<p>
The paper concludes: 
</p>

<blockquote>Our results imply that although many promoters may differ in activity between humans and chimpanzees, it will be difficult to predict physiologically relevant gene-expression differences from promoter activities observed in cell lines, even between two closely related species such as humans and chimpanzees. Further work is necessary to elucidate to what extent this applies also to allelic DNA sequence differences in promoters observed within a species. Further work is also needed to elucidate whether a general paradigm for how genome structure translates to gene expression activity can be derived (Heissig et al. 2005:R57).</blockquote>

<p>
If indeed the expression of genes is the product of a different complex cascade of events <i>for each gene</i>, then it may be that no "general paradigm" will ever be possible. Insights about human evolutionary changes may end up coming not from genomic comparisons, but from experimental work. Look for lots and lots of different human and chimpanzee genes to be turned on and off in cell cultures to determine not only their function but also their effects on the expression of other genes. 
</p>

<h4>References:</h4>

<p class="cite">Heissig F, Krause J, Bryk J, Khaitovich P, Enard W, Paabo S. 2005. Functional analysis of human and chimpanzee promoters. Genom Biol 6:R57. <a href="http://genomebiology.com/2005/6/7/R57/?mkt=83255">Free full text</a></p>

