---
layout: single 
title: "Genetic structure of the chominids" 
category: story
permalink: /weblog/reviews/genomics/divergence/innan_watanabe_chominid_divergence_2006.html
tags: [genetic divergence, genomics] 
comments: false 
author: John Hawks 
---


<p>
In view of the <a href="http://johnhawks.net/weblog/reviews/genomics/divergence/dawn_chumans_patterson_2006.html">previous post</a> on the genetics of the human-chimpanzee divergence, it seems worth going into a bit more detail about the paper by Innan and Watanabe (2006). 
</p>

<p>
I'm going to start using "chominid" for this population, because I'm tired of typing out the whole "pre-human-chimpanzee ancestral divergence population" phrase! I would keep using "chuman", but it has such a connotation of chimerism (no, not "chimperism") -- and "humanzee" is even moreso! 
</p>

<p>
The paper considers a model in which the isolation of the proto-chimpanzee and proto-hominid lineages increases steadily over time at a rate <i>a</i>. They then derived a maximum likelihood method for estimating <i>a</i> from human and chimpanzee sequence data. The time of initiation of divergence is a function of the rate <i>a</i>, so the parameters of the model are <i>a</i>, the speciation time (time of complete isolation) <i>T</i>, the relative size of the proto-chimpanzee and proto-hominid lineages <i>h</i>, and theta which equals four times the mutation rate times the effective population size of the ancestral population. The two values <i>a</i> and <i>h</i> themselves are interdependent, so they may amount to a single parameter for the purposes of the model. 
</p>

<p>
The three-parameter model is very simple compared to the kind of complexities that might actually have affected the ancestral population of the human and chimpanzee lineages. So, it's appropriate to be cautious about the results. The paper discusses some of the possible violations of assumptions. 
</p>

<p>
Recombination is an interesting complicating factor, because it should tend to homogenize the coalescence times for different genomic regions within the chominid population. The authors set up their model in such a way that recombination has a conservative effect -- that is, it tends to inflate the estimate of population structure. 
</p>

<p>
The paper is unique in that it doesn't even try to estimate the pre-divergence effective population size or divergence time. The point is to test the hypothesis of population <i>structure</i>, and these issues are separate from that question. A structure involving progressive isolation from an initially panmictic population will tend to dilate the distribution of recent coalescence times (because coalescence is less likely in a structured population) and leave more ancient coalescence times alone (because the population wasn't structured then). 
</p>

<p>
This effect would be similar to an expansion of the effective population size of the ancestral population, and it might be very difficult to tell population expansion from a progressive increase of isolation. 
</p>

<p>
It is notable that the paper also provides an estimate of the relative contribution of males and females to the mutation rate. Males are generally thought to have a much higher effective mutation rate, because males tend to reproduce a bit later and because there are many more cell generations leading to spermatogenesis in males than to oogenesis in females. Comparing the X chromosome with the autosomes, the paper finds that this is likely true: 
</p>

<blockquote>Our ML estimate of alpha (2.7-4.4) from the human-chimpanzee comparison is close to previous estimates (circa 5) from comparisons of distantly related primates (Li 1997; Makova and Li 2002), rather than low estimates from human-chimpanzee sequence data (Bohossian, Skaletsky, and Page 2000; International Human Genome Sequencing Consortium 2001), which were obtained by ignoring the effect of the coalescent in the ancestral population. It is indicated that taking the effect of the coalescent in the ancestral population into account is important in estimating alpha.</blockquote>

<p>
Another reason for the X chromosome to be a bit different. Probably doesn't explain the low coalescence time compared to the autosomes, though, since it is possible to use local mutation rates in comparison to other primates to estimate those times. 
</p>

<h4>References:</h4>

<p class="cite">Innan H, Watanabe H. 2006. The effect of gene flow on the coalescent time in the human-chimpanzee ancestral population. Mol Biol Evol 23:1040-1047. <a href="http://dx.doi.org/10.1093/molbev/msj109">DOI link</a></p>

