---
layout: single 
title: "The persistence of KIBRA" 
category: story
permalink: /weblog/reviews/genetics/brain/kibra_memory_association_2006.html
tags: [brain] 
comments: false 
author: John Hawks 
---


<p>
Gene Expression <a href="http://www.gnxp.com/blog/2006/10/memory-associated-snp.php">points</a> me to <a href="http://dx.doi.org/10.1126/science.1129837">this article</a> by Andreas Papassotiropoulos and colleagues in <i>Science</i> (10/20/2006): 
</p>

<blockquote><b>Common <i>Kibra</i> Alleles Are Associated with Human Memory Performance</b></blockquote>

<blockquote>Human memory is a polygenic trait. We performed a genome-wide screen to identify memory-related gene variants. A genomic locus encoding the brain protein KIBRA was significantly associated with memory performance in three independent, cognitively normal cohorts from Switzerland and the United States. Gene expression studies showed that <i>KIBRA</i> was expressed in memory-related brain structures. Functional magnetic resonance imaging detected <i>KIBRA</i> allele-dependent differences in hippocampal activations during memory retrieval. Evidence from these experiments suggests a role for KIBRA in human memory.</blockquote>

<p>
A role for KIBRA in functions related to memory was foreshadowed by <a href="http://dx.doi.org/10.1016/j.bbrc.2004.03.107">this paper</a> by Katrin B&uuml;ther and colleagues from 2004: 
</p>

<blockquote><b>KIBRA is a novel substrate for protein kinase C&zeta;</b></blockquote>

<blockquote>WW domain-containing proteins are found in all eukaryotic cells and they are involved in the regulation of a wide variety of cellular functions. We recently identified the neuronal protein KIBRA as novel member of this family of signal transducers. In this report, we describe the identification of protein kinase C (PKC) &zeta; as a KIBRA-interacting protein. PKC&zeta; is known to play an important role in synaptic plasticity and memory formation but its specific targets are not well known. Our studies presented here revealed that KIBRA is a novel substrate for PKC&zeta; and suggest that PKC&zeta; phosphorylation may regulate the cellular function of KIBRA.</blockquote>

<p>
But the current paper by Papassotiropoulos et al. goes further by establishing that genotype for one biallelic SNP covaries with memory performance (with ca. 20 percent better recall for carriers of one allele compared to homozygotes for the other). For the SNP in question, the carriers of a "T" had a higher percentage recall compared to "C" homozygotes. 
</p>

<p>
And even more interesting, the SNP occurs with different frequencies in different human populations: 
</p>

<blockquote>The allelic distribution of rs17070145 differs significantly between ethnic groups according to the National Center for Biotechnology Information database of genetic variation (dbSNP). In populations of European ancestry, the T allele is the minor one with a frequency of 25%, as also shown in this study. In contrast, in Asian populations the T allele is most frequent (75%) and in African-American populations, the T and C alleles are almost equally frequent (54% and 46%, respectively). Therefore, it would be interesting for subsequent studies to assess KIBRA's relation to memory in populations of non-European ancestry.</blockquote>

<p>
Now, here are a couple of questions. First, how do the "T" homozygotes do? There was no indication from the study that their performance was obviously different from heterozygotes, but the proportion of homozygotes was so low (<5%) that they weren't tested alone. It makes some difference to the evolution of the SNP whether the function is dominant or overdominant. 
</p>

<p>
Second, what else does the protein do? After all, most proteins are expressed in many tissues, even "neural" proteins. The evolution of this polymorphism in humans might well have to do with other functions. 
</p>

<p>
Here, there is at least a hint. Check out <a href="http://www.jbc.org/cgi/content/abstract/M600021200v1">this paper</a> from earlier this year: 
</p>

<blockquote><b>Essential Role of KIBRA in Co-activator Function of Dynein Light Chain 1 in Mammalian Cells</b></blockquote>

<blockquote>Suresh K. Rayala et al.</blockquote>

<blockquote>Recently dynein light chain 1 (DLC1), a cytoskeleton signaling component, has been shown to interact with and transactivate estrogen receptor-&alpha; (ER), leading to increased expression of ER target genes and growth stimulation of breast cancer cells. However, the molecular mechanism by which DLC1 regulates the ER pathway remains poorly understood. To gain insights into the putative mechanism, here we set out to identify novel DLC1-interacting proteins. We identified KIBRA, a WW domain- and a glutamic acid stretch-containing protein, as a DLC1-binding protein and showed that it interacts with DLC1 both in vitro and in vivo. We found that KIBRA-DLC1 complex is recruited to ER-responsive promoters. We also found that KIBRA-DLC1 interaction is mandatory for the recruitment and transactivation functions of ER or DLC1 to the target chromatin. Finally we found that KIBRA interacts with histone H3 via its glutamic acid-rich region and that such interaction might play a mechanistic role in conferring an optimal ER transactivation function as well as the proliferation of ligand-stimulated breast cancer cells. Together these findings indicate that DLC1-KIBRA interaction is essential for ER transactivation in breast cancer cells.</blockquote>

<p>
Now, that's interesting. This memory-associated protein is also essential to the activity of estrogen receptor protein. This paper is focused on the cancer-causing potential of the ER pathway, so there is naturally the suspicion that SNP variation in genes like <i>KIBRA</i> might have pathological effects. But to me, the more interesting possibility is that <i>KIBRA</i> might have a normal role in development. 
</p>

<p>
So there are possibly several reasons why a gene like this might be under selection in human populations, and they may not be very easy to sort out -- for estrogen-related functions, it's not real obvious which kind of phenotypes one would expect to be associated with SNPs. 
</p>

<p>
I'm raising the issue of multiple functions because it will increasingly become important to distinguish the neural effects of some gene variants from the selection that caused them to be variable in the first place. 
</p>

<p>
In some instances -- possibly for <i>KIBRA</i> -- these will be exactly the same. For <i>KIBRA</i>, that would mean that human populations are now different in frequencies <i>because</i> of selection on memory performance. 
</p>

<p>
In other instances -- also, possibly for <i>KIBRA</i> -- the target of selection may not be a neural trait. For <i>KIBRA</i>, one possibility is selection on the estrogen receptor pathway. 
</p>

<p>
Or both. Or selection might tend to increase the frequency of a SNP allele for one function but decrease it for another. And so on. 
</p>

<p>
Now, the reason I raise this is that some kinds of research have lots of funding (e.g., <i>brain</i> research, <i>cancer</i> research) and other kinds of research have little funding (e.g., <i>evolutionary biology</i>, <i>comparative genomics</i>) -- that is, except insofar as these latter categories are employed in the pursuit of the well-funded ones. 
</p>

<p>
So for the foreseeable future, we are going to have a lot of knowledge about some specific gene functions in specific tissues, but not a broad knowledge about all of their effects. It seems to me a lot like the biases we get from the fossil record -- we know a lot about bony anatomy of hard bones and teeth, but a lot less about lighter, thinner bones (the pelvis) and very little about soft tissue (did australopithecines have fur?) or behavior (did robust australopithecines make tools?). These biases shape our hypothesis-formation: we tend to focus on the parts we know a lot about. 
</p>

<p>
Nothing wrong with that, of course, but natural selection isn't blind in the same way that we are. Of course, it doesn't have a very good memory...
</p>

<h4>References:</h4>

<p class="cite">B&uuml;ther K, Plaas C, Barnekow A, Kremerskothen J. 2004. KIBRA is a novel substrate for protein kinase C&zeta;. Biochem Biophys Res Commun 317:703-707. <a href="http://dx.doi.org/10.1016/j.bbrc.2004.03.107">DOI link</a></p>

<p class="cite">Papassotiropoulos A and 19 others. 2006. Common <i>Kibra</i> alleles are associated with human memory performance. Science 314:475-478. <a href="http://dx.doi.org/10.1126/science.1129837">DOI link</a></p>

