---
layout: single 
title: "The dawn chumans" 
category: story
permalink: /weblog/reviews/genomics/divergence/dawn_chumans_patterson_2006.html
tags: [Sahelanthropus, genetic divergence, genomics] 
comments: false 
author: John Hawks 
---


<p>
Try to wrap your mind around <a href="http://www.msnbc.msn.com/id/12836649">this one</a>: 
</p>

<blockquote>Humans and chimps diverged from a single ancestral population through a complex process that took 4 million years, according to a new study comparing DNA from the two species.</blockquote>

<blockquote>...</blockquote>

<blockquote>The researchers hypothesize that an ancestral ape species split into two isolated populations about 10 million years ago, then got back together after a few thousand millennia. At that time the two groups, though somewhat genetically different, would have mated to form a third, hybrid population. That population could have interbred with one or both of its parent populations. Then, at some point after 6.3 million years ago, two distinct lines arose.</blockquote>

<p>
That's the writeup of <a href="http://dx.doi.org/10.1038/nature04789">a new paper by Nick Patterson and colleagues</a> in <i>Nature</i>. It's an advance publication, so I'm not sure it's widely available, but here's the first paragraph: 
</p>

<blockquote>The genetic divergence time between two species varies substantially across the genome, conveying important information about the timing and process of speciation. Here we develop a framework for studying this variation and apply it to about 20 million base pairs of aligned sequence from humans, chimpanzees, gorillas and more distantly related primates. Human-chimpanzee genetic divergence varies from less than 84% to more than 147% of the average, a range of more than 4 million years. Our analysis also shows that human-chimpanzee speciation occurred less than 6.3 million years ago and probably more recently, conflicting with some interpretations of ancient fossils. Most strikingly, chromosome X shows an extremely young genetic divergence time, close to the genome minimum along nearly its entire length. These unexpected features would be explained if the human and chimpanzee lineages initially diverged, then later exchanged genes before separating permanently.</blockquote>

<p>
I've read the paper, and I have to say it doesn't deliver on its promises. It fails to cite previous work on the topic, it discards without explanation the hypothesis supported by most previous studies, and it promotes a "provocative" hypothesis for which there is no good evidence. It doesn't even show that the speciation of humans and chimpanzees was "complex". 
</p>

<p>
It's just a mess. 
</p>

<p>
Some reactions: 
</p>

<p>
1. It is a breakthrough <i>Nature</i> has published a figure titled, "Genetic relationships differ from species relationships"! This is akin to pulling out of the dark ages. 
</p>

<p>
2. The dark ages end by baby steps at <i>Nature</i>, at least in terms of failure to demand citation of relevant literature. Consider the two most relevant recent studies of chimp-human divergence times: Wildman et al. (2004) and Yang (2002). Both these studies estimated human-chimpanzee divergence times on the order of 5 to 7 million years. Both studies use a large number of loci in their assessment. Wildman et al. (2004) consider the effects of natural selection on their results. Yang (2002) concludes that the variance among human-chimpanzee genetic divergences for different loci is relatively slight, leading to the conclusion that the effective population size of the human-chimpanzee ancestral population was probably small (on the order of 10,000 effective individuals). The Yang (2002) result is of interest, because it reflects <I>low</i> variance in human-chimpanzee genetic divergence, the opposite of that found in the current study. 
</p>

<p>
Several other studies might have been cited, such as Satta et al. (2004), or the granddaddy of all these kinds of studies, the original <i>Theoretical Population Biology</i> paper by Takahata, Satta, and Klein (1995), which lays out the theory and relation of genetic divergence to pre-speciation effective population size. This original paper by Takahata, Satta and Klein was the first to find a high degree of variance in genetic divergence times between humans and chimpanzees, and its conclusion was that the genetic divergences reflect a large effective population size in the pre-speciation human-chimpanzee ancestor. 
</p>

<p>
Usually I would think it excessively picky to point out papers that should have been cited. And I should note that I don't know who is to blame -- <i>Nature</i> has a limit on the number of references they will accept, so it may be a length constraint. But the thing is that these papers are the fundamental and most recent work in the area. And they are based on large datasets -- not as large as here, but not tiny as the authors imply. And their results are either very much like the result here, or contradict it in ways that should have been explained. There is a long, long literature on human-chimpanzee divergence and the variance of genetic divergences in light of population history. You wouldn't know it from reading this. 
</p>

<p>
3. The paper does cite Wall (2003), a study that concludes that the ancestral effective population size of the human-chimpanzee lineage was between around 50,000 and 75,000. An ancestral N<sub>e</sub> of 50,000 and a human-chimpanzee speciation time of 5 million years would imply an average time of genetic divergence around 7 million years ago, and a likely range from around 10 million to just over 5 million. That is at or more than the amount of variation in genetic divergences found in this study. <i>In other words, it isn't even a new result!</i> The result was <i>entirely anticipated</i> by earlier work on smaller datasets. There's nothing new here, other than the addition of more data. 
</p>

<p>
4. But even though it cites the earlier work that anticipates the finding of high variance in genetic divergence times, the current paper ignores <i>the explanation</i> given by those previous papers. There is hardly any mention of effective population size in this paper (the only place it figures in the text is in the consideration of low X chromosome variation). Instead, the paper assumes that <i>population structure</i> is the explanation for variance in genetic divergence times. 
</p>

<p>
5. But effective population size explains the current data <i>better</i> than population structure. It isn't reasonable to expect that the Patterson et al. paper would cite <a href="http://dx.doi.org/10.1093/molbev/msj109">this paper by Innan and Watanabe</a> in the current issue of <i>Molecular Biology and Evolution</i>, but check out the abstract: 
</p>

<blockquote>The coalescent process in the human-chimpanzee ancestral population is investigated using a model, which incorporates a certain time period of gene flow during the speciation process. a is a parameter to represent the degree and time of gene flow, and the model is identical to the null model with an instantaneous species split when a = infinity. A maximum likelihood (ML) method is developed to estimate a, and its power and reliability is investigated by coalescent simulations. The ML method is applied to nucleotide divergence data between human and chimpanzee. It is found that the null model with an instantaneous species split explains the data best, and no strong evidence for gene flow is detected. The result is discussed in the view of the mode of speciation. Another ML method is developed to estimate the male-female ratio (alpha) of mutation rate, in which the coalescent process in the ancestral population is taken into account.</blockquote>

<p>
In other words, the pattern of variation in genetic divergences between humans and chimpanzees is entirely consistent with a model in which there were no great separations between populations in the ancestral species. The hypothesis of hybridization after a long prior differentiation would predict a different pattern of variation than that observed. There is no current genetic evidence for hybridization among semi-isolated ancient ape species leading up to hominid origins. 
</p>

<p>
Why does the Patterson et al. study conclude that such hybridization may have occurred? Here is the key passage:
</p>

<blockquote>[I]t could explain the wide range of divergence times (more than 4 Myr): at some loci human and chimpanzee lineages share ancestry around initial separation, whereas at others the genetic ancestry is more recent at the time of hybridization. </blockquote>

<p>
But this is an error; there are not only genes with <i>ancient</i> divergences and others with <i>recent</i> divergences, there are also genes at every date in between, as shown by their comparisons. This is not the pattern of early divergence and hybridization, it is the pattern of one large ancestral population. 
</p>

<p>
Now, personally, I don't think it unlikely that the ancestral species consisted of several populations with long histories, possibly even subspecies. That is the pattern in most living hominoid species, and apparently a pattern that gave rise to long-term speciation of chimpanzees and bonobos. But that scenario doesn't necessarily predict long-term isolation and hybridization; it is compatible with recurrent and strong long-term interactions. The current data seem to say that those interactions were strong enough to make a point estimate of speciation just as good a model as a more highly parameterized model with partial isolation and gene flow. 
</p>

<p>
6. The relatively low divergence time of the X chromosome is not a mystery. The paper even says the obvious: 
</p>

<blockquote>[S]trong selection across chromosome X could produce this effect (4).</blockquote>

<p>
There is no need to invoke hybridization or any other hypothesis. In fact, hybridization cannot explain the X chromosome <i>in the absence of selection</i>. Natural selection is the necessary and sufficient hypothesis. And we know from recent genome surveys that selection on X-linked genes is more common than selection on autosomal genes (this seems partly to be attributable to their semidominance in males). The finding is interesting, but there is no mystery here. 
</p>

<p>
7. The paper makes a point of saying that the hybridization scenario provides an explanation for <i>Sahelanthropus</i>. If we have a long, long split with separate hybridizing populations, then you can say <i>Sahelanthropus</i> is a near-hominid. Maybe it's on the incipiently hominid side of the split, but hasn't yet attained many diagnostically hominid features because the incipient hominids are still fundamentally apes hybridizing with other apes. Of course, you can also say that it is on the <i>ape</i> side of the split, an incipient chimpanzee (or gorilla, or extinct ape lineage) that has some <i>hominid</i> features because those ancient apes still haven't shed away their hominid-like ancestral features. I would probably think the incipient-ape hypothesis is more likely than the incipient-hominid hypothesis, but you can see that both cases are a middle ground. 
</p>

<p>
But I have to point out that you don't need a hybridization scenario to account for this. For one thing, the paper demonstrates how widespread human-gorilla and human-orangutan parallelisms at the DNA level are. Why should we think that <i>Sahelanthropus</i>-human morphological parallelisms should be unlikely? For another, a diverse ancestral ape population does not necessarily require isolation and hybridization to account for occasional hominid-like features in individuals or local populations. 
</p>

<p>
8. Here is what I think is the worst part. It is sad that I have to care about it, but this idea of human-chimpanzee hybridization should be very appealing to creationists.
</p>

<p>
Now, I don't think that science should either entertain or reject hypotheses on the basis of creationist arguments. But I do think it is important to be cognizant of those arguments, since we can easily predict the way that some scientific hypotheses will be deliberately misinterpreted. I view it as important in this instance, because the "provocative" theory that early hominids were human-chimpanzee hybrids <i>has no empirical support!</i> It is provided in the paper <i>only</i> because it is provocative. It is making news headlines <i>only because it is provocative!</i> In that light, I think it is appropriate to be vocal about why it has no current scientific support. It is not an impossible hypothesis, but it is not currently justified. 
</p>

<p>
I think the most likely result is that the rest of us scientists will just ignore or move past this specific hypothesis. And it is possible that further evidence will emerge to make it seem more likely. But creationists will now cite Eric Lander in support of the idea that hominid fossils are not transitional between apes and humans, but instead are <i>hybrids</i> of apes and humans. After all, he's not only an author, he says this in <a href="http://www.msnbc.msn.com/id/12836649">the AP article</a>: 
</p>

<blockquote>"For the first time we're able to see the details written out in the DNA," said Eric Lander, founding director of the Broad Institute. "What they tell us at the least is that the human-chimp speciation was very unusual."</blockquote>

<p>
"Written out in the DNA." It's not true. There's no evidence here that the human-chimp speciation was unusual <i>in any way</i>. It is <i>not unusual</i> for two species to have a long period of genetic divergence before they cease reproductive contact with each other. It is <i>the normal mechanism of speciation in mammals.</i>

<p>
I just think it is important not to get carried away thinking that your "all new" hypothesis is going to rewrite evolutionary theory. When it doesn't, and it's not even new!
</p>

<h4>References:</h4>

<p class="cite">Innan H, Watanabe H. 2006. The effect of gene flow on the coalescent time in the human-chimpanzee ancestral population. Mol Biol Evol 23:1040-1047. <a href="http://dx.doi.org/10.1093/molbev/msj109">DOI link</a></p>

<p class="cite">Patterson N, Richter DJ, Gnerre S, Lander ES, Reich D. 2006. Genetic evidence for complex speciation of humans and chimpanzees. Nature (advance online access) <a href="http://dx.doi.org/10.1038/nature04789">DOI link</a></p>

<p class="cite">Satta Y, Hickerson M, Watanabe H, O'hUigin C, Klein J. 2004. Ancestral population sizes and species divergence times in the primate lineage on the basis of intron and BAC end sequences. J Mol Evol 59:478-487. <a href="http://dx.doi.org/10.1007/s00239-004-2639-2">DOI link</a></p>

<p class="cite">Takahata, N., Y. Satta, and J. Klein. 1995. Divergence time and population size in the lineage leading to modern humans. Theor. Popul. Biol. 48:198-221. </p>

<p class="cite">Wall JD. 2003. Estimating ancestral population sizes and divergence times. Genetics 163:395-404. <a href="http://www.genetics.org/cgi/content/abstract/163/1/395">Abstract</a></p>

<p class="cite">Wildman DE, Uddin M, Liu G, Grossman LI, Goodman M. 2004. Implications of natural selection in shaping 99.4% nonsynonymous DNA identity between humans and chimpanzees: Enlarging genus <i>Homo</i>. Proc Nat Acad Sci USA 100:7181-7188. <a href="http://dx.doi.org/10.1073/pnas.1232172100">DOI link</a></p>

<p class="cite">Yang Z. 2002. Likelihood and Bayes estimation of ancestral population sizes in hominoids using data from multiple loci. Genetics 162:1811-1823. <a href="http://www.genetics.org/cgi/content/abstract/162/4/1811">Abstract</a></p>

