---
layout: single 
title: "Microbiomic metagenomics" 
category: story
permalink: /weblog/reviews/genomics/meta/intestinal_microflora_metagenomics_2006.html
tags: [genomics, Late Pleistocene] 
comments: false 
author: John Hawks 
---


<p>
At the movies last night I saw a preview for <i>Superman</i>,  and now <i>Science</i> tells me that humans are <i>superorganisms!</i>

<blockquote><b>Metagenomic Analysis of the Human Distal Gut Microbiome</b></blockquote>

<blockquote>Steven R. Gill et al.</blockquote>

<blockquote>The human intestinal microbiota is composed of 10<sup>13</sup> to 10<sup>14</sup> microorganisms whose collective genome ("microbiome") contains at least 100 times as many genes as our own genome. We analyzed 78 million base pairs of unique DNA sequence and 2062 polymerase chain reaction-amplified 16<i>S</i> ribosomal DNA sequences obtained from the fecal DNAs of two healthy adults. Using metabolic function analyses of identified genes, we compared our human genome with the average content of previously sequenced microbial genomes. Our microbiome has significantly enriched metabolism of glycans, amino acids, and xenobiotics; methanogenesis; and 2-methyl-D-erythritol 4-phosphate pathway-mediated biosynthesis of vitamins and isoprenoids. Thus, humans are superorganisms whose metabolism represents an amalgamation of microbial and human attributes.</blockquote>

<p>
This is really important stuff -- our nutrition is very dependent on these microbes, and there is every reason to think that their ecology affects our overall health status as well. And we know very little about them -- heck, these guys are using the same metagenomic techniques to fine organisms <i>in our bodies</i> that are used to find new <i>unidentified ocean life!</i>

<p>
But notice that even this study with all its comparisons of gross gene function among all these microbes has only included two humans. The next step will be finding the variability among people in intestinal ecologies, how humans have adapted to their microbes by means of genetic evolution, and how different people have different health susceptibilities as a result. 
</p>

<p>
The metagenomic technique gives two assessments at once -- the genes of the microbes and their relative abundances (through the relative numbers of reads). I wonder if there is some sense in treating the entire community as an evolutionary unit -- one that can shift its proportions of different parts over time and in different hosts. 
</p>

<p>
There should be much of interest here, considering the dietary and other evolutionary changes within humans during the Late Pleistocene and Holocene. 
</p>

<h4>References:</h4>

<p class="cite">Gill SR, Pop M, DeBoy RT, Eckburg PB, Turnbaugh PJ, Samuel BS, Gordon JI, Relman DA, Fraser-Liggett CM, Nelson KE. 2006. Metagenomic analysis of the human distal gut microbiome. Science 312:1355-1359. <a href="http://dx.doi.org/10.1126/science.1124234">DOI link</a></p>

