---
layout: single 
title: "The 200 genome challenge" 
category: story
permalink: /weblog/topics/biotech/testing/genomics_x_prize_2006.html
tags: [testing, genomics, biotech] 
comments: false 
author: John Hawks 
---


<p>
Nick Wade has <a href="http://www.nytimes.com/2006/10/04/science/04cnd-genome.html">the story</a>: 
</p>

<blockquote>A $10 million prize for cheap and rapid sequencing of the human genome was announced today by the X Prize Foundation of Santa Monica, Calif.</blockquote>

<blockquote>The terms of the prize require competitors to sequence 100 human genomes of their choice within 10 days, and, within six months, those of a further 100 individuals chosen by the foundation.</blockquote>

<blockquote>Peter Diamandis, the chairman of the foundation, said that the second list would include patients nominated by disease advocacy groups and celebrities. Those who have already signed up include the astrophysicist Steven Hawking; the television interviewer Larry King; Paul Allen, the co-founder of Microsoft; the financier Michael Milken, and Google's co-founder Larry Page.</blockquote>

<p>
Now, the celebrity angle is interesting, but basically frivolous. In fact, probably worse than frivolous, since somebody might think they can actually use that sample to <i>say</i> something. I like the idea of having "disease advocacy groups" nominate people, but it will be most helpful if there are many individuals per disease, so 100 may not be too much help. On the other hand, if they can manage to do 100 people for 5 million dollars or less, these kinds of samples will soon be well within the scope of NIH grants for the study of these diseases. 
</p>

<p>
The real question is, <i>how are they going to test the accuracy of the reconstucted genomes?</i> I mean, anybody can churn out billions of "nucleotides" -- my computer can spit them out in a second or two. Of course, all mine are based on <i>random numbers</i>. Only a very few can do billions of nucleotides from a real genome with 8 or 9 "nines" of accuracy. 
</p>

<p>
So what kinds of detection methods are in place to ensure that the submitted sequences aren't full of errors? I suppose the 100 "picked" individuals help, since they will be sequenced redundantly by different competitors. But if that is the main method of error detection, we may have to wait a while to find out who the winner is. 
</p>

<p>
According to the article, these kinks haven't quite been worked out yet: 
</p>

<blockquote>The X Prize Foundation has not yet determined a critical parameter of its prize, that of how complete the genomes need to be. The present complete human genome has many gaps and is only as complete as present technology can make it.</blockquote>

<p>
And there is this interesting tidbit: 
</p>

<blockquote>The 454 Life Sciences company has been working to decode the genome of Dr. James Watson, co-discoverer of the structure of DNA, but it has not yet completed it.</blockquote>

<p>
Well, it should be interesting to see what <i>he</i> has the genes for...
</p>

<p>
In the meantime, I think this competition sends a pretty clear message to those of us interested in genetics and human evolution. If you're not working with methods that can scale to datasets on the order of hundreds of complete genomes, you're already behind. 
</p>

