---
layout: single 
title: "How much are deletions like SNPs?" 
category: story
permalink: /weblog/reviews/genomics/hinds_2006_intermediate_deletions.html
tags: [genomics] 
comments: false 
author: John Hawks 
---


<p>
Hinds et al. (2006) examine the pattern of common deletion polymorphisms in the human genome. These are genetic variants in which the alleles have different lengths, the shorter resulting from the deletion of genetic material that was originally present (and is today found both in some humans and in other primates). They focused on intermediate-length deletions: those lying between a few base pairs and thousands. 
</p>

<p>
Here's some background: 
</p>

<blockquote>SNPs are the result of errors in DNA replication or repair that occurred once in human history and are shared among individuals by descent. Very small common deletions and insertions, in the range of 15 bp, show strong linkage disequilibrium with common SNPs, which suggests that, although the mechanisms giving rise to them may differ, these polymorphisms share a similar evolutionary history. It is well documented that diseases classified as genomic disorders, such as DiGeorge or velocardiofacial syndrome, alpha-thalassemia, Williams-Beuren syndrome and Charcot-Marie-Tooth disease type 1A, result from recurring mutations involving large deletions, insertions and other genomic alterations. These recurring mutations are the result of non-allelic homologous recombination events that occur between blocks of duplicated sequences (>95% sequence identity, >10 kb in length, and separated by 50 kb to 10 Mb)21. Here we and other concurrent reports in this issue show that intermediate-length deletion polymorphisms contribute to common genetic variation in healthy individuals. Our report focuses on whether these common deletion polymorphisms are the result of single mutation events such as SNPs or are due to recurring mutational events such as those resulting in genomic disorders (Hinds et al. 2006:82, references redacted).</blockquote>

<p>
They find that the intermediate-length deletions (like the short deletions) pretty much behave like SNPs. 
</p>

<blockquote>The set of common intermediate-length deletions identified here has linkage disequilibrium patterns similar to SNPs, indicating that these polymorphisms share a similar evolutionary history and suggesting that most intermediate-length deletions, like SNPs, arose once in human history. High linkage disequilibrium with nearby SNPs suggests that most of these deletions are effectively assayed by proxy in SNP-based association studies, consistent with previous results for short insertion/deletion polymorphisms. On the basis of the fraction of the genome examined and the technical limits of our study, we estimate there are several thousand intermediate insertion/deletion polymorphisms in the human genome, suggesting that they represent an important component of common genetic variation and are likely to contribute to phenotypic variation in complex traits (ibid:85).</blockquote>

<p>
This means that intermediate-length deletions that recur again and again at the same genomic location are probably rare (via <a href="http://evolgen.blogspot.com/2006/01/are-deletions-deleterious-part-1.html">evolgen</a>). 
</p>

<h4>References:</h4>

<p class="cite">Hinds DA, Kloek AP, Jen M, Chen X, Frazer KA. 2006. Common deletions and SNPs are in linkage disequilibrium in the human genome. Nat Genet 38:82-85. <a href="http://www.nature.com/ng/journal/v38/n1/full/ng1695.html">Full text (subscription)</a></p>

