---
layout: single 
title: "Prostate risk allele differs in Europeans and Africans" 
category: story
permalink: /weblog/reviews/genetics/disease/prostate_risk_african_americans_2006.html
tags: [America, disease, risk] 
comments: false 
author: John Hawks 
---


<p>
Thanks to an enterprising student, I have <a href="http://www.msnbc.msn.com/id/12685715/">an AP story</a> about the discovery of a risk allele for prostate cancer that has different frequencies in different groups. 
</p>

<blockquote>Scientists have identified a common genetic marker that signals a 60 percent heightened risk of prostate cancer in men who carry it, and it may help explain why black men are unusually prone to the disease, a new study says.</blockquote>

<blockquote>The DNA variant may play a role in about 8 percent of prostate cancers in men of European extraction and 16 percent of the cancers in blacks, researchers said.</blockquote>

<p>
So, what does somebody like me do with this information? I'm interested in cases where populations are different as illustrations of human biological variation and its evolutionary causes. Here, we have medical data and genomic information that appears to show a geographic difference among men in their risk. But we have to look at the study to see what that means. Here's the abstract:
</p>

<blockquote>With the increasing incidence of prostate cancer, identifying common genetic variants that confer risk of the disease is important. Here we report such a variant on chromosome 8q24, a region initially identified through a study of Icelandic families. Allele -8 of the microsatellite DG8S737 was associated with prostate cancer in three case-control series of European ancestry from Iceland, Sweden and the US. The estimated odds ratio (OR) of the allele is 1.62 (<i>P</i> = 2.7 x 10<sup>-11</sup>). About 19% of affected men and 13% of the general population carry at least one copy, yielding a population attributable risk (PAR) of &tilde; 8%. The association was also replicated in an African American case-control group with a similar OR, in which 41% of affected individuals and 30% of the population are carriers. This leads to a greater estimated PAR (16%) that may contribute to higher incidence of prostate cancer in African American men than in men of European ancestry.</blockquote>

<p>
This is actually a great case study in the complexity of finding the genetic causes of disease: 
</p>

<blockquote>As only the microsatellite allele showed significant association in the African American case-control group, and it is contained in a smaller LD block in African Americans than in populations of European ancestry (Supplementary Fig. 2 online), we propose that the region most likely to contain the functional variant can be narrowed down to positions 128.414-128.474 Mb (NCBI build 34). This region contains one spliced EST (AW183883) and three single-exon ESTs (BE144297, CV364590 and AF119310) in addition to a few predicted genes, but no known genes. No microRNAs have been detected within the block. Expression analysis in various cDNA libraries confirmed only the expression of the AW183883 EST (Supplementary Methods). We identified four different splice variants of AW183883 by 5'and 3' RACE that were verified by RT-PCR and RNA blot analysis (Fig. 2a). Using the AW183883 EST as a probe on an RNA blot, we detected a 1.5-kb signal only in testis, consistent with the size of the two longer forms. The two shorter transcripts harboring exons 6-8 were detected only in normal (0.6-kb transcript) and malignant (0.6- and 0.9-kb transcripts) prostate cell lines, not in the other tissues analyzed (Fig. 2b,c). The predicted ORFs for these transcripts did not show significant homology to known proteins.</blockquote>

<p>
In other words, although they have narrowed down a region around this marker as a risk factor for prostate cancer, they don't know what specific genetic change causes the risk. It isn't any of the SNPs in the HapMap set. They can tell from the HapMap variation that the linkage block is narrower in Africans, so that the causative variant must be fairly near it. There are no characterized genes and no known microRNAs in the region, but there are some possible genes that haven't been characterized. "cDNA" libraries are complements of mRNA expressed within cell lines, only one of these may correspond to the region in question, and one variant is expressed in testis and prostate cell lines. They don't know what it is or what it does. 
</p>

<p>
All of that analysis is bioinformatics -- taking databases of known gene expression in cell lines, genetic variation among people, and identified genes, SNPs and microsatellite markers and integrating them into a picture of associations. What is left is a lot of biology -- what does the key genetic variant change; what gene is it part of; what is the normal role of the gene within prostate (or elsewhere); why does it become pathological; etc. And most important, how can we fix it?
</p>

<p>
My interest is at the opposite end --- why did this allele become common; why does it have different frequencies in different groups; how old is it? 
</p>

<p>
There are no answers to these questions -- there rarely are in studies like this. But put several of them together and we may start to uncover much about human prehistory and the conditions of life in ancient populuations. Does this risk allele correspond to different ancient environments? Different population dynamics? Or is it just different by chance? 
</p>

<p>
That is the breakdown on the one hand between bioinformatics and biology, and on the other between functional biology and evolutionary biology. 
</p>

<h4>References:</h4>

<p class="cite">Amundadottir LT and 45 others. 2006. A common variant associated with prostate cancer in European and African populations. Nature Genet (online early) <a href="http://dx.doi.org/10.1038/ng1808">DOI link</a></p>

