---
layout: single 
title: "Introgression and microcephalin FAQ" 
category: story
permalink: /weblog/reviews/neandertals/neandertal_dna/introgression_faq_2006.html
tags: [Neandertals, introgression, MSA, Middle Pleistocene] 
comments: false 
author: John Hawks 
---


<p>
Considering the <a href="http://www.pnas.org/cgi/content/abstract/0606966103v1">paper</a> by Evans and colleagues, I've come up with a list of questions and answers: 
</p>

<h4>What is introgression?</h4>

<p>
<i>Introgression</i> is the transfer of alleles across species or subspecies boundaries. In other words, it describes gene flow between populations that are partially isolated. For archaic humans, there is no test of the strength or permeability of boundaries between populations; it is common to use the term "introgression" to describe gene flow in such situations, even if such gene flow is fairly common. 
</p>

<p>
The paper by Evans and colleagues describes a scenario of <i>adaptive introgression</i>. In such cases, an allele with a selective advantage moves from one population to another. 
</p>

<h4>Adaptive introgression must be a very unusual event, right? I mean, I've never heard of it before!</h4>

<p>
If you haven't heard of adaptive introgression, you haven't been reading the literature. Adaptive introgression <i>across species boundaries</i> is very common in mammals, and is almost ubiquitous where closely related species are sympatric. It has long been known to happen on the basis of morphological characters that spread through hybrid zones into adjacent populations. But now that molecular surveys have become common, introgressive genes have been found moving out of <i>current</i> hybrid zones, and also in the areas where hybrid zones likely occurred <i>long in the past</i>. 
</p>

<p>
Hybrid zones themselves are often quite obvious. But introgression is not about hybrids. It occurs when backcrosses spread alleles into the other parental species. Hybrids may have a mixture of many genes and characters. Introgression involves a small number of genes, which are much more likely to spread if alleles are adaptive. Where different populations are in reproductive contact, adaptive introgression may often be the most important source of adaptive alleles -- it provides a way for a species or population to benefit from the adaptive evolution of neighboring species. 
</p>

<p>
There is one thing that impedes introgression: linkage to deleterious alleles. Species separated for longer times are more likely to have alleles that are <i>bad</i> on the genetic background of related species, and so potential <i>adaptive</i> alleles must have advantages outweighing all the <i>deleterious</i> alleles they are linked to. In these situations, adaptive introgression may only occur after enough recombination has broken the adaptive allele apart from some or all of its linked deleterious neighbors. 
</p>

<h4>But I thought that "species" means "no interbreeding!"</h4>

<p>
Get with the times, man! Mammal species just don't establish reproductive barriers very quickly. Comparing mammals, postzygotic isolating mechanisms take between 2 and 10 million years to evolve. No primate species pairs have evolved postzygotic isolation on the timescale represented by the evolution of <i>Homo</i>. When archaic and modern humans were in contact, they certainly interbred. 
</p>

<h4>OK, but why is <i>this</i> gene introgression? Why couldn't it just have originated in ancient Africans? </h4>

<p>
The current evidence for introgression comes from the <i>mismatch</i> between the ancient coalescence time for all haplogroups of the <i>microcephalin</i> gene, compared to the very recent selection on the D haplogroup. Now recent selection on an ancient variant could occur within a single population, for example, if the allele was formerly neutral and gained a new advantage with some difference in the genetic background. And an ancient coalescence date would not be unusual in a single population -- several other loci match the 1.7 million years estimated for the <i>microcephalin</i> genealogy. 
</p>

<p>
Two things make this case especially persuasive. First, there is almost no evidence of recombination <i>between</i> the D and non-D haplogroups. If they existed <i>within the same population</i> for 1.7 million years, they should have recombined a lot with each other, and we should see some of those recombinants today. We don't. The best explanation is that the alleles were in different ancient populations, somewhat isolated from each other so that recombination was very rare. 
</p>

<p>
Second, the D haplogroup is common in Europe and Asia, but is very rare in Africa. If it increased under selection from its origin in some ancient African population, then it ought to be most common in Africa now. We might also expect a deeper origin for the D haplogroup in Africa, similar to the structure of many other genetic loci. We observe neither. 
</p>

<h4>Hey, why should this gene be so unique? There's never been any evidence for archaic genes before!</h4>

<p>
Now, this is clearly where I have let you down, by not blogging about these papers as they have been coming out. What can I say, I have to make a living somehow! If I give away all my research, how can I stay a step ahead? 
</p>

<p>
The most similar locus to <i>microcephalin</i> is the region around <i>MAPT</i> on chromosome 17. Hardy and colleagues (2005) suggested that this locus is a Neandertal introgression. Like <i>microcephalin</i>, the locus has an ancient coalescence (>2 million years), and like <i>microcephalin</i>, an allele is under selection, with its highest current frequency in Europe. Like <i>microcephalin</i>, <i>MAPT</i> is brain-active, with most research centered on its possible role in Alzheimer's and Parkinson's disease. Unlike <i>microcephalin</i>, there are <i>no</i> recombinants between the major (H1 and H2) haplogroups; this is due to a chromosomal inversion between them. Evans and colleagues (2006) note that balancing selection might not be statistically ruled out when there is such an inversion preventing recombination. Still, balancing selection doesn't easily explain the recent positive selection, nor the geographic distribution of variation. 
</p>

<p>
Garrigan et al. (2005b) found evidence for an ancient <i>Asian</i> allele being retained in living Asians. This allele was from a non-coding locus, so it seems unlikely that adaptive introgression is the cause, which might suggest even more widespread genetic survival of archaic DNA. Some loci suggest the survival of archaic lineages within Africa, including another X chromosome noncoding region (Garrigan et al. 2005a) and the <i>dystrophin</i> gene (Zietkiewicz et al. 2003). These would presumably be attributable to partial isolation of Middle Pleistocene African populations, with introgressive gene flow among them. 
</p>

<p>
The widest survey for introgression thus far was by Plagnol and Wall (2006), who conclude that around five percent of human genes show some evidence for introgression from archaic humans. Their statistical test was looking for loci with ancient divergence times and in particular divergent alleles centered in Eurasian (non-African) populations. So this is a kind of estimate under the assumption of relatively great genetic differentiation among archaic human populations.
</p>

<p>
I'll end with Templeton (e.g., 2005), who found that human autosomal variation supports a broad ancestry of living humans among Eurasian and African archaics, with evidence of genetic dispersals from Africa several times during the Pleistocene. Under this model, intermixture among archaic populations would have been fairly common, at least intermittently. This is the argument that I made with <a href="http://www-personal.umich.edu/~wolpoff/">Milford Wolpoff</a> several years ago (Hawks and Wolpoff 2001) -- we just don't see a lot of evidence for genetic differentiation among archaic humans. 
</p>

<p>
This kind of model would imply that genes like <i>microcephalin</i> -- with strong evidence for some isolation of populations -- might be fairly rare. The fact that several of them have now cropped up (the 5 percent estimate from Plagnol and Wall, 2006, being the most informative on this score) means that we have a lot about archaic human population structure yet to discover. 
</p>

<p>
But notice the nature of this uncertainty. We have a difference between <i>substantial</i> introgression among populations structured like hominoid subspecies on one side, and <i>ubiquitous</i> genetic exchanges among populations structured like human races on the other side. Complete replacement is completely out. "Mostly" replacement, or "assimilation" is still in, but with the observation that archaic human genes had substantial evolutionary importance in the adaptation of modern humans. 
</p>

<p>
In other words, we have moved the ball down the field. Time to line up for the next play. 
</p>

<h4>What is all this about <i>microcephalin</i> possibly not being from Neandertals? </h4>

<p>
Well, the D haplogroup is common in many areas outside of Africa in addition to Europe. So it isn't possible to really specify in what archaic population it may have originated. There is some chance that it may be found in the Neandertal genome sequence, when that becomes available. In fact, that would be the ultimate test for many candidate introgressive alleles. 
</p>

<p>
But there is a good chance that it <i>won't</i> be found in the Neandertal sequence. After all, Neandertals were probably pretty thin on the ground -- especially in Europe. A sampling of their genes would be sort of unlikely to yield a high proportion of archaic alleles that may have survived to the present day. So there is hope that we will find and document such alleles, but the best evidence for many of them may remain their current pattern of variation in living people. 
</p>

<h4>Now, bear with me here. Neandertals were stupid, right? So why would one of their <i>brain</i> genes be <i>advantageous</i> in modern humans? </h4>

<p>
There are so many possibilities here. 
</p>

<p>
1. Late Neandertals certainly weren't stupid. Consider the Ch&acirc;telperronian. And the European Mousterian includes basically all the elements that are thought to represent cognitive sophistication in MSA Africans. 
</p>

<p>
2. Neandertal brains were big, and their heat generation requirements means that energetic constraints were very different from other archaic populations. The brain doesn't function in isolation -- its development, growth, and ongoing maintenance depend on metabolic constraints. So Neandertals might easily have had brain development alleles that had different responses to their high-energy lifestyles. Considering that early Upper Paleolithic people had much more effective foraging strategies than Neandertals, high-energy brain development may have had an even greater advantage than it had previously enjoyed. 
</p>

<p>
3. Modern humans are variable in brain morphology and cognition. That variability certainly includes alternative strategies (for example, personality types) that may be maintained by frequency-dependent selection. An archaic population that had particular constraints on its behavioral strategies might have given rise to strategies that worked within the modern human mix. In that context, Neandertals are fairly unique in having a very strong dietary dependence on meat, and their means of hunting was both risky and required cooperation. That adaptation may have led to behavioral strategies that succeeded in modern humans, even as Neandertal anatomies disappeared. 
</p>

<p>
Those are some possibilities we are working on. There are probably many others. The key is that we are looking at the function of some genes that survived, through our reconstruction of the total phenome of a population that no longer survives. We are limited by the evidence, but there are many suggestive hypotheses. 
</p>

<h4>Neandertals went extinct! Their features disappeared in later humans! How can any of their genes have survived?</h4>

<p>
This is my favorite one to answer, because it invokes the true paradox of introgression. The features that we <i>recognize</i> as Neandertal features, were <i>defined</i> as Neandertal features by virtue of the fact that they <i>are mostly gone!</i> That means that any alleles correlated with Neandertal morphological features were almost certainly selected <i>against</i>, or were at best neutral. That means that those <i>recognizably</i> Neandertal genes are <i>gone!</i>

<p>
But here we have a gene that looks to have come from some archaic population. Adaptive introgression occurs when adaptive alleles are selected, and broken apart from their genetic background. So even as many (perhaps most) Neandertal alleles disappeared, <i>some</i> of their alleles began to <i>increase</i> in frequency -- slowly at first, then very rapidly. 
</p>

<p>
Some adaptive introgressions may already have been fixed, particularly in Europe (from Neandertals). Others, like <i>microcephalin</i>, are still growing in frequency. The key is to remember Mendel -- this is not <i>blending inheritance</i> of Neandertal traits, it is the extinction of many alleles and the proliferation of some others. 
</p>

<p>
The reduction in frequency of Neandertal-like morphological traits over time is entirely consistent with this scenario. In fact, it shows the widespread importance of Neandertal-modern matings, which led to the emergence of a modern population with many Neandertal traits. The widespread genetic contact is documented by the distribution of the traits -- with different Neandertal-like traits in different specimens. That kind of contact is most likely to enable adaptive introgression to proceed. 
</p>

<p>
UPDATE (11/8/2006): Fixed some citations. 
</p>

<h4>References:</h4>

<p class="cite">Evans PD, Mekel-Bobrov N, Vallender EJ, Hudson RR, Lahn BT. 2006. Evidence that the adaptive allele of the brain size gene <i>microcephalin</i> introgressed into <i>Homo sapiens</i> from an archaic <i>Homo</i> lineage. Proc Nat Acad Sci (early edition) <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.0606966103">DOI link</a></p>

<p class="cite">Garrigan, D., Mobasher, Z., Kingan, S. B., Wilder, J. A., Hammer, M. F. 2005a. Deep haplotype divergence and long-range linkage disequilibrium at Xp21.1 provides evidence that humans descend from a structured ancestral population. Genetics 170:1849-1856. </p>

<p class="cite">Garrigan, D., Mobasher, Z., Severson, T., Wilder, J. A., Hammer, M. F. 2005b. Evidence for archaic Asian ancestry on the human X chromosome. Mol. Biol. Evol. 22:189-192. <a href="http://dx.doi.org/10.1093/molbev/msi013">DOI link</a>. </p>

<p class="cite">Hardy, J., Pittman, A., Myers, A., Gwinn-Hardy, K., Fung, H. C., de Silva, R., Hutton, M. and Duckworth, J. 2005. Evidence suggesting that <i>Homo neanderthalensis</i> contributed the H2 MAPT haplotype to <i>Homo sapiens</i>. Biochemical Society Transactions 33:582-585. </p>

<p class="cite">Hawks, J., Wolpoff, M. H. 2001. The accretion model of Neandertal evolution. Evolution 55:1474-1485. </p>

<p class="cite">Plagnol, V., Wall, J. D. 2006. Possible ancestral structure in human populations. PLoS Genet. 2:e105. <a href="http://dx.doi.org/10.1371/journal.pgen.0020105">DOI link</a>. </p>

<p class="cite">Templeton AR. 2005. Haplotype trees and modern human origins. Yrbk Phys Anthropol 48:33-59. <a href="http://dx.doi.org/10.1002/ajpa.20351">DOI link</a></p>

<p class="cite">Zietkiewicz, E., Yotova, V., Gehl, D., Wambach, T., Arrieta, I., Batzer, M., Cole, D. E., Hechtman, P., Kaplan, F., Modiano, D., Moisan, J. P., Michalski, R., Labuda, D. 2003. Haplotypes in the dystrophin DNA segment point to a mosaic origin of modern human diversity. Am. J. hum. Genet. 73:994-1015. </p>

