---
layout: single 
title: "Human Genome Project afterglow" 
category: story
permalink: /weblog/topics/biotech/human_genome_plus_5_scientist_2006.html
tags: [biotech, metagenomics] 
comments: false 
author: John Hawks 
---


<p>
I was reading <i>The Scientist</i> because <a href="http://scienceblogs.com/evolgen/2006/02/genome_sequencing.php">RPM sent me</a> to <a href="http://www.the-scientist.com/article/display/23065/">this article, titled "The Human Genome Project +5"</a>. 
</p>

<blockquote>And yet the last five years, in Olson's view, have been "a period of a great grinding of gears, kind of shifting of gears." In the terms of the science historian Thomas Kuhn, it's been "a period of consolidation and more normal science." Others, such as Sydney Brenner of the Salk Institute, the Nobel Prize-winning pioneer of the worm, Caenorhabditis elegans, go further, worrying that the genome sequence and the growing lists of sequences and proteins and protein interactions and functional elements don't get very deep into such core problems of biology as the operations of the cell, of development from egg to adult, or the problem of consciousness. "We've become very geno-centric," says Brenner. "The cell must become the focus."</blockquote>

<p>
I would say this is pretty much correct -- there has been a long period of normal science in genetics lately, with new findings pretty much following one after another. There have been no revolutions coming out of the HGP. 
</p>

<p>
But I think this scale of examination is a bit misleading. The HGP opened the deep end of the data pool, and we are still swimming in the toddler tank. 
</p>

<p>
Consider what is happening in terms of new data: 
</p>

<blockquote>One of the most dramatic efforts to push genomics into the realm of complex, multi-genic diseases is the five-year, $138 million haplotype map (HapMap) project, involving samples donated by Japanese, Han Chinese, Yoruba, and Americans of European descent. The project takes advantage of the fact that the millions of single nucleotide polymorphisms (SNPs) found in at least one percent of humans tend to pass between generations in blocks of DNA called haplotypes. The project announced its Phase 1 analysis in October 2005, and said that the analysis of Phase 2, already completed, would be published in 2006. Despite successes, such as using HapMap data to pinpoint a gene for macular degeneration, there remains controversy over HapMap's reach into domains such as rearrangements like deletions and reversals, or the numerous rare mutations that may be involved in diseases.</blockquote>

<blockquote>The minor variations are of central interest to Bentley of Solexa, who has specialized in rare variations. The HapMap, he says, has limitations, capturing only common variations in three target populations, missing the rare mutations. But it may provide a quick way to find more disease genes. Still, in three to five years, he says, the new sequencing machines should open the option of going after virtually all the many genes involved in a disease like diabetes. To be sure, the multiple sequences of patients and "controls" will have to square with what HapMap has found. "Everything that a HapMap captures should also be captured by a technology that aims to do better." Bentley, an early proponent, calls the HapMap "a real benchmark."</blockquote>

<p>
There seems to be a "Moore's law" for genome sequencing: 
</p>

<blockquote>The workhorses of the 2001 human drafts have kept doubling their throughput about every 22 months over 15 years. In September, 454 reported that, in a single run, its system did a shotgun sequence and assembly of the microbe, Mycoplasma genitalium, in four hours. Claire Fraser's team at the Institute for Genomic Research took three months to work out Mycoplasma's sequence in 1995.</blockquote>

<p>
And there are gene expression microarrays and microRNA assays, as described in the article. For people who want to know about gene activity at every stage of life, in every type of cell, and in response to every external stimulus, the tools are in place to figure those things out. 
</p>

<p>
As for myself, I think the accumulating data will have some revolutionary effects. These won't be in genetics itself -- I think the paradigms in place now in terms of gene interactions and regulation are very powerful. No doubt some new twists in gene sequence and function will be found, but I would guess that the current picture will expand rather than being overthrown. 
</p>

<p>
But for other fields, I think genetics has some revolutionary power. Obviously genomic medicine has the potential to radically change the way we approach chronic conditions. And metagenomics is already changing the way biologists study microbial communities in all kinds of environments. It wouldn't surprise me if scientists working in places like <a href="http://johnhawks.net/weblog/topics/humor/garden_lost_world_new_guinea_2006.html">the Foja Mountains</a> work with DNA tag samples before they do traditional taxonomy on new species. 
</p>

<p>
What will happen to anthropology as a result of the HapMap? There are surprises in store...
</p>

