---
layout: single 
title: "African elephant population structure" 
category: story
permalink: /weblog/reviews/genetics/non-primate/elephant_genetic_differentiation_2006.html
tags: [non-primate] 
comments: false 
author: John Hawks 
---


<p>
I got in a conversation today about <a href="http://dx.doi.org/10.1111/j.1096-0031.2004.00044.x">this paper</a> by R&eacute;gis Debruyne: 
</p>

<b>A case study of apparent conflict between molecular phylogenies: the interrelationships of African elephants</b>

<blockquote>Recent molecular phylogenies of the African elephants suggest that there is an evolutionary structure within Loxodonta africana. Some nuclear results (Roca et al., 2001) support the separation of the forest African elephant subspecies L. a. cyclotis as a species distinct from the savannah elephant L. a. africana, on the basis of the recognition of both forming highly divergent (reciprocally monophyletic) clades. Conversely, a mitochondrial survey (Eggert et al., 2002), while admitting a geographic partitioning of the genetic structure within African elephants, suggests retaining the status quo. They recognize three diagnosible entities (western, central and south-eastern Africa) with non-overlapping ranges within L. africana sensu lato. In order to address these conflicting views (historical fragmentation and speciation or isolation by distance, respectively), we have sequenced two datasets of 1961 bp (for 50 elephants) and about 3700 bp, respectively (for 20 elephants) of the mitochondrial DNA for both forms of elephants (cyclotis and africana). They span the cytochrome b gene, the control region and several RNAs. When compared with former mtDNA data, they provide the most comprehensive view of the African elephant phylogeny (78 mtDNA haplotypes, of which 44 are new) and provide the first insight into populations from the Democratic Republic of Congo. The genetic diversity of mtDNA was appraised and the stability of alternative phylogenetic trees was investigated. Our results are inconsistent with both those prior studies. They revealed two highly divergent molecular clades referred to as F and S, that do not conform to the morphological delineations of cyclotis and africana. A non-negligible proportion of specimens of L. a. africana display haplotypes prevailing in forest elephant populations (clade F). The geographic distribution of clades and areas of their co-occurrence support the hypothesis of incomplete isolation between forest and savannah African elephant populations, followed by recurrent interbreeding between the two forms. We state that the conclusions of prior studies resulted from insufficient character and/or geographic sampling. We conclude that there is no satisfying argument which can recognize two or more species of African elephants. We briefly comment on the meaning of such an attitude in a conservation viewpoint.</blockquote>

<p>
The basic problem here was sampling. Earlier, smaller samples suggested strong geographic partitioning; larger samples (in this case, only of a single gene) were sufficient to show lots of interbreeding. 
</p>

<p>
With apes, we see this conflict not only with respect to sampling density but also with respect to different loci. Chimpanzees, gorillas, and orangutans have all been argued to have very high genetic differentiation between subspecies, but this evidence suggesting great genetic differences has been mtDNA and Y chromosomal. Datasets of nuclear loci have more or less looked like more recent genetic differentiation, with more evidence of mixture. 
</p>

<p>
Part of this pattern is entirely expected. Because of the smaller effective size of mtDNA, the effective rate of migration is a fourth that of the autosomes. This means that the genetic differences between populations ought to be four times higher. 
</p>

<p>
But the genetic divergences shouldn't be <i>older</i> -- in fact, the mtDNA should have a more recent coalescence age than the autosomes. In particular, if two popluations diverged from a single ancestral population at time <i>t</i> generations in the past, then the mtDNA divergence date on expectation will be <i>t</i> + <i>N</i>/2, while the mean divergence date for autosomal loci will be <i>t</i> + 2<i>N</i>. 
</p>

<p>
So the greater genetic differentiation of the ape subspecies is quite expected, but the older mtDNA divergence times are not. 
</p>

<p>
Or are they? 
</p>

<p>
Let's alter the assumptions. Instead of supposing that the populations diverged from a single ancestral population at time <i>t</i>, let's instead suppose that they have always had the same population structure as today; with partially isolated subspecies sharing some gene flow. 
</p>

<p>
In this model, the mtDNA differentiation will be higher than the autosomal differentiation. 
</p>

<p>
Also in this model, the mtDNA genetic divergence between the populations will be substantially older than the average divergence of autosomal loci. This is because the more limited gene flow for the mtDNA impedes its coalescence, as we look toward the past. 
</p>

<p>
In other words, a long-term structure of limited gene flow between populations will generate the pattern where the mtDNA has a high divergence date and high population differentiation, at the same time that autosomes have lower average divergence dates and low population differentiation. 
</p>

<p>
Is this important? Stay tuned. 
</p>

<h4>References:</h4>

<p class="cite">Debruyne R. 2005. A case study of apparent conflict between molecular phylogenies: the interrelationships of African elephants. Cladistics 21:31-50. <a href="http://dx.doi.org/10.1111/j.1096-0031.2004.00044.x">DOI link</a></p>

