---
layout: single 
title: "Malaria genetic review" 
category: story
permalink: /weblog/reviews/genetics/disease/kwiatkowski_malaria_review_2005.html
tags: [disease] 
comments: false 
author: John Hawks 
---


<p>
On the <a href="http://johnhawks.net/weblog/reviews/genetics/disease/beutler_1994_g6pd_review_link.html">subject of malaria defenses</a>, I also ran across <a href="http://www.pubmedcentral.gov/articlerender.fcgi?tool=pubmed&pubmedid=16001361">this review article from last year</a> by Dominic Kwiatkowski. 
</p>

<blockquote>What is remarkable is the range of erythrocyte variants, apart from HbS, that have resulted from evolutionary selection by malaria. They include other variants of the HBB gene--namely, HbC and HbE (Agarwal et al. 2000; Modiano et al. 2001b; Chotivanich et al. 2002; Ohashi et al. 2004); regulatory defects of HBA and HBB, which cause alpha and beta thalassemia (Flint et al. 1986; Williams et al. 1996; Allen et al. 1997), variation in the structural protein SLC4A1, which causes ovalocytosis (Foo et al. 1992; Genton et al. 1995; Allen et al. 1999); variation in the chemokine receptor FY, which causes the Duffy-negative blood group (Miller et al. 1976; Chitnis and Miller 1994; Tournamille et al. 1995; Hamblin and Di Rienzo 2000); and polymorphisms of the red-cell enzyme gene G6PD, which causes glucose-6-phosphate dehydrogenase deficiency (Bienzle et al. 1972; Ganczakowski et al. 1995; Ruwende and Hill 1998; Tishkoff et al. 2001; Sabeti et al. 2002b).</blockquote>

<blockquote>This is probably only the tip of the iceberg. Surprisingly little is currently known about the effects of malaria on the evolution of the human immune system, possibly because the phenotypic consequences are more subtle than those of the classic erythrocyte variants; for example, alteration of a splenic dendritic cell receptor is not as easy to visualize as a sickling red cell. However, the last few years have seen a rapid growth in the number of reported genetic associations with susceptibility and resistance to malaria, many of which involve immune system and inflammatory genes.</blockquote>

<blockquote>The purpose of this review is to provide an overview of what is currently known about genetic resistance to malaria and to highlight directions that are likely to see major advances in the next few years.</blockquote>

<p>
I especially like that line about the "tip of the iceberg". So far we have noticed the very strongly selected variants, but there must be many more with weaker levels of selection (and "weaker" in this context includes things with up to 10 percent advantages!) that we haven't yet characterized. 
</p>

<p>
This article has really good coverage -- it doesn't cover everything I learned from Frank Livingstone, but it touches on all the bases. 
</p>

<p>
On the subject of recent selection -- which has been such a hot topic this week -- there is this: 
</p>

<blockquote>The fact that different malaria-resistance alleles have arisen in different places suggests that a great deal of evolutionary selection by malaria has happened relatively recently in human history and certainly since humans started to migrate out of Africa. This is supported by analyses of recent positive selection in the human genome. Haplotype analysis and statistical modeling of an African malaria-resistance allele at the G6PD locus suggests an origin within the last 10,000 years or so (Tishkoff et al. 2001), whereas analysis of the Southeast Asian HbE allele suggests that it originated within the last 5,000 years (Ohashi et al. 2004). Studies of G6PD and CD40L malaria-resistance alleles in West Africa that made use of the long-range haplotype test are also consistent with recent positive selection (Sabeti et al. 2002b).</blockquote>

<p>
There's much more in the review, with lots of pertinent references. It is interesting to speculate how much of the recent selection in the genome may be specific to malarial populations, particularly in view of the powerful mortality. Probably too late to make it into the review is the <a href="http://johnhawks.net/weblog/reviews/genetics/disease/martin_2005_malaria_history.html">evidence for much more ancient resistance</a> in the human lineage, but the review does include this paragraph about the coevolution of Sia-related resistance: 
</p>

<blockquote>Various blood groups are determined by the erythrocyte-membrane sialoglycoproteins glycophorin A and B, and genetic deficiency of glycophorin A or B expression makes erythrocytes relatively resistant to invasion by P. falciparum (Facer 1983). Specific sialic-acid residues on the glycophorin A molecule are recognized by a Duffy-bindinglike domain of P. falciparum erythrocyte-binding antigen 175 (Orlandi et al. 1992; Mayor et al. 2005). Sequence analysis shows evidence of strong evolutionary selection, not only for GYPA and GYPB in the human host (Baum et al. 2002; Wang et al. 2003), but also for EBA-175 in the P. falciparum parasite, which implies an ongoing evolutionary struggle between the parasite ligand and the host receptor (Wang et al. 2003).</blockquote>

<p>
I like the story of ancient resistance mechanisms. It increasingly looks like today's falciparum malaria is a "perfect storm" of adaptations for getting around a fairly effective ancient human malaria resistance. Larger human host populations, larger mosquito populations, and the penetration of previously less-inhabited parts of Africa have all contributed to this evolution. 
</p>

<p>
There's some chance that trypanosomiasis is another old parasite that has learned new tricks to infect humans.  
</p>

<h4>References:</h4>

<p class="cite">Kwiatkowski DP. 2005. How malaria has affected the human genome and what human genetics can teach us about malaria. Am J Hum Genet 77:171-192. <a href="http://www.pubmedcentral.gov/articlerender.fcgi?tool=pubmed&pubmedid=16001361">Free text at PubMed Central</a></p>

