---
layout: single 
title: "Has the dam broken on mtDNA selection?" 
category: story
permalink: /weblog/reviews/genetics/bazin_2006_mtdna_selection_diversity.html
tags: [Pliocene, introgression, Neandertals, Neandertal DNA, natural selection] 
comments: false 
author: John Hawks 
---


<p>
The current <i>Science</i> has <a href="http://dx.doi.org/10.1126/science.1122033">a paper</a> by Eric Bazin and colleagues comparing mtDNA diversity with population size, history and ecology of 3000 animal species. 
</p>

<p>
Here's the conclusion: 
</p>

<blockquote>This study reveals that the mitochondrial diversity of a given animal species does not reflect its population size: No correlation between mtDNA polymorphism and species abundance could be detected, despite the large body of data analyzed. Nuclear data, in contrast, are fairly consistent with intuitive expectations. We conclude that natural selection acting on mtDNA contributes to homogenization of the average diversity among groups, in agreement with the genetic draft theory. <i>mtDNA appears to be anything but a neutral marker and probably undergoes frequent adaptive evolution</i>, e.g., direct selection on the respiratory machinery, nucleo-cytoplasmic coadaptation, two-level selection, or adaptive introgression, perhaps hitchhiking with a maternally transmitted parasite. <i>mtDNA diversity is essentially unpredictable and will, in many instances, reflect the time since the last event of selective sweep, rather than population history and demography</i>. Low-diversity mitochondrial lineages, typically disregarded as important from a conservation standpoint, might sometimes correspond to recently selected, well-adapted haplotypes to be preserved (Bazin et al. 2006:571-572, emphasis added). </blockquote>

<p>
This is a nice empirical comparison, and a very impressive exercise in data mining. To accumulate the dataset, they had to troll large data depositories for cases in which the same DNA segments had been sequenced in multiple individuals of single species, and then had to match those cases with ecological information about the species, as the accompanying <a href="http://dx.doi.org/10.1126/science.1127182">perspective</a> by Adam Eyre-Walker describes. 
</p>

<p>
But, aside from the very persuasive presentation here, the fact has been obvious <i>for years</i>. I <a href="http://johnhawks.net/weblog/reviews/neandertals/neandertal_dna/weaver_roseman_2005.html">blogged about mtDNA selection</a> last year. Finding such widespread mtDNA selection across taxa -- even into invertebrates -- is certainly strong support for the idea that it evolved adaptively in humans. And finding that the chance of adaptive evolution in mtDNA is proportional to population size enhances the likelihood of <i>recent</i> mtDNA selection in humans even more. 
</p>

<p>
Eyre-Walker draws exactly the opposite conclusion than I do: 
</p>

<blockquote>Interestingly, humans are an exception to the pattern seen by Bazin et al. If the authors are correct, then the effective population size estimated from mitochondrial DNA should be lower than that estimated from autosomal DNA. This is not what we see in humans; the effective population sizes estimated from autosomal DNA, Y-chromosome DNA, and mitochondrial DNA are all approximately 10,000. Does this mean that Bazin et al. are incorrect? Probably not. It may be that humans have such small effective population sizes that adaptive evolution in the mitochondrial genome is very rare; the neutrality index in human mitochondrial DNA, and perhaps nuclear DNA, certainly gives no indication of adaptive evolution. (Eyre-Walker 2006:538).</blockquote>

<p>
But of course, this is quite backward -- <i>low</i> mtDNA diversity cannot be evidence for <i>neutrality</i>; at best it can <i>fail to refute</i> a hypothesis of selection. With our long generation lengths, autosomal DNA would have to have coalescence dates in the Pliocene to make the low mtDNA diversity stand out statistically. It is not a question of them all being neutral, it is a question of packing most of human evolution into a space of 2 million years. 
</p>

<p>
Supporting that is the observation that Eyre-Walker points out next: 
</p>

<blockquote>Although nuclear diversity follows the expected pattern, with more diversity in organisms that are expected to have bigger population sizes, the differences are remarkably small; synonymous diversity varies by less than a factor of 10, and allozyme diversity by less than a factor of 4. This is striking given that the population sizes of marsupials and mussels, for example, must differ by many orders of magnitude, and one would expect diversity to be linearly related to population size. This observation is not new for allozyme data (4), but it is the first time this pattern has been so clearly illustrated for synonymous diversity in nuclear genes. The lack of a strong correlation between diversity and population size in nuclear DNA may also reflect the effects of genetic hitchhiking (ibid.). </blockquote>

<p>
In other words, selection has restricted mtDNA diversity, and it has also restricted nuclear DNA diversity -- just not as much. The "not as much" here is a function of recombination, which makes the nuclear genes true subjects of genetic draft. 
</p>

<p>
This isn't news either. We've known about the restricted allozyme diversity since 1984. A few voices crying in the wilderness have been reminding us from time to time, like Gillespie. 
</p>

<p>
I would note that some of their ecological substitutes for population sizes may themselves induce selective effects. For example, Bazin and colleagues note that marine molluscs have more allozyme variation than terrestrial molluscs, which they view as consistent with the greater dispersal of marine species. But greater dispersal might also involve the necessity to maintain diversity for dispersing into to different local environments, which would tend to drive frequency-dependent or balancing selection for traits responding to these local forces. 
</p>

<p>
So there is more to be done on the nuclear DNA side of this equation, probably much more. But the mtDNA comparison is very important, and hopefully will drive some reevaluation of the use of mtDNA diversity as a proxy for genetic diversity in conservation and ecological studies. 
</p>

<h4>References:</h4>

<p class="cite">Bazin E, Gl&eacute;min S, Galtier N. 2006. Population size does not influence mitochondrial genetic diversity in animals. Science 312:570-572. <a href="http://dx.doi.org/10.1126/science.1122033">DOI link</a></p>

<p class="cite">Eyre-Walker A. 2006. Size does not matter for mitochondrial DNA. Science 312:537-538. <a href="http://dx.doi.org/10.1126/science.1127182">DOI link</a></p>


