---
layout: single 
title: "What is up with the prion gene?" 
category: story
permalink: /weblog/reviews/genetics/brain/prion_ascertainment_soldevilla_2006.html
tags: [brain] 
comments: false 
author: John Hawks 
---


<p>
In 2003, Mead and colleagues (<a href="http://www.sciencemag.org/cgi/content/full/300/5619/640?maxtoshow=&HITS=10&hits=10&RESULTFORMAT=&fulltext=mead+prion&searchid=1137031520988_19913&FIRSTINDEX=0&journalcode=sci">Full text</a>) suggested that the prion protein gene (<i>PRNP</i> -- <a href="http://www.ncbi.nlm.nih.gov/entrez/dispomim.cgi?id=176640">OMIM</a>) in humans had been under long-term balancing selection in humans worldwide. 
</p>

<p>
A mutant form of the prion protein is the agent the causes Mad Cow disease, Kuru, and other spongiform encephalopathies. Kuru is a disease that can be transmitted by cannibalism -- if you eat a person's flesh who has the disease, you can contract it yourself. Mead et al. (2003) give a good short review:
</p>

<blockquote>Kuru came to the attention of Western medicine in the 1950s as the affected area of the Eastern Highlands of Papua New Guinea came under Australian administrative control. The Fore and neighboring linguistic groups occupied a remote highland area that had had no direct contact with the outside world before this. It was the practice in these communities for kinship groups to consume deceased relatives at mortuary feasts. From the evidence of local oral history, this practice was not ancient among the Fore and is thought to have started around the end of the 19th century. The first remembered case of kuru was around 1920, and the disease rapidly increased in incidence. Adult women and children of both sexes were primarily affected, reflecting their selective exposure -- adult males participated little at feasts. At its peak, kuru killed around 1% of the population annually, and some villages were almost devoid of young adult women. Kuru was the first human prion disease shown to be transmissible, by inoculation of chimpanzees with autopsy-derived brain tissue (2). It is hypothesized that kuru originated from consumption of an individual with sporadic CJD (3), a disease with a remarkably uniform worldwide incidence of around 1 per million and a lifetime risk of around 1 in 50,000. The ban on cannibalism imposed by the Australian authorities in the mid-1950s led to a decline in kuru incidence, and although rare cases still occur, these are all in older individuals and reflect the long incubation periods possible in human prion disease -- kuru has not been recorded in any individual born after the late 1950s (4) (Mead et al. 2003:640).</blockquote>

<p>
A major polymorphism in humans at one nucleotide site of <i>PRNP</i> appears to give heterozygotes an advantage in resisting kuru, and homozygotes of one allele are the only people who have contracted "Mad Cow", or variant Creutzfeld-Jakob disease (CJD). So the hypothesis of balancing selection suggested that humans may have had a long history of Kuru or other diseases transmitted by prions. <i>PRNP</i> heterozygote cannibals would have been less susceptible to bad prions. 
</p>

<p>
Kreitman and Di Rienzo (2004) noted a problem with the study: ascertainment bias. This <a href="http://www.the-scientist.com/news/display/22927/">article in <i>The Scientist</i></a> does a really good job of explaining the ascertainment bias problem. 
</p>

<blockquote>However, in July 2004, [Martin] Kreitman and Anna Di Rienzo, also of the University of Chicago, published a criticism of the Mead et al. paper in <i>Trends in Genetics</i>, citing an ascertainment bias in haplotype sampling. Mead and colleagues sequenced the full PRNP gene in a few people of European ancestry, and then used the single nucleotide polymorphisms (SNPs) they discovered to genotype a larger, global population. But without resequencing every individual included in the final study, the authors likely biased their results toward common polymorphisms, the signature of balancing selection, Kreitman told <i>The Scientist</i>. "They had in fact excluded the low-frequency variants and only looked at the common ones -- and then came to the conclusion that there are too many common ones," Kreitman said.</blockquote>

<p>
Soldevila et al. (2006) sequenced a large sample to settle the issue of blancing selection: 
</p>

<blockquote>Our analyses reveal the worldwide pattern of variation at the PRNP gene to be inconsistent with neutral expectations, indicating instead an excess of low-frequency variants, a footprint of the action of either positive or purifying selection. A comparison of neutrality test statistics for PRNP with other human genes indicates that the signal of positive selection on PRNP is stronger than expected from a possible confounding genome-wide background signal of population expansion. Two main conclusions arise from our analysis. First, the existence of an ancient, stable, balanced polymorphism that has been claimed in a previous study and related to cannibalism can be rejected and is shown to be due to ascertainment bias. Second, our results are consistent with a complex history of selection including mainly positive selection, even if short local periods of balancing selection (Kuru-like episodes), or even a weak purifying selection model, are consistent with our data (Soldevila et al. 2006:1). </blockquote>

<p>
OK, so no long-term balancing selection. The origin of the key coding variant (site 129) is inferred to have happened within the last 100,000 -- 200,000 years or so. There is not great resolution to the genealogy: there are two major variants (differentiated by site 129) and lots of minor ones branching off these two. 
</p>

<p>
My hypothesis would be that both the major alleles came under positive selection at their origins. The first of these was variant S1, which is distinguished from the chimpanzee allele by an A at site 171. Two African haplotypes (S18 and S28) also have the G at 171, but coupled with the derived G at 129 -- this means that the presence of the presumably ancestral nucleotide must either be a recurrent mutation or a recombinant with an older haplotype not present in the sample. The selective sweep of this allele is complete. As S1 increased, it kicked off a few minor alleles, each of which differs from it at a single site. 
</p>

<p>
Why do I think that S1 was not the ancestral haplotype? Mainly because it looks -- if anything -- <i>more</i> selected than the newer haplotype S2. The absence of any clear tree structure to the haplotypes with 129A just emphasizes that the allele hasn't been around that long. So if it was not ancestral, it was a substitution. This older allele is more common in the samples from Europe and East Asia, less so in Africa, West Asia, and America. 
</p>

<p>
One of the variants kicked off of S1 was haplotype S2, with a G variant at site 129. This <i>also</i> came under positive selection upon its origin. It happened long enough ago to spread into all these populations, including America. It <i>might</i> be in selective balance, but that balance isn't very old -- it is recent enough that what we see is the increase in frequency of a new allele. That means that the variant might also be a simple positively selected allele with no balance or heterozygote advantage, at least in most populations. 
</p>

<p>
The question is, what were these two variants selected <i>for</i>?
</p>

<p>
Phylogenetic comparisons show that <i>PRNP</i> is conserved in primates, with no excess of coding substitutions in humans. The comparison with the chimpanzee sequence emphasizes that: if the S1 haplotype was derived from a more ancient haplotype in humans, that would most parsimoniously be the chimpanzee haplotype itself, meaning that chimpanzees had no substitutions at all, and humans had none before the selection on S1. 
</p>

<p>
So for a gene that had no coding changes in humans or chimpanzees, why would there recently have been two successive selected changes in humans? 
</p>

<p>
For a highly conserved brain-expressed gene, why would there recently have been two successive selected changes in humans? 
</p>

<p>
Hmmm....
</p>

<h4>References:</h4>

<p class="cite">Kreitman M, Di Rienzo A. 2004. Balancing claims for balancing selection. Trends Genet. 20:300-304.</p>

<p class="cite">Mead S et al. 2003. Balancing selection at the prion protein gene consistent with prehistoric kurulike epidemics. Science 300:640-643. <a href="http://www.sciencemag.org/cgi/content/full/300/5619/640?maxtoshow=&HITS=10&hits=10&RESULTFORMAT=&fulltext=mead+prion&searchid=1137031520988_19913&FIRSTINDEX=0&journalcode=sci">Full text (subscription)</a></p>

<p class="cite">Soldevila M et al. 2006. The prion protein gene in humans revisited: Lessons from a worldwide resequencing study. Genome Res epub. <a href="http://www.genome.org/cgi/content/abstract/gr.4345506v1">Abstract</a></p>

