---
layout: single 
title: "Gene expression in the mouse brain" 
category: story
permalink: /weblog/reviews/genomics/brain/brain_atlas_gene_expression_2007.html
tags: [brain, genomics] 
comments: false 
author: John Hawks 
---


<p>
Last week's <i>Nature</i> included a report on the final draft of the gene expression atlas of the mouse brain. I wish this had come out last semester -- I could have devoted four lectures to it. 
</p>

<p>
There is some very interesting stuff in this if you read past the jargon. I wrote about the <a href="http://johnhawks.net/weblog/reviews/genomics/expression/mouse_brain_atlas_announcement_2006.html">brain atlas</a> last year; this paper is a report on the project's completion. 
</p>

<p>
One main system-level finding is that a surprisingly large proportion of the genome is expressed in brain -- around 80 percent of all identified genes. Most of these are expressed in only a relatively small subset of genes ("70.5% of genes are expressed in less than 20% of total cells", not together in the <i>same</i> 20%). In other words, the distribution of gene expression is very broad, with a long tail. 
</p>

<p>
My interpretation is that the large number of genes suggests a substantial role for pleiotropy between other tissues and brain. The representation of most of these genes in only a small proportion of the brain suggests that mammalian brain evolution has diversified neural function by utilizing genes already existing in the genome for small, specialized purposes in the brain. 
</p>

<p>
Another system-level finding supports this notion, by noting that "large numbers of genes [have] restriced expression in specific cell populations." The specificity of gene expression in certain brain areas must underlie functional differentiation of brain regions. 
</p>

<p>
More interesting, the gene expression patterns help to identify distinct regions that are not necessarily recognizable anatomically. For example, this section describes some of the gene expression organization of the cerebellum: 
</p>

<blockquote>The basic structure of the cerebellum is well known and consists of several functionally discrete gross divisions. Additionally, the cerebellar cortex exhibits a bilaterally symmetric series of sagittally oriented bands<sup>31</sup> mirrored by a number of genes, most notably zebrin<sup>32, 33</sup>. Strong (although not complete) correlation between patterns of cerebellar afferent segregation and zebrin expression<sup>34</sup> indicate that molecular markers can delineate functionally discrete regions in the cerebellum.</blockquote>

<blockquote>A number of genes display heterogeneity within cerebellar granule and Purkinje cell populations. For example, <i>Rasgrf1</i> defines a previously unrecognized large, contiguous domain with sharp boundaries in the granule cell layer of the rostral (Fig. 8a) and dorsal cerebellum (Fig. 8b). More complex regional patterns are observed in the Purkinje cell layer, such as that of <i>Opn3</i>, a non-canonical opsin (Fig. 8c, d) whose expression in the cerebellum has been described as a rostro-caudal gradient with radial stripes<sup>35</sup>. Rather than a gradient, three-dimensional reconstruction of ISH data for <i>Opn3</i> reveals a more coherent pattern (Fig. 8e) involving a sharply delineated diagonal band lacking expression extending across the entire cerebellum. The overall pattern of <i>Opn3</i> is both complex and discrete, with regionalized expression in distinct lobules and sagittal banding in the posterior vermis.</blockquote>

<p>
Seeing these "hidden boundaries" pop out in the gene expression data is sort of like on CSI where they shine the UV light on a crime scene. Suddenly lots of things are apparent that weren't visible to the naked (or microscope-aided) eye. 
</p>

<p>
On the whole, the paper concludes that gene expression correlations (different genes expressed in similar patterns) tend to occur for structures around the scale of functional nuclei within larger brain regions, and gross brain regions themselves are represented by "single tight clusters" of genes. In this context, the paper discusses the differences in gene expression between different subregions of the hippocampus. Interestingly, a high proportion of genes involved in cell adhesion are among those expressed in hippocampus: 
</p>

<blockquote>The top over-represented functional category within the regional gene set is cell adhesion (P < 1.79<sup>-10</sup>). Differential cell adhesion may be important for establishment and maintenance of topographic connectivity, or, as described recently, different forms of synaptic plasticity and remodelling<sup>27</sup>.</blockquote>

<p>
An interesting part of the paper to me is its consideration of gene expression <i>within</i> individual neurons. 
</p>

<blockquote><b>Subcellular mRNA targeting<b></blockquote>

<blockquote>Subcellular localization and translation of mRNA transcripts in dendrites is increasingly recognized as a widespread phenomenon<sup>36</sup>, and is thought to be involved in certain forms of synaptic plasticity<sup>37, 38</sup>. Dendritic mRNA targeting is particularly obvious in the hippocampus (Fig. 9a-f) and cerebellum (Fig. 9g, h), where clear distinctions can be made between cell-dense layers, dendritic molecular layers and white matter. Targeting throughout the entire dendritic field is exemplified by the well-characterized patterns of <i>Camk2a</i> and <i>Dnd1</i> in the hippocampus (Fig. 9b, c)<sup>39</sup>. Although often subtle, this distribution is independent of expression level, thereby distinguishing targeting from passive diffusion of mRNA into dendrites. For example, labelling for the highly expressed gene <i>Nptx1</i> is confined to the soma of CA3 pyramidal cells (Fig. 9d), whereas microtubule associated proteins 1A (<i>Mtap1a</i>) and 2 (<i>Mtap2</i>) also label proximal or proximal and distal dendrites (Fig. 9e, f). Similar forms of targeting are seen in cerebellar Purkinje cells (Fig. 9h, i), as well as in oligodendrocytes (Mbp; Fig. 9k)<sup>40</sup>.</blockquote>

<blockquote>Many genes exhibiting dendritic targeting are involved in cytoskeletal organization and biogenesis, as well as in regulating synaptic plasticity. There appear to be multiple cis-acting sequence elements that mediate mRNA targeting<sup>41</sup>. Identification of sets of dendritically targeted mRNAs with shared features (Supplementary Table 4), such as regional specificity and distribution in dendrites, may aid in the identification of conserved sequence elements that correlate with cellular and intracellular transport specificity.</blockquote>

<p>
So subcellular function in neurons is identifiable by looking at gene expression in different microanatomical regions. That's pretty cool. It goes to show that the neurons are the most specialized adaptive part of this whole party. The neuron is a finely tuned machine. It remains to be seen to what extent neurons are different machines in different brain regions -- exactly how expression differences may influence intracellular operation. But it is tempting to speculate that the formation of different regions of the brain was a relatively simple operation compared to the 
</p>

<p>
Now this atlas represents expression in the mouse brain, so not everything will apply into primates or humans. I would guess that additional functions have arisen during primate evolution by the proliferation of these nucleus-level structures, and some relative expansion of pre-existing functional regions. That kind of evolution would probably involve increasing the representation of the genome in brain expression, by using additional genes in brain development, by duplication and later functional differentiation of some brain-expressed genes, and by an increase in alternative splicing. 
</p>

<h4>References:</h4>

<p class="cite">Lein ES and many, many others. 2007. Genome-wide atlas of gene expression in the adult mouse brain. Nature 445:168-176. <a href="http://dx.doi.org/10.1038/nature05453">doi:10.1038/nature05453</a></p>

