---
layout: single 
title: "Neandertal genome damage profile" 
category: story
permalink: /weblog/reviews/genetics/ancient/neandertal_sequencing_damage_2007.html
tags: [ancient DNA, Neandertals] 
comments: false 
author: John Hawks 
---


<p>
It's another week, which means it's time for another of the mysteriously-not-yet-appeared <i>PNAS</i> papers. But this time, a friendly source <i>sent me the paper</i> (which should be <a href="http://dx.doi.org/10.1073/pnas.0704665104">here</a>, when it appears). 
</p>

<p>
You can see my comments on it in David Biello's <a href="http://www.sciam.com/article.cfm?articleID=64DFDA17-E7F2-99DF-32A2ED5FF323905D">article</a> at <i>Scientific American</i>. 
</p>

<p>
Basically, this is the analysis of DNA damage in the genome sequence coming out on the 454 sequencing platform. They show the damage as limited to two kinds of chemical changes, which generate spurious C->T and G->A substitutions. Also, the position of the nucleotide in the sequencing read makes a difference, as terminal nucleotides have more than 5 times as many spurious errors than some internal ones. There is a lot of detail in the paper about all of this. 
</p>

<p>
The interesting part is near the end, where the paper discusses ways to quantify contamination and correct for misincorporated bases. Basically, one can quantify and correct all such problems with multiple coverage. The more copies of a given sequence are produced, the more it becomes possible to estimate the proportion of contaminant sequence -- at least in areas where the contaminant may be differentiated based on its sequence. But even where contaminants are possibly identical to the endogenous sequence, it may be possible to find them in various ways (for example, if three homologous sequences were present instead of two, or if a significant excess of some particular sequence were present. Likewise, misincorporations can be found and corrected with multiple reads. 
</p>

<p>
The demonstration of these problems helps to increase confidence in the outcome: if we understand the errors, we can better evaluate results. The important (and unanswered) question is how much fossil grinding it will take to get good genomes. 
</p>

<h4>References:</h4>

<p class="cite">Briggs AW and 10 others. 2007. Patterns of damage in genomic DNA sequences from a Neandertal. Proc Nat Acad Sci USA (early) <a href="http://dx.doi.org/10.1073/pnas.0704665104">

