---
layout: single 
title: "Alternative splicing" 
category: story
permalink: /weblog/reviews/genomics/variation/alternative_splicing_tress_2007.html
tags: [genomics, evo-devo] 
comments: false 
author: John Hawks 
---


<p>
The <i>PNAS</i> early edition includes <a href="">a paper</a> by Michael Tress and (many) others about the frequency of alternative splicing across the genome. I <a href="http://johnhawks.net/weblog/reviews/genomics/duplication/intron_recent_evolution_zhuo_2007.html">wrote about alternative splicing</a> in January, and <a href="http://johnhawks.net/weblog/reviews/genomics/evo-devo/science_special_rna_issue_2005.html">this post</a> about new developments in the study of RNA rewards reading if you missed it the first time. 
</p>

<p>
The new paper examines a small subset of known functional loci for which the splicing pathways have been well-studied. Here's the abstract: 
</p>

<blockquote>Alternative premessenger RNA splicing enables genes to generate more than one gene product. Splicing events that occur within protein coding regions have the potential to alter the biological function of the expressed protein and even to create new protein functions. Alternative splicing has been suggested as one explanation for the discrepancy between the number of human genes and functional complexity. Here, we carry out a detailed study of the alternatively spliced gene products annotated in the ENCODE pilot project. We find that alternative splicing in human genes is more frequent than has commonly been suggested, and we demonstrate that many of the potential alternative gene products will have markedly different structure and function from their constitutively spliced counterparts. For the vast majority of these alternative isoforms, little evidence exists to suggest they have a role as functional proteins, and it seems unlikely that the spectrum of conventional enzymatic or structural functions can be substantially extended through alternative splicing.</blockquote>

<p>
The short description of the ENCODE project is worthwhile: 
</p>

<blockquote>The pilot project of the Encyclopedia of DNA Elements (ENCODE) (11), which aims to identify all the functional elements in the human genome, has undertaken a comprehensive analysis of 44 selected regions that make up 1% of the human genome. One valuable element of the project has been the detailing of a reference set of manually annotated splice variants by the GENCODE consortium (12). The annotation by the GENCODE consortium is an extension of the manually curated annotation by the Havana team at The Sanger Institute (Tress et al 2007:5495). </blockquote>

<p>
The main conclusion of the paper is that, even though alternative splicing is very common, in most cases the splicing "isoforms" are nonfunctional. Splicing provides a mechanism for the evolution of many distinct proteins from a small number of genes, but the variation in splicing only rarely leads to two or more functional proteins as an outcome. This raises the question of why cells tolerate multiple splices for a gene. We pretty much have to conclude that the nonfunctional splices do not have a fitness cost. 
</p>

<p>
Still, splicing may provide a mechanism for creative evolutionary changes in some cases: 
</p>

<blockquote>The standard path of protein evolution is usually conceived of as <br />
stepwise single base-pair mutations. In contrast alternative splicing typically involves large insertions, deletions, or substitutions of segments that may or may not correspond to functional domains, subcellular sorting signals, or transmembrane regions. The deletion and substitution of multiple exons seen in many of these transcripts suggests that splicing is not always a mechanism for delicate and subtle changes and, as a process, may be rather more revolution than evolution (Tress et al. 2007:5499). </blockquote>

<p>
They describe a few cases where alternative splices lead to disease, or are associated with abnormal cellular metabolism during -- particularly in cancer. In this sense, we have the usual range of functional implications from any kind of genetic change: rarely, something good results; often the change makes no difference, but sometimes it's really bad. The observation that a fairly large set of nonfunctional spliced proteins may be floating around cells means that this class of variants joins many of the transcribed RNA elements as part of the cytoplasmic menagerie. 
</p>

<h4>References:</h4>

<p class="cite">Michael L. Tress and 46 others. 2007. The implications of alternative splicing in the ENCODE protein complement. Proc Nat Acad Sci USA 104:5495-5500. <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.0700800104">doi:10.1073/pnas.0700800104</a></p>

