---
layout: single 
title: "Neandertal, other ancient DNA review" 
category: quickbit
permalink: /node/1582
tags: [Neandertal DNA, metagenomics] 
comments: false 
author: John Hawks 
---

Last week, <a href="http://dx.doi.org/10.1126/science.1161890">a short article</a> in <i>Science</i> by Rachel Mackelprang and Edward Rubin discussed some of the recent advances in ancient DNA extraction. Of most interest is the paragraph that discusses ways to probe for particular genes while avoiding some drawbacks of PCR amplification:

<blockquote>Microarray-based hybridization, coupled with high-throughput sequencing of recovered DNA, has recently been used to capture thousands of targets in parallel from modern DNA samples. With these strategies, a DNA sample is directly applied to an array of specifically designed oligonucleotide probes immobilized on a chip. Complementary fragments hybridize to the probes while the remaining nonbound DNA is washed away. The hybridized DNA can then be eluted from the chip and sequenced, resulting in enrichment of targeted genomic regions (11). Alternatively, chip-synthesized oligonucleotide probes have been released from the chip and used to capture molecules in solution (12). A purely solution-based method, where sets of probes are designed against a reference genome and used as a bait to "hook" corresponding sequences from a DNA pool (13), has been used to recover specific regions of nuclear DNA from Neandertal and cave bear genomic sequence libraries (1). These various capture approaches hold promise for economically investigating the same sequence in multiple different samples as well as examining multiple independent molecules of an allele isolated from a single sample.</blockquote>

The short review also mentions problems with contamination and some of the results that indicate contamination of the Neandertal sequences, which I've discussed before (<a href="http://johnhawks.net/weblog/reviews/neandertal_dna">Complete Neandertal DNA files</a>). 

