---
layout: single 
title: "Domesticated yeast face ancient amber-yeast competitors" 
category: story
permalink: /weblog/reviews/genomics/non-primate/yeast-amber-moses-2008.html
tags: [yeast, ancient DNA, genomics, phylogenetics] 
comments: false 
author: John Hawks 
---


So, they've resurrected 45-million-year-old yeast from amber, and are using it to brew beer under the "Fossil Fuels" label: 

<blockquote>The beer has received good reviews at the Russian River Beer Festival and from other reviewers. The Oakland Tribune beer critic, William Brand, says the beer has "a weird spiciness at the finish," and The Washington Post said the beer was "smooth and spicy."</blockquote>

<blockquote>Part of that taste comes from the yeast's unique metabolism. "The ancient yeast is restricted to a narrow band of carbohydrates, unlike more modern yeasts, which can consume just about any kind of sugar," said Cano.</blockquote>

I have this incredible compulsion to taste the stuff. Actually, I really like the entire idea of the taste qualities of these micro-adapted yeast strains -- which go into everything from San Francisco sourdough bread to the wild yeasts living on grape skins in Bordeaux. 

But last week I went to a talk by Alan Moses, who's been working in "population genomics" -- essentially, comparative population genetics taken across the genome. The major model organisms for this work so far have been yeast -- the domesticated <i>Saccharomyces cerevisiae</i> and its wild relative <i>Saccharomyces paradoxus</i>. There were a lot of interesting things coming out of that work (not only about yeast, but also about people and mice). 

One of the easiest analyses was a simple overlay of phylogeny on geography for the two yeast species, described in a preprint by Moses' collaborators on Nature Precedings. The wild <i>S. paradoxus</i> has a very clear population structure, with genetically distinct populations in Europe, America, East Asia, and Hawaii. These populations are easily pulled out of a STRUCTURE analysis, so there are a large set of distinctive alleles that characterize them. 

In contrast, the domesticated <i>S. cerevisiae</i> has been extensively crossed, hybridized, and managed by humans. This has led to a population structure that does not clearly distinguish strains by their geographic locations -- and in fact, strains are not easily distinguished by their use (wine versus sake, for instance). The geographic structure has largely been smudged out of existence by human intervention and recombination between strains. Moreover, some changes of adaptive importance, such as nonsense or missense mutations in the galactase pathway, have been repeated more than once in these human-managed strains. That's a very clear contrast, and other aspects of the genomic variation bear the marks of selection. 

To me, there is a practical import to these analyses. If the domesticated strains have tuned in countless ways to operate in our food, what are the chances that genuine "wild" strains of yeast have actually persisted for years and years in the face of this boundary-crossing contact between yeasts? We say that these are original wild yeasts, because a lump of starter has been replenished and renewed for years and years, or because the natural yeasts enter from the grape skins, or some other story. But that starter is not in a hermetically sealed chamber, and people don't put on bunny suits to work on it. Our gregarious domestic yeast strains have gotten in there, and they've probably replaced the original yeast strains many times over. 

So, if we take yeast out of 45-million-year-old amber, what are the chances it's going to last long in culture? Maybe it will do surprisingly well. But this is a case where genetic barcoding might be in order. 





<h4>References:</h4>

<p class="cite">Carter D, Liti, G, Moses A and 22 others. Population genomics of domestic and wild yeasts. Preprint. <a href="http://precedings.nature.com/documents/1988/version/1">Available at Nature Precedings</a></p>

