---
layout: single 
title: "An MAPT review" 
category: story
permalink: /weblog/reviews/genomics/17q-pennisi-review-2008.html
tags: [genomics, introgression, brain, Neandertal DNA, recent selection] 
comments: false 
author: John Hawks 
---

Elizabeth Pennisi <a href="">writes this week</a> a news focus in <i>Science</i> about the genome region labeled 17q21.31. I'm probably one of the few people who would recognize that address right away: A recently selected inversion in this region is one of the best candidates for introgression of Neandertal genes into recent Europeans. 

<blockquote>Eichler suspects that when H1 appeared, it somehow provided a strong fitness bonus and became much more common over time at the expense of H2. In Africans, H2 almost disappeared, except in the relatively few people who migrated to Europe 50,000 to 100,000 years ago. Then, for as-yet-unknown reasons, H2 provided its own advantage in the European population--as Stefnsson's data show--and the pendulum has begun to swing in the other direction.</blockquote>

<blockquote>Hardy and, to a lesser extent, Stefnsson give credence to a more extreme explanation for the distribution of H2. Hardy thinks that H2 had disappeared from the modern humans moving out of Africa to populate the Northern Hemisphere but not from Neandertals, who reintroduced the inversion into the European gene pool through interbreeding with Homo sapiens 28,000 to 40,000 years ago. This view is not supported by the genetic evidence emerging from sequencing Neandertal DNA, and "I realize it's an off-the-wall idea," says Hardy. But he nonetheless thinks it's plausible.</blockquote>

We covered the locus in <a href="http://johnhawks.net/weblog/reviews/evolution/introgression/introgression_tig_hawks_2007.html">our <i>Trends in Genetics</i> review</a> earlier this year. Unless there are new data I don't know about, there is not yet any confirmation or test possible from the Neandertal genome.  I'm not at all confident that there will be one, since detecting structural variants like inversions in the fragmented ancient DNA will not be trivial. 

This is very interesting also: 

<blockquote>The sequence comparisons also reveal that independently in humans, chimps, and orangutans, this 900,000-base region has reoriented itself into the H1 orientation, which explains why Eichler found both orientations in these primates. "This bit of DNA has been flip-flopping up and down. There must be an evolutionary reason for that, but we don't know what it is," says Hardy.</blockquote>

Inversions shouldn't be trivial; on the average they should be deleterious. So finding inversion parallelism is curious. I wonder if there is some strategy variant here that might explain the flipping -- sort of like MHC alleles that can emerge in parallel? 

Pennisi spends much of the article describing the current medical research linking rare deletions in 17q21.31 with mental retardation: 

<blockquote>Now they have joined forces to describe 22 patients in molecular and clinical detail in a paper published online 15 July by the Journal of Medical Genetics. They calculate the prevalence of this new genomic disorder to be 1 in 16,000 newborns, and it may account for up to 0.64% of unexplained mental retardation in Europeans. "This is the first novel microdeletion syndrome identified and one of the most frequent ones," says collaborator Joris Veltman, a molecular geneticist at RUNMC.</blockquote>

There are also links between the inversion polymorphism and schizophrenia and Alzheimer's, although these remain "enigmatic" because neither a biochemical nor a clear mutational explanation for the correlations has been found. 

Anyway, it's a good story and well worth reading as an example of how a single genetic region can fall subject to population genomics, medical genetics, contrasts of rare and common variants, structural variability, primate comparisons, and all the rest. 

It also shows how scientific attention can dogpile onto single genomic locations. That doesn't necessarily mean these are the best or only relevant candidates. Maybe more than anything, it means that grant reviewers recognize loci that have been interesting in prior studies, and reward further attention with more research dollars. 

UPDATE (2008-11-8): According to a reader, there's a rumor that the 17q21.31 region has been found in the Neandertal genome, and that the inversion (the putative selected version) is not present. That would weigh against the selected allele having been ubiquitous in Neandertals, although it cannot exclude that it may have been present. In any event, the practical effect would be to remove this as a likely case of introgression. 

The recurrence of inversions in this region in other hominoids remains very interesting...

<h4>References:</h4>

<p class="cite">Pennisi E. 2008. 17q21.31: Not your average genomic address. Science 322:842-845. <a href="http://dx.doi.org/10.1126/science.322.5903.842">doi:10.1126/science.322.5903.842</a></p>



