---
layout: single 
title: "The genetic networks underlying disease" 
category: story
permalink: /weblog/topics/evolution/disease/genetic-classification-disease-genes-2008.html
tags: [disease] 
comments: false 
author: John Hawks 
---


<p>
Last week, the <i>NY Times</i> printed <a href="http://www.nytimes.com/2008/05/06/health/research/06dise.html">an interesting article by Andrew Pollack</a>, titled "Redefining disease, genes and all." The article explores recent (and ongoing) attempts to map the genetic networks underlying common disease phenotypes. 
</p>

<p>
Many people, including me, have criticized the HapMap and other attempts to catalog disease alleles because they depend on an evolutionary fallacy. These projects are best at finding genetic variations that are relatively common in today's human populations -- common because the number of people surveyed in such studies is ultimately limited. But if an allele were <i>bad</i> in an evolutionary sense -- that is, if it lowers fitness -- then it shouldn't be common. So we really shouldn't expect to find alleles associated with common disease phenotypes. 
</p>

<p>
Naturally, there are exceptions, which we can find if we consider some population genetics. A bad allele may become common if its bad effect doesn't really lower fitness. Disease phenotypes that occur late in life, such as Alzheimer's, have a minimal impact on their victims' fitness, because for the most part the childrearing years are long past. 
</p>

<p>
Or, an allele that yields one unpleasant phenotype may actually increase fitness -- sickle cell and other instances of heterozygote advantage are examples of this. 
</p>

<p>
Or, the disorder associated with the allele may occur only in the presence of some new environmental factor. Obesity is one such phenotype: lately on the increase, it cannot have been common throughout most of our evolutionary past because of resource limitations. 
</p>

<p>
But aside from these exceptions, we can expect that any common allele is likely to explain relatively little of the overall risk of any given disorder. And indeed, so far, this is precisely what genome surveys have found. Many disorders now have known genetic associations, but these are almost always alleles of relatively weak effect on the phenotype. 
</p>

<p>
Another reason to look for risk alleles, even if they are of weak effect, is that they may help to identify the genetic interactions that lead to disease. This idea has been around for a long time. Mapping the genes that cause Mendelian disorders in a given phenotype led to our current understanding of genetic pathways underlying skin color, inner ear function, blood clotting, and many other biological functions. Finding some genes associated with Alzheimer's is helping to unravel the physiological observations on people with the disorder, particularly the role of beta-amyloid plaques in the disease progression. 
</p>

<p>
Many workers are currently trying to facilitate these kinds of discoveries by developing new functional maps of gene-disease associations and interactions. Pollack's article focuses on the growing discrepancy between symptomology and etiology -- the consequences of a disease and its causes: 
</p>

<blockquote>Scientists are finding that two tumors that arise in the same part of the body and look the same on a pathologist's slide might be quite different in terms of what is occurring at the gene and protein level. Certain breast cancers are already being treated differently from others because of genetic markers like estrogen receptor and Her2, and also more complicated patterns of genetic activity.</blockquote>

<blockquote>"In the not too distant future, we will think about these diseases based on the molecular pathways that are aberrant, rather than the anatomical origin of the tumor," said Dr. Todd Golub, director of the cancer program at the Broad Institute in Cambridge, Mass.</blockquote>

<p>
The process is really one of atomizing disease. As Pollack notes, diseases that once were considered different kinds of cases of a single disorder -- like hemophilia -- were later shown to be due to distinct defects in different genes. A diffuse grouping of "like with like" has given way to much greater specificity at the biochemical and genetic level for many disorders. 
</p>

<p>
Complex disease phenotypes that involve interactions among many genes will fall the last, but computational methods are starting to attack them: 
</p>

<blockquote>Other scientists use data on which genes appear to cause disease or contribute to the risk of contracting it.</blockquote>

<blockquote>Using such data, Marc Vidal, a biologist at Harvard, and Albert-Laszlo Barabasi, now a physicist at Northeastern University, created a map of what they called the "diseasome" that was published last year in The Proceedings of the National Academy of Sciences.</blockquote>

<blockquote>Diseases were represented by circles, or nodes, and linked to other diseases by lines that represent genes they have in common -- something like the charts linking actors to one another (and ultimately to Kevin Bacon) based on the movies they appeared in together.</blockquote>

<p>
But obviously if the genetic associations are weak, they do not give great hope of simple effective treatments for such complex diseases. And some genetic similarities may lead people to infer equivalences that do not really exist. 
</p>

<p>
What is lacking from this story -- and in general from the field -- is an understanding of evolution. If there is one thing that <i>can</i> deal with the genetics underlying complex phenotypes, it is natural selection. Population genetics has been dealing with the theory underlying genetic interactions for a hundred years. Now we have empirical observations on gene networks, all of them products of our evolutionary history. 
</p>

<p>
So it is disheartening to see that some prominent figures in the field of human genetics (and who hold the purse strings for so much funding) have little familiarity with the evolutionary dynamics of gene interactions:
</p>

<blockquote>"I'm shaking my head with disbelief that two genes would pop up in these two diseases that have absolutely nothing in common," said Dr. Francis S. Collins, the director of the National Human Genome Research Institute. He said another gene, cyclin-dependent kinase inhibitor 2A, seemed to be involved in cancer, diabetes and heart disease.</blockquote>

<p>
I'm shaking my head in disbelief that Collins doesn't seem to be aware of  pleiotropy. That's another of the exceptions I pointed out above -- the rare instances where a common allele might really be associated with a common disease.  In antagonistic pleiotropy, an allele that has a good effect on one function may proliferate despite having a bad effect on some other function. There's nothing at all surprising about a single gene having different alleles that may have adverse impacts on two different bodily processes, and in fact some have been well known for fifty years or more. 
</p>

<p>
Um, could somebody brief Collins about <i>ABO</i>?
</p>


