---
layout: single 
title: "Probing for the alien within" 
category: story
permalink: /weblog/reviews/genomics/meta/pathogen-screening-metagenomics-2008.html
tags: [genomics, metagenomics] 
comments: false 
author: John Hawks 
---


<p>
Laura MacConaill and Matthew Meyerson present a cool short review in <i>Nature Genetics</i> of metagenomics applications in pathogen discovery. 
</p>

<p>
The basic principle is to extract DNA from a tumor or sore, do intensive sequencing of all the DNA in it, and use the computers to subtract out everything human. What's left after you subtract out the human DNA is any pathogen that might be in the sample: 
</p>

<blockquote>The two recent studies combined computational subtraction with microreactor-based pyrosequencing to identify viral signatures associated with human disease. Feng et al. used high-throughput pyrosequencing15 and comparison to the human transcriptome to identify a viral sequence in a library of cDNAs generated from individuals with Merkel cell carcinoma, a rare but aggressive human skin cancer. The authors sequenced over 395,000 reads of 150-200 bp in length. After digital transcriptome subtraction, 2,395 sequences remained. Among these, conceptual translation of one sequence showed similarity to a polyomavirus. By cloning the complete viral genome and carrying out further analyses, the authors found that the Merkel cell polyomavirus sequence was present in eight of ten Merkel cell carcinomas.</blockquote>

<blockquote>A second group used the same high-throughput DNA sequencing technology to identify a previously undiscovered arenavirus that likely caused the deaths of three transplant recipients who all received organs from a single donor.</blockquote>

<p>
I don't know if sequencing will ever get so cheap that this will become practical diagnostic method, but it really doesn't need to be. As soon as you suspect a pathogen, you can probe directly for that pathogen's DNA in a sample -- and there's no barrier to testing for hundreds of pathogens at once. Heck, there ought to be a SNP chip for it. 
</p>

<p>
But this is a potentially important way of identifying new pathogens in unknown samples from scratch. The article mentions that the current cost of this kind of sequencing is around $10,000 per sample, and that is rapidly falling. For that cost, you get the sequence on your computer, even if you can't identify it yet, and who knows -- it might pop up two years later when somebody else finds it in some unexpected place. 
</p>

<h4>References:</h4>

<p class="cite">MacConaill L, Meyerson M. 2008. Adding pathogens by genomic subtraction. Nat Genet 40:380-382. <a href="http://dx.doi.org/10.1038/ng0408-380">doi:10.1038/ng0408-380</a></p>

