---
layout: single 
title: "Why have variants influencing recombination rate been selected in non-Africans?" 
category: story
permalink: /weblog/reviews/genomics/selection/recombination-rate-rnf212-kong-2008.html
tags: [natural selection, genomics] 
comments: false 
author: John Hawks 
---


<p>
A complicated story is tangled through this paper by Augustine Kong and colleagues, and I don't see where it may end. But here's the abstract: 
</p>

<blockquote>The genome-wide recombination rate varies between individuals, but the mechanism controlling this variation in humans has remained elusive. A genome-wide search identified sequence variants in the 4p16.3 region correlated with recombination rate in both males and females. These variants are located in the <i>RNF212</i> gene, a putative ortholog of the ZHP-3 gene that is essential for recombinations and chiasma formation in <i>Caenorhabditis elegans</i>. It is noteworthy that the haplotype formed by two single-nucleotide polymorphisms (SNPs) associated with the highest recombination rate in males is associated with a low recombination rate in females. Consequently, if the frequency of the haplotype changes, the average recombination rate will increase for one sex and decrease for the other, but the sex-averaged recombination rate of the population can stay relatively constant.</blockquote>

<p>
Perhaps it's not so curious that alleles of this gene have opposite effects on recombination in males and females. The mechanisms of gamete production are obviously different in the two sexes, and we might expect some kind of frequency-dependent mechanism to regulate recombination. At least, it's a hypothesis. 
</p>

<p>
What I find mysterious is this: 
</p>

<blockquote>A phylogenetic analysis of a 55-kb region containing rs3796619 and rs1670533 in the HapMap data (24) revealed three well-differentiated clusters of haplotypes showing notable differences in frequency between the Yoruban Nigerians (YRI) and CEU and East Asians (CHB and JPT) (fig. S6). The [C,T] and [T,C] haplotypes that associate most strongly with recombination rate have a combined frequency of only 17% in the YRI sample, but reach a frequency of 91% and 98% in the CEU and East Asian samples, respectively. Several SNPs in this region show an unusual degree of divergence among the HapMap groups, on the basis of the rank percentile of their FST values (Wright's coefficient, a measure of variance in allele frequencies among populations) among all autosomal SNPs with the same overall frequency in the HapMap. Specifically, we identified eight SNPs whose FST values are in the top 0.5% for differences between the YRI and East Asian HapMap samples and also in the top 5% of differences between the YRI and CEU samples. Each of these SNPs differentiated a subset of [T,T] haplotypes from the rest, perhaps indicating an episode of positive selection (or a severe founder effect) that increased the frequency of [C,T] and [T,C] haplotypes in the ancestors of European and East Asian populations.</blockquote>

<p>
The [C,T] and [T,C] haplotypes are the ones associated with increased recombination rate in males and females, respectively. The markers are in strong disequilibrium (no [C,C] haplotypes were observed), and seem to have been selected outside of Africa. 
</p>

<p>
I have no idea why. 
</p>

<p>
The recombination rates were all inferred from a large Icelandic sample, so maybe the rates don't really characterize the haplotypes in other populations. Maybe recombination rate is incidental to the real reason for the selection. Or maybe in populations roaring with positive selection on many genes at once, it is a good thing to break them apart more often. 
</p>

<h4>References:</h4>

<p class="cite">Kong A and 16 others. 2008. Sequence variants in the <i>RNF212</i> gene associate with genome-wide recombination rate. Science 319:1398-1401. <a href="http://dx.doi.org/10.1126/science.1152422">doi:10.1126/science.1152422</a></p>

