---
layout: single 
title: "&quot;The worm in the fruit of the mitochondrial DNA tree&quot;" 
category: story
permalink: /weblog/reviews/genetics/mtdna_migrations/mtdna-selection-balloux-2009.html
tags: [mtDNA, mtDNA migrations, genetic drift, recent selection, phylogeography] 
comments: false 
author: John Hawks 
---

Fran&ccedil;ois Balloux (2009) has a polemic in the online access area of <i>Heredity</i> presenting references about mtDNA selection, and arguing that the use of this single genetic marker is no longer warranted without support from other loci. 

Yay! I've been saying that both here, and in peer-reviewed articles, for several years. I think serious workers know that one gene is not enough; two genes (mtDNA and Y chromosome, for example) aren't enough -- we have to integrate information across every possible source, genetic, skeletal, and anthropological, to really test hypotheses about the past. 

Still, an industry of mtDNA sequencing has grown up, reviewing each others' grants and papers, and shutting down any discussion of adaptive changes. Balloux's commentary addresses this problem -- I'm going to quote the same paragraph as <a href="http://dienekes.blogspot.com/2009/09/balloux-in-heredity-on-mitochondrial.html">Dienekes</a>:

<blockquote>Let us assume I gave a seminar. I would tell the audience about my latest results on the population history of the pigmy shrew. My findings would be based on a stretch of DNA comprising several metabolic genes, showing no signs of genetic recombination. Armed with sequences from a large number of individuals sampled over a broad geographical area, I would make some inference on the colonization routes and times. To make life easier, I would restrict my analysis to the mutations I liked best, with nice names having been given to related sequences, rather than relying on dull mathematical quantities. As I reach one of the key conclusions of the lecture, which would go as follows: <i>'It is obvious from the distribution of haplotypes Amanda, Eugenie* and Hector_2 that the Outer Hebrides were colonised about 50,000 years ago, this was followed by considerable population fluctuations, a bottleneck during the last Ice Age, a swift recovery and a dramatic recent expansion over the last 200 years and...'</i>. Imagine that, at that climactic stage I was interrupted by someone in the audience. The impertinent would say, <b>'Sir, can I just ask you whether this confidence in your conclusions may not be misplaced; your analysis is based on a single genetic marker, which comprises genes with a central role in metabolism and is thus likely to have been affected by natural selection'.</b> An awkward silence may ensue, as I would find it difficult to dismiss this criticism easily.</blockquote>

Well, let me tell you, <b>I've been in dozens of audiences, and have raised that exact point</b>. Here is a sample of the bogus responses I've gotten to this question:

Bogus answer 1: <i>There are no functional differences between humans and chimpanzees in the mtDNA, so it can't have been selected during human evolution</i>. False, false false!

Bogus answer 2: <i>Metabolic processes are highly conserved, and humans couldn't have changed much.</i> Hello? Have you noticed that your breakfast <b>didn't exist in the Paleolithic?</b>

Bogus answer 3: <i>But the pattern of variation can be equally explained by a bottleneck.</i> Some aspects can, others can't so easily. 

Bogus answer 4: <i>We examined only noncoding parts of the mtDNA, so there could be no selection.</i> Yes, believe it or not, this is the most common response. I guess they don't teach people about linkage anymore. 

Bogus answer 5: <i>There's little or no evidence of selection on <b>any</b> gene in recent human evolution. Human evolution may have stopped entirely. </i> Oh, lord. Yes, I've gotten this one many times. 

There have been others over the years. Yet mtDNA is a big business -- people seem to be worried that the slightest criticism will bring down the whole thing like a house of cards. That's not true, even if mtDNA has sometimes been selected during human prehistory or history, that doesn't mean it isn't a useful marker for many purposes. But many seem more comfortable avoiding the issue entirely. 

I think that taking the hypothesis of selection seriously would improve most of the work in this field. The possibility of selection doesn't eliminate demographic interpretation -- for example, the high ancient African mtDNA variation <a href="http://johnhawks.net/weblog/reviews/genetics/mtdna_migrations/sub-saharan-africa-population-size-behar-2008.html">allows us to test hypotheses about African demography before 50,000 years ago</a>, and there the data appear to reject the hypothesis of selection, at least after around 150,000 years ago. Gene genealogies don't allow us to see the whole past, just the time and forces that they experienced. If we ignore one of the major forces, we are reducing our knowledge. 


There is an obvious problem testing the hypothesis of selection with mtDNA. When we consider any one single locus, it's always possible to find some demographic scenario that yields exactly the same predictions as selection. It's just a mathematical necessity -- selection is fundamentally a demographic phenomenon, and the increase in frequency of selected alleles looks similar to exponential growth of a small population. 

So what can we do? Fortunately we have lots of options. We can test the proposed demographic hypotheses against the historical record. When we make observations that show that people 1000 years ago had very different frequencies of common haplotypes, well, we know it was selection. There hasn't been any genetically significant bottleneck in the last 1000 years! When we see small Neolithic population samples dominated by haplotypes that are very rare today, again, no historically possible bottleneck could have caused that. 


Balloux with his colleagues (2009) has shown that one aspect of mtDNA patterning -- the association of haplogroup diversity with geography -- is very unlikely to have arisen by genetic drift. Here's part of their abstract: 

<blockquote>We show that populations living in colder environments have lower mitochondrial diversity and that the genetic differentiation between pairs of populations correlates with difference in temperature. These associations were unique to mtDNA; we could not find a similar pattern in any other genetic marker. We were able to identify two correlated non-synonymous point mutations in the ND3 and ATP6 genes characterized by a clear association with temperature, which appear to be plausible targets of natural selection producing the association with climate. The same mutations have been previously shown to be associated with variation in mitochondrial pH and calcium dynamics. Our results indicate that natural selection mediated by climate has contributed to shape the current distribution of mtDNA sequences in humans.</blockquote>

They took a dual approach to testing the hypothesis of selection. First, they modeled the evolution of haplotype diversity under neutrality, and showed that the empirical distribution lies significantly outside that range of results. But even so, we might imagine some bottleneck scenario that would cause low diversity in high-latitude peoples, and this would be difficult to refute historically because many of those populations have poor historical documentation. But demography should have similar effects on other genes, and they were able to show that the rest of the genome doesn't share the mtDNA pattern. 

It's really not that hard to test demographic hypotheses, using comparative genomics and anthropological knowledge. That's what anthropological genetics should be doing more and more. There was a time when obtaining a reasonable sample of mtDNA was an accomplishment, and comparing that sample to other genes was not feasible. But that time is past, and hopefully the review process -- journals and grants -- will start demanding some integration of mtDNA phylogeography with results from the rest of the genome. 

Back to Balloux's conclusion: 

<blockquote>Exploiting these new resources of autosomal variation will present significant challenges, but it will not help overcoming them if a large fraction of the community of human population biologists persists in sticking to mtDNA as the marker of choice.</blockquote>

Mitochondrial DNA isn't the tip of the iceberg -- it's an ice cube on top of the tip of the iceberg. 



Related: 

<a href="http://johnhawks.net/weblog/reviews/genetics/mtdna_migrations/selection-mtdna-dowling-tree-review-2009.html">"Mitochondrial DNA selection review"</a>

<a href="http://johnhawks.net/weblog/reviews/genetics/mtdna_migrations/selection-sperm-mtdna-2009.html">"Mitochondrial DNA and sperm"</a>

<a href="http://johnhawks.net/weblog/reviews/genetics/mtdna_migrations/selection-mtdna-iceland-2009.html">"mtDNA selection in Iceland?"</a>

<a href="http://johnhawks.net/weblog/reviews/neandertals/neandertal_dna/green-2008-complete-mtdna-sequence.html">"Complete Neandertal mitochondrial sequence, and selection on human (not Neandertal) mtDNA"</a>

<a href="http://johnhawks.net/weblog/reviews/neandertals/neandertal_dna/orlando_2006_scladina_dna.html">"Did Neandertals need better mitochondria?"</a>

<a href="http://johnhawks.net/weblog/reviews/genetics/bazin_2006_mtdna_selection_diversity.html">"Has the dam broken on mtDNA selection?"</a>

<a href="http://johnhawks.net/weblog/reviews/neandertals/neandertal_dna/ruiz-pesini_selection_mtdna_2004.html">Mitochondrial DNA adaptations in living human populations"</a>

OK, that's enough related posts. But you can find a whole lot more by <a href="http://johnhawks.net/search/node/mtdna%20selection">searching the topic!</a>




<h4>References:</h4>

<p class="cite">Balloux F. 2009. Mitochondrial phylogeography: The worm in the fruit of the mitochondrial DNA tree. Heredity (advance online): <a href="http://dx.doi.org/10.1038/hdy.2009.122">doi:10.1038/hdy.2009.122</a></p>

<p class="cite">Balloux F, Lawson Handley L-J, Jombart T, Liu H, Manica A. 2009. Climate shaped the worldwide distribution of human mitochondrial DNA sequence variation. Proc Roy Soc Lond B 276:3447-3455. <a href="http://dx.doi.org/10.1098/rspb.2009.0752">doi:10.1098/rspb.2009.0752</a></p>




