---
layout: single 
title: "Spatial variation and near-fixed selected alleles" 
category: story
permalink: /weblog/topics/evolution/selection/acceleration/coop-2009-geography-recent-selection.html
tags: [recent selection, HGDP, HapMap, acceleration, 1000 Genomes Project, spatial dynamics, dispersal] 
comments: false 
author: John Hawks 
---

I couple of people have asked me about a new paper in <i>PLoS Genetics</i> by Graham Coop and colleagues, titled, <a href="http://dx.doi.org/10.1371/journal.pgen.1000500">"The role of geography in human adaptation."</a> The paper is open access, and while the details of genetic measures and simulations can be hard to follow, I think it's a great example of the way recent work on selection and human diversity has been structured.

I'll just expand on a few of the topics in the paper, and discuss how they relate to the previous findings about the number and age of selected variants in human populations.
<!--break-->
Here's the paper's abstract: 

<blockquote>Various observations argue for a role of adaptation in recent human evolution, including results from genome-wide studies and analyses of selection signals at candidate genes. Here, we use genome-wide SNP data from the HapMap and CEPH-Human Genome Diversity Panel samples to study the geographic distributions of putatively selected alleles at a range of geographic scales. We find that the average allele frequency divergence is highly predictive of the most extreme F<sub>ST</sub> values across the whole genome. On a broad scale, the geographic distribution of putatively selected alleles almost invariably conforms to population clusters identified using randomly chosen genetic markers. Given this structure, there are surprisingly few fixed or nearly fixed differences between human populations. Among the nearly fixed differences that do exist, nearly all are due to fixation events that occurred outside of Africa, and most appear in East Asia. These patterns suggest that selection is often weak enough that neutral processesespecially population history, migration, and driftexert powerful influences over the fate and geographic distribution of selected alleles.</blockquote>

The paper looks for "nearly fixed" genetic differences between populations, and finds relatively few of them. That's relatively well-known; the F<sub>ST</sub>-based test has been done on fewer populations with similar results (e.g., Williamson et al. 2007; Barreiro et al. 2008). This paper has the HGDP panel, which includes many more populations, and therefore is able to add geographic resolution to these older results. They find that the geographic distribution of near-fixed alleles is clinal; there aren't strong boundaries delimiting the geographic distributions of most apparently selected alleles. That means that the same demographic forces affecting neutral genetic variation have also affected recently selected alleles. 

Is that surprising? As we pointed out in our 2007 paper, the recent demographic history of human populations has included a lot of population growth. This means that the number of adaptive mutations should have increased during the last 10,000--20,000 years. High-F<sub>ST</sub> selected alleles can only reflect selected mutations that are older than this (old enough to reach near fixation in one population), or are extraordinarily strong. A few mutations <i>are</i> exceptionally strong in their selective advantages -- <i>SLC24A5</i> and lactase persistence seem to be examples. But as long as adaptive mutations are intrinsically rare, very few of them could have occurred in the small populations of 20,000 years ago or earlier, even if many happened in the large populations of the Holocene. So I think the new paper actually reinforces the interpretation of acceleration. The pattern we're seeing today with new mutations just can't be a feature of human evolution before around 20,000 years ago. 

If selection is affected by demographic processes, does that mean that it is "weak"? Clearly, "weak" is a matter of scale. Adaptive genes disperse through a spatially structured population very slowly, even if they confer very large fitness advantages. That means that their dispersal is highly dependent upon demographic conditions, such as the disproportionate growth of some populations or occasional long-distance gene flow. Locally, an allele may rapidly increase under selection, but that effect may have little influence on the evolution of distant populations. 

We see that pattern with genes known to be under strong selection in humans, like the ones that help some people resist malaria. Sickle cell, hemoglobin C and E, alpha- and beta-thalassemia, ovalocytosis, G6PD deficiency all have restricted geographic ranges that parallel the clinal pattern of neutral genes. There is an important difference: the patterns of these genes diverge in areas where malaria risk changes rapidly with geography (like coastal versus inland areas of Mediterranean Europe), and some of them have wide geographic distributions compared to their young haplotype ages (like sickle cell). But even in the latter cases, most are too rare to elevate the F<sub>ST</sub> of surrounding SNP markers. Malaria adaptations are a tremendous example of the way that demographic conditions limit strong selection. 


<h4>Africa versus other populations</h4>

Derived alleles are expected to have lower frequencies on average than ancestral alleles. So if a population has a bias toward higher-frequency derived alleles, that may be evidence against neutral evolution. The paper finds that this bias is greater in non-African populations than within Africa: 

<blockquote>The overall genic enrichment is present in all three population comparisons, and each tail seems to be similarly enriched for high- F<sub>ST</sub> genic SNPs. However, the number of derived alleles in each tail does differ substantially and is biased towards derived alleles outside Africa and especially in east Asia. Thus, the statistical evidence for enrichment of events inside Africa is weaker than for the other two populations (we return to this point later).</blockquote>

In general, populations outside Africa have a genome-wide bias toward higher frequencies of derived alleles. The causes of that bias aren't clear -- ascertainment may account for some of the bias but cannot account for all of it; it's possible that early demographic events may explain some of the bias but the pattern isn't obvious. 

The F<sub>ST</sub>-based tests of neutrality are most powerful when a new allele has swept several rare mutations with it to near-fixation. Rare mutations tend to be derived ones. So the power of the test depends on how many rare mutations there are to start with, and what their frequencies are in other populations that didn't have the same selected allele. 

It's one of many issues that make finding selection in African populations slightly different from elsewhere. I think that Africans have undergone as much, and very possibly more, selection by new adaptive mutations as other populations. But our 2007 work suggested that the modal age of the selection we ascertain in Africa may be older than in other regions. That would be consistent with demographic history, since Late Pleistocene African populations were larger than others. But it's possible that genome-wide features like faster LD decay, higher heterozygosity, and more ancestral versus derived variants may also influence our estimates of the timing and number of selected alleles in Africa. 


<h4>Polygenic adaptation</h4>


Toward the end of the paper, the authors discuss the pattern of local adaptation in a more general sense. Why should there be relatively few near-fixed genetic differences between populations, if human ecological changes suggest that local adaptation should have been a powerful force in our recent evolution? One possibility is acceleration -- most of the variants are too recent to have reached near-fixation in any single population. 

But the authors mention another possible influence that we've also been thinking about: epistatic interactions among new variants. For example, lots of skin pigmentation loci are known to have been under recent selection, but only a couple of them have reached near-fixation in any population. The rest are at lower frequencies. Since these alleles all affect the same phenotype, they're subject to diminishing returns. As one lighter-pigment allele becomes common, it reduces the strength of selection on the others. The population doesn't have to fix for any of them; in fact, selection probably cannot drive more than one or two up to fixation since the rest of them compete with each other. 

Over the very long term, this situation would be sorted out. A handful of loci that optimize skin pigmentation might ultimately go to high frequencies or fixation, for some alleles the costs may exceed the benefits and they will disappear. Others, relatively neutral to each other, may fix by drift. But the "very long term" is a span of hundreds of thousands of generations. Here we're talking about a few hundred generations at most. So human populations aren't anywhere near an optimum, they're in a transient where epistatic interactions may be quite important. 

Greg Cochran and I have been discussing this idea for some time. We call it the "Stooge effect". Think of the Three Stooges all trying to run through a door at the same time and getting stuck in the middle. That's what these genes are doing -- all of them are competing to respond to selection, but each is slowed by the presence of the others. 

It's not a new idea -- Frank Livingstone used to talk about this general concept with different malaria adaptations. What's new is the increasing evidence that humans are really in a transient with a lot of genes out of equilibrium. It's very possible that for some phenotypes, standing variation has been an epistatic block on the selection of new mutations. For others, the emergence of some new mutations has limited the trajectory of selection on others. 


<h4>Conclusion</h4>

All in all, I think this paper is a nice contribution to our understanding of the pattern and rate of recent positive selection in human populations. Certainly, the HGDP sample will continue to be a very informative addition to our understanding of spatial dynamics in ancient humans. The addition of the new HapMap v.3 samples may be even more important, because these represent further regions with roughly the same discovery power as the initial three HapMap samples. And of course, we have the 1000 Genomes sample coming up, adding significant potential for discovering rarer selected variants.

<h4>References:</h4>

<p class="cite">Coop G, Pickrell JK, Novembre J, Kudaravalli S, Li J, et al. 2009. The Role of Geography in Human Adaptation. PLoS Genet 5(6): e1000500.<a href="http://dx.doi.org/10.1371/journal.pgen.1000500"> doi:10.1371/journal.pgen.1000500</a></p>

