---
layout: single 
title: "Gene regulation and its evolution" 
category: story
permalink: /weblog/topics/evolution/development/hoekstra-coyne-regulation-2009.html
tags: [gene regulation, selection, modularity] 
comments: false 
author: John Hawks 
---

I wrote early this week about Hopi Hoekstra's work on pigmentation evolution in mice (<a href="http://johnhawks.net/weblog/topics/evolution/selection/hoekstra-2009-adaptive-pigmentation.html">"The color of mice"</a>). The linked article focusing on this empirical work didn't mention her interesting involvement in the debate over the nature and importance of gene regulation as a target of selection. 

I wanted to point out some articles on the topic, by Hoekstra and others, because more and more, paleoanthropological hypotheses are being found to involve the evolution of gene regulation. I'll just note a couple of examples (diet and pigmentation), and hint that there are more coming in the next year. 

If the topic of gene regulatory evolution, or <i>cis</i>-regulation in particular, are obscure, let me recommend a 2008 primer article by Wisconsin geneticist Sean B. Carroll: (<a href="http://dx.doi.org/10.1016/j.cell.2008.06.030">"Evo-Devo and an Expanding Evolutionary Synthesis: A Genetic Theory of Morphological Evolution "</a>). Carroll is probably the most well-known advocate of an "evo-devo" perspective on morphological evolution, and in particular the hypothesis that most morphological evolution may be explained by changes to <I>cis</i>-regulation -- "self-acting" sequence elements that affect gene transcription, such as promoter or enhancer elements. 

In a 2007 commentary, Hoekstra and Jerry Coyne presented a critique of the idea that <i>cis</i>-regulation is a central mechanism of adaptation. Here's a quote from their conclusion (Hoekstra and Coyne 2007:1006): 

<blockquote>While the study of <i>cis</i>-regulatory evolution is an important endeavor, justifiably championed by [evo-devotee Sean B.] Carroll and others, our survey of the theory and empirical data shows that the widespread enthusiasm for the importance of <i>cis</i>-regulatory change in evolution is at best premature. Analyzing the verbal theory, one finds no compelling reason to draw a distinction between the genetic basis of anatomical versus physiological evolution. Nor is there good reason to accept the a priori argument thatfor either anatomy or physiologychanges in <i>cis</i>-regulatory genes are more likely to be fixed in evolution than are changes in the coding region of genes. </blockquote>

Everyone agrees that changes in <i>cis</i>-regulation, <i>trans</i>-regulation and good old-fashioned changes to protein sequences may all be selected, and there are examples of each being involved in the evolution of new adaptive phenotypes. So at that level, the theoretical disagreement is relatively sterile -- all of them are possible and cases are known for each. 

The question is whether any of them account for a preponderance of adaptive evolution. Is there anything <i>special</i> about <i>cis</i>-regulation, or any other kind of change? Are they coequal, do they occur in proportion to the number of regulatory elements, amino acid-coding positions, gene duplications? Do any of them release constraints on adaptive changes, allowing more rapid evolution?
<!--break-->
Why does anybody care? Well, there is a mercenary answer: They all have their own empirical research agendas to look out for, and some of them work mainly with experimental models and techniques effective for studying <i>cis</i>-regulation, others on <i>trans</i>-regulation and still others on classical polymorphisms. To me, these are totally boring topics, since I'm not hoping for any funding to do molecular work on gene regulation. Hopefully, the funding conflict will become less important as genomic methods get cheaper. Of course, when it's no longer difficult to find out the answers, we'll have a decent survey of empirical cases! 


<b>Search strategies.</b> A second explanation for why we should care is a practical one. Now that we are able to get genomes from any species we like, the question arises: what is a sensible strategy for forming hypotheses about adaptive (and non-adaptive) change? What should we be looking for? 

Coyne and Hoekstra wrote a 2007 perspective on an article about amylase adaptive evolution inhuman populations. They returned to the issue of whether we should expect a predominant target of adaptive change: <i>cis</i>-regulatory or so-called "structural" mutations to coding sequences. 

<blockquote>The amylase results [showing adaptive change in gene regulation by duplication] followa related study on the genetics of human dietary differences. In 2006,Tishkoff and colleagues [11] identified a mutation in the upstream regulatory region of the gene for lactase, an enzyme important for digesting milk, in pastoral African populations. Using an in vitro system, they showed that this mutation could increase gene expression. The relevant mutation, however, is not a duplication, but probably a change in <i>cis</i>-regulation. (An independent <i>cis</i>-regulatory mutation at this locus, also conferring lactose tolerance, was identified earlier in European populations [12].)</blockquote>

<blockquote>Even in the simplest cases of adaptation, then  increased enzyme production to handle newdiets  evolution works in multiple ways. Obviously, no amount of <i>a priori</i> speculation will tell us which sorts of mutations willbe important; the answer, unfortunately, requires meticulous, case-by-case analysis of putative adaptations.</blockquote>

Humans may be a poor model organism for considering this question. For one thing, we have a large store of loss-of-function mutations that have been selected for resistance to disease. The same thing probably occurs in other species, but the exceptional number of new diseases in humans may tilt the scales in favor of "structural" mutations. 

Still, these diet-related examples show pretty clearly that multiple mechanisms of gene regulation may be targets of recent selection. That's also evident when we consider human pigmentation variation, a system that is relatively well-understood now from a genetic perspective, for the same reason that Hoekstra's deer mice pigment variations are tractable. In a genome-wide context, it now looks like <i>cis</i>-regulation has been a frequent target of recent adaptive evolution (Kudaravalli et al. 2009), but most of the well-studied examples of recent adaptive change are amino-acid coding


<b>The breadth of pleiotropy.</b> Pleiotropy ought to impede adaptation. If genes are solving multiple problems -- by interacting with distinct functional networks -- then changes that make one function better may often make others worse. If genes interact widely enough, then optimization becomes extremely difficult or impossible -- what Stuart Kauffman (1993) called a "complexity catastrophe." Make a system complicated enough, and the probability falls to nil that a random change might <i>improve</i> it. 

If evolution were <i>generally</i> mutation-limited in this way, you might well expect to see a highly modularized system of gene regulation evolve, and that's precisely the argument for the <i>cis</i>-regulatory evo-devo model. I don't have an opinion on the general question of how often adaptations may make use of this modular system of regulation as opposed to <i>trans</i>-regulation, duplication, or straight-on coding substitutions. It seems like toolkit genes, which are both highly conserved and strongly pleiotropic, may have evolved by altering <i>cis</i>-regulation more often than other means. Those genes have highly modularized "cassettes" of <i>cis</i>-regulatory elements that control their expression in different contexts. One regulatory element can change without necessarily impeding the function of the gene in other contexts.  

Empirical pigmentation research helps to illuminate the dispersal (and limits to dispersal) of recently selected mutations. That makes it a very relevant model system for understanding recent human evolution. Many human (and Neandertal) mutations to <i>MC1R</i> are trans-regulatory -- by altering the sequence of the hormone receptor, these mutations downregulate the pathway that converts pheomelanin to eumelanin. The nature of this regulatory change is structural -- it's an actual change in the gene product that affects pigmentation. 

From the deeper perspective of paleoanthropology, the evolution of form is the central topic. How did the distinctive conformation of the bipedal pelvis evolve? Some paleoanthropologists have already laid out scenarios in which morphological evolution took a small number of very broad changes -- pelvis, spine, and femora as integrated units that may have had correlated effects on arms and other morphological structures. Such hypotheses make the background assumption of strong pleiotropy on a hard-to-explore adaptive landscape. If human evolution was a product of a few hard-to-get mutations, which might not have happened at all, then our emergence was contingent on a series of unlikely events. 

Pleiotropic constraints may help to explain why we see a pulse of rapid adaptive evolution in humans along with population growth. Adaptive mutations rarely appeared during the earlier Pleistocene, making many possible changes were mutation-limited. If certain kinds of regulatory changes were very easily evolvable, then we might expect to see a different pattern of recent evolution. 



Still, most folks don't think very much about pleiotropy as a constraint on human evolutionary change. The "one gene, one trait" model is really universal out there. Certainly, if you ask people, they'll give you the textbook answer -- genes have more than one function; phenotypes are influenced by many genes. But I can't tell you how many times I've heard people refer to "skin color genes" as if they did nothing else. 




<h4>References:</h4>

<p class="cite">Carroll SB. 2008. Evo-Devo and an Expanding Evolutionary Synthesis: A Genetic Theory of Morphological Evolution. Cell 134:25-36. <a href="http://dx.doi.org/10.1016/j.cell.2008.06.030">doi:10.1016/j.cell.2008.06.030</a></p>

<p class="cite">Coyne JA, Hoekstra HE. 2007. Evolution of protein expression: New genes for a new diet. Curr Biol 17:R1014-R1016. <a href="http://dx.doi.org/10.1016/j.cub.2007.10.009">doi:10.1016/j.cub.2007.10.009</a></p>

<p class="cite">Hoekstra HE, Coyne JA. 2007. The locus of evolution: Evo devo and the genetics of adaptation. Evolution 61:995-1016. <a href="http://dx.doi.org/10.1111/j.1558-5646.2007.00105.x">doi:10.1111/j.1558-5646.2007.00105.x</a></p>

<p class="cite">Kauffman SA. 1993. <a href="http://www.amazon.com/gp/product/0195079515?ie=UTF8&tag=johnhawksanth-20&linkCode=as2&camp=1789&creative=390957&creativeASIN=0195079515">The Origins of Order: Self-Organization and Selection in Evolution</a>. Oxford University Press, New York. </p>

<p class="cite">Kudaravalli S, Veyrieras J-B, Stranger BE, Dermitzakis ET, Pritchard JK. 2009. Gene expression levels are a target of recent natural selection in the human genome. Mol Biol Evol 26:649-658. <a href="http://dx.doi.org/10.1093/molbev/msn289">doi:10.1093/molbev/msn289</a></p>

