---
layout: single 
title: "Leprosy evolution in humans" 
category: story
permalink: /weblog/reviews/genomics/disease/leprosy-monot-2009-phylogeography.html
tags: [disease, paleogenomics, genomics, health, phylogeography, pathogens] 
comments: false 
author: John Hawks 
---


Where did leprosy come from as a human pathogen, and how did it spread through the world? Two years ago, this new research would have merited a whole book. Now it's all packed into a single <i>Nature Genetics</i> paper by Marc Monot and coworkers. 

I mean, there's a lot in here: 

1. They used next-gen sequencing platforms to get three additional whole-genome sequences for the pathogen that causes leprosy, <i>Mycotuberculum leprae</i>. 

2. By comparing the different strains together with an already-available one, representing patients in four countries, they measured the genome diversity and found SNPs between strains. 

3. They then genotyped the resulting SNPs in 400 isolates, building a phylogeny of worldwide strains of <i>M. leprae</i> today. 

4. They did a phylogeographic analysis of the strains, testing hypotheses about past transfers of the bacterium among regions. 

5. And then, on top of all that, they recovered skeletal remains from "leprosy graveyards" in six countries, diagnosed the skeletal correlates of leprosy in 13 cases and genotyped the resulting extracts for <i>M. leprae</i>, placing them on the global phylogenetic tree. 

Whew!

Well, I assume that the skeletal work was done separately, with samples being sent to the lab folks to do their DNA extraction. 


This would be a really good topic for a documentary. There's all the historical information about leprosy to draw upon, including of course its prominent appearance in the Bible and Father Damien. There's the triumph of effective treatments in developed parts of the world -- an aspect that this paper emphasizes, as it attempted to find out whether regions of the world that now lack <i>M. leprae</i> once had the strains expected from their geographic placement. And there's the continuing tragedy of the disease in many less developed parts of the world, with the need to deliver treatment more effectively. Meanwhile, the phylogeographic aspects of this paper provide another historical angle, about the spread of leprosy around the world on human trade routes. 

Plus there's the whole mystery of how it got into humans in the first place:

<blockquote>Finally, it is worth discussing the enormous discrepancy between the period at which pseudogene formation is thought to have arisen and the origin of early humans. It has been estimated recently that the bulk of the pseudogenes in M. leprae arose no earlier than 9 million years ago. Pseudogene formation is an indicator of radical change in the lifestyle of the host bacterium, such as from the free-living to pathogenic state or of adaptation to life within a particular tissue or cell type. In the case of M. leprae, obligate parasitism of humans or another primate species would represent such a change. Although modern humans represented by H. sapiens have existed only since approximately 250,000 years ago and left Africa within the last 100,000 years to settle other regions, earlier hominids are thought to have diverged from chimpanzees over 5 million years ago. Reconciliation of the estimated time of pseudogene formation with human evolution could be achieved if an ancestor of M. leprae infected an early primate and then underwent genome decay and was subsequently transmitted verticallyalthough this seems unlikely, given that more genetic diversity among M. leprae isolates would be expected if this were true. Alternatively, the genome decay could well be ancient, but M. leprae may only recently have become a human pathogen. For instance, it is conceivable that an ancestral form of M. leprae infected an invertebrate host such as an insect, which later acted as a vector for transmitting the bacillus to humans. Support for the latter scenario is provided by studies of the related pathogen Mycobacterium ulcerans, which is at an early stage of reductive evolution and appears to be transmitted to humans by water bugs and/or mosquitoes. Further insight into the timing of pseudogene formation in M. leprae will be provided by microbiology and paleomicrobiology and by deeper genome sequence analysis.</blockquote>

In rough outline, you "date" a pseudogene by counting the number of nonsynonymous substitutions in comparison to some other species where the gene is functional. When the gene was functional, most substitutions of nonsynonymous mutations would have been prevented by purifying selection. You generally apply more detailed assumptions, but that's the basic process. I raise the point because dating a 9-million-year-old event in a bacterial species on the basis of nonsynonymous mutations is probably not going to give a very tight confidence interval, to put it charitably. Maybe 9 million is 4 million? 

In any event, leprosy is one more addition to a growing story about the coevolution of pathogens with <i>Homo</i>. It may have a long history with us, like its congener, tuberculosis. It apparently doesn't have a long history of coevolution within different regionally variable human populations -- tuberculosis does. Possibly it is a relatively recent invasion from another species, which would make it maybe more like the evolutionary dynamics of vivax malaria. 

We don't lack for examples, and tabulating the histories of all of these pathogens may give us a better picture of the population ecology of <i>Homo</i> in Pleistocene times. 





<h4>References:</h4>

<p class="cite">Monot M and many others. 2009. Comparative genomic and phylogeographic analysis of <i>Mycobacterium leprae</i>. Nat Genet 41:1282-1289. <a href="http://dx.doi.org/10.1038/ng.477">doi:10.1038/ng.477</a></p>




