---
layout: single 
title: "Mailbag: Genome fantasy league" 
category: mailbag
permalink: /weblog/mailbag/genome-fantasy-league-2009.html
tags: [genomics] 
comments: false 
author: John Hawks 
---

<blockquote>Hi John.   Here is a variant that I like because the question points at reasons and strategy that the one-genome question does not:
 
If you could pick any organism to have 100 individuals fully sequenced ? what would it be?  How would you choose the 100 candidates?  What would you do with the data?
 
My own first thought is that it should be one of several disease organisms, and sampled both temporally and geographically.
 
Some years ago there was a controversial "50 Drosophila Genomes" project afoot.  I'm not sure how far that has gone.  At the time it was thought absurdly expensive for the given goal.  It still might be, if FULL sequencing still requires clone-by-clone methods.  Which of course it does.  All these new, fast shotgun methods miss repetitive stuff.  The guiding idea, I think, was to get some handle on the effects of natural selection on all levels and kinds of DNA variation - - - .  I can't recall who was pushing it.  It was a fairly prominent Drosophila guy.</blockquote>

Yes, I think that's a good addendum. Of course, we don't have that for humans yet....

I would dearly love to see as many early modern (or even Neolithic) humans sequenced as possible. We could do a lot of work testing how things have changed in the last 10,000 years. And then there are the apes!

I'd be interested to have two genomes each from fifty different primates. The one-genome-per-species strategy really doesn't make sense anymore -- understanding fixed differences doesn't work unless you understand the pattern of polymorphism. 



