---
layout: single 
title: "Mutual information between strings of loci" 
category: story
permalink: /weblog/topics/information/theory/strings-of-loci-multiinformation.html
tags: [effective population size, recent selection, HapMap, recombination, information theory] 
comments: false 
author: John Hawks 
---

<div class="quote">      <p class="noindent" >Fourth in a series on mutual information and genetic linkage. If      you&#8217;re happening upon it for the first time, you can find the <a  href="http://johnhawks.net/taxonomy/term/306" >entire      series</a> or the first post, <a  href="http://johnhawks.net/weblog/topics/information/theory/information-theory-intro-2008.html" >&#8220;Information theory: a short introduction&#8221;</a>.</p> 
 
</div> <p class="noindent" >After <a  href="http://johnhawks.net/weblog/topics/information/theory/genetic-drift-reduces-entropy-2009.html" >the last post</a>, you might wonder what the big deal is about these information theoretic measures of linkage. After all, we&#8217;ve got lots of other measures of linkage to choose in population genetics, with many years of theory behind them. The basic conclusion about genetic drift was that it adds mutual information to samples over short regions, but that recombination over longer areas washes it out. If the net effect is no linkage, why would we bother to come up with some non-standard linkage measure? </p> 
 
<p class="indent" >   One answer: If the existing linkage measures were so great for testing neutrality, then we might expect some of the recent genome-wide selection scans to have used them. But they didn&#8217;t &#8211; instead we have several partially incompatible methods, all of which eschew the usual measures of linkage. </p>
<!--break-->
<p class="indent" >   I&#8217;m not going to summarize all the reasons for this, at least not yet. But there is one thing that the current positive selection tests have in common: they all attempt to quantify the decay of linkage across long strings of loci. Each of them&#8212;LRH, iHS, LDD&#8212;distills into a single number the pattern of reduction of linkage across physical distance. </p> 
 
<p class="indent" >   After <a  href="http://johnhawks.net/weblog/topics/information/theory/genetic-drift-reduces-entropy-2009.html" >the last post</a>, that may seem odd. If we know the physical distance (or more to the point, the map distance), we can predict the amount of linkage under drift. That really ought to be enough to test neutrality. Yet the existing tests insist on <span  class="cmti-10">a second dimension</span>: the rate of decline of linkage. Partly, that&#8217;s because these tests try to separate positive selection from <span  class="cmti-10">other </span>things that violate neutrality, like inversions. But it also reflects a real limitation of the methods: they ignore some of the information available in the data, and thereby limit their power to test the hypothesis of genetic drift based on linkage. So they must turn to the <span  class="cmti-10">rate </span>of linkage decay as a test of drift. <a   id="section*.2"></a></p> 
 
    <h4 class="subsectionHead"><a   id="x1-2000"></a>The trouble with homozygosity</h4> <p class="noindent" >Massive genotyping projects are not sequencing projects. They provide long lists of genotypes, not sequences. If you&#8217;ve got a long list of genotypes, one of the measures of variation immediately available to you is homozygosity. Homozygosity among                                                                                                                                      multiple sites, or <span  class="cmti-10">joint homozygosity </span>is not mathematically the same as the mutual information between sites, but it also is increased by linkage. To the extent that genetic drift predicts the distribution of long-range linkage, you can test neutrality by looking at the joint homozygosity of two or more sites. For example, the extended haplotype heterozygosity (EHH) is the probability that a pair of gametes that share a single core haplotype will also share identical haplotypes at longer distances. Homozygosity itself is not a test of linkage&#8212;for that we have to combine the probabilities of identity of all alleles in some way. </p> 
 
<p class="indent" >   Homozygosity-based tests of neutrality set up this combination as a ratio: the ratio of homozygosity for one allele or core haplotype versus the homozygosity of other alleles or haplotypes. That procedure throws away a lot of information carried by the fraction of haplotypes that are nonidentical at distance. Since that fraction of non-homozygote mutual information, unlike homozygosity, actually <span  class="cmti-10">increases </span>with distance, it may in some circumstances provide a more powerful test of neutrality. The use of a ratio of homozygosity also reduces the chance to detect certain classes of non-neutral loci&#8212;most obviously, the ones that have two or more selected alleles. </p> 
 
<p class="indent" >   To put some numbers to these ideas, let&#8217;s consider we have two loci, <span  class="cmmi-10">A </span>and <span  class="cmmi-10">B</span>. The alleles of <span  class="cmmi-10">A </span>are <span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub> and <span  class="cmmi-10">a</span><sub><span  class="cmr-7">2</span></sub>, <span  class="cmmi-10">B </span>has <span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub> and <span  class="cmmi-10">b</span><sub><span  class="cmr-7">2</span></sub>. Each of these alleles has an associated frequency, for example <span  class="cmmi-10">p</span>(<span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub>), and the frequency of the <span  class="cmti-10">haplotype </span><span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub><span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub> will be written <span  class="cmmi-10">p</span>(<span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub><span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub>). If <span  class="cmmi-10">A </span>and <span  class="cmmi-10">B </span>are independent (unlinked) then the expected value of <span  class="cmmi-10">p</span>(<span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub><span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub>) would simply be <span  class="cmmi-10">p</span>(<span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub>)<span  class="cmmi-10">p</span>(<span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub>). </p> 
 
<p class="indent" >   The usual measure of linkage, <span  class="cmmi-10">D</span>, is calculated as </p> 
 
    <table  class="equation"><tr><td>    <center class="math-display" > <img  src="/graphics/strings-of-loci-multiinformation0x.png" alt="D = p(a b)p(a b) - p(a b )p(a b )        1 1   2 2     1 2   2 1 " class="math-display"  /><a   id="x1-2001r1"></a></center></td><td class="equation-label">(1)</td></tr></table> <p class="nopar" > </p> 
 
<p class="indent" >   It thus involves the frequencies of all four possible haplotypes. At linkage equilibrium, the two terms are expected to be the same, so that <span  class="cmmi-10">D </span>= 0. </p> 
 
<p class="indent" >   Several other measures of linkage are in common use (and sometimes lead to confusion). There are several reasons why you might want a different measure, and one of the biggest is that <span  class="cmmi-10">D </span>depends on the frequencies of the alleles <span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub> and <span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub>&#8212;low-frequency polymorphisms will have systematically lower linkage values for                                                                                                                                      the same evolutionary scenario. A very good thing about <span  class="cmmi-10">D </span>is that it does not require us to know the frequencies of the individual alleles <span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub> or <span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub>, only the frequencies of their joint haplotypes. But the expectation that the products in the equation are equal depends on a 2 <span  class="cmsy-10">&#x00D7; </span>2 contingency table. Three-, four- or more-locus haplotypes demand some other measure. </p> 
 
<p class="indent" >   Using the same terminology, the homozygosity of the haplotype <span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub><span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub> is given as <span  class="cmmi-10">p</span>(<span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub><span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub>)<sup><span  class="cmr-7">2</span></sup>. By itself, this tells us nothing at all about linkage. If we combine it with the homozygosity of one or the other allele, (for example with the difference <span  class="cmmi-10">p</span>(<span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub>)<sup><span  class="cmr-7">2</span></sup> <span  class="cmsy-10">- </span><span  class="cmmi-10">p</span>(<span  class="cmmi-10">a</span><sub><span  class="cmr-7">1</span></sub><span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub>)<sup><span  class="cmr-7">2</span></sup>, we will have a measure of the <span  class="cmti-10">reduction </span>in homozygosity with distance. That reduction in homozygosity still doesn&#8217;t tell us about linkage, without considering <span  class="cmmi-10">p</span>(<span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub>). But if we scale the reduction by taking it as a ratio with the reduction in homozygosity of one or more other haplotypes, those other haplotypes give us <span  class="cmti-10">indirect </span>information about <span  class="cmmi-10">p</span>(<span  class="cmmi-10">b</span><sub><span  class="cmr-7">1</span></sub>). So we have an <span  class="cmti-10">indirect </span>measure of linkage, based on the relative rate of linkage decay. </p> 
 
<p class="indent" >   Relying on the rate of decay of linkage isn&#8217;t such a bad idea if we don&#8217;t know the rate of recombination between markers. In that case, we can use the rate of decay of homozygosity of <span  class="cmti-10">other </span>haplotypes as a scaling factor. But there is a problem lurking: The decay of homozygosity around a marker or core haplotype depends on its frequency. Under drift, higher-frequency haplotypes decay over shorter distances. So our test will be biased in a way that depends on the frequency spectrum of haplotypes. </p> 
 
<p class="indent" >   It seems much more direct to estimate the mutual information between <span  class="cmmi-10">A </span>and <span  class="cmmi-10">B</span>. We don&#8217;t need to use an indirect linkage measure when we have all the frequencies needed to calculate a direct measure. Mutual information is a useful measure because it extends easily to many loci. But it has a disadvantage: it requires us to obtain an independent estimate of the recombination fraction between our markers. </p> 
 
<p class="indent" >   Maybe that&#8217;s not such a problem. High-resolution recombination maps are available for the HapMap markers. If we use those maps, then we can test neutrality with markers at fixed map distances instead of physical distances. That ought to let us easily quantify the genome-wide fraction of mutual information that originated in a given time interval in the past. But we&#8217;ll have to remain cautious, since the map distances are worked out under the assumption of neutrality. <a   id="section*.3"></a></p> 
 
    <h4 class="subsectionHead"><a   id="x1-3000"></a>Why multilocus haplotypes are useful</h4> <p class="noindent" >Why do we care about many loci, when we can test neutrality using only one? Suppose we&#8217;re interested in the mutual information across a 500 kb stretch of the genome. We could pick a single marker on each end of the 500 kb, and measure the mutual information between them. That&#8217;s basically what I did in the case of drift in the last post, <a  href="http://johnhawks.net/weblog/topics/information/theory/genetic-drift-reduces-entropy-2009.html" >&#8220;When genetic drift reduces entropy&#8221;</a>.                                                                                                                                      </p> 
 
<p class="indent" >   That isn&#8217;t slicing the sample very finely. Suppose that our two markers both have major alleles at 80 percent. That makes the expected frequency of the most common joint haplotype 64 percent, and the least common 4 percent. In our sample of 120 gametes, we&#8217;ll conclude that the two are significantly associated if those values go to around 70 and 10 percent, respectively. That&#8217;s like salting the sample with seven or eight copies of the rare haplotype. </p> 
 
<p class="indent" >   On the other hand, we could salt our sample with more than a dozen copies, evenly split between the two intermediate-frequency haplotypes, and never get a significant result. Even if the expected rare haplotype were completely absent, it wouldn&#8217;t be enough to reject the hypothesis of random association, much less the hypothesis of genetic drift. Natural selection can cover its tracks, if two or more linked haplotypes have both increased in frequency. </p> 
 
<p class="indent" >   Is this likely to be a problem very often? Past some distance, positive selection loses its impact on linked haplotypes, because there&#8217;s too much recombination. Near a selected site, linkage is strong enough that there will usually be a single major haplotype hitchhiking upward. In between, there may be minor haplotypes hitchhiking more weakly but we will generally be able to look closer to find a major haplotype. But if there are multiple haplotypes under selection, perhaps because more than one adaptive mutation have occurred, the locus may well look completely neutral. </p> 
 
<p class="indent" >   An example in human populations may be <span  class="cmti-10">MC1R</span>. <a  href="#XHarding:2000">Harding et&#x00A0;al.</a>&#x00A0;(<a  href="#XHarding:2000">2000</a>) sequenced the gene in 106 Africans and over 356 Europeans. They found a diverse array of amino acid mutations in Europeans that were absent in their African sample, concluding that purifying selection on skin pigmentation had prevented such mutations from becoming common in Africa. In contrast, purifying selection was relaxed in Europe, allowing several loss-of-function variants to increase in frequency. They found no statistical evidence for positive selection on these loss-of-function variants in Europe. Likewise, no other recent tests for positive selection have shown <span  class="cmti-10">MC1R </span>to stand out as selected. </p> 
 
<p class="indent" >   But wait a minute. Where did all those European redheads come from? </p> 
 
<p class="indent" >   Harding and colleagues found five different coding polymorphisms with frequencies more than 7 percent in Europe, but completely absent in their sample of Africans. That adds up to roughly half the copies of <span  class="cmti-10">MC1R </span>in Europe, all derived alleles. That kind of emergence of new alleles doesn&#8217;t happen by chance, at least not in a population history like Europe&#8217;s. But none of the usual tests will reject neutrality when five different alleles have increased under selection. And individually, each of these alleles is too rare to register a rejection of neutrality, at least, if we apply the usual tests. </p> 
 
<p class="indent" >   What makes this gene so troublesome? There&#8217;s no problem estimating the frequency of any single derived allele in Europe, and it&#8217;s easy to figure out its extended homozygosity. But if we then compare that to other alleles at the same locus, well, some of them have long haplotypes, too. So none of them have an unusual pattern of LD decay compared to the others.                                                                                                                                      </p> 
 
<p class="indent" >   What to do? We could try to get a much larger genetic sample, or a large sample of individuals from a part of Europe where one of these alleles is especially common. </p> 
 
<p class="indent" >   But since I&#8217;m lazy, I figure I&#8217;ll just consider the mutual information carried by haplotypes of multiple loci. Individually, no single allele may have a frequency high enough to reliably show linkage in a two-locus comparison. Individually, no single allele will reject neutrality by a homozygosity-based test, because the <span  class="cmti-10">other </span>selected haplotypes decay over very long areas, making the ratio for any single haplotype insignificant. Collectively, the selected haplotypes may carry linkage over long distances. Together they will reject neutrality, even if individually they don&#8217;t. </p> 
 
<p class="indent" >   But first, we need to know the distribution of multilocus mutual information under neutrality. <a   id="section*.4"></a></p> 
 
    <h4 class="subsectionHead"><a   id="x1-4000"></a>Degrees of freedom</h4> <p class="noindent" >If you can calculate a <span  class="cmmi-10">&#x03C7;</span><sup><span  class="cmr-7">2</span></sup> and its degrees of freedom, you can skip next four paragraphs. </p> 
 
<p class="indent" >   In the <a  href="http://johnhawks.net/weblog/topics/information/theory/chi-square-mutual-information-2008.html" >second post</a> I showed that the mutual information is distributed as a <span  class="cmmi-10">&#x03C7;</span><sup><span  class="cmr-7">2</span></sup>. That&#8217;s a statistical distribution with one parameter&#8212;the <span  class="cmti-10">degrees of freedom</span>. The concept of degrees of freedom is one of the trickiest definitions in statistics. Roughly, it means the number of independent factors that contribute to a single observation. </p> 
 
<p class="indent" >   Let&#8217;s take the case of a contingency table, where we have a number of rows, <span  class="cmmi-10">r</span>, and a number of columns, <span  class="cmmi-10">c</span>, where the total number of cells is <span  class="cmmi-10">r </span><span  class="cmsy-10">&#x00D7;</span><span  class="cmmi-10">c</span>. If all the cells were independently distributed, then the number in each cell is expected to be the fraction in its row times the fraction in its column. That&#8217;s the expected value. If you take the number you observed in a cell, subtract the number you expected, square that difference, divide it by the expected value, and add up the result for every cell in the table, that&#8217;s the <span  class="cmmi-10">&#x03C7;</span><sup><span  class="cmr-7">2</span></sup> statistic. This statistic is high when the observed numbers are far from the expected ones. It&#8217;s low when they are close to expected. </p> 
 
<p class="indent" >   For a 2 <span  class="cmsy-10">&#x00D7; </span>2 table, we can ask, how many ways are there for the <span  class="cmmi-10">&#x03C7;</span><sup><span  class="cmr-7">2</span></sup> statistic to be high? Any one of the cells may have an unusually large number of observations, so we might be tempted to say there are four different ways to get a high value. But it&#8217;s obvious that if one value is too high, the value in the same row and the <span  class="cmti-10">other</span> column must be too low. Likewise, the observed number in the other <span  class="cmti-10">row </span>and the same column must be too low. And if both those diagonal cells are too low, well then the observed value in the cell diagonal to our <span  class="cmti-10">high </span>cell must also be high. In other words, all four cells depend on each other. Push one, and you push them all. So there&#8217;s really only one axis along which the cells can vary&#8212;one degree of freedom. </p> 
 
<p class="indent" >   With a 3 <span  class="cmsy-10">&#x00D7; </span>2 table, things are a little different. We can hold one row entirely constant, and just change the proportions in the other two. If one cell has a high                                                                                                                                      observed value, that might be because both of the other rows were shortchanged. Or it might be just a single row that&#8217;s low, the other being high as well. That makes two distinct ways that we could get a high <span  class="cmmi-10">&#x03C7;</span><sup><span  class="cmr-7">2</span></sup> statistic, two degrees of freedom. In general, there are (<span  class="cmmi-10">r </span><span  class="cmsy-10">- </span>1)(<span  class="cmmi-10">c </span><span  class="cmsy-10">- </span>1) degrees of freedom in a contingency table. It&#8217;s always one less than the number of rows or columns because <span  class="cmti-10">at</span> <span  class="cmti-10">least one </span>row and column must be shortchanged when a cell is higher than expected. <a   id="section*.5"></a></p> 
 
    <h4 class="subsectionHead"><a   id="x1-5000"></a>Degrees of freedom with comparisons of multiple haplotypes</h4> <p class="noindent" >From the <a  href="http://johnhawks.net/weblog/topics/information/theory/genetic-drift-reduces-entropy-2009.html" >previous post</a>, you might have anticipated how we could do simulations of drift in samples of multiple markers. For the following, I&#8217;m going to examine the mutual information between two sets of three SNP markers. That is, I have a stretch of chromosome roughly 50 kb long from end to end (or more properly, 0.05 cM), with three markers tightly linked at either end. Using the <a  href="http://johnhawks.net/weblog/topics/information/theory/genetic-drift-reduces-entropy-2009.html" >same methods as the previous post</a>, I&#8217;m applying the same three-stage population history with genetic drift only to these six markers. </p> 
 
<p class="indent" >   Here are the results for 10,000 trials: </p> 
 
<div class="middle-picture">   <img  src="/graphics/drift_only_three_stage_rho_4000_six_markers_50kb.png" alt="PIC"    /> </div> 
 
<p class="indent" >   That doesn&#8217;t look very much like the chart that I got in the previous post, using the same population history. Here is that one: </p> 
 
<div class="middle-picture">   <img  src="/graphics/drift_only_N_simple_growth_rho_4000_markers_50kb.png" alt="PIC"    /> </div> 
 
<p class="indent" >   What&#8217;s going on? That red line in the second chart is the <span  class="cmmi-10">&#x03C7;</span><sup><span  class="cmr-7">2</span></sup> distribution with one degree of freedom. The first chart doesn&#8217;t match that at all. </p> 
 
<p class="indent" >   Why not? The second chart shows the mutual information between two markers, with two alleles each. That&#8217;s one degree of freedom. The first chart shows the mutual information between two sets of three markers. Three markers give the possibility of eight distinct haplotypes (2<sup><span  class="cmr-7">3</span></sup>). So we have an 8 <span  class="cmsy-10">&#x00D7; </span>8 contingency table, with 49 degrees of freedom. Here is that distribution, in red, along with the result of our 10,000 simulated samples: </p> 
 
<div class="middle-picture">   <img  src="/graphics/drift_only_three_stage_rho_4000_six_markers_50kb_49df.png" alt="PIC"    /> </div> 
 
<p class="indent" >   Except, oops, it doesn&#8217;t look like 49 degrees of freedom either! What the heck is going on? </p> 
 
<p class="indent" >   If you&#8217;ll remember from the <a  href="http://johnhawks.net/weblog/topics/information/theory/chi-square-mutual-information-2008.html" >second post in the series</a>, our empirical distribution isn&#8217;t going to fit a chi-square unless we have enough observations. Sure, in theory, three markers could give us 8 haplotypes. But in practice, the three markers on each end of our 50 kb region are so tightly linked that we get many fewer haplotypes. Sometimes we get only three or four haplotypes on each end. If each set of three markers were independent of the other, they might fit a chi-square with four degrees of freedom, or nine, or six or twelve. In fact, the mutual information observed in these 10,000 simulations accords with a mixture of distributions. Here are the number of degrees of freedom in the 10,000 simulations, as a histogram: </p> 
 
<div class="middle-picture">   <img  src="/graphics/drift_only_three_stage_rho_4000_six_markers_50kb_df_histogram.png" alt="PIC"    />                                                                                                                                      </div> 
 
<p class="indent" >   Most of the trials have six or nine degrees of freedom, with 12 being a substantial proportion as well. No primes, there, except for 2 and 3. Only a tiny fraction of trials have as many as 20 degrees of freedom between the 3-marker sets. None get anywhere close to the theoretical 49, because the three-marker sets are too tightly linked to express all possible haplotypes. </p> 
 
<p class="indent" >   So if we want a theoretical distribution to match to our simulated one, we are going to need a <span  class="cmti-10">mixture </span>of different <span  class="cmmi-10">&#x03C7;</span><sup><span  class="cmr-7">2</span></sup> in proportion to the number represented in the simulated set. Here&#8217;s that comparison: </p> 
 
<div class="middle-picture">   <img  src="/graphics/drift_only_three_stage_rho_4000_six_markers_50kb_mix_df.png" alt="PIC"    /> </div> 
 
<p class="indent" >   The red line is a mixed distribution, in which different <span  class="cmmi-10">&#x03C7;</span><sup><span  class="cmr-7">2</span></sup> distributions are blended in the proportions represented by the histogram above. That&#8217;s the distribution of mutual information between the two marker sets, conditioned on the haplotypes actually present in each set, under the hypothesis of independence. The simulations lie to the right of this distribution, meaning that small-sample bias and genetic drift have added mutual information to the simulations, compared to the expectation in the absence of linkage. </p> 
 
<p class="indent" >   The only difference between these simulations and the ones in the previous post is the number of markers. Here, we&#8217;re looking at mutual information between two three-marker sets; there, we were looking at mutual information between two one-marker loci. Both examples used the same distance and the same population history. Genetic drift has had a similar effect&#8212;in both sets of trials, there is a slight excess of cases with high mutual information. With both, there are around twice as many at the tail as expected if the markers were independent. </p> 
 
<p class="indent" >   This leads to a couple of conclusions:      </p> 
 
<ol  class="enumerate1" >      <li    class="enumerate" id="x1-5002x1">Under this population history, genetic drift is not sufficient to cause very      large amounts of mutual information to be shared by distant sites.      </li>      <li    class="enumerate" id="x1-5004x2">A slight alteration to the <span  class="cmmi-10">&#x03C7;</span><sup><span  class="cmr-7">2</span></sup> test&#8212;for example, adding some proportion      to the critical values&#8212;might allow us to test the hypothesis of neutrality      for any given case without the necessity of all those simulations.      </li></ol> <p class="indent" >   If we were to add more and more markers, we would eventually find that the bias in mutual information due to small sample size would get bigger. So there&#8217;s some maximum number of markers giving useful information in any given sample. But I just picked 3 markers out of a hat. Adding more markers, up to some point, should increase our ability to see rare haplotypes that might diverge from the expectations of genetic drift. </p> 
 
<p class="indent" >   <span  class="cmbx-10">Next: Conditional mutual information: finding the haplotypes that explain linkage disequilibrium</span>                                                                                                                                      </p> 
 
<p class="indent" >                                                                                                                                      </p> 
 
    <h2 class="likechapterHead"><a   id="x1-6000"></a>References</h2> <a   id="likesection.6"></a><a   id="Q1-1-7"></a>   <div class="thebibliography">   <p class="bibitem" ><span class="biblabel"> <a   id="XHarding:2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Harding  RM,  et&#x00A0;al.  2000.   Evidence  for  variable  selective  pressures  at   MC1R. Am J Hum Genet 66:1351&#8211;1361. </p> 
 
   </div>

