---
layout: single 
title: "mtDNA, purifying selection and &quot;distorted&quot; genealogies" 
category: story
permalink: /weblog/reviews/genetics/mtdna_migrations/seger-2010-distorted-genealogies.html
tags: [mutation rate, effective population size, theory, mtDNA, mtDNA migrations, population genetics] 
comments: false 
author: John Hawks 
---



I'm going to pass along this paper without much comment, it's by Jon Seger and colleagues and it came out earlier this year in <i>Genetics</i> <bib>Seger:2010</bib>: 

<blockquote>Gene Genealogies Strongly Distorted by Weakly Interfering Mutations in Constant Environments</blockquote>

<blockquote>Neutral nucleotide diversity does not scale with population size as expected, and this "paradox of variation" is especially severe for animal mitochondria. Adaptive selective sweeps are often proposed as a major cause, but a plausible alternative is selection against large numbers of weakly deleterious mutations subject to HillRobertson interference. The mitochondrial genealogies of several species of whale lice (Amphipoda: Cyamus) are consistently too short relative to neutral-theory expectations, and they are also distorted in shape (branch-length proportions) and topology (relative sister-clade sizes). This pattern is not easily explained by adaptive sweeps or demographic history, but it can be reproduced in models of interference among forward and back mutations at large numbers of sites on a nonrecombining chromosome. A coalescent simulation algorithm was used to study this model over a wide range of parameter values. The genealogical distortions are all maximized when the selection coefficients are of critical intermediate sizes, such that Muller's ratchet begins to turn. In this regime, linked neutral nucleotide diversity becomes nearly insensitive to N. Mutations of this size dominate the dynamics even if there are also large numbers of more strongly and more weakly selected sites in the genome. A genealogical perspective on HillRobertson interference leads directly to a generalized background-selection model in which the effective population size is progressively reduced going back in time from the present.</blockquote>

The topic arises for me at the moment because of some inconsistencies between the apparent timing of events from mtDNA estimates compared to nuclear DNA estimates. Across the crucial "out of Africa" time interval between 200,000 and 50,000 years ago, the mtDNA is not really giving the same chronology as might be expected from nuclear DNA comparisons. 

The mutation rate of mtDNA genome-wide is very high, giving rise to the possibility of interaction between weakly deleterious mutations on the same sequence. It is widely known that the apparent rate of mtDNA mutation depends on the timescale of the comparison in humans. Mothers and their offspring differ by much more than would be predicted by longer pedigrees or by comparisons between populations. Recently diverged populations (such as those in island Polynesia) differ much more than would be predicted from the difference between humans and Neandertals or humans and chimpanzees. 

This apparent "speed-up" of rate as we get closer to the present is consistent with the action of strong purifying selection. So establishing the other genealogical effects of this selection should help us understand the patterns of mtDNA sequence differences found in humans. 

