---
layout: single 
title: "Genetic lapidaries" 
category: story
permalink: /weblog/reviews/genomics/variation/desmond-tutu-genomics-2010.html
tags: [genetics, genomics, health, genetics and society] 
comments: false 
author: John Hawks 
---

<i>Nature News</i> has a short piece on yesterday's Desmond Tutu and other South African genomes: <a href="http://www.nature.com/news/2010/100217/full/463857a.html">"Africa yields two full human genomes."</a>

Has anybody else noticed how it has become routine to print a short list of phenotypic associations whenever a new genome is published? Last week, the press about the <a href="http://johnhawks.net/weblog/reviews/genomics/ancient/greenland-eskimo-willerslev-2010.html">Greenland genome</a> emphasized mostly uninformative data about earwax type and hair form. It was all stuff you would have predicted anyway. Now, what pearls of information have come from the African genomes? 

<blockquote>Several of the DNA changes in the Khoisan may reflect adaptations to the rigorous life of a hunter-gatherer in the Kalahari Desert. Three of the Khoisan carry a version of the muscle-expressed ACTN3 gene linked to faster sprinting. !Gubi carries one copy of a form of the CLCNKB gene that encodes a cell-membrane channel capable of reabsorbing chloride ions in the kidney, a possible advantage in the desert. And all have a form of the taste-receptor gene TAS2R38 that would enable them to taste the bitter compound phenylthiocarbamide. This form of the gene may help hunter-gatherers to avoid toxic plants.

The Khoisan also lack a gene variant giving protection against malaria infection by <i>Plasmodium vivax</i>. This could reflect their lifestyle, Schuster says: with both water and cattle scarce, the Khoisan face little threat from malaria. But as agricultural communities encroach on the Khoisan habitat, or more Khoisan take up farming, the population may find itself increasingly vulnerable to infection.</blockquote>

OMG! They're PTC tasters! I'm so happy they sequenced the genome for that; <b>those test strips really put a dent in the budget at $2.50 per hundred!</b>

The information is accurate, but these genotypes just aren't very interesting. We already <i>knew</i> that Duffy O is rare in southern Africa. That's another cheap phenotyping test. Are they under threat from <i>vivax</i> malaria? A little -- but that certainly won't help them when <i>falciparum</i> is the main threat. <I>ACTN3</i> may not have been known already, but isn't surprising at all. 

I think there are two problems here. One: putting out this list of variants really misrepresents the purpose of the sequencing. Genotyping known variants is not the point of these sequences. If it were, the geneticists could have accomplished that with a $400 chip. Finding unknown variants is more worthwhile (and the number of new polymorphisms has been listed in some articles), but we don't know what phenotypic correlations a newly-discovered polymorphism may have. So, we get this catalog of known variations -- written like a little lapidary of genetic gems -- which is just window-dressing for the real science. 

Two: The point of having whole genomes or genome-wide genotypes is that they give us the opportunity to test hypotheses about <i>systems</i> of genes instead of individual genotypes. But for the most part, we don't know how to do that. Certainly on my end, looking for the causes of recent human evolution, we just haven't made great methodological progress on how to study the simultaneous evolution of many genes and their joint effect on human biology. Here in my lab, and elsewhere, we're trying really hard -- and that is to me the exciting part of this science. But we're far from the point where we can predict much from a genome. 

One aspect that hasn't gotten the attention it deserves: These genomes have a tremendous amount to tell us about the Pleistocene history of African populations. This is like an ancient record of thousands of genes, representing this area of the world where only a handful of ancient human fossils exist. Much more on that later. 


