---
layout: single 
title: "The other story about the mammoth DNA" 
category: story
permalink: /weblog/reviews/genetics/non-primate/mammoth-mtdna-phylogeny-gilbert-2008.html
tags: [mammoths, demography, paleogenomics, mtDNA migrations, non-primate, spatial dynamics] 
comments: false 
author: John Hawks 
---

I got to writing about a story a couple of years ago, and then stalled out. That happens every so often -- remember, most of my research-related entries are my own notes. You can only imagine how many half-written posts I have, but the AI on my computer has gotten better and better at archiving them. 

In this case, the half-written post lately has grown in relevance, so I've revisited it. In the summer of 2008, Thomas Gilbert and (many) colleagues reported on a phylogenetic analysis of 18 mtDNA genomes from extinct woolly mammoths. 

That's pretty cool, by the way. <b>We now know a lot more about woolly mammoth mtDNA variation than we knew about human mtDNA variation in 1980</b>. 

The mammoth mtDNA is an example of something slightly different than the usual phylogeography -- it adds the dimension of time. Call it phylotemporogeography, if you like. The best comparison? Neandertals -- a group for which the number of mtDNA sequences is very similar, over a similarly wide Palearctic geographic range. I wrote about Neandertal phylogeography last year (<a href="http://johnhawks.net/weblog/reviews/neandertals/neandertal_dna/fabre-2009-population-structure-mtdna-neandertals.html">"Neandertal races?"</a>), and the topic will surely return sometime this year. 


Different mammoth mtDNA clades, which originated millions of years ago, apparently became extinct at different times. The paper divided the mammoth mtDNA variation into two clades, which diverged approximately 1.7 million years ago. These two clades have different geographic distributions. One, which the authors termed, "clade I," was broadly distributed across Siberia and Beringia. The other, "clade II," appears to have been restricted to one area of Arctic Siberia, between the Taymyr Peninsula and the Lena River. Each of these clades has highly restricted diversity, and taking all the mammoth mtDNA sequences together, they are roughly as diverse as the within-subspecies diversity in living elephants. So that deep branch dividing the two clades accounts for a lot of the restricted diversity within mammoths. 

The interesting thing is that the two clades also have different <i>temporal</i> distributions, based on the radiocarbon dates associated with the remains. The geographically restricted clade II is systematically earlier. The time distributions overlap somewhat, but there is no clade II mtDNA after 30,000 years ago, while clade I lasts up to the extinction of the mammoths in the early Holocene. 

First question: why the deep branch? The simple answer is probably that it's just one of those things. It's difficult to weigh the importance of different parts of the geographic range of mammoths, so I hesitate to guess whether the relatively smaller region of clade II mammoths is "peripheral". It's not at a geographic extreme, but it's hard to judge the migration potential among these regions. 

The region occupied by a minor clade doesn't have to be peripheral or geographically isolated. The oldest branch point in a mtDNA tree is unlikely to be evenly balanced, and given that one clade is likely to be less numerous than the other, it is also likely to be geographically restricted. For all we know, the <i>spatial</i> distribution found among these mammoth mtDNAs is perfectly consistent with neutrality. 

Moreover, given the disappearance of clade II after 30,000 years ago, there aren't very many contemporary sequences that are clade I. We don't really know that they weren't evenly balanced at that time -- nor do we know what mtDNA clades may have been present in the broader range of mammoths across Europe and Beringia (although subsequent papers may have given some information on this). 


Second question: why the replacement of one clade by another? The authors first considered whether the mammoth mtDNA might have undergone a selective sweep: 


<blockquote>All of the observed substitutions appear to be between closely related amino acids. For those proteins having a close homolog with an experimentally determined structure (namely, <i>COX1</i>, <i>COX2</i>, <i>COX3</i>, and <i>Cytb</i>), we also modeled the structure of the mammoth proteins. All substitutions appear in regions on the surface or in loop regions that neither seem essential for proper folding nor would be expected to alter protein function in any obvious way. Therefore, the evidence from the modeled structures suggest [sic] that it is unlikely that the nonsynonymous differences found in the mitochondrial genomes of the two mammoth clades have resulted in any physiological disparities, and thus a selective advantage for clade I based on mtDNA sequence differences alone is not expected (Gilbert et al. 2008:8331).</blockquote>

I think the authors have done as much analysis of this question as possible, given the available data, but I still think this is very weak evidence against selection as an explanation for the clade II extinction. After all, <i>positively</i> selected mtDNA variants are unlikely to change function in a major way -- big changes being much more likely to be bad under the usual Fisher model of adaptation. 

At any rate, the alternative hypothesis is local extinction, taking a geographically-localized clade with it. 

<blockquote>A more likely alternative is that the loss of clade II is a consequence of its restricted geographical distribution, because taxa with small ranges are generally more prone to extinction compared with widespread taxa. It is therefore conceivable that clade II was lost because of a demographic bottleneck resulting in genetic drift or a local population extinction. </blockquote>

This seems contradictory. Given that there are no noticeable phenotypic differences between these clades, and that mtDNA clades I and II <i>coexisted</i> in the Lena-Kolyma region, a purely local demographic bottleneck doesn't make much sense. Now, there are alternatives that retain mtDNA neutrality -- for example, a demographic <i>replacement</i> of the Arctic Siberian mammoths by populations expanding from elsewhere (either east or south). This might have been driven by selection involving other aspects of physiology, enhanced by climate forcing. For instance, a long-lasting locally adapted population might give way to a more generalized form due to climate oscillations. 

Bottom line: mammoths were a dynamic population, capable of high mobility and rapid clade replacements on the scale of tens of thousands of years. And the Late Pleistocene was a time of high population turnover even across what should have been ideal mammoth habitat. That dynamism is not unusual for large, long-lived mammals, and is something we should be looking for in the DNA phylogeography of Late Pleistocene hominins. 



<h4>References:</h4>


<p class="cite">Gilbert MTP and 32 others. 2008. Intraspecific phylogenetic analysis of Siberian woolly mammoths using complete mitochondrial sequences. Proc Nat Acad Sci USA 105:8327-8332. <a href="http://dx.doi.org/10.1073/pnas.0802315105">doi:10.1073/pnas.0802315105</a></p>


