---
layout: single 
title: "A low human mutation rate may throw everything out of whack" 
description: "Whole genome sequencing of a family finds a very low number of mutations, suggesting evolution doesn&#39;t have the timescale we thought." 
category: story
permalink: /weblog/reviews/genetics/divergence/roach-family-quartet-mutation-rate-2010.html
tags: [sequencing, mutation rate, coalescent, chumans, genetic divergence, mutation, 1000 Genomes Project, genomics] 
comments: false 
author: John Hawks 
---

Last week, a paper looking for the genetic causes of Miller syndrome reported the whole genomes of four members of a single family: two siblings with the disorder and their two parents without. The idea was that they would simply compare the affected and unaffected genomes. They would then find candidate loci that might account for Miller syndrome in the affected siblings. By exploiting some other sources of information, they found what they were looking for. Daniel MacArthur covered the story in his post, <a href="http://scienceblogs.com/geneticfuture/2010/03/whole_genome_sequences_dont_al.php#more">"Disease hunting with whole genome sequences: the good news, and the bad news"</a>. 

I got interested in another aspect of the story. With whole-genome sequences of parents and offspring, it becomes possible to <b>directly determine the rate of mutations</b> in each generation. The paper by Roach and colleagues did just that -- they counted 28 in the 2.3 billion bases of sequence they included in their comparison. That makes a per-site mutation rate of 1.1 x 10<sup>-8</sup> per generation. 

Which is a pretty interesting number. You see, it's less than half what it ought to be: 

<blockquote>[O]ur estimated human mutation rate is lower than previous estimates, the most widely cited of which is 2.5 x 10<sup>-8</sup> per generation (10) based on three parameters: a human-chimpanzee nucleotide divergence per site (Kt) of 0.013, a species divergence time of five million years ago, and an ancestral effective population size of 10,000. More recent estimates indicate a nucleotide divergence of 0.012 (9), species divergence time between six and seven million years ago (1115), and ancestral effective population size between 40,000 and 148,000 (1619). With these parameter ranges and a generation length of 15 to 25 years, the mutation rate estimate is between 7.6 x 10<sup>-9</sup> and 2.2 x 10<sup>-8</sup> per generation, which is consistent with our intergenerational estimate of 1.1 x 10<sup>-8</sup>. Our estimate is within one standard deviation (SD) of an earlier estimate of 1.7 x 10<sup>-8</sup> (SD: 9 x 10<sup>-9</sup>) based on 20 disease-causing loci (20). The rate we report is for autosomes, and should be several-fold lower than that of the Y chromosome, as in the male germline more cell divisions occur per generation. Though our rate differs approximately as expected from the recently reported estimate of 3.0 x 10<sup>-8</sup> (95% CI: 8.9 x 10<sup>-9</sup>  7.0 x 10<sup>-8</sup>) for the Y chromosome, the error rates make this difference not significant (21). </blockquote>

You can see the obvious implication: If this mutation rate is accurate, then the average human-chimpanzee <b>gene</b> divergence has to be up around 11 million years ago. That can be accommodated with a 7-million-year-old <b>species</b> divergence only if we assume a very large ancestral population -- on the order of 50,000 or higher. Or, the ancestral effective size could be lower -- but that would make the <b>species</b> divergence substantially older -- 9 million years or more. 

There is a second implication. Most studies of human genetic variation have assumed that 5-million-year-old human-chimpanzee divergence and the high associated rate of mutations. If the true rate is less than half that, then <b>the coalescence times of human genes are more than double most estimates</b>. That would include our estimates of human-Neandertal genetic differences. 

Well, that's a fine pickle. 

I'm not quite ready to believe the very low rate estimate. The analysis in this paper uncovered tens of thousands of false positives, and had to filter through those to arrive at 28 true mutations. The filtering involved resequencing all the positives to determine which were true and which were false, but maybe there's room in there for a substantial number of false negatives, too. 

If this low estimate were true of the human-chimpanzee divergence, it would imply vastly higher ages for other primate divergences, or a much lower rate on the human lineage specifically. So that allows another check on the process. 

But generally, I'll be looking at whole-genome family comparisons with great interest, because they will give us a much more precise understanding of the rate of mutations and recombinations across the genome. 




<h4>References:</h4>

<p class="cite">Roach JC and 14 others. 2010. Analysis of Genetic Inheritance in a Family Quartet by Whole-Genome Sequencing. Science (early online) <a href="http://dx.doi.org/10.1126/science.1186802">doi:10.1126/science.1186802</a></p>


