---
layout: single 
title: "The Denisova mtDNA sequence: The X-Woman" 
category: story
permalink: /weblog/reviews/neandertals/neandertal_dna/denisova-krause-2010.html
tags: [Homo floresiensis, mammoths, Neandertal DNA, paleogenomics, Denisova, Asia, Neandertals, ancient DNA] 
comments: false 
author: John Hawks 
---

In this week's copy of <i>Nature</i>, Johannes Krause and colleagues <bib>Krause:Denisova:2010</bib> report on the complete mitochondrial sequence of a pinky bone from Denisova Cave, in the Altai Mountains of Siberia. 

You might expect this sequence would look like a Neandertal. After all, two other specimens from a little further to the West have both produced mitochondrial sequences very similar to those of Neandertals from Europe. 

But you would be wrong. This sequence turns out to be a surprise.

Instead of falling within the Neandertal clade, the sequence in this pinky bone lies as an outgroup to Neandertals and as an outgroup to modern humans.

<blockquote>Assuming an average divergence of human and chimpanzee mtDNAs of 6 million years ago, the date of the most recent common mtDNA ancestor shared by the Denisova hominin, Neanderthals and modern humans is approximately one million years ago (mean = 1,040,900 years ago; 779,3001,313,500 years ago, 95% highest posterior density (HPD)), or twice as deep as the most recent common mtDNA ancestor of modern humans and Neanderthals (mean = 465,700 years ago; 321,200618,000 years ago, 95% HPD) (Fig. 3). Although the absolute dates depend on several assumptions and are subject to uncertainty (Supplementary Information), the fact that the divergence of the Denisova hominin mtDNA is about twice as old as the divergence of Neanderthal and modern human mtDNAs is robust to most assumptions (Krause et al. 2010: 2).</blockquote>

If you are sharp-eyed, you may notice that mean value from the Neandertal-human comparison, at 465,700 years ago, is rather substantially lower than has previously been reported -- Green and colleagues <bib>Green:2008</bib> put this divergence at 660,000 years ago. Including the new Denisova specimen into the comparison provides a much more recent branch point than the human-chimpanzee divergence date. That means some of the ambiguity in the long branch between the chimpanzees and the human-Neandertal ancestor can be resolved, effectively pushing the Neandertal a little bit closer to us.

As you might have guessed from the paper's title, the authors interpret the deep divergence of the new Denisova sequence as evidence of a previously unknown, "genetically distinct" lineage of hominins. I want to be very precise about what they say and don't say, because it is a very short paper. Nowhere in the paper do they use the word "species". But in the conclusion, they do discuss lineages and "forms".

<blockquote>We note that the stratigraphy and indirect dates indicate that this individual lived between 30,000 and 50,000 years ago. At a similar time individuals carrying Neanderthal mtDNA were present less than 100 km away from Denisova Cave in the Altai Mountains, whereas the presence of an Upper Palaeolithic industry at some sites, such as Kara-Bom and Denisova, has been taken as evidence for the appearance of anatomically modern humans in the Altai before 40,000 years ago. Although these dates are associated with large and unknown errors, this temporal concurrence suggests that complete and successive replacements of distinct hominin forms, similar to what occurred in Western Europe, may not have taken place in southern Siberia. Rather, representatives of three genetically distinct hominin lineages may all have been present in this region at about the same time. Thus, the presence of Homo floresiensis in Indonesia about 17,000 years ago and of the Denisova mtDNA lineage in southern Siberia about 40,000 years ago suggest that multiple Late Pleistocene hominin lineages coexisted for long periods of time in Eurasia. </blockquote>

The mention of <i>Homo floresiensis</i> in this conclusion seems unlikely to be an accident, particularly in <i>Nature</i>, the hobbits' birthplace. I haven't seen any press coverage of this yet, obviously, as I'm writing before the embargo breaks. But I can only imagine the likely spin: just as <i>Homo floresiensis</i> has demonstrated the diversity of archaeologically recent hominins in Asia, this new mitochondrial sequence adds even more to that diversity.

<b>One of my long-time correspondents is already calling it "the Yeti".</b>

<h4>Is this a new species?</h4>

As my students have heard me say many, many times, gene trees are not species trees. The different genetic loci within a population have diverse genealogies. Often, when two populations diverge from each other, their gene genealogies will show similar patterns of divergence. But not always.

When we look within a single population, gene genealogies are likewise diverse. but within a single population, there is no population divergence. There must be an oldest branch point in the genealogy of any single gene. Here's a question: how many individuals do you have to sample so that you are sure you will find this deepest branch point? The answer to that question depends on the frequencies of the lineages on either side of that branch. If one of them happens to be rare, you're unlikely to find it unless you sample lots and lots of individuals.

And if the population is spread across a substantial amount of space, it is very likely that one of the clades will be geographically limited compared to the other.

Put these two things together, and apply them to a widespread population like the Neandertals. It is pretty likely that if we sample a dozen Neandertals across a subset of their range, that we will miss the deepest divergence in the genealogy of a single gene. That may be what has happened here. By extending the known mitochondrial sample of Neandertals even further to the east, this study may have discovered a deeper branch point than was previously known within the Neandertal population.

Indeed, a million-year-old clade divergence would be entirely normal for a large mammal. That's what we see in chimpanzees, and <a href="http://johnhawks.net/weblog/reviews/genetics/non-primate/mammoth-mtdna-phylogeny-gilbert-2008.html">as I pointed out yesterday</a>, it's smaller than the clade divergence we see among mammoth mtDNA across a similar time range and geographic extent. 

I think the mammoth paper makes a really nice comparison to this one. In that case, they discovered a deep clade divergence in an ancient population, one branch of which was geographically restricted within a part of northern Siberia. They didn't conclude that multiple species of mammoths had been sampled -- despite the fact that one mtDNA lineage significantly outlasted the other. That was variation within one geographically diverse species, consistent with what we know about other species' mtDNA variation. 

So it is unnecessary to posit the existence of an unknown species of hominins in southern Siberia, based on the mitochondrial evidence alone. Whether we're talking about an unexpected diversity of forms -- well, I want to see something other than a pinky bone. 

<h4>Does it add to our understanding of Neandertal phylogeography?</h4>

Well, first we need to know if it's a Neandertal. We don't. It's a pinky bone. 

But if it were a Neandertal, then the appearance of a deep clade at the very eastern extent of the population's range might suggest something about its diversification. The western Neandertals in that scenario have relatively restricted diversity, as if they had descended from a recent mtDNA ancestor. That pattern would be consistent with a range expansion from the east to the west. So maybe the Late Pleistocene Neandertals invaded Europe from elsewhere? 

<h4>Could this be <i>Homo erectus?</i></h4>

Of course, at the very furthest eastern extreme of the Neandertal range, we might well be running out of Neandertals and running into another kind of hominin. Even as recently as 40,000 years ago, it is not entirely obvious who those hominins would have been. The archaeological transition is nowhere near as clear in the east as in Europe, and even in Europe the archaeological transition to Upper Paleolithic industries is not the same as the biological transition.

Before  100,000 years ago, the humans in China could plausibly be assigned to <i>Homo erectus</i>. It seems likely that much, if not most, of the genetic heritage of the pre-40,000 year population of China would have been derived from these ancient Chinese hominins. It is unknown how much genetic exchange there would have been between east and west at this time. I suspect that there were substantial genetic exchanges, both along the southern coast of Asia and across Central Asia. So China might well provide an alternative geographical origin for this mitochondrial clade.

If we look to China as the ancestor population for this mitochondrial sequence, we can ask whether the roughly million year divergence date makes sense. As a marker of populations, a single gene can inform us about the maximum time of population divergence, not the minimum. The minimum is in effect zero: in other words, a million-year-old divergence genetically could occur within a single human population. So a widespread human population across much of Asia could contain such a deep branch, just as Neandertal's -- equally widespread across West an and Central Asia -- could have contained such a branch.

But a million-year-old divergence does tell us one thing: this cannot represent a <i>Homo erectus</i> population that originated in Africa 2 million years ago, colonized Asia around the time of Dmanisi, and was isolated after that time.

In other words, it would argue strongly against the hypothesis of a deep divergence of eastern and western hominin species, starting with the initial dispersal of humans from Africa in the Early Pleistocene. It argues in favor of continued genetic exchanges or a more complex history of population movements. 

I hesitate to take this line of reasoning too far. It's a pinky bone. 


<h4>Could this be a modern human?</h4>

Even though the date of the cave could be as recent as 30,000 years ago, it is very unlikely that this mitochondrial sequence would have occurred within the growing population of "modern" humans. A growing population is relatively unlikely to lose mitochondrial variants. An ancient clade like this one, which survived in the population for a million years, might have been just at the edge of extinction at the time the population started to grow and therefore might just have missed its opportunity to survive. But it seems sort of unlikely.




<h4>Do they know more than they are letting on?</h4>

In the back of my mind I'm thinking this: if Krause's team has done enough sequencing to do the entire mitochondrial genome, they surely already know something about what the nuclear genome looks like. The increasing success of DNA recovery from these very fragmentary fossils has been stunning over the last several years. It is incredible that we are likely to recover a substantial amount of autosomal sequence from the distal phalanx of a (did I mention?) pinky. A quick comparison against raw data, without much systematic analysis, would be enough to check the mtDNA result. 

I wonder if this is only the first shoe, and there is another left to drop? These guys know as well as I do the gene trees are not species trees, and that such an obvious point that -- even though this is <i>Nature</i> we're talking about -- the reviewers should have caught it.

So maybe there are already hints that the autosomal comparison will fall in the same direction as the mitochondrial comparison with Neandertals: different from them, different from us.

Maybe it's a Yeti after all.


UPDATE (2010-03-24): Man, the press is worse than I imagined. <i>Nature</i>'s news article goes fully with the "new species" interpretation -- <b>even though the paper itself does not include the word "species"</b> -- and every other outlet I've seen is following suit. 

I have to teach my class this afternoon where we'll be talking about this mtDNA sequence, so I don't have time for a longer update. Let me say very clearly: <b>nothing about this sequence requires there to have been an undiscovered hominin species.</b>

UPDATE (2010-08-10): References updated.



