---
layout: single 
title: "Ancient genomes review" 
description: "A new review article by Mark Stoneking and Johannes Krause presents some useful information." 
category: story
permalink: /weblog/reviews/neandertals/neandertal_dna/stoneking-krause-2011-review.html
tags: [Denisova, ancient DNA, paleogenomics, Neandertal DNA] 
comments: false 
author: John Hawks 
---

Mark Stoneking and Johannes Krause present a review article in the current <i>Nature Reviews Genetics</i> <bib>Stoneking:Krause:2011</bib> that gives an overview of the science of ancient genomes. 

I think the article is very good about presenting aspects of ancient genome sequencing and assembly, and the attendant problems and biases. I find myself explaining this stuff a lot and it's useful to have the concise descriptions that Stoneking and Krause provide here. For example, here's a paragraph that describes mapping bias: 

<blockquote>However, there are important limitations to current approaches to ancient genome assembly owing to the short length of ancient DNA fragments and the repetitive nature of large parts of mammalian genomes (which creates ambiguities in sequence read mapping). For example, short fragments can cause mapping bias, as highly divergent short fragments cannot be accurately mapped to a reference genome. Fragments may also map to different locations in different reference genomes depending on the completeness and accuracy of the reference genomes. For example, to calculate divergence times between an ancient hominin genome sequence, modern humans and chimpanzees, it is important to first verify that the ancient DNA sequences map to orthologous positions in both the human and chimpanzee genomes. These issues mean that even at 20-fold coverage (which was the coverage obtained for the Saqqaq genome) not more than 85% of the genome could be reconstructed; full genome sequences from fossil samples can probably never be achieved with current methods.</blockquote>

The article discusses chemical changes in ancient genomes, methods to detect contamination, and specialized methods such as targeted DNA hybridization capture. 

I'm less happy with the second half of the article, which discusses population genetics. A few computational techniques are very briefly described (for example, unsupervised versus model-based approaches) and Stoneking and Krause give quick synopses of some population genetic inferences reported during the last year. 

I guess where I perceive a difference between the first (sequencing) and second (population genetics) parts of the article, is that the sequencing part emphasizes the many problems with analysis and describes approaches to overcome them. It seems as if there's a vibrant discussion of sequencing and biochemistry, giving rise to a fuller account. Meanwhile, the second part, discussing human population history, seems to accept results relatively uncritically. There is very little citation of anthropological or archaeological work, and little indication that the methods of population genetic inference may have weaknesses or assumptions that color their results. 

It's great to see review articles on this topic, given the broad interest I expect we'll see more of them soon. A flood of ancient genetic data means a lot of new results that need to be summarized. But a summary is really not enough -- we need critical examination of the assumptions underlying population genetic inferences and a discussion of how they accord with what we know from archaeology and paleontology. 

