---
layout: single 
title: "Denisovan DNA in the islands, and an Australian genome" 
description: "The hypothesis of a single out-of-Africa dispersal is rejected by new data about Denisovan mixture and whole-genome sequencing of an Aboriginal Australian." 
category: story
permalink: /weblog/reviews/denisova/reich-2011-denisova-asia.html
tags: [population expansion, founder effect, introgression, Denisova, migrations, Neolithic, Philippines, East Asia, Australia, China, New Guinea, gene flow, Asia] 
comments: false 
author: John Hawks 
---

David Reich and colleagues today report on the persistence of Denisova-like ancestry in island Southeast Asia and Australia (citation not yet available). Meanwhile, Morten Rasmussen and colleagues (citation not yet available) report on the whole-genome sequencing of hair from an Aboriginal Australian who lived some 100 years ago. 

The most obvious story: These data utterly destroy the hypothesis of a single out-of-Africa colonization of Southeast Asia by modern humans. Many human geneticists have argued our present pattern of diversity originated in a wave of successive founder effects coming from a single recent African origin. They were wrong.

Instead, we can turn to a complex model with successive dispersals and episodes of population mixture. This is not a static model of isolation-by-distance; it is a dynamic model in which populations grow and spread across large spans of the Old World, again and again and again. By my count, at least three massive episodes of population dispersal and mixture are necessary in Reich and colleagues' model. A picture of their admixture hypothesis: 

<div class="middle-picture">
<img src="/graphics/denisova-admixture-model-reich-2011.png" alt="Denisova admixture model from Reich et al. 2011" />
</div>

This model depicts (a) an early divergence of an African (represented by Yoruba) and Asian/Australasian populations. These mix with first Neandertals and then (for the Australian/New Guinea/Mamanwa populations) with Denisova-like people. Later (b), after the initial habitation of the Philippines by the ancestors of Mamanwa, a population like Andamanese Onge pushes into the islands, mixing with the ancestors of New Guinea and Australian populations. Later still (c), a population ancestral to today's Chinese people mixes with Philippines and other Southeast Asian people. 

As complicated as it looks, even this model must be a vast oversimplification. I don't like or attribute much belief to mixture models like this, as they assume too much about relative population sizes and the timing of mixture. Many recent hunting and gathering populations of Southeast Asia are not included in the current samples, and the Chinese sample is itself the result of very recent demographic events, covering what once may have been a wider diversity of peoples. Depicting Australian and New Guinean populations as monolithic is an artifact of the small sample; these places themselves housed a tremendous diversity of peoples. Nevertheless, the true model won't be <em>simpler</em> than this one; it will involve many more events that the data cannot yet resolve. 


Hints of that complexity emerge from the Aboriginal Australian whole genome. Rasmussen and colleagues show that this individual shares some ancestry with East Asian peoples, but on the whole populations in Europe and East Asia are much more genetically similar to each other than to this genome. The picture from the whole genome is essentially the same as that drawn by the SNP comparisons by Reich and colleagues, but with the potential (in the long run) to actually trace the histories of individual genes. And I think the gene-by-gene account of history will be important, because we already have some evidence that a few Denisovan genes do persist in mainland Asia, even though most are gone. 

To explain why, we can look at the proportion of Denisovan ancestry in different populations as depicted in a map by Reich and colleagues. The pie charts are confusing here, because they report the fraction of ancestry from Denisovans in each population <em>relative to the 5% estimate for New Guinea</em>. So Australians also have 5% in this figure, Timorese have around 2.5%, and Bougainville has more than 4%. 

<div class="middle-picture">
<img src="/graphics/denisova-snp-proportions-reich-2011.png" />
</div>

Notice the apparent lack of Denisovan ancestry in anyone who lives anywhere that was once connected by land with mainland Asia. I say "apparent" deliberately: Abi-Rached and colleagues reported last month on the widespread distribution of Denisovan HLA types among today's Asian populations, and those may well be products of Denisovan genes that were later selected. I've already identified a handful of other loci that seem to reflect Denisovan ancestry in mainland Asian people. According to the comparisons by Reich and colleagues, such loci must be exceptions. 

At the same time, the mixture model presents an important idea: Once there were people in Southeast Asia who had much more Denisovan ancestry than any populations still remaining today. Both Australian/New Guinea populations and Philippine populations like the Mamanwa have subsequently mixed with new immigrants who lacked any sign of Denisovan ancestry. Prior to this later mixture, the ancestors of those populations must have been more Denisovan -- Reich and colleagues estimate 7%. This is the first evidence that ancestry from archaic people of Eurasia was diluted to a lower value by later population movements. If the population mixture originally happened somewhere in mainland Asia, any traces of Denisovan ancestry in those areas has been diluted almost to nonexistence. But the persistence of some genes would be predicted if natural selection were maintaining them in the face of demographic pressure from elsewhere. 

About the Australian genome, there will be much more interesting analyses to come, I expect. As whole-genome data come to represent more of the variation within human populations, we get a larger store of information about how we came to be variable. Variation traces not only to population movements and demography, but also to natural selection. Australia's population history has been very different from many populations of the Old World, and this genome should give us new perspective on the effects of that demographic history. 






