---
layout: single 
title: "The immune systems of archaic humans" 
description: "We&#39;re gathering the first information on the immune systems of ancient humans. Some challenges await." 
category: story
permalink: /weblog/reviews/neandertals/neandertal_dna/hla-parham-2011.html
tags: [HLA, immune, Neandertal DNA, Denisovans] 
comments: false 
author: John Hawks 
---

I've just submitted an abstract for a conference in the fall, with the title, "Immunogenetics of archaic humans." 

Ten years ago, it would have been beyond imagining that this kind of science would be possible. Now, my graduate student Aaron Sams has been working directly with HLA and other immune system genes in ancient DNA sequences. It's pretty tough to work with the HLA region because of the low coverage of the ancient genomes and the high variation and repetitiveness of the HLA. But it is possible to find some of the basic human alleles in the ancient sequences, and those open the possibility of examining the coevolution of pathogens and human immunity in our recent evolution. 

Turns out we're not alone: According to <i>New Scientist</i>, Peter Parham has also been looking at HLA in archaic humans: <a href="http://www.newscientist.com/article/mg21028174.000-breeding-with-neanderthals-helped-humans-go-global.html">"Breeding with Neanderthals helped humans go global"</a>. 

<blockquote>One allele, HLA-C*0702, is common in modern Europeans and Asians but never seen in Africans; Parham found it in the Neanderthal genome, suggesting it made its way into H. sapiens of non-African descent through interbreeding. HLA-A*11 had a similar story: it is mostly found in Asians and never in Africans, and Parham found it in the Denisovan genome, again suggesting its source was interbreeding outside of Africa.</blockquote>

HLA-A*11 is actually the most common allele of HLA-A in Papua New Guinea, the population that otherwise shows significant evidence of ancestry from a Denisova-like genome. However, I don't agree with the main idea of the article. The major human HLA alleles are evolutionarily ancient -- most of them predate the origins of modern human groups and are older than the founding of the Denisova-Neandertal populations. This is actually  perhaps the worst region to look for evidence of interbreeding among these populations because the probability of incomplete lineage sorting (maintained by balancing selection) is very high. 

As a case in point, HLA-A*11 is very common in Papua New Guinea, but it is also very common in north India and in China. These two areas otherwise show no significant evidence of Denisova ancestry. We might conclude that the HLA-A gene just has an unusually high level of introgression into Asian populations, not typical of the genome as a whole. That's certainly possible. But without finding any substantial number of derived mutations in the HLA-A*11 variant in the Denisova genome and in living Asians, it is hard to rule out that the sharing of HLA-A*11 in all these populations is just coincidence. 

Of course, if the allele were absent in Africa, that would weigh in favor of the idea it is shared by Late Pleistocene interbreeding outside Africa. But HLA-A*11 <i>is</i> in Africa, just very rare. And it's in Europe. This is the kind of locus that is difficult to interpret: if it has any tiny disadvantage against malaria, for instance, its rarity in Africa is easily explained as a function of recent evolution, while its presence almost everywhere outside Africa would be no surprise even if there were never any interbreeding. This is not a case where the geographic distribution is an unusual coincidence -- it's present in Africa and relatively more common everywhere outside sub-Saharan Africa. So the distribution outside Africa cannot simply be explained by interbreeding with Denisovans -- not without selection -- leaving us stuck. Parham's hypothesis may be correct, but the data are really not sufficient to decide. 

HLA-C*07:02 -- the one apparently mentioned in the story -- is all over sub-Saharan Africa at low frequencies. <a href="http://www.allelefrequencies.net/hla6006a.asp?hla_locus_type=Classical&hla_locus=&hla_allele1=C*07%3A02&hla_allele2=C*07%3A02&hla_selection=&hla_pop_selection=&hla_population=&hla_country=&hla_dataset=&hla_region=Sub-Saharan+Africa&hla_ethnic=&hla_study=&hla_order=order_1&hla_sample_size_pattern=equal&hla_sample_size=&hla_sample_year_pattern=equal&hla_sample_year=&hla_level_pattern=equal&hla_level=&hla_show=">Allelefrequencies.net</a> has a dozen entries for the frequency of HLA-C*07:02 in sub-Saharan Africa, they all have it at frequencies up to around 7 percent (except for the small (n<15) samples where it is actually higher!) I can't understand the assertion that it isn't in Africa. There must be some miscommunication. 

What about the question of hybrid vigor that the article raises? Is it possible that modern humans got HLA mojo from Neandertals and Denisovans? 

<blockquote>While only 6 per cent of the non-African modern human genome comes from other hominins, the share of HLAs acquired during interbreeding is much higher. Half of European HLA-A alleles come from other hominins, says Parham, and that figure rises to 72 per cent for people in China, and over 90 per cent for those in Papua New Guinea.</blockquote>

I just don't think it's clear that these HLA alleles in humans have actually come from the archaic genomes. 

We've tried to match these at more precise levels (in the HLA system, that would be four- or six-digit haplotypes) and have not found the quality of the data high enough to manage a close match. That leaves us with the most superficial classification, which isn't enough to argue that the present human types are derived from the archaic genomes. Incomplete lineage sorting remains a good explanation for the similarities. In fact, we're thinking it makes a nice case study of just how hard it is to work with these genomes, which have lower than 2x coverage. Just typing the Denisova genome requires an assumption about whether the individual was a homozygote or heterozygote across the locus -- an assumption that we can test easily with higher coverage, but not so much with 1x and many gaps. It also requires greater trust in the mapping quality of the reads than we probably should have. With those caveats, the match to HLA-A*11 is likely but not totally solid. Saying that HLA-A*11 in modern humans <i>came from Denisovans</i> is simply premature. And while I've focused here on HLA-A, this is also true of all the other loci. There's a tipping point at higher coverage where typing becomes more secure, and the archaic data are not there. 

Anyway, I imagine that anyone typing HLA in whole genome data knows all this. The press account isn't going to go into the complexity, and I think it's worth noting the real difficulty of making inferences in this region of the genome on the archaic data. It's a tough problem and I've spoken to many human geneticists who thought we were foolhardy to start. But with the first information about the immune systems of archaic humans as the goal, you can see it's a worthwhile problem to tackle. 

