---
layout: single 
title: "Archaic genome snooping from GWAS" 
category: quickbit
permalink: /weblog/reviews/neandertals/neandertal_dna/gwas-diabetes-south-asia-2011.html
tags: [GWAS, type 2 diabetes, Denisova, health, Neandertal DNA] 
comments: false 
author: John Hawks 
---

The 23andMe blog reports on a recent genome-wide association study of type 2 diabetes in South Asian people: <a href="http://spittoon.23andme.com/2011/10/18/snpwatch-genetic-variants-associated-with-type-2-diabetes-in-south-asians-and-europeans/">"SNPWatch: Genetic Variants Associated with Type 2 Diabetes in South Asians and Europeans"</a>. The study was published in August in <em>Nature Genetics</em>, by Kooner and colleagues <bib>Kooner:2011</bib>. As described in the post: 

<blockquote>The authors behind this study carried out one of the largest type 2 diabetes studies to date, scanning the genomes of nearly 19,000 people with the disease and 40,000 without it, all of South Asian descent. Their analysis identified six SNPs linked to this condition. When they combined their results with previously published findings in other ethnicities, they found suggestive evidence that five of the six SNPs were also associated with type 2 diabetes in European populations. Similarly, there was some evidence that the majority of the genetic risk factors in Europeans were also linked to disease in South Asians. Only three genetic factors were not shared at all between the two groups.</blockquote>

Type 2 diabetes is presently a very interesting topic from an evolutionary viewpoint, and we're beginning to think about it very seriously now. Whenever I see a study like this, I quickly look at the Neandertal and Denisovan genomes to see if any interesting patterns emerge. Sharing GWAS SNP alleles is not necessarily very interesting, because the GWAS risk alleles are mostly not causative themselves; each may be linked to some causative allele that remains to be discovered. The linkage is a function of the evolutionary history of that chromosome region, and many of the key historical events that affect linkage happened within the last 10,000 years. So we really shouldn't expect GWAS alleles to be predictive of phenotypes in Neandertals or Denisovans. 

Still, these alleles are associated with disease in living people, and their genotypes in ancient humans may illuminate cases where the evolutionary history links the population across the gene networks that influence disease. A closer examination of the genealogy around these loci will be more informative, but as a first look I often just genotype the archaic genomes for SNPs in a study. The six SNPs reported here include two cases where the archaic genomes have the derived risk alleles, one of them present in Neandertals but not the Denisova genome. Again, that doesn't tell us anything about the phenotype of the ancient people, but worth a closer look to see if one or both of these is an introgressive allele. 

We have here the GWAS Catalog genotypes for all the archaic genomes. Not much actionable information but there are some interesting phenotypes in there. I'll share some more of those later this week. 



