---
layout: single 
title: "A longer timescale for human evolution" 
description: "A commentary on a new paper describing the effects of longer generation time estimates in chimpanzees and gorillas. " 
category: research
permalink: /research/langergraber-generation-time-comment-2012.html
tags: [chimpanzees, mutation rate, John Hawks, mutation] 
comments: false 
author: John Hawks 
---


During the last few years, the best estimates of the human single nucleotide mutation rate have been cut in half. Until recently, estimates of mutation rate have relied on counting substitutions between primate species and assuming that fossil relatives of living species can accurately pin dates onto phylogenetic branches. This procedure allows very precise estimates, but introduces systematic bias toward higher substitution rates and longer branch lengths because a new lineage can leave a fossil record only after its origin, never beforehand <bib>Steiper:Young:2008</bib>. Now, widespread resequencing, initially of de novo Mendelian genetic disorders <bib>Kondrashov:rate:2003</bib>,<bib>Lynch:spectrum:2010</bib> and later of whole genomes in parent-offspring trios <bib>Roach:whole:2010</bib>, has allowed direct comparisons of parent and offspring genomes. The most commonly-used but now outdated estimate of the mutation rate was 2.4 times 10^-8 changes per nucleotide per generation <bib>Nachman:Crowell:2000</bib>. Current estimates of the same value based on resequencing data are much lower, around 1.1-1.28 times 10^-8 <bib>Roach:whole:2010</bib>, <bib>Lynch:spectrum:2010</bib>. 

Langergraber et al. (this issue) <bib>Langergraber:2012</bib> seal one of the remaining holes in this emerging understanding, by providing the most accurate estimates of generation length yet possible for wild chimpanzees and gorillas. They determined the parentage of chimpanzees and gorillas in wild study populations, which in concert with field data on births allows an accurate measure of the mean generation length. Chimpanzees average more than 24 years per generation; gorillas more than 19, substantially longer than indicated by earlier life history assessments <bib>Teleki:1976</bib>. Long generations, with few genetic mutations in each, mean that the clock of genetic substitutions has ticked very slowly during the evolution of humans and apes. 



<h4>Breathing easier</h4>

Some paleoanthropologists will welcome the new, slower mutation rate. For twenty years, they have been unearthing Late Miocene fossils that purport to represent the lineage leading to recent hominins. Candidates including the 7-million-year-old <em>Sahelanthropus tchadensis</em>, 6 million-year-old <em>Orrorin tugenensis</em>, and 5.5-million-year-old <em>Ardipithecus kadabba</em> vie for a place in our ancestry. Genetic comparisons once pegged the human-chimpanzee common ancestor as recently as 4 million years ago, pruning these fossil limbs out of our family tree <bib>Wildman:2003</bib>. As Langergraber et al. report, a slower rate places the human-chimpanzee common ancestor more than 7 and possibly as early as 13 million years ago, reopening the case for these and other fossils. 

A longer timescale has many other consequences. The 10.5-million-year-old <em>Chororapithecus abyssinicus</em> may really be an early member of the gorilla lineage, as its dental anatomy suggests <bib>Suwa:Chororapithecus:2007</bib>. For the orangutan lineage, the prospect of a much deeper genetic estimate of divergence illuminates the relation between phylogenetics and population genetics. Genetic divergence between two species is a function not only of the time that the species became isolated, but also of the genetic variation within their ancient common ancestral population. Whole-genome analysis of apes and humans has uncovered abundant evidence of complex population structure in the common ancestors of living species <bib>Seipel:2009</bib>. Hobolth et al. <bib>Hobolth:2011</bib> assessed incomplete lineage sorting of orangutan similarity in human and chimpanzee genomes, showing that the ancestral population of the orangutans and African apes must have been large and diverse. A fast mutation rate and this complex ancient structure made the origin of the orangutan branch uncomfortably recent, only 9-13 million years ago, barely old enough to accommodate the earliest known orangutan-like fossil evidence, the 12.5-million-year-old <em>Sivapithecus indicus</em>. A slower mutation rate appears to be a better fit to both fossil evidence and the genetic structure of this ancient population. 


<h4>Beyond branches</h4>

The genomes of the African apes and humans have opened a new way of studying population history. In addition to the cladistic relations among species plodding along phyletic branches, we can now test hypotheses about the diversity and structure of dynamic populations. We depend on accurate estimates of mutation and recombination to examine introgression, partial population replacement, continuing gene flow and changes in population size. A slower mutation rate demands that we revisit the population histories of humans and our close relatives. The histories of the present subspecies of chimpanzees may go back to nearly a million years ago. As Langergraber et al. show, the genetic differences between western and eastern gorillas may be 1.5 million years or older. A longer timescale shows that the present subspecies of primates have survived multiple episodes of climate change in tropical Africa, events should also have shaped human evolution in complementary ways. More interesting, the depth of gene genealogies in these primates may reflect ancient episodes of partial population replacement and introgression. 


Along these lines, genomes from Neandertals and from Denisova Cave <bib>Green:draft:2010</bib>,<bib>Reich:Denisova:2010</bib> demonstrate the complexity of human population history. Ten years ago, many scientists argued that the population history of living humans converged to a recent strong bottleneck in a single African population. Today we work to refine a richer and more complex model with multiple episodes of dispersal, genetic differentiation and introgression. So much is left for us to discover as we are far from achieving the full potential of billions of base pairs of new data. 


A mere two years ago, genomic evidence from Neandertals suggested that they had originated within the last 270,000-440,000 years <bib>Green:draft:2010</bib>. This troublesome date excludes specimens that have appeared to be strong candidates for Neandertal ancestors, including the large sample of skeletal remains from Sima de los Huesos, Atapuerca, Spain, possibly more than 530,000 years old. Now the maximum value for Neandertal-human common ancestry from 2010 seems instead closer to a minimum date. Langergraber et al. suggest a range from 420,000-780,000 years, bringing much of the Middle Pleistocene record of Europe into the scope of Neandertal ancestry. 


<h4>Moving out</h4>

Across this same timescale, the archaic ancestors of today's Africans had already developed an intricate population structure. Genomic investigation of African hunter-gatherers has opened new windows onto this deep genetic history of differentiation and introgression <bib>Lachance:2012</bib>, <bib>Hammer:archaic:2011</bib>, bringing the origin of modern African diversity into the population structure of the early Middle Pleistocene. A simple hypothesis of modern human origins in a bottlenecked population cannot account for this diverse genetic history. 


The mitochondrial DNA timescale now poses a hanging question. Mitochondrial mutations occur much more often than nuclear DNA mutations, with greater heterogeneity among sites <bib>Soares:2009</bib>. Still, our estimate of mtDNA substitution rates depends on our estimates of branch lengths of the primate phylogeny. Up to now, mitochondrial comparisons have been the strongest evidence in favor of a short timescale for the dispersal and differentiation of non-African peoples, within the last 70,000 years <bib>Endicott:2009</bib>. Some recent attempts to examine the relationships of non-African populations using nuclear genome data have led to timescales in excess of 100,000 years <bib>Gutenkunst:2009</bib>, others favor more recent estimates <bib>Lukic:Hey:2012</bib>. Despite the recency of this work, most authors have continued to use outdated fast molecular clock and short generation time estimates. As we move forward, such results will need to be corrected or adjusted to enable comparisons with current work.


<h4>A common language</h4>

It may seem surprising that such a basic parameter as the mutation rate could have been inaccurately estimated for so long. An accurate per-genome estimate of mutation rate depends on large amounts of sequence data, observed for a large number of parent-offspring pairs. Whole genome sequencing has become very widespread during the last two years, but low-coverage genomes have a high rate of false positive changes, which have delayed acceptance of the lower rate estimates. Stronger evidence about mutation rate comes from the even broader sample of parent-offspring trios from surveillance of de novo Mendelian diseases <bib>Lynch:spectrum:2010</bib>. These values will be subject to continuing refinement, as geneticists add more and more primate and human genomes and closer examination of their biology. 

Sampling DNA from other primates effectively collates thousands of generations of time into a single comparison, allowing the substitution rate to be estimated from relatively short DNA sequences. For mutations not under selection, the substitution rate estimates the mutation rate very precisely. 

But precision is not accuracy. Radiometric ages are often very precise, and paleontologists can constrain the provenience of some Miocene primate fossils to ranges less than a hundred thousand years. Accuracy about the time of speciation would require evidence the fossil record can never provide. We cannot say how many orangutan ancestors may have lived before the 12.5-million-year-old <em>Sivapithecus indicus</em>; we can only hope to discover more of them. Fossils have limited value even as minimum estimators of speciation time. Steiper and Young <bib>Steiper:Young:2006</bib> estimated a relatively slow rate of mutations in primates, by assuming that a series of fossils represent minimum ages for various phylogenetic branches of primates. Their slow rate estimate depended upon placing the 7-million-year-old <em>Sahelanthropus tchadensis</em> as a member of the hominin lineage, an assumption that has been challenged on morphological grounds <bib>Sahelanthropus</bib>. This challenge could not necessitate a higher mutation rate, but could delay acceptance of a slower rate. A slow mutation rate does not settle the phylogenetic position of <em>Sahelanthropus</em> or other fossil specimens, it merely refocuses study upon anatomical and ecological evidence. 

Mutation rates estimated from pedigree and phylogenetic data may still prove to be significantly different, as they are for mitochondrial DNA <bib>Soares:2009</bib>. The nuclear mutation rate varies among sites and regions (e.g., CpG nucleotides) <bib>Subramanian:Kumar:2003</bib>, and discovery of functional elements will bring to light some amount of previously unrecognized purifying selection. The average mutation rate across the genome is only a starting point. Still, as genomes have begun to reveal the kind of complexity long evidenced by the fossil record, we can begin to seek a new anthropological synthesis that ties together genomes, morphology, and life history data. 

















