---
layout: single 
title: "Taking the mtDNA pulse of Neandertal populations" 
description: "Neandertals in western Europe have a recent mtDNA ancestor, pointing to the dynamics within their population." 
category: story
permalink: /weblog/reviews/neandertals/neandertal_dna/dalen-2012-neandertal-diversity.html
tags: [Neandertal DNA, population dynamics, Neandertals, migration, mtDNA, phylogeography] 
comments: false 
author: John Hawks 
---

Neandertals have strikingly limited genetic variation. They once lived across a range from Spain to Siberia. Yet when we compare sequences across their whole genomes, we find them to be much less different across this geographic range than people living in the same regions today. 

I think this is one of the most fascinating findings of ancient genomics. It may tell us something about Neandertal populations that we did not begin to suspect without their DNA. 

But there is one explanation for this fact that I and others pointed out long before DNA evidence: The Neandertal population was surely much, much smaller than Holocene population of Europe. Small population size over a long time can restrict genetic diversity. So maybe the Neandertals preserved little genetic variation simply because there were so few of them.

<h4>Neandertal mtDNA dynamism</h4>

Love Dal&eacute;n and colleagues <bib>Dalen:2012</bib> add some perspective to this question. The paper adds one novel mtDNA sequence, of the Neandertal from Valdegoba, Spain, to the record of Neandertals. This builds on earlier work by Ludovic Orlando and colleagues, who performed some analysis of Neandertal variation over time when they reported the sequence of the 100,000-year-old Scladina mtDNA sequence <bib>Orlando:2006</bib>. The main contribution of the current paper is its separation of Neandertals into earlier and later subsamples, showing that the Neandertals after 48,000 years ago in Western Europe have greatly restricted mtDNA diversity compared to the earlier sample of Neandertals. 

That's a tricky comparison. The paper illustrates it with this figure: 

<div class="middle-picture">
<img src="/graphics/dalen-neandertal-mtdna-tree-2012.png" alt="Neandertal mtDNA phylogeny from Dalen et al. 2012" />
<p class="caption">Figure 1 from Dal&eacute;n et al. 2012. Original caption: "Figure 1. Phylogenetic relationships and geographic distribution of Neandertals. Recent (&lt;48 kyr) western Neandertals are placed within a well defined monophyletic group (blue box), whereas specimens older than 48 kyr constitute a paraphyletic group together with eastern Neandertals (red box). The sampling locations for the specimens are shown with corresponding colour coding."</p>
</div>

The blue clade includes all Neandertals after 48,000 years ago from Western Europe; the red clade includes earlier Neandertals from the west as well as both earlier and later Neandertals from the east. 

The meat in this phylogram is not only that the later western Neandertals are close relatives, but that they share an ancestor only around 60,000 years ago. That's a mere 20,000 to 25,000 years before the later western Neandertals lived. The variation within these Neandertals is roughly the same as that within a single mtDNA clade within Europe today, such as clade H1. 

Comparing the later Neandertal diversity to the variation of present-day Europeans helps to clarify the meaning of low diversity. Low mtDNA diversity doesn't necessarily imply that the later Neandertals in western Europe were few in number. Certainly there are millions of Europeans today who carry clade H1, for example. Low mtDNA diversity tells us something more limited about the ancestors of these Neandertals. Sometime after 60,000 years ago, a pulse of mitochondria came from the east and were remarkably successful in the west. 

Looking at the red clade in the figure is also illustrative. Eastern Neandertals and earlier western Neandertals had a lot more diversity than the later western Neandertals. We have to remember that the Scladina individual lived 40,000 years before the <em>common ancestor</em> of the blue clade, so that the greater ages of these specimens matters. Still, when we look at the diversity in that red clade, it is greater than the mtDNA diversity today in the most widespread basal clade outside Africa, the M clade. Taking the mtDNA phylogeny alone, we would say that the 13 Neandertals had a greater sequence difference than all the people who with ancestry outside Africa today. Only when we look at the predominantly African clades today (the L clades) do we start to see sequence differences as great as among these Neandertals. 

I began the post by pointing out that small population size alone might explain the low mtDNA diversity of Neandertals. Dal&eacute;n and colleagues provide a key comparison that helps to reject that hypothesis. Small population size alone cannot explain the discrepancy of mtDNA diversity of these Neandertals across space and time. 

<h4>The whole-genome perspective</h4>

Now, the question is whether this pattern holds true only for mtDNA, or whether the rest of the genome also shows some dynamic within Neandertal populations. 

We have quite a lot of information on this point, because the initial sequencing of the Vindija Neandertals was accompanied by a smattering of sequencing of the nuclear genomes of one individual from El Sidr&oacute;n, the original Feldhofer specimen and the Mezmaiskaya Neandertal specimen. The inclusion of Mezmaiskaya is important, because it alone is not included in the "low mtDNA diversity" red clade pictured above. If the pattern observed for mtDNA is reflected by the rest of the genome, the comparison between Mezmaiskaya and western Neandertal genomes should show substantially more diversity. 

When they published the draft Denisova genome, Reich and colleagues <bib>Reich:Denisova:2010</bib> used it as an outgroup to investigate variation among the Neandertals, and they focused initially on Mezmaiskaya:

<blockquote>Using the 56 Mb of autosomal DNA sequences determined from [the Mezmaiskaya specimen], we estimate that the DNA sequence divergence between the Vindija and Mezmaiskaya Neanderthals corresponds to a date of 140,000 +/- 33,000 years ago (Supplementary Information section 6) (Fig. 1). This remarkably low divergencewhich is about one-third of the closest pair of present-day humans that we analysedis in agreement with the observation that diversity among Neanderthal mtDNAs is low relative to present-day humans and indicates that the Vindija and Mezmaiskaya Neanderthals descend from a common ancestral population that experienced a drastic bottleneck since separating from the ancestors of the Denisova individual.</blockquote>

That adds substantially to the mtDNA picture. The mtDNA variation of western Neandertals may reflect population turnover after 50,000 years ago. But the nuclear genome comparison cannot be explained by this single event. The variation of nuclear genomes between Mezmaiskaya and El Sidr&oacute;n spans across more than half the Neandertal geographic range and requires mechanisms that restricted genetic variation across at least the period after 140,000 years ago. 

I think we can do quite a bit better using the nuclear genetic information already available, keeping an explicit phylogeographic model in mind. My view is that Neandertal populations were dynamic throughout their existence, with repeated waves of population turnover across broad geographic scales. The mtDNA of later western Neandertals may reflect a large, recent event. But there must also have been earlier ones to limit variation of the nuclear genome. The implication is that early Neandertals like Krapina may have had relatively little genetic connection to later Neandertals in the same region, like Vindija. 

That picture matches what we are beginning to understand about the population history of Europe during the last 30,000 years. I think that's how human populations have always behaved. 


<h4>Revisiting Neandertal races</h4>

I wrote extensively about Neandertal mtDNA in 2009, noting the work of Virginie Fabre and colleagues <bib>Fabre:2009</bib>, which showed the geographic structure of Neandertal mtDNA variation (<a href="http://johnhawks.net/weblog/reviews/neandertals/neandertal_dna/fabre-2009-population-structure-mtdna-neandertals.html">"Neandertal races?"</a>). Fabre and colleagues showed that Neandertal mtDNA variation is apportioned unequally across space, and made sense of the variation using a phylogeographic model with three broad geographic groups. I pointed out then that an alternative explanation might be that the specimens represent different times:  

<blockquote>Many have pointed out, going back to McCown and Keith (1939), that time is another possible cause of morphological differentiation of Neandertals. The mtDNA sequences cover a wide range of times -- the Scladina sequence comes from roughly 100,000 years ago, the others cover the span from 50,000 down to 29,000 years ago. Why not test temporal groups instead of geographic groups? Temporal clusters might reflect interglacial colonizations, differential gene flow, or natural selection. There is a good precedent -- last year a report of complete mtDNA sequences from woolly mammoths found evidence for geographic structure among mtDNA lineages, one of which apparently replaced the other (Gilbert et al. 2008).</blockquote>

<blockquote>Time is just one example of an alternative model for variation. But I think it helps to clarify the basic problem of the a priori models -- you have to draw boundaries between the specimens somewhere. </blockquote>

The problem still remains even in the current paper. Why should we divide time arbitrarily at 48,000 years ago? Why divide time in western Europe but not across the eastern part of the Neandertals' range? 

Combining space and time into a single phylogeographic picture is complicated. We end up using a null model to generate millions of pseudosamples to represent the exact time and place we found specimens, hoping to show the null model wrong. Refuting a null model doesn't necessarily tell us much about the behavior of ancient populations that flowed across space and interacted at different times. I think that life was more complicated rather than less, and look to models from more recent populations to understand it. 



<h4>How not to publicize your work</h4>


The paper by Dal&eacute;n and colleagues is such a neat piece of work, I think it's a shame that Uppsala University had to go and spoil it with this silly press release: <a href="http://www.sciencedaily.com/releases/2012/02/120225110942.htm">"European Neanderthals Were On the Verge of Extinction Even Before the Arrival of Modern Humans"</a>. 

The paper pointedly does <em>not</em> show that Neandertals were on the "verge of extinction". Neandertals in the eastern part of their range show no sign of any demographic collapse, and the western part of the range arguably only shows signs of recovery and expansion. 

What the paper actually tells us is about the dynamism of Neandertal populations, which is very comparable to that of the Europeans of the last 10,000 years. Keeping this comparison in mind helps remind us that very large groups of people may still have low mtDNA diversity, reflecting the history of population movements and interactions in the past. Comparing the mtDNA with nuclear genetic evidence is also essential to this picture. Neither of these tell us that Neandertals were near extinction. 

Please, if you're putting together a press package about Neandertals, <strong>stop framing it around the concept of Neandertal extinction</strong>. You aren't going to say anything novel about this, and it just encourages lazy science writing. And it's a false concept. The Neandertals didn't become extinct. 

UPDATE (2012-03-06): A reader points out that several of the dates for specimens in the paper are different than reported in the literature. I noticed that too, and don't know quite what to make of it. I don't think that the differences in dates affect the general result, that later specimens in Western and Central Europe are relatively invariant compared to the Eastern European and Asian sample. But it is a reminder that the results do depend on a certain ordering and geographic sampling of specimens and may change if we fill in the gaps. 

