---
layout: single 
title: "When genes break: validating loss-of-function variants" 
description: "A &quot;punishing&quot; resequencing project validates mutations in the 1000 Genomes Project individuals that deactivate protein-coding genes." 
category: story
permalink: /weblog/reviews/genomics/macarthur-2012-loss-of-function-variants.html
tags: [1000 Genomes Project, pseudogenes, positive selection, recent selection, fitness effects, mutation] 
comments: false 
author: John Hawks 
---


Daniel MacArthur and colleagues have an important paper in <em>Science</em>, titled "A Systematic Survey of Loss-of-Function Variants in Human Protein-Coding Genes" <bib>MacArthur:LOF:2012</bib>. They took 1000 Genomes Project pilot data and systematically looked at every allelic variant in the sample that appeared to cause the loss of function of a protein-coding gene. Mutations that de-activate genes in this way are not rare, but they are often eliminated from the population rapidly by purifying natural selection, because the normal function of a protein is necessary to survival or reproduction. However, not every protein is so important, and MacArthur and colleagues confirmed that more than 1200 alleles in this sample genuinely occur in one or more of the 1000 Genomes Project individuals. 

Some of these are common but most occur in fewer than 2% of individuals in the sample, as expected if purifying selection were affecting many of them. 

MacArthur is one of the authors of the Genomes Unzipped group blog, and has written a great summary and introduction to his research paper: <a href="http://www.genomesunzipped.org/2012/02/all-genomes-are-dysfunctional-broken-genes-in-healthy-individuals.php">"All genomes are dysfunctional: broken genes in healthy individuals"</a>. It's free and well-written, so it will probably work better for many readers than the original paper. 

<em>Science</em> is running a commentary to accompany the research article, by Lluis Quintana-Murci <bib>Quintana-Murci:MacArthur:2012</bib>. This paragraph encompasses a lot of the numerical facts about these loss-of-function variations, and discusses the idea that some of them were positively selected -- that is <em>advantageous</em> in recent human populations. 

<blockquote>MacArthur et al. estimated that, depending on ethnic background, each individual's genome carries 26 to 37 variants that introduce a stop codon (which signals the termination of translation of nucleic acids into protein), with up to 6 present in the homozygous state. When considering other types of LoF variants, including those that disrupt splice-sites, large deletions, or insertions or deletions of nucleotides that change the DNA reading frame, the total number per individual is extended to 103 to 121, with ?20 present in homozygosity. A large proportion of LoF variants were enriched in low-frequency alleles, suggesting that the removal of deleterious alleles has prevented them from increasing to high frequencies. Furthermore, some have already been associated with severe human diseases, supporting the less-is-less hypothesis. Other LoF variants, which can reach higher population frequencies, fall into poorly evolutionarily conserved genes or belong to multigene families displaying high paralogous sequence identity. This suggests that the functions of the corresponding genes are highly redundant, explaining their greater tolerance for LoF variants and supporting a less-is-nothing scenario. Also, although no substantial enrichment in positive selection signals was observed among LoF variants at the genomewide level, 20 of them fell into regions displaying signatures of positive selection, as predicted by the less-is-more hypothesis, suggesting that they may have conferred a selective advantage in human evolution.</blockquote>

Common loss-of-function variants that are evolutionarily recent are very interesting to us as we work to understand the changes that accompanied modern human origins and the later invention and spread of agriculture. I am really excited that these analyses were carried out using the 1000 Genomes samples because that means we can use the sequence data to estimate the ages of these functional losses. We can do quite a lot better than to say that they "fall into regions displaying signatures of positive selection": In fact, we can determine whether these variants themselves were selected, or hitchhiked to high frequency along with some other variant that was selected. 

Many of loss-of-function variants are in genes that may not matter much to selection. Olfactory receptor genes, for example, comprise a very large family with recurrent duplications and pseudogenizations during primate evolution. We have scores of olfactory receptor pseudogenes, many of which are polymorphic in living human populations. Some may continue to make a noticeable difference to the phenotype, such as the asparagus-urine-smelling polymorphism. But many are probably invisible to us. Still, a few of these do look like they've been positively selected in recent human populations. 

Sometimes less really is more. 



