---
layout: single 
title: "Speaking of Neandertal FOXP2" 
description: "A discovery of gene regulation differences in FOXP2 may explain the variation of the gene in recent and archaic people." 
category: story
permalink: /weblog/reviews/neandertals/neandertal_dna/maricic-foxp2-regulation-2013.html
tags: [Neandertals, Denisova, language evolution, brain, Neandertal DNA, brain development] 
comments: false 
author: John Hawks 
---

Tomislav Maricic and colleagues from Svante P&auml;&auml;bo's group have reported finding a regulatory change in the gene <em>FOXP2</em> that may be of relevance to the evolution of human speech <bib>Maricic:2013</bib>. To telegraph the conclusion, the paper does not demonstrate that Neandertals or Denisovans were different from humans in speech or language-relevant phenotypes. 



Most important, a substantial number of living people share the ancestral genotype inferred for Neandertals and Denisovans for the site considered in the study. It is a genetic change within living people that may have been important, but it is an instance where human variation includes the Neandertal genotype. 



I'm going to let the paper's mini-review do the work of describing the background to the study: 



<blockquote>Among humans, sequence variation around exon 7 shows an excess of derived nucleotide variants at high frequencies and of rare nucleotide variants, indicating that the region has been affected by a selective sweep (Enard et al. 2002; Zhang et al. 2002; Yu et al. 2009). It has been estimated that this happened within the last 200,000 years (Enard et al. 2002) or 55,000 years (Coop et al. 2008). Because it was initially assumed that at least one of the two amino acid substitutions were the cause of the sweep, it was expected that at least one of them would not be present in Neandertals, who shared a common ancestor with modern humans 370450,000 years ago (Green et al. 2010). However, both nucleotide substitutions were found in two Neandertals from Spain (Krause et al. 2007) as well as in Neandertals from Croatia (Green et al. 2010), and in Denisovans, an extinct Asian hominin group related to Neandertals (Reich et al. 2010). Furthermore, it was found that linkage disequilibrium extends across exon 7 in present-day humans, which is not expected if one of the two amino acid changes in exon 7 was the target of selection (Ptak et al. 2009). Hence, although at least one of the two amino acid changes is very likely evolutionarily relevant given the functional data and the conservation of FOXP2, they are not likely to be the cause for the selective sweep. Assuming that a sweep did occur, it must therefore be caused by some other variant in the region, possibly affecting the regulation or splicing of FOXP2.</blockquote>



It was a big story that humans had a recent sweep in this gene, eliminating most of the variation, and that humans are different from other primates in the coding sequence. But the apparent timing of the sweep did not make sense in combination with the observation that Neandertals share the human coding sequence.



One resolution of these observations is the hypothesis that the human version of <em>FOXP2</em> simply came from Neandertals. I wrote about a short paper by Graham Coop and colleagues in 2008 that went along similar lines (<a href="http://johnhawks.net/weblog/reviews/neandertals/neandertal_dna/coop-foxp2-recent-selection-2008.html">"FOXP2 is really recent, it really did introgress (if it's not contamination)"</a>). Coop and colleagues substantiated the hypothesis of a recent selective sweep, but at the same time they did acknowledge that selection on some other linked locus might account for the evidence.



Maricic and colleagues have found another linked genetic change that could account for the sweep. In their scenario, the sweep was only the most recent of possibly several changes under selection to this gene. This most recent one involved a regulatory change within exon 7 of the gene that did not affect the coding sequence at all. 



The sequence analysis carried out by Maricic and colleagues is very straightforward. They simply resequenced the gene region from Neandertal specimens to get a list of sites where Neandertals and Denisovans do not carry a derived human variant, and then resequenced the gene in 50 humans to see how many of the derived human mutations are high-frequency. The one they identify is both high-frequency and affects a candidate regulatory site. The site is a binding site for the transcription factor POU3F2. The rest of the paper documents their attempts to demonstrate an effect of this site on gene regulation in tissue culture. They conclude: 



<blockquote>The transcription factor POU3F2 is expressed exclusively in the central nervous system (Schreiber et al. 1993), more specifically in postmitotic neurons and glia (Hagino-Yamagishi et al. 1997). Within the central nervous system, FOXP2 is expressed in postmitotic neurons (Ferland et al. 2003). Thus, it is reasonable to assume that POU3F2 regulates expression of FOXP2 in neurons. It is furthermore interesting that position 114076877 is located at the point in intron 8 of the FOXP2 gene where the pattern of allele frequencies among humans indicates that a functional change occurred that could be responsible for a positive selective sweep affecting the FOXP2 gene during the last 50,000 years (Coop et al. 2008). It is noteworthy that this is the only nucleotide variant in that region where the majority of present-day people carry a derived variant that is not present in Neandertals and Denisovans. Thus, it is possible that this change was positively selected recently during the evolution of fully modern humans.</blockquote>



However, the ancestral allele shared by Neandertals and Denisovans is also fairly common in some human populations today. As Maricic and colleagues conclude, the obvious thing to do is look at homozygote carriers of the allele to see if they're different from noncarriers:





<blockquote>The ancestral allele occurs at frequencies of ?10% in some African populations (supplementary table S6, Supplementary Material online). Therefore, individuals homozygous for the ancestral allele can be expected to occur at a frequency of approximately 1% in the population. In such individuals, the phenotype of the ancestral allele should be observable even if is recessive to the derived allele. Further work will explore the phenotypes of such homozygous carriers of the ancestral allele and the consequences of the substitution at position 114076877 on FOXP2 transcription in model systems.</blockquote>



This all seems logical. We may not be able to say that Neandertals were just like us in <em>FOXP2</em> -- but that's because we're not all alike. They're just like <em>some of us</em>. 



The only thing I would add is that the number of humans covered by the study is still quite small. The paper examined only 50 individuals from the HGDP set; additionally they considered the 1000 Genomes data. It is interesting that the Neandertal-Denisovan ancestral allele at this site is not present in several of the samples outside Africa in the 1000 Genomes data, but it is present in two of the American samples, and in all the African samples. So although the region looks like it was positively selected at some point during the last 100,000 years or so, we still can't yet say that the ancestral allele carried by Neandertals was disadvantageous within later populations. 



Larger samples would settle that question. In the meantime, this study does point the way toward a wider analysis of differences in gene regulation among archaic human genomes. 





