---
layout: single 
title: "Cave bear DNA from Sima de los Huesos" 
description: "The oldest-ever recovery of DNA outside permafrost comes from Atapuerca." 
category: story
permalink: /weblog/reviews/genomics/non-primate/cave-bear-atapuerca-2013.html
tags: [mtDNA, non-primate, ancient DNA, cave bears, phylogeography] 
comments: false 
author: John Hawks 
---

Jesse Dabney and colleagues, including Svante P&auml;&auml;bo from the Max Planck Institute for Evolutionary Anthropology, report on the assembly of a complete mitochondrial genome from a 300,000-year-old cave bear, from Sima de los Huesos, Atapuerca, Spain. This is the oldest DNA recovered outside of permafrost contexts, and as the conclusion of the paper points out, this is a demonstration of the potential for ancient DNA techniques in Middle Pleistocene specimens. The DOI for the paper is not yet working but the <a href="http://www.pnas.org/content/early/2013/09/04/1314445110.abstract">abstract is available online from <em>PNAS</em></a>. 

Of course, the most newsworthy aspect of the study is what it promises for sequencing the hominin remains from Sima de los Huesos in the near future. This is one of the densest accumulations of hominin bone anywhere in the world, through the entire record of our evolution, and so has lots and lots of specimens that might yield genetic evidence if the technology can cope. Dabney and colleagues show a new ability to recover and examine the short fragments of DNA that survive after hundreds of thousands of years: 

<blockquote>We note that, although the vast majority of cave bear sequences are only 3050 bp in length, we have not yet systematically explored the lower size limit of DNA fragments surviving in ancient bone. It is therefore possible that even shorter molecules can be made available for sequencing in the future, by using library-based techniques as described here or directly via single-molecule sequencing (8, 25). However, in addition to further optimizations of the DNA extraction method, such attempts will have to include improvements to hybridization enrichment of very short molecules and the development of new sequence analysis strategies that allow for confidently aligning very short sequences to a reference genome while discriminating endoge- nous sequences from contaminating environmental DNA. We hope that the methodology presented here will help to retrieve ancient DNA sequences from additional organisms of the Middle Pleistocene period. The fossil remains from Sima de los Huesos will undoubtedly remain in the focus of such efforts, because they include the largest assembly of Middle Pleistocene hominin fossils in the world (42).</blockquote>

The breakthrough here is in using library preparation methods that reduce the bias against shorter sequence fragments, and computationally filtering those sequence fragments in ways that enhance their assembly against a reference genome.

<blockquote>Strikingly, despite a bias toward hybridizing longer molecules (Fig. S1), 94% of the sequences are no longer than 50 bp and 76% are no longer than 40 bp, respectively (Fig. 2A). The vast majority of sequenced DNA fragments are thus in a size range that was not efficiently recovered with previous methods.</blockquote>

This seems very promising. As Dabney and coworkers show, it is enough to assemble a mitochondrial genome. It will be more challenging to get nuclear DNA evidence in this manner, because the coverage will be much lower for a given amount of material, but several different approaches can yield useful hypothesis tests from such fragments interspersed across the genome. What they won't do is allow PCR probing for particular loci, because short fragments don't leave room for primers. 

So what can we learn from a 300,000-year-old cave bear mtDNA genome? The paper gives a great short mini-review of how ancient DNA has changed our knowledge of cave bear phylogeography: 

<blockquote>The earliest fossil evidence of cave bear-derived morphological features is found ?1.2 Ma in Ursus dolinensis, a species that was defined in Atapuerca Gran Dolina (TD4) (27) but is also recorded at Atapuerca Trinchera Elefante (TE9) (28) and Untermassfeld (29, 30). An abundant fossil record in Europe and parts of Asia indicates that subsequent cave bear evolution proceeded through the Middle Pleistocene form U. deningeri, which transitioned into the Late Pleistocene form Ursus spelaeus sensu lato (31), before cave bears went extinct 28 ka (32). Genetic and morphological analyses support a further differentiation of three types of Late Pleistocene cave bears. The first two, U. spelaeus sensu stricto and Ursus ingressus, are predominantly found in Europe and are thought to have become reproductively isolated (33). The third type has been found only in the Caucasus and the Yana river region in Eastern Siberia and was designated U. deningeri kudarensis (34) based on its more ancestral dental morphology. It also shows a divergent mitochondrial haplotype (35, 36).</blockquote>

This is not exactly like the shifts in Neandertal phylogeography that have emerged from mtDNA sequences in the last few years, but nonetheless the cave bear story is comparable in many ways. Maybe most interesting is that cave bears from the easternmost part of their Late Pleistocene range in the Caucasus form an mtDNA clade separate from the more western cave bears, which additionally retains a more ancestral dental configuration. Cave bear Denisovans? The mtDNA phylogeny is very much like the hominin one, which has the Denisova mtDNA sequence as a relatively distant outgroup to both living people and all known Neandertals. 

The new sequence from Sima de los Huesos is not an outgroup, despite being the most ancient specimen, and despite its plesiomorphic anatomy. That renders the "ancestral" cave bear species, <em>Ursus deningeri</em>, as a paraphyletic taxon, at least for mtDNA sequences. 

Of course, in the hominin case the nuclear DNA does not show the same picture as the mtDNA relationships, so it is hard to predict what the future of cave bear ancient DNA may hold. The current samples are recognized as several distinct species, and it will be interesting to see whether DNA supports that interpretation, or whether instead we see a hominin-like mixture of different populations across Europe during the Middle and Late Pleistocene. Personally, I can't wait until we have a thicker sampling of the Middle Paleolithic time slice for a number of species, because that will enable us to understand the population dynamics in response to at least two and possibly more glacial cycles in Europe. 




