---
layout: single 
title: "Taking a blender to the skin" 
category: quickbit
permalink: /weblog/reviews/genomics/meta/mathieu-microbiome-skin-2013.html
tags: [metagenomics, skin, ecological community, microbiome] 
comments: false 
author: John Hawks 
---

A new paper by Alban Mathieu and colleagues looked at the functional pathways represented by the metagenomes of microbial communities living on the skin of two humans <bib>Mathieu:2013</bib>. The study of skin microbiomes among humans is quickly growing. One approach is to study the kinds of species that are present in skin microbial assemblages, and compare that species community with different people, different parts of the body, and different kinds of microbial communities (<a href="http://johnhawks.net/weblog/reviews/genomics/meta/roller-derby-microbiome-2013.html">"Close contact skin microbiome smashup"</a>). 

A different approach is to study the function of an ecosystem is to look more collectively across species at the parts that interact with each other. Sharp teeth mark the presence of animal flesh, lactase marks the presence of lactose as an energy source. At the microbial level, looking across an entire community at the functioning genes gives some notion of the structure of resources across the entire collection of species represented: 

<blockquote>The role of the microbiota in regulating another critical healthy state parameter (skin acidity), which controls the permeability barrier homeostasis, is also highlighted by numerous functional subsystems associated with acid resistance detected in the databases. For instance, acidification ecosystem preservation could explain the bacterial adaptive strategy of using the butanediolic fermentation as deduced from detection of alpha acetolactate and acetoin butanediol metabolism genes for transforming pyruvate into the final product (2,3-butanediol) rather than a mixed acid fermentation. The predominance of genes involved in the arginine deiminase metabolism [17] in the metagenome datasets confirms the tolerance of bacteria to skin acidity [15]. The skin metagenome analysis also brings new clues about the extensive spread of antibiotic resistance genes among bacteria. Within the human skin Staphylococcus populations of the two individuals, various Staphylococci seem to be intrinsically resistant to methicillin (fig. 3), although neither of the two individuals had recent contact with a methicillin-rich environment (hospitalization and/or methicillin treatment). Moreover, the level of teicoplanin and bacitracin resistance genes was particularly high in the sequence datasets.</blockquote>

The analysis is very coarse-grained, without the ability to show the relationships between distinct components of this skin microbial community and other microbiota. It's a little like sticking a forest into a gigantic blender and studying the resulting mush. One advantage of this approach is that it is less easily distracted by very rare components of the biota. The results show the important energy and nutrient flows. In particular, turning from this approach (pooled sequencing over a long time) to transcriptome sequencing (sequencing of RNA expressed in cells) can give a picture of what is metabolizing at a particular time slice in a microbial community. Mapping the overall metabolic breakdowns of more and more communities will help to clarify what resources are structuring the microbial colonization of humans and their evolution over time. 

