---
layout: single 
title: "How next-gen sequencing changes the work of a small biology lab" 
category: story
permalink: /topics/biotech/sequencing/smith-next-gen-op-ed-2014.html
description: "A lament from a PI facing a massive increase in data with a concomitant increase in complexity."
tags: [sequencing, biotech, metascience, biology] 
comments: false 
author: John Hawks 
---

David Roy Smith in the current <em>Frontiers in Genetics</em> has an opinion article that reflects on the way that next-generation sequencing technologies have changed biology: <a href="http://dx.doi.org/10.3389/fgene.2014.00146">Last-gen nostalgia: a lighthearted rant and reflection on genome sequencing culture.</a>. 

<blockquote>Sequencing nuclear DNAs has been a different story. Even with huge datasets, state-of-the-art assembly programs, and intricate annotation pipelines, I'm incapable of producing decent nuclear genome assemblies. It doesn't help that the species I choose to investigate are poorly studied and poorly sequenced. For researchers investigating organisms for which high-quality nuclear genome assemblies already exist (i.e., assemblies based on Sanger sequencing), the payoffs of NGS have been great (Koboldt et al., 2013). Perhaps as sequencing technologies improve, personal computing power increases, and bioinformatics software become more user friendly, it will soon be easier for small labs to assemble publication-quality nuclear genomes of non-model taxa. For now, however, the promises of NGS have, at least for me, not lived up to their hype and often resulted in disappointment, frustration, and a loss of perspective.</blockquote>

When technology leads the science, scientists run into practical problems. The problems Smith describes here are problems that consortia solve. Armed with a large log of high-quality assemblies and postdocs who can reprogram bioinformatics tools if necessary, a consortium can straighten out data quality problems that will bedevil a small isolated lab. 

But it is simply not practical for most biologists to work in large consortia. Smith works on the genetics of algae. If there were to be an alga genome consortium, it would have to include most of the people working on the genetics of algae. 

Maybe that's the best way to move forward. Certainly it makes little sense to have fifty small labs beating their head against the same wall when they could be collaborating. But many scientists find a strong appeal in being independent, formulating their own independent research questions that they can tackle on the scale that makes sense for their labs. 

Which makes this passage sadly ironic: 

<blockquote>I was taught to approach research with specific hypotheses and questions in mind. In the good ol' Sanger days it was questions that drove me toward the sequencing data. But now it's the NGS data that drive my questions. I recently sequenced the transcriptome of a saltwater Chlamydomonas alga and have been knocking my head against the laboratory door asking, “What is the best way to market, package, and publish these data?” I'm trapped in a cycle where hypothesis testing is a postscript to senseless sequencing (Smith, 2013).</blockquote>

The technology promises to enable a smaller lab to take on more interesting projects. But the technology is limited in a way that requires the lab to shoehorn its work into a very limited set of empirical investigations. That transforms the lab from a hypothesis-testing lab to a technology-justifying lab. 

This is where science goes to die. 



### References

<p class="cite">Smith DR. 2014. Last-gen nostalgia: a lighthearted rant and reflection on genome sequencing culture. Frontiers in Genetics Front. 5:146. <a href="http://dx.doi.org/10.3389/fgene.2014.00146">doi:10.3389/fgene.2014.00146</a></p>
