---
layout: single 
title: "The genome from Ust'-Ishim" 
description: "A femur from Siberia produces the oldest modern human genome sequence to date." 
category: story
permalink: /weblog/reviews/ancient-genomes/ust-ishim-fu-2014.html
tags: [ancient DNA, Neandertal DNA, Upper Paleolithic, Siberia, out-of-Africa] 
comments: false 
author: John Hawks 
---

Qiaomei Fu and colleagues from Svante Pääbo's lab have reported on a genome from northern Siberia that dates to 45,000 years ago. The genome comes from a human femur that was eroding out of a bank of the river Irtysh, which is in the Siberian plain near Omsk. The specimen is the shaft of a femur lacking its ends, and has no special similarities to archaic humans such as the Neandertals. It would certainly be unsurprising for a modern human to have this anatomy, and would likely be categorized as a modern human femur even without the genome. But the genome is the earliest ever recovered from a modern human individual, and therefore gives us some very useful information that we could not get in any other way. 



<h3>Another mystery population?</h3>

To me, the most interesting part of the paper is the <em>D</em>-statistic comparison with other ancient and modern genomes. The <em>D</em>-statistic is basically a measure of genome-wide similarity, considering only those alleles derived relative to some outgroup. The results are useful for assessing mixture between ancient populations. For example, people who have ancestry within sub-Saharan Africa have fewer derived alleles in common with Neandertals than people who lack recent sub-Saharan ancestry. The <em>D</em>-statistic in a comparison of a non-African and a sub-Saharan African genome with Neandertals will show a skew toward Neandertal similarity in the non-African genome. That is one way of assessing Neandertal ancestry of living people. 

<strong>The Ust'-Ishim genome seems to be ancestral to nobody.</strong> Consider the result of <em>D</em>-statistic comparisons with living Asians, Native Americans and Europeans, and two ancient genomes from Eurasia -- the Mal'ta and La Braña specimens. Here is the figure showing the <em>D</em>-statistics as compared with other specimens:


<figure>
<img src="/images/fu-2014-ust-ishim-d-statistics.png" alt="Figure 3 from Fu et al 2014 showing D statistics with Ust-Ishim specimen" />
<figcaption><p class="caption">Figure 3 from Fu et al. 2014. Original caption: "Statistics testing whether the Ust’-Ishim genome shares more derived alleles with one or the other of two modern human genomes (X, Y). We computed D statistics of the form D (X, Y, Ust’-Ishim, Chimpanzee) using a subset of the genome-wide SNP array data from the Affymetrix Human Origins array and restricting the analysis to transversions. Error bars correspond to three standard errors. Red bars indicate that the D statistic is significantly different from 0 (|Z|.2), such that the Ust’-Ishim genome shares more derived alleles with the genome on the right (X) than the left (Y). Ancient genomes are given in italics."</p></figcaption>
</figure>

In that figure, the value of zero (with a vertical line across comparisons) indicates that the genomes listed on left and right sides of the figure have the same proportion of derived similarities with the index genome (in all cases, the Ust'-Ishim genome). A positive value skews toward the right genome; a negative value toward the left. The top panel shows that Ust'-Ishim is more similar to genomes of non-Africans than it is to sub-Saharan African genomes.

The case of Mal'ta is most interesting as a comparison. This individual from eastern Siberia near Lake Baikal dates to around 24,000 years ago. Its genome was described earlier this year by Raghavan and colleagues (2014), and one of their major conclusions was that the similarity of this genome with Europeans on the one hand, and Native Americans on the other, probably shows a common source population in Siberia that had contributed to the rise of both these living populations. That is not to say that both these populations derive fully from ancient Siberians, but that some ancient Siberians related to Mal'ta later became part of the population that entered the Americans, and that some other ancient Siberians related to Mal'ta would ultimately migrate into Europe.

Ust'-Ishim shows no such pattern. It is not more like the Mal'ta genome than it is like any other genomes of Asians or Native Americans. It is not like any living population of Asians or Native Americans more than any other. 

At the same time, Ust'-Ishim is <em>less</em> like living Europeans than it is like any of these Asian populations, and <em>less</em> like living Europeans than it is like the Mesolithic-era La Braña individual, even though that individual lived in Iberia. Fu and colleagues suggest that the recent population of Europe must have gotten some of its ancestry from other populations that were not part of the initial out-of-Africa spread represented by the Ust'-Ishim individual. That could mean later Africans, or it could mean one or more West Asian populations that contributed neither to the later dispersal of humans into the rest of Eurasia and the New World <strong>nor to the Mesolithic occupation of Europe by modern humans</strong>. I wrote "Mesolithic" there very consciously, because it seems plausible until demonstrated otherwise that earlier Upper Paleolithic Europeans might represent yet some other population.

Considering the fact that Ust'-Ishim is equally similar to all Asian and Native American populations and equally similar to the two ancient genomes, Fu and colleagues write this: 


<blockquote>This suggests that the population to which the Ust’- Ishim individual belonged diverged from the ancestors of present-day West Eurasian and East Eurasian populations before—or simultaneously with—their divergence from each other. </blockquote>

I would give a strong interpretation to this. It seems unlikely that Chinese (Han) and Andaman Island (Onge) populations could be uniquely descended from this ancient Siberian individual, so Ust'-Ishim is not at the stem of the later diversification of Eurasian people. That means that these later people derive from a <em>different group</em> than that represented by Ust'-Ishim. The initial dispersal of humans into Eurasia contained at least one dead-end population that contributed at most some very small amount of ancestry to living people. 

This is not an isolated case, it is another example of what we see throughout the world: Ancient people represented by DNA that seem to have very little to do with the people who live in the same areas today. We're not finding the ancestors of living populations so much as we are finding branches of populations we did not know existed.  


<h3>Mutation rate</h3>

I've written about the human mutation rate many times here, and in print (Hawks 2012). The new paper is interesting because it is the first modern human we have at high coverage, where the age is sufficient to estimate the number of <strong>missing mutations</strong> that would be expected in a descendant living today. Even though Ust'-Ishim may have no living descendants, this is a measure of how short his DNA branches are compared to the branches connecting living humans to their common ancestors. 

Given that, the authors find that the mutation rate would need to be fairly low, on the order of 1 times 10<sup>-8</sup> mutations per site per generation, to account for the Ust'-Ishim data. That low rate is more or less the same as estimated from looking at parents and offspring living today, which is a good piece of evidence that the per-generation rate we measure in living people is not different from the per-generation rate averaged over longer periods of time. As the authors point out, this means that a lot of ancient events estimated from DNA only must have happened longer ago than was claimed prior to a couple of years ago. 



<h3>Neandertal mixture</h3>

Several notable aspects of this genome can tell us more about Neandertal ancestry than we knew before. 

1. The genome has no greater Neandertal ancestry than in living non-Africans. At least in this part of Asia, there was no greater opportunity to interbreed with Neandertals than elsewhere for ancient people. 

2. Shared alleles with Neandertals are co-inherited across longer distances in the Ust'-Ishim genome than in living people. That is, there is greater linkage disequilibrium between Neandertal-derived alleles in the Ust'-Ishim genome. That indicates that Ust'-Ishim lived much sooner after its Neandertal ancestors than we do. 

3. The pattern of linkage disequilibrium indicated by the Neandertal segments of his genome allows us to estimate the time that the Ust'-Ishim individual had Neandertal ancestors. This time seems to have been between 50,000 and 60,000 years ago.

Here's an excerpt from the section of the paper where they describe the identification of Neandertal-derived segments: 


<blockquote>To test if this is indeed the case, we identified putative Neanderthal DNA segments in the Ust’-Ishim and present- day genomes based on derived alleles shared with the Neanderthal ge- nome at positions where Africans are fixed for ancestral alleles. Figure 5 shows that fragments of putative Neanderthal origin in the Ust’-Ishim individual are substantially longer than those in present-day humans. We use the covariance in such derived alleles of putative Neanderthal origin across the Ust’-Ishim genome to infer that mean fragment sizes in the Ust’-Ishim genome are in the order of 1.8–4.2 times longer than in present-day genomes and that the Neanderthal gene flow occurred 232–430 generations before the Ust’-Ishim individual lived (Supplementary Information section 18; Fig. 6). Under the simplifying assumption that the gene flow occurred as a single event, and assuming a generation time of 29 years, we estimate that the admixture between the ancestors of the Ust’-Ishim individual and Neanderthals occurred approximately 50,000 to 60,000 years BP, which is close to the time of the major expansion of modern humans out of Africa and the Middle East. How- ever, we also note that the presence of some longer fragments (Fig. 5) may indicate that additional admixture occurred even later. Nevertheless, these results suggest that the bulk of the Neanderthal contribution to present-day people outside Africa does not go back to mixture between Neanderthals and the anatomically modern humans who lived in the Middle East at earlier times; for example, the modern humans whose remains have been found at Skhul and Qafzeh</blockquote>

As usual, we should be cautious about assumptions. In particular, the assumption that admixture happened at a single time is almost certainly wrong. This amounts to the idea that the last Neandertal ancestors of Ust'-Ishim were his only Neandertal ancestors. As the paper suggests, there may have been more recent ancestors. I would point out that a small amount of Neandertal ancestry going further back in time would be indistinguishable from this pattern. If we imagine that Neandertal mixture into Eurasian modern human populations may have happened over some range of times, these data show that the modal time was probably in the time range from 50,000 to 60,000 years ago but do not exclude tails going further back in time or closer to the present. 

That being said, it is pretty important that this genome adds resolution to the timing of Neandertal ancestry. We are not looking at a situation in which the bulk of mixture happened earlier than 100,000 years ago. We're looking at mixture that happened not too long before humans reached China, Australia, and obviously Arctic Siberia. 

In contrast to its Neandertal DNA heritage, the Ust'-Ishim genome has no evidence of ancestry from the Denisovan population. That may or may not be surprising -- the genome may be from the same time zone but that doesn't make it especially close to Denisova; we don't know when the Denisovans lived there or where other modern humans may have mixed with them. But it is at least unfortunate, as it deprives us of the opportunity to examine the date of Denisovan ancestry with the same method as Fu and colleagues used for Neandertal ancestry. 




<h3>References:</h3>

<p class="cite">Fu, Q. and many others. (2014). Genome sequence of a 45,000-year-old modern human from western Siberia. Nature, 514, 445-450. <a href="http://dx.doi.org/10.1038/nature13810">doi:10.1038/nature13810</a></p>

<p class="cite">Hawks, J. (2012). Longer time scale for human evolution. Proceedings of the National Academy of Sciences, 109(39), 15531-15532. <a href="http://dx.doi.org/10.1073/pnas.1212718109">doi:10.1073/pnas.1212718109</a></p>

<p class="cite">Raghavan, M., Skoglund, P., Graf, K. E., Metspalu, M., Albrechtsen, A., Moltke, I., ... & Willerslev, E. (2014). Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans. Nature, 505, 87–91. <a href="http://dx.doi.org/10.1038/nature12736">doi:10.1038/nature12736</a></p>
