---
layout: single
title: "How will ancient proteins change paleoanthropology?"
description: "A news feature in Nature by Matthew Warren looks at the developing science of ancient proteomics"
category: story
permalink: /weblog/reviews/proteomics/warren-proteomics-extinct-hominins-2019.html
tags: [proteomics, ancient DNA, species, Denisovans]
comments: false
modified: 2019-06-26
author: John Hawks
---

<em>Nature</em> has a news feature by Matthew Warren that provides a nice background to recent work on proteomics of fossil hominins: <a href="https://www.nature.com/articles/d41586-019-01986-x">"Move over, DNA: ancient proteins are starting to reveal humanity’s history"</a>.

Like most areas of new technology, news stories are training the public to expect that lab folks will wave a magic wand and soon answer all questions. The reality is more complicated.

Protein sequences from teeth and bones provide tremendous promise for understanding the evolution and relationships of ancient hominins. But it's important to separate the science from the hype. Protein sequences provide information that is orders of magnitude more limited than genome sequences. A single low-coverage genome sequence built from three Neanderthal specimens led the observation that humans today have Neanderthal ancestry, many of them as much as 2 or 3 percent. That observation---arguably the single most important ancient DNA result---is outside the power of protein work. Another area where ancient DNA has transformed our knowledge is the divergence times of ancient hominin groups. Protein data will provide some information about such divergences in cases where DNA is unobtainable. But the precision of such dates will be very low, because the number of changes in amino acid sequences among hominoids is small.

That being said, those of us who work with fossil hominin material are excited about the potential of proteins. They are already transforming some areas, and they have much to offer in others. For example:

1. Proteins are great for identifying hominin fragments too small for reliable anatomical identification. In this area, proteins are already prime-time science. This has been one important application of the ZooMS approach, with tremendous success under the leadership of Katerina Douka on the Denisova Cave bone collection.

2. Along similar lines, ZooMS has started to make important contributions to understanding the species composition of fragmented faunal collections. We could use additional investigation of how comparable such results are to traditional, less-destructive quantification of faunal collections based on identifiable fragments.

3. As related in Warren's story, protein sequences have brought new information to phylogenetic questions of extinct groups, such as sloths, South American ungulates, and rhinoceroses. Hominin systematics is a mess. Few studies agree on the shape of the phylogenetic tree of human relatives. This confusion comes from a combination of too little morphological data about some fossil fragments, and too many instances of morphological convergence or parallelism in various hominin lineages. Protein sequences don't necessarily solve these problems---there will be many lineages with no data, few changes among most species, and convergent amino acid changes are by no means impossible in hominins. But everyone who works on hominins is interested in any approach that can squeeze more information out of the record.

The article spends much time on this third area of future promise, which is the most susceptible to hype. This is partly driven by the recent recognition of the Denisovan affinity of the mandibular fragment from Xiahe, China, based upon protein sequence. That similarity is compelling, but it is based upon very little information---one single change in the collagen sequence that has been observed in a Denisova Cave genome and not in living humans or Neandertals.

The enamel proteome has more information than collagen, and many ancient hominins are more different from us than Neandertals and Denisovans. That means when we look at ancient hominins like <em>Homo erectus</em>, we can expect a bit more information about their evolutionary divergence from us. That's a time frame during which we know much less about relationships:

<blockquote>Go back one million years or more, and things get even less clear. <em>H. erectus</em>, for example, first emerged in Africa around 1.9 million years ago, but without DNA evidence, it remains uncertain exactly how it is related to later hominins, including <em>H. sapiens</em>.</blockquote>

I think the most likely result of this research is that "it" is going to be "they". Protein analysis is not the only new approach shedding new light on the relationships of Early and Middle Pleistocene hominins, and all of the new information is raising new questions about how we recognize and understand species and populations.

Work on internal structure of the teeth, involving Maria Martinón-Torres and others, is starting to provide some fascinating evidence that <em>"Homo erectus"</em> in Asia is a complicated story, involving branches that haven't been previously been recognized. Gross morphological evidence has long suggested that African <em>"Homo erectus""</em> is likewise complicated, and in Africa work on the internal anatomy of teeth is only just getting started.

The big challenge is this: We live today in a world where "lumping versus splitting" is no longer an interesting question about extinct hominins. We know that populations of hominins existed for hundreds of thousands of years in relative isolation, with very little or no gene flow, and yet still interbred with each other. That interbreeding shaped the evolution of later populations, including people living today, even when it makes up a fairly small proportion of their ancestry.

So, how do we examine the evolution of earlier populations? Even with DNA evidence, there is a limit to our ability to test for layers of hybridization and introgression, because of the small and geographically limited samples we have. With morphology and low-information biomolecules like proteins, we are going to need a new synthesis to understand the connections between biomolecules, morphology, and development.

This is nothing to be afraid of. New sources of evidence are going to make it possible to fill in some broad strokes that are currently monochrome. We're going to see multiple populations of <em>H. erectus</em>---whether we will call those species, or paleodemes, or populations, is not yet clear, and will depend on their temporal and spatial patterning.

I'm less sure that we will resolve much about hominin phylogeny. It's very bad now. Three different methods of looking at the phylogenetic placement of <em>Homo naledi</em> have led to three very different results, and similar problems have emerged with <em>Australopithecus sediba</em>, <em>Homo floresiensis</em>, and other species. These are some of the most complete skeletal samples of any hominin species, and our field cannot reliably place them on a tree. Proteomics will provide some new evidence to add to the tree, but it may only deepen some of the problems.

Meanwhile, as the science of proteomics develops, we need to be vigilant to avoid some of the mistakes that have been made by ancient DNA researchers. The <em>Nature</em> article includes a link to last year's story by Ewen Callaway: <a href="https://www.nature.com/articles/d41586-018-03773-6">"Divided by DNA: The uneasy relationship between archaeology and ancient genomics"</a>. That story went into some details about the conflicts that have arisen between ancient DNA specialists and archaeologists. These groups have different histories, and few ancient DNA specialists have made the effort to understand some of the deeper, darker history of archaeology, with troubling results for their work. Meanwhile, both groups have a history of failure to work effectively with descendant communities and local researchers in many countries.

Although Warren's story new story about proteomics does not focus on this element, ancient proteins will pose many of the same problems as ancient DNA sampling. The sampling is destructive, and they have possibly greater application to cultural material than to the human remains themselves. Meanwhile, integrating protein results--which are limited--into a more complex picture involving many kinds of information will be a challenge.

I look at those challenges as opportunities. What a single laboratory might perceive as a "problem" of interpretation is a place where leadership from other researchers is most valuable. I know many of the people who are making progress in proteomics, and I think they're on a good track at the moment to make better science with these kinds of collaborations.

