---
layout: single
title: "Genomes and the complicated history of baboons"
description: "A new paper by Jeffrey Rogers and coworkers looks at hybridization and population history."
category: story
permalink: /weblog/reviews/primates/baboons/baboon-genome-hybridization-rogers-2019.html
tags: [baboons, primates, hybridization, introgression, speciation]
image:
  feature: banner-1500-baboon-gape-pauen-cc-by.jpg
  credit: Hamadryas baboon, by Sonja Pauen (Flickr) CC-BY 2.0
  credit-link: https://www.flickr.com/photos/dutch-tiger/4868459279
comments: false
author: John Hawks
---


Today, <em>Science Advances</em> has released a paper by Jeffrey Rogers and coworkers on the genome diversity of six species of baboons: <a href="https://doi.org/10.1126/sciadv.aau6947">"The comparative genomics and complex population history of <em>Papio</em> baboons"</a>.

This paper represents a significant advance in scientific knowledge of the history and evolution of baboons across Africa. The genus <em>Papio</em> arose around the same time as our own genus, <em>Homo</em>, and the diversity of baboons across Africa today reflects a history of divergence and mixture during the last million years.

Our team pointed to an earlier stage of knowledge of baboon population history in our 2017 paper, <a href="https://doi.org/10.7554/eLife.24234">"<em>Homo naledi</em> and Pleistocene hominin evolution in subequatorial Africa"</a>. Our interest has been the geographic distribution of <em>Homo</em> species in Africa, in light of the occurrence of <em>H. naledi</em> in the later Middle Pleistocene.

Anthropologists have recognized baboons as a very relevant comparison to hominins for more than a hundred years, and in particular the work of Clifford Jolly has focused upon baboon population differences as useful models for the adaptive differences that may have separated ancient hominins. I wrote about Jolly's perspectives on hybrid zones and introgression in baboons back in 2005: <a href="http://johnhawks.net/weblog/reviews/neandertals/neandertal_dna/jolly_hybridization_2001.html">"Look to the baboons; there will you your insights find!"</a>

Baboons of course are not alone as relevant comparisons to hominins. Carnivores and ungulates both show some similar biogeographic patterns to baboons, at least based upon mitochondrial DNA variation. But the whole-genome analysis of these species within Africa is only beginning. So getting a clear whole-genome picture of baboon population history is one of the first views of species that lived in the same habitats as ancient humans.

First, a quick introduction to baboon species. There are six of them:

1. <em>Papio papio</em> in extreme west Africa,
2. <em>Papio anubis</em> across most of the Sahel to Sudan and Ethiopia, and from there south into the Lake Tanganyika area,
3. <em>Papio hamadryas</em> in Eritrea, the Afar region, and across the Red Sea in the Arabian Peninsula,
4. <em>Papio cynocephalus</em> on the eastern coast of Africa from Somalia to Mozambique,
5. <em>Papio kindae</em> from Angola to Zambia, and
6. <em>Papio ursinus</em> in southern Africa.


<figure>
<img src="/images/papio-map-rogers-2019.jpg" alt="Map showing geographic distribution of six species of baboons in Africa from Rogers et al. 2019" />
</figure>

Rogers and coworkers looked at the genomes of a total of 17 baboons, which represented "2 to 4 individuals" from each of the 6 species. This was not a sample chosen to probe geographic diversity within each species. Only two individuals (both from the Aberdare region of Kenya) were chosen to examine historically recent hybridization and introgression. So it is a very restricted picture of variation, and that is important to keep in mind when trying to make sense of the phylogenetic inferences in the paper.

The study provides a composite tree giving a topology for the relationships of the six species as well as approximate times when they diversified:

<figure>
<img src="/images/papio-composite-phylogeny-rogers-2019.jpg" alt="Baboon composite phylogeny from Rogers et al. 2019" />
</figure>

The main features of this picture include:

1. A primary split between northern (<em>P. hamadryas</em>, <em>P. anubis</em> and <em>P. papio</em>) and southern (<em>P. cynocephalus</em> and <em>P. ursinus</em>) lineages around 1.4 million years ago.
2. A subsequent hybridization of a northern and southern branch to form <em>P. kindae</em>. Neither of these branches is closely aligned with any of the other five extant lineages.
3. A ghost lineage from the base of the genus contributing around 10% of the ancestry of <em>P. papio</em>.
4. The speciation of today's species date to between around 400,000 and 800,000 years ago, with the exception of the hybrid origin of <em>P. kindae</em> which was within the last 100,000 years.

These events were sketched out using <em>f</em>-statistics and the CoalHMM software, both of which have been used for hominins as well as chimpanzees and gorillas, so this tree is very comparable to the tree presented for chimpanzees and bonobos by de Manuel and coworkers (2016), for example.

Additional analyses in the paper look at phylogeny using different methods, including Bayesian and parsimony phylogenetic analyses. These give rise to various results that are mostly unreconcilable, and it's not obvious to me that they add anything to the paper, since none of them are capable of handling the degree of mixture that the <em>f</em>-statistics infer for <em>P. kindae</em>.

The big limitation of the study is the lack of geographic coverage of variation within each species. The small sample size should give rise to more tree-like phylogenetic results than a broader sample. The same is true of ancient hominin genomes: We have only a handful of ancient genomes, and the results are very treelike. But the introgression from Neanderthals and Denisovans in living people samples a broader number of populations from these groups and is not so simply treelike as the high-coverage ancient genomes themselves.

So I think we still have a way to go to really understand the importance of hybridization in the ancestry of today's baboons.

A hint of upcoming results comes from the two Kenya individuals of <em>P. anubis</em> examined in the study. Both individuals reflect historically recent introgression from <em>P. cynocephalus</em>. The passage describing this is very interesting:

<blockquote>Our results also shed new light on the historical dynamics of hybridization between <em>P. anubis</em> (a northern clade species) and <em>P. cynocephalus</em> (a southern clade species), which has previously been reported in southern Kenya near Amboseli National Park (17). Behavioral observations and microsatellite-based analyses support recent introgression from <em>P. anubis</em> into <em>P. cynocephalus</em> since the 1980s (25, 26). Our analysis of genome-wide haplotype block sharing indicates that a <em>P. anubis</em> individual from the Aberdare region of Kenya, more than 200 km north of Amboseli, is also admixed with <em>P. cynocephalus</em>, carrying ~546 Mb of nuclear DNA derived from <em>P. cynocephalus</em> (fig. S7). If we assume that this resulted from a single admixture event, then it is estimated to have occurred about 21 generations (~220 years) ago. However, other more complex explanations are also possible. The second individual from the <em>P. anubis</em> Aberdare population also carries <em>P. cynocephalus</em> haplotypes, but these shared genomic segments are fewer and shorter and likely result from more ancient introgression. Consistent with other studies (27), our findings suggest that there have been multiple episodes of gene flow involving these two species over a considerable time span and that the effects of past hybridization extend far beyond the current hybrid zone. This complexity may well be representative of the complexity of other known baboon hybrid zones (10, 12, 15, 18, 19, 28).</blockquote>

The heterogeneity between two individuals in the same population with respect to recent introgression is really striking. These two genomes emphasize that the movement of individuals and spread of genes between two hybridizing species are more of a turbulent interface than a smooth cline. We have a hint of this turbulent interface in the Oase genome results from Romania, an individual with a high and recent degree of Neanderthal ancestry.

The taxonomy of baboons seems to be relatively stable now. Twenty years ago, there was substantial debate about how many species should be recognized across Africa. At that time much less was known about the reproductive fitness of hybrids. Jolly (2001) emphasized the "indiscriminate" hybridization of different populations of baboons in captivity :

<blockquote>In captivity, all allotaxa appear to hybridize indiscriminately (Jolly, unpublished data), and there is no evidence for hybrid breakdown, behavioral incompatibility, or intrinsic sterility. Similarly, there is no evidence that <em>Papio</em>, baboon allotaxa ever avoid interbreeding when they meet in the wild, though many boundary areas have yet to be investigated. The fact that documented baboon hybrid zones are narrow, in spite of the lack of obvious, intrinsic barriers to gene-flow, strongly suggests that they are the result of secondary contact following range oscillations (Barton and Hewitt, 1985; Hewitt, 2001; Harrison, 1993).</blockquote>

The current story is very different from Jolly's (2001) account. As discussed by Rogers and coworkers, scientists have observed a number of indications that hybridization among species of baboons is not fitness neutral.


<blockquote>Another topic of broad interest is the origin of reproductive isolation among incipient species (1). One expectation for the genus <em>Papio</em> is that, given the timing of the radiation and the degree of morphological and behavioral differentiation among species, incipient barriers to gene flow may be evident between some pairs of species. Studies of the present-day hybrid zone between northern clade <em>P. anubis</em> and southern clade <em>P. cynocephalus</em> find no readily apparent barriers to reproduction between these species (17, 26). However, studies of captive <em>P. anubis</em> × <em>P. cynocephalus</em> hybrids document significantly elevated frequencies of craniodental anomalies in hybrids, especially hybrid males, indicating some degree of genetic incompatibility (39). Field studies of the hybrid zone between <em>P. ursinus</em> and <em>P. kindae</em> describe a deficit of hybrid individuals carrying Y chromosomes from <em>P. ursinus</em> and mtDNA from <em>P. kindae</em> compared to the converse (18). This suggests that when hybridization began between these two forms, some type of barrier (premating or postmating) reduced the frequency or fertility of matings by male <em>P. ursinus</em> with female <em>P. kindae</em>, while the converse mating type was more successful (18). Last, <em>P. anubis</em> and <em>P. hamadryas</em> differ substantially in their social organization and social structure (11, 28, 40). Among anubis baboons, both males and females are polygamous. Hamadryas societies are multi-level, with “harem”-like, one-male breeding units (OMUs) as basal social entities. In these OMUs, the single adult male defends exclusive access to one or more adult females. Other differences in sex-specific dispersal and social relationships are also observed (11). Despite the dramatic differences in social systems, these species hybridize in the wild (28). Hybrid males can achieve substantial reproductive success, at least in groups consisting mainly of hybrids (19). There is no clear evidence for a barrier to gene flow between the species, although the geographic distribution of phenotypically recognizable hybrids is narrow.</blockquote>

The various regional populations of chimpanzees (<em>Pan troglodytes</em>) are classified as subspecies, and they originated across roughly the same time span as these species of baboons. Chimpanzee regional populations do not exhibit the same variation in mating system, coat coloration, and morphology as baboons. So there is a good phylogenetic species concept (PSC) argument for baboons being different species that is not there for chimpanzees. Meanwhile, the observations noted by Rogers and colleagues are evidence that the baboon species should be recognized under the biological species concept (BSC). Chimpanzees and bonobos are different species under both BSC and PSC criteria, but neither concept suggests that the various subspecies of chimpanzees should be recognized as species instead.

The divergence among Neanderthals, Denisovans, and African ancestors of modern humans also took place across approximately the same time period, around 600,000--700,000 years. With both chimpanzees and Neanderthals, there is evidence for occasional introgression of genes among the ancient populations.

Aside from recent hybridization and the possible "ghost lineage" contributing to <em>P. papio</em>, the baboon picture of ancient gene flow is not clear from the results presented by Rogers and colleagues. The mismatches among Bayesian and parsimony phylogenetic results, and the different results obtained by looking at <em>Alu</em> insertion data, all suggest that incomplete lineage sorting or ancient reticulations may have been very important to the baboon pattern of variation.

To summarize, this paper provides important new data about the divergence of baboon species. Much more data would be valuable, especially samples that would tell us more about the geographic variation within baboon species and the long-term record of hybridization and introgression at the boundaries of these species.
