---
layout: single
title: "Chicken ancient DNA and natural selection during medieval times"
description: "A time-series look at domestication genes that increase egg-laying in European chickens."
category: story
permalink: /weblog/reviews/non-primate/ancient-dna/chicken-ancient-dna-loog-2020.html
tags: [ancient DNA, natural selection, domestication, non-primate]
comments: false
modified: 2020-04-14
author: John Hawks
---

Three years ago, Liisa Loog and coworkers published a fascinating paper quantifying natural selection from ancient DNA data in chickens: <a href="https://doi.org/10.1093/molbev/msx142">"Inferring Allele Frequency Trajectories from Ancient DNA Indicates That Selection on a Chicken Gene Coincided with Changes in Medieval Husbandry Practices"</a>.

Ancient DNA approaches are more and more being applied to zooarchaeological remains, especially in the study of domestication. Our commensal domesticated animals have some great opportunities for examining evolutionary questions. Geneticists have invested a lot in understanding their genetics today, including very good whole-genome drafts for all of them, making it easier to look at ancient genomes. We have a good idea of what some SNP polymorphisms do in these populations today, so that gives an opportunity to study the evolution of functional variation.

Finally, because of a history of strong human selection, the genetic structure of some phenotypic variation is strongly biased toward large-effect alleles. That's pretty ideal when we are considering ancient DNA data, because small sample size makes it pretty much impossible to see very slight changes in allele frequencies over time. So if you're studying a trait that is affected by hundreds of loci with small effect sizes, you are unlikely to be able to statistically demonstrate evolutionary change, even it if really happened.

But there are some problems using ancient DNA data to look at selection all the same. The authors discuss these at some length. This is an important concept to understand:

<blockquote>Ancient DNA data can provide direct information on changes in allele frequencies through time, and as such allows us to directly link past selection to contemporaneous ecological factors. However, ancient DNA sample sizes are typically small, and samples tend to be sparsely and unevenly distributed in space and time. Several methods exist for studying selection using ancient DNA (reviewed in Malaspinas 2016), but typically they lack the ability to model the confounding effects of gene flow—a process that could lead to overestimation of selection coefficients (Mathieson et al. 2015). </blockquote>

The confounding effects of gene flow are what slowed some of my work on Holocene natural selection around 10 years ago. When we look at the skeletons of post-Neolithic human populations in Europe, it is clear that people changed substantially in many skeletal phenotypes over time. But it is hard to tell how much of that phenotypic evolution was the result of natural selection, because there was also large-scale migration into Europe from other parts of the world. Ancient DNA has made that recent migration even more apparent, with some of today's populations deriving much more of their ancestry from Bronze Age central Asia and the Near East than from Neolithic people in Europe.

As long as you only have one population to look at, it will be hard to tell whether change was caused by <em>in situ</em> natural selection. Change might instead have resulted from immigration from populations that you haven't sampled.

This isn't a unique problem to ancient DNA. It is true of the skeleton, of teeth, of anything where you want to study change over time. You need to know something about the whole spectrum of populations that have contributed genes into later people, to be confident about the relative roles of gene flow and selection.

(Of course, these are not mutually exclusive forces. Change that looks like it comes from migration may be caused by natural selection increasing the survival and reproduction of immigrants. Or natural selection upon one population may increase its fitness to the extent that individuals begin to emigrate to other populations. Demography and selection are intertwined.)

If anything, ancient DNA should make it somewhat easier to untangle the effects of selection from those of migration. The reason is that your sample of DNA can rapidly grow larger than your sample of morphology. All you need to determine a genome is a small part of a bone, while examining a morphological character requires that exact part to have been preserved and recovered.

At the moment, the situation may seem like the opposite. But that's only because ancient DNA right now is relatively expensive. Once it becomes practical to get genomes from tiny bone fragments at minimal expense, we will know much more about ancient allele frequencies than we know about ancient morphological traits.

Anyway, back to the chickens. Loog and coworkers focused on two genes previously implicated in domestication, as they describe in their background:

<blockquote>The domestication of chickens (<em>Gallus gallus domesticus</em>) from wild junglefowl took place over a relatively short evolutionary timeframe (Rubin et al. 2010). In addition, the genetic architectures underlying many of the phenotypic traits that distinguish domestic populations are well understood (Eriksson et al. 2008; Rubin et al. 2010). As such, chickens present an ideal opportunity to correlate the timing of allele frequencies shifts with changes in human imposed selective pressures over the past several thousand years. To do so, we focused on two loci: thyroid-stimulating hormone receptor (TSHR) and β-carotene dioxygenase 2 (BCDO2), both previously identified as targets of selection in domesticated chickens (Rubin et al. 2010).</blockquote>

Having already identified candidate loci, they focused on these in the ancient samples. But they faced the problem that Asian chickens have different frequencies of selected alleles, and Asia is a possible source population for the chickens. So they had to straighten out whether <em>in situ</em> selection or gene flow from Asia was a better explanation for any frequency changes they found.

The authors compared the time course of frequency change in European chickens to their inferred time course of gene flow from Asia. For both genes, they infer that the frequency changes were initiated before substantial gene flow from Asian chickens was underway.


<figure>
<img src="/images/loog-chicken-allele-frequencies-2017.png" alt="Summary of chicken allele frequencies over time from Loog 2017" />
<figcaption>Figure 1 from Loog et al. 2017. Original caption: "Observed allele counts through time for the TSHR locus (a) and the BCDO2 locus (b). Wild type alleles in ancient samples are represented by yellow dots and derived alleles by red dots. Modern allele proportions are shown as solid bars to the right of each panel."</figcaption>
</figure>

I don't think the approach they took is especially satisfying. It would be better to test whether their loci of interest (<em>TSHR</em> and <em>BCDO2</em>) have different dynamics than the genome-wide pattern.

Still, the genes themselves are pretty interesting in the way they impact chicken biology. They give a good idea of how zooarchaeological information from many sites might be integrated into a coevolutionary picture of domesticate evolution and human cultural practices.


